<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Lifan Liang" />

<meta name="date" content="2024-07-24" />

<title>Aggregation of cTWAS results</title>

<script src="site_libs/header-attrs-2.30/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/htmltools-fill-0.5.9/fill.css" rel="stylesheet" />
<script src="site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.34.0/datatables.js"></script>
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.2.2/css/crosstalk.min.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.2.2/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-6.5.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">neuron_stimulation_100line_eQTL</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/LifanLiang/neuron_stim_eQTL100">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Aggregation of cTWAS results</h1>
<h4 class="author">Lifan Liang</h4>
<h4 class="date">2024-07-24</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-12-09
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>neuron_stim_eQTL100/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.2). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231109code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231109)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231109code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231109)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomLifanLiangneuronstimeQTL100tree10addbfe5e25eca0f031c74767c6ad4a00ee95ddtargetblank10addbfa">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/tree/10addbfe5e25eca0f031c74767c6ad4a00ee95dd" target="_blank">10addbf</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomLifanLiangneuronstimeQTL100tree10addbfe5e25eca0f031c74767c6ad4a00ee95ddtargetblank10addbfa"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/tree/10addbfe5e25eca0f031c74767c6ad4a00ee95dd" target="_blank">10addbf</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    analysis/figure/
    Ignored:    data/.DS_Store
    Ignored:    data/single_group_ctwas/.DS_Store

Untracked files:
    Untracked:  Fig15SE_dynamic_count_eqtl_ctwas.pdf
    Untracked:  GTEx_vs_SCZ.pdf.2024-08-26_21-04-15.log
    Untracked:  GTEx_vs_SCZ.png
    Untracked:  GTEx_vs_SCZ.tiff
    Untracked:  GTEx_vs_SCZ.tiff.2024-08-26_21-04-43.log
    Untracked:  Rplot.png
    Untracked:  geneMapping.rds
    Untracked:  mouse_sesenory_experince_inh.csv
    Untracked:  mouse_sesenory_experince_inh.csv.numbers
    Untracked:  plot4pdf.R

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/aggregate_ctwas_res.Rmd</code>)
and HTML (<code>docs/aggregate_ctwas_res.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/10addbfe5e25eca0f031c74767c6ad4a00ee95dd/analysis/aggregate_ctwas_res.Rmd" target="_blank">10addbf</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-12-09
</td>
<td>
wflow_publish(c("analysis", "data"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/bcec9455b663a80918225173ddf1b04aaca549c2/docs/aggregate_ctwas_res.html" target="_blank">bcec945</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/81ffbde0d68da376bd68f72f7906077018675eb2/analysis/aggregate_ctwas_res.Rmd" target="_blank">81ffbde</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
<td>
wflow_publish(c("analysis", "data"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/ba6ddfe48e75e6a2b3d9f36ef005cd3bb8ab982b/docs/aggregate_ctwas_res.html" target="_blank">ba6ddfe</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-12-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/8d6a3836c8157e699adab62beada134ca7638e59/analysis/aggregate_ctwas_res.Rmd" target="_blank">8d6a383</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-12-09
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/904d9445c3c9ddd5a2dd2be9a4d0dbcf98a804f9/docs/aggregate_ctwas_res.html" target="_blank">904d944</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-12-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/91abc486281e9c84cc8e0e7f03807dfbcfd31028/analysis/aggregate_ctwas_res.Rmd" target="_blank">91abc48</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-12-06
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/a4e0ba51139ff121a443e23d111f7c74bb0cd0fd/docs/aggregate_ctwas_res.html" target="_blank">a4e0ba5</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-12-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/a04679de525106b13e233b6a7a3127d5df87ed56/analysis/aggregate_ctwas_res.Rmd" target="_blank">a04679d</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-12-05
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/93fda5cd9a4ec04dc65bc3813ba90cba7e713fa8/docs/aggregate_ctwas_res.html" target="_blank">93fda5c</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-12-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/8b6a42541343ef50f737f1c40ae22ffa1a21fcb0/analysis/aggregate_ctwas_res.Rmd" target="_blank">8b6a425</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-12-02
</td>
<td>
wflow_publish(c("analysis", "docs", "data"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/16e21361c31a714c34aa38b6724b4efb7755cf3b/docs/aggregate_ctwas_res.html" target="_blank">16e2136</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-11-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/454eba45ef3400df5f0a3ecdd62d393735a02bec/analysis/aggregate_ctwas_res.Rmd" target="_blank">454eba4</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-11-14
</td>
<td>
wflow_publish(c("docs", "data", "analysis"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/6d4efe8ac0b7272df7faccecf6054eed259a9272/docs/aggregate_ctwas_res.html" target="_blank">6d4efe8</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-09-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/4d20f389fc89b96731f79beefe564a9b6d75bd2a/analysis/aggregate_ctwas_res.Rmd" target="_blank">4d20f38</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-09-25
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/4d20f389fc89b96731f79beefe564a9b6d75bd2a/docs/aggregate_ctwas_res.html" target="_blank">4d20f38</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-09-25
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/ef89bedf0b33c8ccac40a501984d4fb3a2d187c1/docs/aggregate_ctwas_res.html" target="_blank">ef89bed</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-09-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/c693b4cae6b42a429e5f5f88059e9231760a890b/analysis/aggregate_ctwas_res.Rmd" target="_blank">c693b4c</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-09-19
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/c693b4cae6b42a429e5f5f88059e9231760a890b/docs/aggregate_ctwas_res.html" target="_blank">c693b4c</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-09-19
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/0b2f14d5edf33464d3d68c104808b3b2893fbf8a/docs/aggregate_ctwas_res.html" target="_blank">0b2f14d</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/5df41b0430868e27337e4696d783bbafaa57d48d/analysis/aggregate_ctwas_res.Rmd" target="_blank">5df41b0</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-23
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/5df41b0430868e27337e4696d783bbafaa57d48d/docs/aggregate_ctwas_res.html" target="_blank">5df41b0</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-23
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/ccbf827af648135903f1a35bd394114b9e16cd6c/docs/aggregate_ctwas_res.html" target="_blank">ccbf827</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/574a14f9f5ae486c7da03848f66082140c90ff88/analysis/aggregate_ctwas_res.Rmd" target="_blank">574a14f</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-16
</td>
<td>
wflow_publish(c("analysis", "data"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/2f59d291e6ee2212ca84d7d01a3aa4d5624c6504/docs/aggregate_ctwas_res.html" target="_blank">2f59d29</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/d064713dd35ed0ebb8ec074c1ebef180af9c6a86/analysis/aggregate_ctwas_res.Rmd" target="_blank">d064713</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-16
</td>
<td>
wflow_publish(c("analysis", "data"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/6d74a6484602335f7cf0d459871dc2520fcadc4d/docs/aggregate_ctwas_res.html" target="_blank">6d74a64</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/c6dd0777e8b9bbb4a72c157b803f58a50cf44687/analysis/aggregate_ctwas_res.Rmd" target="_blank">c6dd077</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-16
</td>
<td>
wflow_publish(c("analysis", "data"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/c9bb44c28b2767cb24c906998f5ab03c6009226e/docs/aggregate_ctwas_res.html" target="_blank">c9bb44c</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/4f05c154aa05cbd25467ff769f3f48135e9b0793/analysis/aggregate_ctwas_res.Rmd" target="_blank">4f05c15</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-05
</td>
<td>
wflow_publish("analysis")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/c255b081903658d8cfe9666eba4458b290d79939/docs/aggregate_ctwas_res.html" target="_blank">c255b08</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/7d70d1a246cfdd0f983b820d2eff381cf7ecdd8e/analysis/aggregate_ctwas_res.Rmd" target="_blank">7d70d1a</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-05
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/7d70d1a246cfdd0f983b820d2eff381cf7ecdd8e/docs/aggregate_ctwas_res.html" target="_blank">7d70d1a</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-05
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/3242ba61bacd56b9877476b60953ef2ed8055ef0/docs/aggregate_ctwas_res.html" target="_blank">3242ba6</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-07-26
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/0e6b329808a511fe04b48403d8411059366b4852/analysis/aggregate_ctwas_res.Rmd" target="_blank">0e6b329</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-07-26
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="introduction" class="section level2">
<h2>Introduction</h2>
<p>Aggregate cTWAS results from NPD traits and investigate sharing</p>
<pre><code>quartz_off_screen 
                2 </code></pre>
<p><img src="figure/aggregate_ctwas_res.Rmd/unnamed-chunk-6-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-6-1">
Past versions of unnamed-chunk-6-1.png
</button>
</p>
<div id="fig-unnamed-chunk-6-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bcec9455b663a80918225173ddf1b04aaca549c2/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-6-1.png" target="_blank">bcec945</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/a4e0ba51139ff121a443e23d111f7c74bb0cd0fd/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-6-1.png" target="_blank">a4e0ba5</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-12-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/4d20f389fc89b96731f79beefe564a9b6d75bd2a/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-6-1.png" target="_blank">4d20f38</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-09-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/5df41b0430868e27337e4696d783bbafaa57d48d/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-6-1.png" target="_blank">5df41b0</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/6d74a6484602335f7cf0d459871dc2520fcadc4d/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-6-1.png" target="_blank">6d74a64</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="time-point-distribution-of-enrichment" class="section level3">
<h3>Time point distribution of enrichment</h3>
<pre class="r"><code>enrich1 &lt;- as.data.frame(enrichment)
dat.tp &lt;- data.frame(enrichment=c(unlist(enrich1[1:3,]),unlist(enrich1[4:6,]),unlist(enrich1[7:9,])), timepoint=c(rep(&quot;0hr&quot;,15), rep(&quot;1hr&quot;,15),rep(&quot;6hr&quot;,15)))
boxplot(enrichment~timepoint,dat.tp)</code></pre>
<p><img src="figure/aggregate_ctwas_res.Rmd/unnamed-chunk-7-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-7-1">
Past versions of unnamed-chunk-7-1.png
</button>
</p>
<div id="fig-unnamed-chunk-7-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bcec9455b663a80918225173ddf1b04aaca549c2/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-7-1.png" target="_blank">bcec945</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/a4e0ba51139ff121a443e23d111f7c74bb0cd0fd/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-7-1.png" target="_blank">a4e0ba5</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-12-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/93fda5cd9a4ec04dc65bc3813ba90cba7e713fa8/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-7-1.png" target="_blank">93fda5c</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-12-02
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/4d20f389fc89b96731f79beefe564a9b6d75bd2a/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-7-1.png" target="_blank">4d20f38</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-09-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/5df41b0430868e27337e4696d783bbafaa57d48d/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-7-1.png" target="_blank">5df41b0</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/6d74a6484602335f7cf0d459871dc2520fcadc4d/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-7-1.png" target="_blank">6d74a64</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/c9bb44c28b2767cb24c906998f5ab03c6009226e/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-7-1.png" target="_blank">c9bb44c</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/7d70d1a246cfdd0f983b820d2eff381cf7ecdd8e/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-7-1.png" target="_blank">7d70d1a</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="cell-type-distribution-of-significant-genes"
class="section level2">
<h2>Cell type distribution of significant genes</h2>
<p><img src="figure/aggregate_ctwas_res.Rmd/unnamed-chunk-8-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-8-1">
Past versions of unnamed-chunk-8-1.png
</button>
</p>
<div id="fig-unnamed-chunk-8-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bcec9455b663a80918225173ddf1b04aaca549c2/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-8-1.png" target="_blank">bcec945</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/a4e0ba51139ff121a443e23d111f7c74bb0cd0fd/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-8-1.png" target="_blank">a4e0ba5</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-12-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/93fda5cd9a4ec04dc65bc3813ba90cba7e713fa8/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-8-1.png" target="_blank">93fda5c</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-12-02
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/4d20f389fc89b96731f79beefe564a9b6d75bd2a/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-8-1.png" target="_blank">4d20f38</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-09-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/5df41b0430868e27337e4696d783bbafaa57d48d/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-8-1.png" target="_blank">5df41b0</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="dynamic-gene-counts-across-cell-types" class="section level2">
<h2>Dynamic gene counts across cell types</h2>
<p><img src="figure/aggregate_ctwas_res.Rmd/unnamed-chunk-9-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-9-1">
Past versions of unnamed-chunk-9-1.png
</button>
</p>
<div id="fig-unnamed-chunk-9-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bcec9455b663a80918225173ddf1b04aaca549c2/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-9-1.png" target="_blank">bcec945</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/a4e0ba51139ff121a443e23d111f7c74bb0cd0fd/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-9-1.png" target="_blank">a4e0ba5</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-12-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/93fda5cd9a4ec04dc65bc3813ba90cba7e713fa8/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-9-1.png" target="_blank">93fda5c</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-12-02
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/4d20f389fc89b96731f79beefe564a9b6d75bd2a/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-9-1.png" target="_blank">4d20f38</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-09-25
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/aggregate_ctwas_res.Rmd/unnamed-chunk-10-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-10-1">
Past versions of unnamed-chunk-10-1.png
</button>
</p>
<div id="fig-unnamed-chunk-10-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bcec9455b663a80918225173ddf1b04aaca549c2/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-10-1.png" target="_blank">bcec945</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/ba6ddfe48e75e6a2b3d9f36ef005cd3bb8ab982b/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-10-1.png" target="_blank">ba6ddfe</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-12-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/904d9445c3c9ddd5a2dd2be9a4d0dbcf98a804f9/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-10-1.png" target="_blank">904d944</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-12-06
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/a4e0ba51139ff121a443e23d111f7c74bb0cd0fd/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-10-1.png" target="_blank">a4e0ba5</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-12-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/4d20f389fc89b96731f79beefe564a9b6d75bd2a/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-10-1.png" target="_blank">4d20f38</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-09-25
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="high-confidence-and-plausible-gene-counts-across-traits"
class="section level3">
<h3>High confidence and plausible gene counts across traits</h3>
<p><img src="figure/aggregate_ctwas_res.Rmd/unnamed-chunk-11-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-11-1">
Past versions of unnamed-chunk-11-1.png
</button>
</p>
<div id="fig-unnamed-chunk-11-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bcec9455b663a80918225173ddf1b04aaca549c2/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-11-1.png" target="_blank">bcec945</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/a4e0ba51139ff121a443e23d111f7c74bb0cd0fd/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-11-1.png" target="_blank">a4e0ba5</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-12-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="significant-gene-distribution-across-the-five-traits"
class="section level2">
<h2>Significant gene distribution across the five traits</h2>
<p>Below is a heatmap of genes over PIP&gt;50% in at least one trait. It
shows not many shared genes among traits.</p>
<p><img src="figure/aggregate_ctwas_res.Rmd/unnamed-chunk-12-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-12-1">
Past versions of unnamed-chunk-12-1.png
</button>
</p>
<div id="fig-unnamed-chunk-12-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bcec9455b663a80918225173ddf1b04aaca549c2/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-12-1.png" target="_blank">bcec945</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>
Attaching package: &#39;dplyr&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:stats&#39;:

    filter, lag</code></pre>
<pre><code>The following objects are masked from &#39;package:base&#39;:

    intersect, setdiff, setequal, union</code></pre>
<div class="datatables html-widget html-fill-item" id="htmlwidget-7657cc3e42637b46d6e4" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-7657cc3e42637b46d6e4">{"x":{"filter":"none","vertical":false,"data":[["DNPH1","EFHB","GGNBP2","HLA-DMA"],[0.6371740503923569,0.885489351807218,0.580841382879703,0],[0.6753938406185035,0.642515402315624,0,0.7672702597392497],[0,0,0,0.913600019811909],[0,0,0,0],[0,0,1.070308818920317,0]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>SCZ<\/th>\n      <th>Bipolar<\/th>\n      <th>Depression<\/th>\n      <th>Neuroticism<\/th>\n      <th>ADHD<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":1,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":2,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":5,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[1,2,3,4,5]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"SCZ","targets":1},{"name":"Bipolar","targets":2},{"name":"Depression","targets":3},{"name":"Neuroticism","targets":4},{"name":"ADHD","targets":5}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render","options.columnDefs.2.render","options.columnDefs.3.render","options.columnDefs.4.render"],"jsHooks":[]}</script>
</div>
<div id="enrichment-analysis" class="section level2">
<h2>Enrichment analysis</h2>
<div id="three-traits-with-pip0.8" class="section level3">
<h3>Three traits with PIP&gt;0.8</h3>
<pre><code>Welcome to enrichR
Checking connections ... </code></pre>
<pre><code>Enrichr ... Connection is Live!
FlyEnrichr ... Connection is Live!
WormEnrichr ... Connection is Live!
YeastEnrichr ... Connection is Live!
FishEnrichr ... Connection is Live!
OxEnrichr ... Connection is Live!</code></pre>
<pre><code>Uploading data to Enrichr... Done.
  Querying GO_Biological_Process_2023... Done.
Parsing results... Done.</code></pre>
<div class="datatables html-widget html-fill-item" id="htmlwidget-427ea3e24281aa554de8" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-427ea3e24281aa554de8">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10"],["Amino-Acid Betaine Metabolic Process (GO:0006577)","Carnitine Metabolic Process (GO:0009437)","Peripheral Nervous System Development (GO:0007422)","ATP Metabolic Process (GO:0046034)","Proton Motive Force-Driven Mitochondrial ATP Synthesis (GO:0042776)","Proton Motive Force-Driven ATP Synthesis (GO:0015986)","Oxidative Phosphorylation (GO:0006119)","Aerobic Electron Transport Chain (GO:0019646)","Mitochondrial ATP Synthesis Coupled Electron Transport (GO:0042775)","CD4-positive, Alpha-Beta T Cell Activation (GO:0035710)"],["2/6","2/9","2/20","2/34","2/53","2/60","2/63","2/68","2/70","1/5"],[3.943661060076414e-05,9.435561918879866e-05,0.0004923687563769557,0.001432947791192216,0.003451591837737016,0.004401659958467019,0.004841789853430103,0.005618568554412981,0.005944244950633995,0.008223547969126478],[0.01108168757881473,0.01325696449602621,0.04611854018064152,0.1006645823312532,0.1103009210916455,0.1103009210916455,0.1103009210916455,0.1103009210916455,0.1103009210916455,0.1103009210916455],[321.983870967742,183.963133640553,71.50179211469533,40.19153225806452,25.19418089816572,22.14571746384872,21.05341089370704,19.4535679374389,18.87950664136622,155.9609375],["CROT;CPT1C","CROT;CPT1C","SERPINI1;ASIC2","OLA1;ATP5PO","NDUFA6;ATP5PO","NDUFA6;ATP5PO","NDUFA6;ATP5PO","NDUFA6;COX6A1","NDUFA6;COX6A1","HMGB1"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>P.value<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Odds.Ratio<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[3,4,5]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Term","targets":1},{"name":"Overlap","targets":2},{"name":"P.value","targets":3},{"name":"Adjusted.P.value","targets":4},{"name":"Odds.Ratio","targets":5},{"name":"Genes","targets":6}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render"],"jsHooks":[]}</script>
</div>
<div id="three-traits-with-pip0.8-and-input-egene-as-background-genes."
class="section level3">
<h3>Three traits with PIP&gt;0.8 and input eGene as background
genes.</h3>
<pre class="r"><code>background &lt;- readRDS(&quot;data/background_gene_ctwas.rds&quot;)
enrich.res &lt;- enrichr(gene.univ, &quot;GO_Biological_Process_2023&quot;,background = background)</code></pre>
<pre><code>Uploading data to Speedrichr...
 - Your gene set... Done.
 - Your background... Done.
Getting enrichment results...
 - GO_Biological_Process_2023... Done.
Parsing results... Done.</code></pre>
<pre class="r"><code>head(enrich.res$GO_Biological_Process_2023[-c(5,6,8)],10) %&gt;%
  DT::datatable() %&gt;%
  DT::formatRound(columns=c(&#39;P.value&#39;, &#39;Adjusted.P.value&#39;), digits=3)</code></pre>
<div class="datatables html-widget html-fill-item" id="htmlwidget-a50efc78b4c04b4a25da" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-a50efc78b4c04b4a25da">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10"],["Amino-Acid Betaine Metabolic Process (GO:0006577)","Carnitine Metabolic Process (GO:0009437)","Peripheral Nervous System Development (GO:0007422)","ATP Metabolic Process (GO:0046034)","Regulation Of Apoptotic Cell Clearance (GO:2000425)","Proton Motive Force-Driven Mitochondrial ATP Synthesis (GO:0042776)","Purine Ribonucleotide Metabolic Process (GO:0009150)","Aerobic Electron Transport Chain (GO:0019646)","Oxidative Phosphorylation (GO:0006119)","Proton Motive Force-Driven ATP Synthesis (GO:0015986)"],["1","2","3","4","5","6","7","8","9","10"],[0.000130871247878495,0.000363911908216701,0.000849913966395625,0.00173126814199373,0.00368627189261547,0.00573176118529838,0.0068676043220461,0.0068676043220461,0.00726711797169322,0.00726711797169322],[0.0367748206538571,0.0511296231044465,0.0796086081857236,0.111790107696605,0.111790107696605,0.111790107696605,0.111790107696605,0.111790107696605,0.111790107696605,0.111790107696605],[191.741935483871,95.8387096774194,57.4774193548387,38.2967741935484,null,19.7775305895439,17.9173387096774,17.9173387096774,17.3724340175953,17.3724340175953],["CROT;CPT1C","CROT;CPT1C","SERPINI1;ASIC2","ATP5PO;OLA1","HMGB1","NDUFA6;ATP5PO","OLA1;CROT","NDUFA6;COX6A1","NDUFA6;ATP5PO","NDUFA6;ATP5PO"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Rank<\/th>\n      <th>P.value<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Odds.Ratio<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[3,4,5]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Term","targets":1},{"name":"Rank","targets":2},{"name":"P.value","targets":3},{"name":"Adjusted.P.value","targets":4},{"name":"Odds.Ratio","targets":5},{"name":"Genes","targets":6}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render"],"jsHooks":[]}</script>
</div>
<div id="three-traits-with-pip0.5" class="section level3">
<h3>Three traits with PIP&gt;0.5</h3>
<pre><code>Uploading data to Enrichr... Done.
  Querying GO_Biological_Process_2023... Done.
Parsing results... Done.</code></pre>
<div class="datatables html-widget html-fill-item" id="htmlwidget-1c14e9d616c779d1b646" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-1c14e9d616c779d1b646">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10"],["Carnitine Metabolic Process (GO:0009437)","Amino-Acid Betaine Metabolic Process (GO:0006577)","Regulation Of Epithelial Cell Migration (GO:0010632)","Positive Regulation Of Vascular Endothelial Cell Proliferation (GO:1905564)","Regulation Of Short-Term Neuronal Synaptic Plasticity (GO:0048172)","Medium-Chain Fatty Acid Metabolic Process (GO:0051791)","Regulation Of Vascular Endothelial Cell Proliferation (GO:1905562)","Peripheral Nervous System Development (GO:0007422)","Positive Regulation Of Endothelial Cell Migration (GO:0010595)","Positive Regulation Of Epithelial Cell Migration (GO:0010634)"],["3/9","2/6","3/50","2/12","2/13","2/14","2/18","2/20","3/85","3/95"],[3.857340151791266e-06,0.0001952441783803584,0.0008084332971786784,0.0008469756722727654,0.0009986092713926314,0.001162295606906837,0.001935823870251673,0.002392649522599408,0.003718912702727769,0.005078056933364429],[0.0021639678251549,0.05476599203569053,0.1086746392457892,0.1086746392457892,0.1086746392457892,0.1086746392457892,0.1551424558873126,0.1677845477722835,0.2182252025158051,0.2182252025158051],[142.2928571428571,140.3028169014084,18.12765957446809,56.10422535211268,51.00128040973112,46.74882629107982,35.05457746478874,31.15649452269171,10.37195121951219,9.239906832298137],["ACADM;CROT;CPT1C","CROT;CPT1C","GLIPR2;PLCG1;SASH1","HMGB1;PLCG1","SYNGR1;PPFIA3","CROT;ACADM","HMGB1;PLCG1","SERPINI1;ASIC2","HMGB1;PLCG1;SASH1","GLIPR2;PLCG1;SASH1"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>P.value<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Odds.Ratio<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[3,4,5]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Term","targets":1},{"name":"Overlap","targets":2},{"name":"P.value","targets":3},{"name":"Adjusted.P.value","targets":4},{"name":"Odds.Ratio","targets":5},{"name":"Genes","targets":6}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render"],"jsHooks":[]}</script>
</div>
<div id="five-traits-with-pip0.8" class="section level3">
<h3>Five traits with PIP&gt;0.8</h3>
<pre><code>Uploading data to Enrichr... Done.
  Querying GO_Biological_Process_2023... Done.
Parsing results... Done.</code></pre>
<div class="datatables html-widget html-fill-item" id="htmlwidget-771ad2536610c1179489" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-771ad2536610c1179489">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10"],["Amino-Acid Betaine Metabolic Process (GO:0006577)","Carnitine Metabolic Process (GO:0009437)","Peripheral Nervous System Development (GO:0007422)","Glycolipid Metabolic Process (GO:0006664)","ATP Metabolic Process (GO:0046034)","Carbohydrate Catabolic Process (GO:0016052)","Proton Motive Force-Driven Mitochondrial ATP Synthesis (GO:0042776)","Proton Motive Force-Driven ATP Synthesis (GO:0015986)","Oxidative Phosphorylation (GO:0006119)","Aerobic Electron Transport Chain (GO:0019646)"],["2/6","2/9","2/20","2/33","2/34","2/41","2/53","2/60","2/63","2/68"],[8.408303430078136e-05,0.0002008745228693487,0.00104246932126757,0.002839972184710252,0.003012866504729657,0.004357041855378435,0.00718928839712134,0.009136597521628502,0.01003535415090053,0.01161675978304017],[0.02757923525065628,0.03294342175057319,0.113976645791921,0.1591789154420074,0.1591789154420074,0.1591789154420074,0.1591789154420074,0.1591789154420074,0.1591789154420074,0.1591789154420074],["CROT;CPT1C","CROT;CPT1C","SERPINI1;ASIC2","GLB1;NAGA","OLA1;ATP5PO","NAGA;ENO4","NDUFA6;ATP5PO","NDUFA6;ATP5PO","NDUFA6;ATP5PO","NDUFA6;COX6A1"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>P.value<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[3,4]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Term","targets":1},{"name":"Overlap","targets":2},{"name":"P.value","targets":3},{"name":"Adjusted.P.value","targets":4},{"name":"Genes","targets":5}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render"],"jsHooks":[]}</script>
</div>
<div id="five-traits-with-pip0.5" class="section level3">
<h3>Five traits with PIP&gt;0.5</h3>
<pre><code>Uploading data to Enrichr... Done.
  Querying GO_Biological_Process_2023... Done.
Parsing results... Done.</code></pre>
<div class="datatables html-widget html-fill-item" id="htmlwidget-118ca324790aac282ebf" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-118ca324790aac282ebf">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10"],["Carnitine Metabolic Process (GO:0009437)","Amino-Acid Betaine Metabolic Process (GO:0006577)","Positive Regulation Of Vascular Endothelial Cell Proliferation (GO:1905564)","Regulation Of Short-Term Neuronal Synaptic Plasticity (GO:0048172)","Regulation Of Epithelial Cell Migration (GO:0010632)","Medium-Chain Fatty Acid Metabolic Process (GO:0051791)","Regulation Of Vascular Endothelial Cell Proliferation (GO:1905562)","Peripheral Nervous System Development (GO:0007422)","Regulation Of Autophagy (GO:0010506)","Positive Regulation Of Endothelial Cell Migration (GO:0010595)"],["3/9","2/6","2/12","2/13","3/50","2/14","2/18","2/20","5/241","3/85"],[1.121537397168748e-05,0.0003962753614965705,0.001708444506433439,0.002012233979929768,0.002241885879850449,0.002339659956910014,0.003880772627961155,0.004786757771404614,0.008533408476139146,0.00991100827582375],[0.007547946682945677,0.133346659143596,0.2624318585000732,0.2624318585000732,0.2624318585000732,0.2624318585000732,0.3159075618166758,0.3159075618166758,0.3159075618166758,0.3159075618166758],["ACADM;CROT;CPT1C","CROT;CPT1C","HMGB1;PLCG1","SYNGR1;PPFIA3","GLIPR2;PLCG1;SASH1","CROT;ACADM","HMGB1;PLCG1","SERPINI1;ASIC2","FKBP8;GPR137;BMF;HMGB1;ATP6V0C","HMGB1;PLCG1;SASH1"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>P.value<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[3,4]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Term","targets":1},{"name":"Overlap","targets":2},{"name":"P.value","targets":3},{"name":"Adjusted.P.value","targets":4},{"name":"Genes","targets":5}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render"],"jsHooks":[]}</script>
</div>
</div>
<div id="analysis-of-three-genes-enriched-in-lipid-metabolism"
class="section level2">
<h2>Analysis of three genes enriched in lipid metabolism</h2>
<div id="cpt1c" class="section level3">
<h3>CPT1C</h3>
<p>CPT1C was a top gene in SCZ with a high PIP in 1hr_NMglut.</p>
<div class="datatables html-widget html-fill-item" id="htmlwidget-7af0e23c13b7c4def9a9" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-7af0e23c13b7c4def9a9">{"x":{"filter":"none","vertical":false,"data":[["4559245","4559257","4559278"],[19,19,19],["CPT1C|1hr_nmglut","CPT1C|1hr_npglut","CPT1C|6hr_GABA"],[49644795,49695897,49693661],["1hr_nmglut","1hr_npglut","6hr_GABA"],[19,19,19],[34,34,34],[1,0,0],[0.905347681624163,0.00151059831002931,0.00435343825355738],[41.7889387916842,15.4953450954328,18.9961589999033]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>chrom<\/th>\n      <th>id<\/th>\n      <th>pos<\/th>\n      <th>type<\/th>\n      <th>region_tag1<\/th>\n      <th>region_tag2<\/th>\n      <th>cs_index<\/th>\n      <th>susie_pip<\/th>\n      <th>mu2<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,3,5,6,7,8,9]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"chrom","targets":1},{"name":"id","targets":2},{"name":"pos","targets":3},{"name":"type","targets":4},{"name":"region_tag1","targets":5},{"name":"region_tag2","targets":6},{"name":"cs_index","targets":7},{"name":"susie_pip","targets":8},{"name":"mu2","targets":9}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<p>The SNP related to CPT1C in 1 hour NMglut was 50K bps away from the
other two SNPs. Looking into this gene-SNP pair, 1hr_nmglut is the only
condition with FDR &lt; 0.1.</p>
<p><img src="figure/aggregate_ctwas_res.Rmd/unnamed-chunk-21-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-21-1">
Past versions of unnamed-chunk-21-1.png
</button>
</p>
<div id="fig-unnamed-chunk-21-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/6d4efe8ac0b7272df7faccecf6054eed259a9272/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-21-1.png" target="_blank">6d4efe8</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-09-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/4d20f389fc89b96731f79beefe564a9b6d75bd2a/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-21-1.png" target="_blank">4d20f38</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-09-25
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>The SNP, rs12104272, has a nominal P value around <span
class="math inline">\(10^{-8}\)</span> in SCZ GWAS. It was mapped to
SCAF1 in (another study)[<a
href="https://pubmed.ncbi.nlm.nih.gov/31740837/"
class="uri">https://pubmed.ncbi.nlm.nih.gov/31740837/</a>]. However, in
our dataset, this gene-SNP pair was not significant in any
conditions.</p>
<div class="datatables html-widget html-fill-item" id="htmlwidget-3989c53c1df7f640b079" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-3989c53c1df7f640b079">{"x":{"filter":"none","vertical":false,"data":[["0hr_GABA","0hr_nmglut","0hr_npglut","1hr_GABA","1hr_nmglut","1hr_npglut","6hr_GABA","6hr_nmglut","6hr_npglut"],["rs12104272","rs12104272","rs12104272","rs12104272","rs12104272","rs12104272","rs12104272","rs12104272","rs12104272"],["SCAF1","SCAF1","SCAF1","SCAF1","SCAF1","SCAF1","SCAF1","SCAF1","SCAF1"],[0.574774166034331,-0.3729246192638403,0.8458122621678535,1.072171376125389,-0.03809029492735802,2.079516886032295,0.738321254212311,-0.8137213143143791,0.01113420405838825],[0.5673691053925456,0.7106124106069782,0.4005395258238228,0.2873256255594018,0.9697324354462731,0.04113162719482017,0.4627191025751316,0.4190222384186835,0.9911467469354346],[0.9456052082105234,0.9764401585267503,0.8968566879051304,0.8549122948936176,0.9977924225830014,0.5104435101399567,0.9152515221236248,0.9274249815453185,0.9990454819829944],[0.0603484177072146,-0.04523522590392092,0.06025847991259258,0.1028994953321254,-0.004832422695816465,0.1389533203738191,0.06519446156539602,-0.1251347059145657,0.0008955215597614747]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>snps<\/th>\n      <th>gene<\/th>\n      <th>statistic<\/th>\n      <th>pvalue<\/th>\n      <th>FDR<\/th>\n      <th>beta<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[3,4,5,6]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"snps","targets":1},{"name":"gene","targets":2},{"name":"statistic","targets":3},{"name":"pvalue","targets":4},{"name":"FDR","targets":5},{"name":"beta","targets":6}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<div id="eqtl-plots" class="section level4">
<h4>eQTL plots</h4>
<p><img src="assets/CPT1C_rs12104272_nmglut.png" /></p>
</div>
<div id="ld-across-top-eqtls-input" class="section level4">
<h4>LD across top eQTLs input</h4>
<p><img src="assets/CPT1C_ctwas_LD.png" /></p>
</div>
</div>
<div id="crot" class="section level3">
<h3>CROT</h3>
<p>CROT has PIP=0.87 in SCZ.The corresponding SNP was rs13233308</p>
<div class="datatables html-widget html-fill-item" id="htmlwidget-f8241748eb28a8a54961" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-f8241748eb28a8a54961">{"x":{"filter":"none","vertical":false,"data":[["2349367","2349377"],[7,7],["CROT|0hr_GABA","CROT|6hr_GABA"],[87610918,87615644],["0hr_GABA","6hr_GABA"],[7,7],[54,54],[0,2],[0.00788059564431587,0.868267869654942],[15.7492305816438,29.0437847027247]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>chrom<\/th>\n      <th>id<\/th>\n      <th>pos<\/th>\n      <th>type<\/th>\n      <th>region_tag1<\/th>\n      <th>region_tag2<\/th>\n      <th>cs_index<\/th>\n      <th>susie_pip<\/th>\n      <th>mu2<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,3,5,6,7,8,9]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"chrom","targets":1},{"name":"id","targets":2},{"name":"pos","targets":3},{"name":"type","targets":4},{"name":"region_tag1","targets":5},{"name":"region_tag2","targets":6},{"name":"cs_index","targets":7},{"name":"susie_pip","targets":8},{"name":"mu2","targets":9}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<p><img src="figure/aggregate_ctwas_res.Rmd/unnamed-chunk-24-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-24-1">
Past versions of unnamed-chunk-24-1.png
</button>
</p>
<div id="fig-unnamed-chunk-24-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bcec9455b663a80918225173ddf1b04aaca549c2/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-24-1.png" target="_blank">bcec945</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/a4e0ba51139ff121a443e23d111f7c74bb0cd0fd/docs/figure/aggregate_ctwas_res.Rmd/unnamed-chunk-24-1.png" target="_blank">a4e0ba5</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-12-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="eqtl-plots-1" class="section level4">
<h4>eQTL plots</h4>
<p><img src="assets/CROT_rs13233308_GABA.png" /></p>
</div>
<div id="ld-across-top-eqtls-input-1" class="section level4">
<h4>LD across top eQTLs input</h4>
<p><img src="assets/CROT_ctwas_LD.png" /></p>
</div>
</div>
<div id="acadm" class="section level3">
<h3>ACADM</h3>
<p>ACADM has PIP=0.61 in 6 hour NMglut for the trait</p>
<div class="datatables html-widget html-fill-item" id="htmlwidget-224ae002e7acfef1f0c4" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-224ae002e7acfef1f0c4">{"x":{"filter":"none","vertical":false,"data":[["454819","454827","454833","454840","454847","454855","454865","454868"],[1,1,1,1,1,1,1,1],["ACADM|0hr_GABA","ACADM|1hr_GABA","ACADM|6hr_GABA","ACADM|1hr_nmglut","ACADM|6hr_nmglut","ACADM|0hr_npglut","ACADM|1hr_npglut","ACADM|6hr_npglut"],[75677758,75759948,75683496,75724884,75540257,75724884,75707395,75724884],["0hr_GABA","1hr_GABA","6hr_GABA","1hr_nmglut","6hr_nmglut","0hr_npglut","1hr_npglut","6hr_npglut"],[1,1,1,1,1,1,1,1],[47,47,47,47,47,47,47,47],[0,0,0,0,1,0,0,0],[0.00330392328003271,0.00517454295759545,0.00433177977478938,0.0243651173068259,0.611441219347865,0.00492732890601999,0.00206115565407428,0.00297551556964404],[6.03239080935922,4.7440602970202,4.45146586343824,6.19599166541378,21.039254037044,6.19599166541377,4.28362686469841,6.19599166541378]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>chrom<\/th>\n      <th>id<\/th>\n      <th>pos<\/th>\n      <th>type<\/th>\n      <th>region_tag1<\/th>\n      <th>region_tag2<\/th>\n      <th>cs_index<\/th>\n      <th>susie_pip<\/th>\n      <th>mu2<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,3,5,6,7,8,9]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"chrom","targets":1},{"name":"id","targets":2},{"name":"pos","targets":3},{"name":"type","targets":4},{"name":"region_tag1","targets":5},{"name":"region_tag2","targets":6},{"name":"cs_index","targets":7},{"name":"susie_pip","targets":8},{"name":"mu2","targets":9}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<p><img src="figure/aggregate_ctwas_res.Rmd/unnamed-chunk-26-1.png" width="672" style="display: block; margin: auto;" /></p>
<div id="eqtl-plots-2" class="section level4">
<h4>eQTL plots</h4>
<p><img src="assets/ACADM_rs211745_nmglut.png" /></p>
</div>
<div id="ld-across-top-eqtls-input-2" class="section level4">
<h4>LD across top eQTLs input</h4>
<p><img src="assets/ACADM_ctwas_LD.png" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.1.2 (2021-11-01)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur 10.16

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] enrichR_3.4     dplyr_1.1.2     pheatmap_1.0.13 ggplot2_4.0.1  
[5] workflowr_1.7.2

loaded via a namespace (and not attached):
 [1] tidyselect_1.2.1   xfun_0.54          bslib_0.3.1        vctrs_0.6.5       
 [5] generics_0.1.4     htmltools_0.5.9    yaml_2.3.11        rlang_1.1.6       
 [9] jquerylib_0.1.4    later_1.3.1        pillar_1.11.1      glue_1.8.0        
[13] withr_3.0.2        RColorBrewer_1.1-3 S7_0.2.1           lifecycle_1.0.4   
[17] stringr_1.6.0      gtable_0.3.6       ragg_1.2.5         htmlwidgets_1.6.4 
[21] evaluate_1.0.5     WriteXLS_6.8.0     labeling_0.4.3     knitr_1.50        
[25] callr_3.7.6        fastmap_1.1.1      crosstalk_1.2.2    httpuv_1.6.10     
[29] ps_1.9.1           curl_7.0.0         Rcpp_1.0.10        DT_0.34.0         
[33] promises_1.5.0     scales_1.4.0       jsonlite_2.0.0     farver_2.1.1      
[37] otel_0.2.0         fs_1.6.2           systemfonts_1.0.4  textshaping_0.3.6 
[41] rjson_0.2.21       digest_0.6.31      stringi_1.7.12     processx_3.8.6    
[45] getPass_0.2-4      rprojroot_2.1.1    grid_4.1.2         cowplot_1.2.0     
[49] cli_3.6.5          tools_4.1.2        magrittr_2.0.4     sass_0.4.6        
[53] tibble_3.3.0       dichromat_2.0-0.1  whisker_0.4.1      pkgconfig_2.0.3   
[57] rmarkdown_2.30     httr_1.4.7         rstudioapi_0.17.1  R6_2.6.1          
[61] git2r_0.32.0       compiler_4.1.2    </code></pre>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
