<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Lifan Liang" />

<meta name="date" content="2023-12-08" />

<title>caQTL_cTWAS</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">neuron_stimulation_100line_eQTL</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/LifanLiang/neuron_stim_eQTL100">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">caQTL_cTWAS</h1>
<h4 class="author">Lifan Liang</h4>
<h4 class="date">2023-12-08</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-02-20
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>neuron_stim_eQTL100/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231109code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231109)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231109code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231109)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomLifanLiangneuronstimeQTL100tree3a382e8bb2e20451fd68db7f664221ec3f5a20bdtargetblank3a382e8a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/tree/3a382e8bb2e20451fd68db7f664221ec3f5a20bd" target="_blank">3a382e8</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomLifanLiangneuronstimeQTL100tree3a382e8bb2e20451fd68db7f664221ec3f5a20bdtargetblank3a382e8a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/tree/3a382e8bb2e20451fd68db7f664221ec3f5a20bd" target="_blank">3a382e8</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    data/.DS_Store
    Ignored:    data/single_group_ctwas/.DS_Store

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/caQTL_cTWAS.Rmd</code>) and HTML
(<code>docs/caQTL_cTWAS.html</code>) files. If you’ve configured a
remote Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/3a382e8bb2e20451fd68db7f664221ec3f5a20bd/analysis/caQTL_cTWAS.Rmd" target="_blank">3a382e8</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-02-20
</td>
<td>
wflow_publish(c("analysis", "data"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/68005459ad1f6b4e47e8fc5b4c581707135460af/docs/caQTL_cTWAS.html" target="_blank">6800545</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-02-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/03d7f697653909f5872f0c4360d8b18b1ddd7921/analysis/caQTL_cTWAS.Rmd" target="_blank">03d7f69</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-02-13
</td>
<td>
wflow_publish("analysis")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/ef4395e10a7a838a47f2ebe9b18e1b72d3e9b34a/docs/caQTL_cTWAS.html" target="_blank">ef4395e</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-02-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/2f98863d20d0950e11293d86b91bbaf703854bbd/analysis/caQTL_cTWAS.Rmd" target="_blank">2f98863</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-02-09
</td>
<td>
wflow_publish("analysis")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/eedfee5819b640e6616ca4b8aea86b52983b5b9c/docs/caQTL_cTWAS.html" target="_blank">eedfee5</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-02-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/78c0cbc5d0856a39a8e12258f53088f989b9e00a/analysis/caQTL_cTWAS.Rmd" target="_blank">78c0cbc</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-02-09
</td>
<td>
wflow_publish(c("analysis", "data"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/1ac12f16783147280374195f6555cce438534140/docs/caQTL_cTWAS.html" target="_blank">1ac12f1</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-01-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/5bbe54f08a652760f2b238187523ea44c929a51f/analysis/caQTL_cTWAS.Rmd" target="_blank">5bbe54f</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-01-05
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/d0538285cf560197dfdde04d45ee1bf90b7aa973/docs/caQTL_cTWAS.html" target="_blank">d053828</a>
</td>
<td>
Lifan Liang
</td>
<td>
2023-12-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/e2bc65fb1294b9a2a87f81213617f9e262b0191b/analysis/caQTL_cTWAS.Rmd" target="_blank">e2bc65f</a>
</td>
<td>
Lifan Liang
</td>
<td>
2023-12-20
</td>
<td>
wflow_publish(c("analysis", "data"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/1f5299991f8a988fc30a150a821bde531885f5e2/docs/caQTL_cTWAS.html" target="_blank">1f52999</a>
</td>
<td>
Lifan Liang
</td>
<td>
2023-12-08
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/feb17830faf52041ed0ad97f06150c705aa15dcb/analysis/caQTL_cTWAS.Rmd" target="_blank">feb1783</a>
</td>
<td>
Lifan Liang
</td>
<td>
2023-12-08
</td>
<td>
wflow_publish(c("analysis", "data"))
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="peak-selection" class="section level2">
<h2>Peak selection</h2>
<p>cPeaks was defined as FDR &lt; 0.05. The inclusion of peaks for each
condition (a unique combination of time points and cell types) is: (1) a
caPeak in any time point in the same cell type; (2) nominal P value &lt;
0.1 in the current condition. The number of peaks was shown in the table
below</p>
<pre><code>Loading required package: stats4</code></pre>
<pre><code>Loading required package: BiocGenerics</code></pre>
<pre><code>
Attaching package: &#39;BiocGenerics&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:stats&#39;:

    IQR, mad, sd, var, xtabs</code></pre>
<pre><code>The following objects are masked from &#39;package:base&#39;:

    anyDuplicated, append, as.data.frame, basename, cbind, colnames,
    dirname, do.call, duplicated, eval, evalq, Filter, Find, get, grep,
    grepl, intersect, is.unsorted, lapply, Map, mapply, match, mget,
    order, paste, pmax, pmax.int, pmin, pmin.int, Position, rank,
    rbind, Reduce, rownames, sapply, setdiff, sort, table, tapply,
    union, unique, unsplit, which.max, which.min</code></pre>
<pre><code>Loading required package: S4Vectors</code></pre>
<pre><code>
Attaching package: &#39;S4Vectors&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:base&#39;:

    expand.grid, I, unname</code></pre>
<pre><code>Loading required package: IRanges</code></pre>
<pre><code>Loading required package: GenomeInfoDb</code></pre>
<table>
<caption>Number of peaks included in cTWAS</caption>
<thead>
<tr class="header">
<th align="left"></th>
<th align="right">0hour</th>
<th align="right">1hour</th>
<th align="right">6hour</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">GABA</td>
<td align="right">11580</td>
<td align="right">12059</td>
<td align="right">12110</td>
</tr>
<tr class="even">
<td align="left">nmglut</td>
<td align="right">6322</td>
<td align="right">6593</td>
<td align="right">6603</td>
</tr>
<tr class="odd">
<td align="left">npglut</td>
<td align="right">13107</td>
<td align="right">13633</td>
<td align="right">13632</td>
</tr>
</tbody>
</table>
</div>
<div id="ctwas-parameter-estimation" class="section level2">
<h2>cTWAS parameter estimation</h2>
<p>For each peak selected, the lead caQTL was used to build the
prediction model. cTWAS first performed parameter estimation without
fine mapping.</p>
<div id="gaba-parameters" class="section level3">
<h3>GABA parameters</h3>
<p><img src="figure/caQTL_cTWAS.Rmd/unnamed-chunk-2-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-2-1">
Past versions of unnamed-chunk-2-1.png
</button>
</p>
<div id="fig-unnamed-chunk-2-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/1f5299991f8a988fc30a150a821bde531885f5e2/docs/figure/caQTL_cTWAS.Rmd/unnamed-chunk-2-1.png" target="_blank">1f52999</a>
</td>
<td>
Lifan Liang
</td>
<td>
2023-12-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>library(RColorBrewer)
myPalette &lt;- brewer.pal(4, &quot;Set2&quot;) 
pie(GABA.params$group_pve, col=myPalette, border=&quot;white&quot;,
    main = paste0(sprintf(&quot;%.2f&quot;, 100-(GABA.params$group_pve[1]/GABA.params$total_pve)*100), &quot;% PVE for GABA peaks&quot;))</code></pre>
<p><img src="figure/caQTL_cTWAS.Rmd/unnamed-chunk-3-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-3-1">
Past versions of unnamed-chunk-3-1.png
</button>
</p>
<div id="fig-unnamed-chunk-3-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/d0538285cf560197dfdde04d45ee1bf90b7aa973/docs/figure/caQTL_cTWAS.Rmd/unnamed-chunk-3-1.png" target="_blank">d053828</a>
</td>
<td>
Lifan Liang
</td>
<td>
2023-12-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/1f5299991f8a988fc30a150a821bde531885f5e2/docs/figure/caQTL_cTWAS.Rmd/unnamed-chunk-3-1.png" target="_blank">1f52999</a>
</td>
<td>
Lifan Liang
</td>
<td>
2023-12-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="npglut-parameters" class="section level3">
<h3>NPglut parameters</h3>
<p><img src="figure/caQTL_cTWAS.Rmd/unnamed-chunk-4-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-4-1">
Past versions of unnamed-chunk-4-1.png
</button>
</p>
<div id="fig-unnamed-chunk-4-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/d0538285cf560197dfdde04d45ee1bf90b7aa973/docs/figure/caQTL_cTWAS.Rmd/unnamed-chunk-4-1.png" target="_blank">d053828</a>
</td>
<td>
Lifan Liang
</td>
<td>
2023-12-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/1f5299991f8a988fc30a150a821bde531885f5e2/docs/figure/caQTL_cTWAS.Rmd/unnamed-chunk-4-1.png" target="_blank">1f52999</a>
</td>
<td>
Lifan Liang
</td>
<td>
2023-12-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/caQTL_cTWAS.Rmd/unnamed-chunk-5-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-5-1">
Past versions of unnamed-chunk-5-1.png
</button>
</p>
<div id="fig-unnamed-chunk-5-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/d0538285cf560197dfdde04d45ee1bf90b7aa973/docs/figure/caQTL_cTWAS.Rmd/unnamed-chunk-5-1.png" target="_blank">d053828</a>
</td>
<td>
Lifan Liang
</td>
<td>
2023-12-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="nmglut-parameters" class="section level3">
<h3>NMglut parameters</h3>
<p><img src="figure/caQTL_cTWAS.Rmd/unnamed-chunk-6-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-6-1">
Past versions of unnamed-chunk-6-1.png
</button>
</p>
<div id="fig-unnamed-chunk-6-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/d0538285cf560197dfdde04d45ee1bf90b7aa973/docs/figure/caQTL_cTWAS.Rmd/unnamed-chunk-6-1.png" target="_blank">d053828</a>
</td>
<td>
Lifan Liang
</td>
<td>
2023-12-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/caQTL_cTWAS.Rmd/unnamed-chunk-7-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-7-1">
Past versions of unnamed-chunk-7-1.png
</button>
</p>
<div id="fig-unnamed-chunk-7-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/d0538285cf560197dfdde04d45ee1bf90b7aa973/docs/figure/caQTL_cTWAS.Rmd/unnamed-chunk-7-1.png" target="_blank">d053828</a>
</td>
<td>
Lifan Liang
</td>
<td>
2023-12-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="significant-peaks" class="section level2">
<h2>Significant peaks</h2>
<p>A peak is significant for the cell type if its sum of PIP across time
points is more than 0.8. There are 22 peaks in GABA, 6 peaks in NPglut,
and 11 peaks in NMglut.</p>
<div id="gaba-significant-peaks" class="section level3">
<h3>GABA significant peaks</h3>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; ">
<table>
<caption>
13 significant peaks for the GABA cell type
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
PeakRange
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
PIP_sum
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
PIP_0hr_GABA
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
PIP_1hr_GABA
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
PIP_6hr_GABA
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
PIP_diff
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
chr12_57096240_57096740
</td>
<td style="text-align:right;">
1.012
</td>
<td style="text-align:right;">
0.429
</td>
<td style="text-align:right;">
0.126
</td>
<td style="text-align:right;">
0.457
</td>
<td style="text-align:right;">
0.153
</td>
</tr>
<tr>
<td style="text-align:left;">
chr13_35764141_35764641
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
1.000
</td>
</tr>
<tr>
<td style="text-align:left;">
chr18_29920961_29921461
</td>
<td style="text-align:right;">
0.964
</td>
<td style="text-align:right;">
0.467
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
0.497
</td>
<td style="text-align:right;">
0.030
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1_115314324_115314824
</td>
<td style="text-align:right;">
0.947
</td>
<td style="text-align:right;">
0.402
</td>
<td style="text-align:right;">
0.117
</td>
<td style="text-align:right;">
0.428
</td>
<td style="text-align:right;">
0.143
</td>
</tr>
<tr>
<td style="text-align:left;">
chr7_2010548_2011048
</td>
<td style="text-align:right;">
0.945
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
0.945
</td>
<td style="text-align:right;">
0.945
</td>
</tr>
<tr>
<td style="text-align:left;">
chr7_157665058_157665558
</td>
<td style="text-align:right;">
0.915
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
0.915
</td>
<td style="text-align:right;">
0.915
</td>
</tr>
<tr>
<td style="text-align:left;">
chr19_5977436_5977936
</td>
<td style="text-align:right;">
0.903
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
0.903
</td>
<td style="text-align:right;">
0.903
</td>
</tr>
<tr>
<td style="text-align:left;">
chr3_175410533_175411033
</td>
<td style="text-align:right;">
0.900
</td>
<td style="text-align:right;">
0.181
</td>
<td style="text-align:right;">
0.155
</td>
<td style="text-align:right;">
0.564
</td>
<td style="text-align:right;">
0.538
</td>
</tr>
<tr>
<td style="text-align:left;">
chr5_90929346_90929846
</td>
<td style="text-align:right;">
0.872
</td>
<td style="text-align:right;">
0.370
</td>
<td style="text-align:right;">
0.108
</td>
<td style="text-align:right;">
0.394
</td>
<td style="text-align:right;">
0.132
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2_65539480_65539980
</td>
<td style="text-align:right;">
0.857
</td>
<td style="text-align:right;">
0.390
</td>
<td style="text-align:right;">
0.052
</td>
<td style="text-align:right;">
0.415
</td>
<td style="text-align:right;">
0.077
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11_64044962_64045462
</td>
<td style="text-align:right;">
0.848
</td>
<td style="text-align:right;">
0.656
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
0.192
</td>
<td style="text-align:right;">
-0.464
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11_113577991_113578491
</td>
<td style="text-align:right;">
0.842
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
0.260
</td>
<td style="text-align:right;">
0.582
</td>
<td style="text-align:right;">
0.842
</td>
</tr>
<tr>
<td style="text-align:left;">
chr5_89432966_89433466
</td>
<td style="text-align:right;">
0.823
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
0.823
</td>
<td style="text-align:right;">
0.823
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="npglut-significant-peaks" class="section level3">
<h3>npglut significant peaks</h3>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; ">
<table>
<caption>
5 significant peaks for the npglut cell type
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
PeakRange
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
PIP_sum
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
PIP_0hr_npglut
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
PIP_1hr_npglut
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
PIP_6hr_npglut
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
PIP_diff
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
chr12_57096240_57096740
</td>
<td style="text-align:right;">
0.985
</td>
<td style="text-align:right;">
0.134
</td>
<td style="text-align:right;">
0.851
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
0.718
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2_172098931_172099431
</td>
<td style="text-align:right;">
0.904
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
0.904
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
0.904
</td>
</tr>
<tr>
<td style="text-align:left;">
chr18_29920961_29921461
</td>
<td style="text-align:right;">
0.893
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
0.893
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
0.893
</td>
</tr>
<tr>
<td style="text-align:left;">
chr10_86244301_86244801
</td>
<td style="text-align:right;">
0.852
</td>
<td style="text-align:right;">
0.106
</td>
<td style="text-align:right;">
0.672
</td>
<td style="text-align:right;">
0.073
</td>
<td style="text-align:right;">
0.640
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11_64044962_64045462
</td>
<td style="text-align:right;">
0.835
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
0.785
</td>
<td style="text-align:right;">
0.050
</td>
<td style="text-align:right;">
0.835
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="nmglut-significant-peaks" class="section level3">
<h3>nmglut significant peaks</h3>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; ">
<table>
<caption>
9 significant peaks for the nmglut cell type
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
PeakRange
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
PIP_sum
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
PIP_0hr_nmglut
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
PIP_1hr_nmglut
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
PIP_6hr_nmglut
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
PIP_diff
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
chr6_92378607_92379107
</td>
<td style="text-align:right;">
0.975
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
0.975
</td>
<td style="text-align:right;">
0.975
</td>
</tr>
<tr>
<td style="text-align:left;">
chr22_41362794_41363294
</td>
<td style="text-align:right;">
0.969
</td>
<td style="text-align:right;">
0.969
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
-0.969
</td>
</tr>
<tr>
<td style="text-align:left;">
chr7_1986364_1986864
</td>
<td style="text-align:right;">
0.966
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
0.966
</td>
<td style="text-align:right;">
0.966
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2_172098931_172099431
</td>
<td style="text-align:right;">
0.935
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
0.237
</td>
<td style="text-align:right;">
0.699
</td>
<td style="text-align:right;">
0.935
</td>
</tr>
<tr>
<td style="text-align:left;">
chr7_2008327_2008827
</td>
<td style="text-align:right;">
0.877
</td>
<td style="text-align:right;">
0.877
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
-0.877
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11_113577991_113578491
</td>
<td style="text-align:right;">
0.874
</td>
<td style="text-align:right;">
0.203
</td>
<td style="text-align:right;">
0.197
</td>
<td style="text-align:right;">
0.474
</td>
<td style="text-align:right;">
0.468
</td>
</tr>
<tr>
<td style="text-align:left;">
chr12_2191122_2191622
</td>
<td style="text-align:right;">
0.836
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
0.034
</td>
<td style="text-align:right;">
0.803
</td>
<td style="text-align:right;">
0.836
</td>
</tr>
<tr>
<td style="text-align:left;">
chr5_135670508_135671008
</td>
<td style="text-align:right;">
0.822
</td>
<td style="text-align:right;">
0.248
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
0.574
</td>
<td style="text-align:right;">
0.326
</td>
</tr>
<tr>
<td style="text-align:left;">
chr5_90929346_90929846
</td>
<td style="text-align:right;">
0.816
</td>
<td style="text-align:right;">
0.224
</td>
<td style="text-align:right;">
0.176
</td>
<td style="text-align:right;">
0.416
</td>
<td style="text-align:right;">
0.368
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="cell-type-overlap" class="section level2">
<h2>Cell type overlap</h2>
<div id="overlap-of-the-ctwas-input" class="section level3">
<h3>Overlap of the cTWAS input</h3>
<p><img src="figure/caQTL_cTWAS.Rmd/unnamed-chunk-11-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-11-1">
Past versions of unnamed-chunk-11-1.png
</button>
</p>
<div id="fig-unnamed-chunk-11-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/d0538285cf560197dfdde04d45ee1bf90b7aa973/docs/figure/caQTL_cTWAS.Rmd/unnamed-chunk-11-1.png" target="_blank">d053828</a>
</td>
<td>
Lifan Liang
</td>
<td>
2023-12-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="overlap-of-the-ctwas-significant-peaks" class="section level3">
<h3>Overlap of the cTWAS significant peaks</h3>
<p><img src="figure/caQTL_cTWAS.Rmd/unnamed-chunk-12-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-12-1">
Past versions of unnamed-chunk-12-1.png
</button>
</p>
<div id="fig-unnamed-chunk-12-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/d0538285cf560197dfdde04d45ee1bf90b7aa973/docs/figure/caQTL_cTWAS.Rmd/unnamed-chunk-12-1.png" target="_blank">d053828</a>
</td>
<td>
Lifan Liang
</td>
<td>
2023-12-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="dynamic-cpeaks" class="section level2">
<h2>Dynamic cPeaks</h2>
<p>In addition to PIP sum &gt; 0.8, dynamic cpeaks must satisfy
that:</p>
<p><span class="math display">\[
PIP_{GABA6} + PIP_{GABA1} - PIP_{GABA0} \geq 0.5
\]</span> for the GABA cell type. The same condition applies to the
other cell types .</p>
<p>Peaks not included in one time point are regarded as <span
class="math inline">\(PIP=0\)</span>.</p>
<p><img src="figure/caQTL_cTWAS.Rmd/unnamed-chunk-13-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-13-1">
Past versions of unnamed-chunk-13-1.png
</button>
</p>
<div id="fig-unnamed-chunk-13-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/d0538285cf560197dfdde04d45ee1bf90b7aa973/docs/figure/caQTL_cTWAS.Rmd/unnamed-chunk-13-1.png" target="_blank">d053828</a>
</td>
<td>
Lifan Liang
</td>
<td>
2023-12-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="cell-type-correlation" class="section level2">
<h2>Cell type correlation</h2>
<div
id="comparing-top-cpeaks-across-cell-types-nominated-in-the-same-locus"
class="section level4">
<h4>Comparing top cPeaks across cell types nominated in the same
locus</h4>
<p>This table shows how different cell types may nominate different
peaks in the same locus.</p>
<p>All shared peaks are consistent except 7_4 in chr7. Also note that
both “7_3” and “7_4” nominate chr7_2010548_2011048.</p>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; ">
<table>
<caption>
Dynamic genes by locus
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
locus_tag
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
GABA
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
npglut
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
nmglut
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
1_71
</td>
<td style="text-align:left;">
chr1_115314324_115314824
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
2_42
</td>
<td style="text-align:left;">
chr2_65539480_65539980
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
3_108
</td>
<td style="text-align:left;">
chr3_175410533_175411033
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
5_52
</td>
<td style="text-align:left;">
chr5_89432966_89433466
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
5_53
</td>
<td style="text-align:left;">
chr5_90929346_90929846
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
chr5_90929346_90929846
</td>
</tr>
<tr>
<td style="text-align:left;">
7_3
</td>
<td style="text-align:left;">
chr7_2010548_2011048
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
7_4
</td>
<td style="text-align:left;">
chr7_2010548_2011048
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
chr7_2008327_2008827
</td>
</tr>
<tr>
<td style="text-align:left;">
7_98
</td>
<td style="text-align:left;">
chr7_157665058_157665558
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
11_36
</td>
<td style="text-align:left;">
chr11_64044962_64045462
</td>
<td style="text-align:left;">
chr11_64044962_64045462
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
11_67
</td>
<td style="text-align:left;">
chr11_113577991_113578491
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
chr11_113577991_113578491
</td>
</tr>
<tr>
<td style="text-align:left;">
12_35
</td>
<td style="text-align:left;">
chr12_57096240_57096740
</td>
<td style="text-align:left;">
chr12_57096240_57096740
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
13_13
</td>
<td style="text-align:left;">
chr13_35764141_35764641
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
18_16
</td>
<td style="text-align:left;">
chr18_29920961_29921461
</td>
<td style="text-align:left;">
chr18_29920961_29921461
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
19_6
</td>
<td style="text-align:left;">
chr19_5977436_5977936
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
2_104
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
chr2_172098931_172099431
</td>
<td style="text-align:left;">
chr2_172098931_172099431
</td>
</tr>
<tr>
<td style="text-align:left;">
10_55
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
chr10_86244301_86244801
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
5_81
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
chr5_135670508_135671008
</td>
</tr>
<tr>
<td style="text-align:left;">
6_62
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
chr6_92378607_92379107
</td>
</tr>
<tr>
<td style="text-align:left;">
12_2
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
chr12_2191122_2191622
</td>
</tr>
<tr>
<td style="text-align:left;">
22_17
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
chr22_41362794_41363294
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="weak-signal-aggregation" class="section level2">
<h2>Weak signal aggregation</h2>
<p>We tried to aggregate PIP of each peak in several ways: * merge peaks
to the same lead variants * merge peaks to the nearest the gene in terms
of distance between lead variants and gene bodies * merge peaks in the
same locus</p>
<div id="power-gain-via-aggregation" class="section level3">
<h3>Power gain via aggregation</h3>
<p><img src="figure/caQTL_cTWAS.Rmd/sigcount-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>Figure @ref(fig:sigcount) shows that significant drops after SNP
aggregation and then increases through gene and locus level aggregation.
SNP level detects less significant units because most cPeaks at one
timepoint correspond to one SNP only (Figure @ref(fig:peak2snp).</p>
<p><img src="figure/caQTL_cTWAS.Rmd/peak2snp-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="snp-level-significant-units" class="section level3">
<h3>SNP level significant units</h3>
<p>Aggregate PIPs of peaks by their shared lead variants.</p>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
4 SNPs with a sum of PIP&gt;0.8 in npglut
</caption>
<thead>
<tr>
<th style="text-align:left;">
snp
</th>
<th style="text-align:right;">
0hr
</th>
<th style="text-align:right;">
1hr
</th>
<th style="text-align:right;">
6hr
</th>
<th style="text-align:right;">
PIP_sum
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
12_57096317
</td>
<td style="text-align:right;">
0.1336861
</td>
<td style="text-align:right;">
0.8514970
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.9851832
</td>
</tr>
<tr>
<td style="text-align:left;">
2_172091721
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.9043174
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.9043174
</td>
</tr>
<tr>
<td style="text-align:left;">
18_29920663
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.8932926
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.8932926
</td>
</tr>
<tr>
<td style="text-align:left;">
10_86233362
</td>
<td style="text-align:right;">
0.1060814
</td>
<td style="text-align:right;">
0.6722493
</td>
<td style="text-align:right;">
0.0734215
</td>
<td style="text-align:right;">
0.8517522
</td>
</tr>
</tbody>
</table>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
10 SNPs with a sum of PIP&gt;0.8 in GABA
</caption>
<thead>
<tr>
<th style="text-align:left;">
snp
</th>
<th style="text-align:right;">
0hr
</th>
<th style="text-align:right;">
1hr
</th>
<th style="text-align:right;">
6hr
</th>
<th style="text-align:right;">
PIP_sum
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
12_57096317
</td>
<td style="text-align:right;">
0.4294953
</td>
<td style="text-align:right;">
0.1255602
</td>
<td style="text-align:right;">
0.4567561
</td>
<td style="text-align:right;">
1.0118116
</td>
</tr>
<tr>
<td style="text-align:left;">
13_35742112
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.9999771
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.9999771
</td>
</tr>
<tr>
<td style="text-align:left;">
18_29920663
</td>
<td style="text-align:right;">
0.4672065
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.4967968
</td>
<td style="text-align:right;">
0.9640032
</td>
</tr>
<tr>
<td style="text-align:left;">
1_115309383
</td>
<td style="text-align:right;">
0.4023598
</td>
<td style="text-align:right;">
0.1169965
</td>
<td style="text-align:right;">
0.4281076
</td>
<td style="text-align:right;">
0.9474639
</td>
</tr>
<tr>
<td style="text-align:left;">
7_2027958
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.9453575
</td>
<td style="text-align:right;">
0.9453575
</td>
</tr>
<tr>
<td style="text-align:left;">
7_157672322
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.9150728
</td>
<td style="text-align:right;">
0.9150728
</td>
</tr>
<tr>
<td style="text-align:left;">
19_5998231
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.9028704
</td>
<td style="text-align:right;">
0.9028704
</td>
</tr>
<tr>
<td style="text-align:left;">
5_90920989
</td>
<td style="text-align:right;">
0.3701993
</td>
<td style="text-align:right;">
0.1080968
</td>
<td style="text-align:right;">
0.3937391
</td>
<td style="text-align:right;">
0.8720353
</td>
</tr>
<tr>
<td style="text-align:left;">
5_89448145
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.8225518
</td>
<td style="text-align:right;">
0.8225518
</td>
</tr>
<tr>
<td style="text-align:left;">
2_65542897
</td>
<td style="text-align:right;">
0.3903023
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.4149691
</td>
<td style="text-align:right;">
0.8052714
</td>
</tr>
</tbody>
</table>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
7 SNPs with a sum of PIP&gt;0.8 in nmglut
</caption>
<thead>
<tr>
<th style="text-align:left;">
snp
</th>
<th style="text-align:right;">
0hr
</th>
<th style="text-align:right;">
1hr
</th>
<th style="text-align:right;">
6hr
</th>
<th style="text-align:right;">
PIP_sum
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
6_92367782
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.9751299
</td>
<td style="text-align:right;">
0.9751299
</td>
</tr>
<tr>
<td style="text-align:left;">
22_41357599
</td>
<td style="text-align:right;">
0.9685206
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.9685206
</td>
</tr>
<tr>
<td style="text-align:left;">
7_1990232
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.9660026
</td>
<td style="text-align:right;">
0.9660026
</td>
</tr>
<tr>
<td style="text-align:left;">
2_172092378
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.2367689
</td>
<td style="text-align:right;">
0.6986057
</td>
<td style="text-align:right;">
0.9353746
</td>
</tr>
<tr>
<td style="text-align:left;">
7_2027958
</td>
<td style="text-align:right;">
0.8765450
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.8765450
</td>
</tr>
<tr>
<td style="text-align:left;">
5_135671420
</td>
<td style="text-align:right;">
0.2480799
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.5741231
</td>
<td style="text-align:right;">
0.8222031
</td>
</tr>
<tr>
<td style="text-align:left;">
12_2188830
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.8026575
</td>
<td style="text-align:right;">
0.8026575
</td>
</tr>
</tbody>
</table>
</div>
<div id="gene-level" class="section level3">
<h3>Gene level</h3>
<p>After aggregating PIP to SNP, we assign each SNP to its nearest gene
by TSS. SNP-gene pair with distance over one million bps were removed.
Then the PIP of SNPs were attributed to its nearest genes.</p>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
14 genes with a sum of PIP&gt;0.8 in GABA
</caption>
<thead>
<tr>
<th style="text-align:left;">
entrez
</th>
<th style="text-align:left;">
gene_name
</th>
<th style="text-align:right;">
0hr
</th>
<th style="text-align:right;">
1hr
</th>
<th style="text-align:right;">
6hr
</th>
<th style="text-align:right;">
PIP_sum
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
153478
</td>
<td style="text-align:left;">
PLEKHG4B
</td>
<td style="text-align:right;">
2.0646827
</td>
<td style="text-align:right;">
0.3718286
</td>
<td style="text-align:right;">
3.3946297
</td>
<td style="text-align:right;">
5.8311410
</td>
</tr>
<tr>
<td style="text-align:left;">
79850
</td>
<td style="text-align:left;">
TLCD3A
</td>
<td style="text-align:right;">
0.9692924
</td>
<td style="text-align:right;">
0.2260241
</td>
<td style="text-align:right;">
2.2990269
</td>
<td style="text-align:right;">
3.4943434
</td>
</tr>
<tr>
<td style="text-align:left;">
56940
</td>
<td style="text-align:left;">
DUSP22
</td>
<td style="text-align:right;">
2.3674471
</td>
<td style="text-align:right;">
0.1199076
</td>
<td style="text-align:right;">
0.5430639
</td>
<td style="text-align:right;">
3.0304186
</td>
</tr>
<tr>
<td style="text-align:left;">
6539
</td>
<td style="text-align:left;">
SLC6A12
</td>
<td style="text-align:right;">
1.6090881
</td>
<td style="text-align:right;">
0.2436348
</td>
<td style="text-align:right;">
1.1708797
</td>
<td style="text-align:right;">
3.0236026
</td>
</tr>
<tr>
<td style="text-align:left;">
642273
</td>
<td style="text-align:left;">
FAM110C
</td>
<td style="text-align:right;">
0.5570514
</td>
<td style="text-align:right;">
0.2387649
</td>
<td style="text-align:right;">
2.1563550
</td>
<td style="text-align:right;">
2.9521713
</td>
</tr>
<tr>
<td style="text-align:left;">
116033993
</td>
<td style="text-align:left;">
FOXL3
</td>
<td style="text-align:right;">
0.1027212
</td>
<td style="text-align:right;">
0.1937026
</td>
<td style="text-align:right;">
1.9770449
</td>
<td style="text-align:right;">
2.2734688
</td>
</tr>
<tr>
<td style="text-align:left;">
147199
</td>
<td style="text-align:left;">
SCGB1C1
</td>
<td style="text-align:right;">
0.6795426
</td>
<td style="text-align:right;">
0.5435868
</td>
<td style="text-align:right;">
0.7992410
</td>
<td style="text-align:right;">
2.0223704
</td>
</tr>
<tr>
<td style="text-align:left;">
81099
</td>
<td style="text-align:left;">
OR4F17
</td>
<td style="text-align:right;">
0.2781236
</td>
<td style="text-align:right;">
0.0304635
</td>
<td style="text-align:right;">
1.3081981
</td>
<td style="text-align:right;">
1.6167853
</td>
</tr>
<tr>
<td style="text-align:left;">
79501
</td>
<td style="text-align:left;">
OR4F5
</td>
<td style="text-align:right;">
0.5149275
</td>
<td style="text-align:right;">
0.1656049
</td>
<td style="text-align:right;">
0.8886191
</td>
<td style="text-align:right;">
1.5691515
</td>
</tr>
<tr>
<td style="text-align:left;">
10752
</td>
<td style="text-align:left;">
CHL1
</td>
<td style="text-align:right;">
0.3500099
</td>
<td style="text-align:right;">
0.4381771
</td>
<td style="text-align:right;">
0.7770477
</td>
<td style="text-align:right;">
1.5652347
</td>
</tr>
<tr>
<td style="text-align:left;">
152687
</td>
<td style="text-align:left;">
ZNF595
</td>
<td style="text-align:right;">
0.9551037
</td>
<td style="text-align:right;">
0.1154915
</td>
<td style="text-align:right;">
0.4748242
</td>
<td style="text-align:right;">
1.5454195
</td>
</tr>
<tr>
<td style="text-align:left;">
441308
</td>
<td style="text-align:left;">
OR4F21
</td>
<td style="text-align:right;">
0.5373962
</td>
<td style="text-align:right;">
0.1722232
</td>
<td style="text-align:right;">
0.6962693
</td>
<td style="text-align:right;">
1.4058887
</td>
</tr>
<tr>
<td style="text-align:left;">
51728
</td>
<td style="text-align:left;">
POLR3K
</td>
<td style="text-align:right;">
0.1981058
</td>
<td style="text-align:right;">
0.1470473
</td>
<td style="text-align:right;">
0.6939295
</td>
<td style="text-align:right;">
1.0390826
</td>
</tr>
<tr>
<td style="text-align:left;">
260334
</td>
<td style="text-align:left;">
TUBB8B
</td>
<td style="text-align:right;">
0.4672065
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
0.4967968
</td>
<td style="text-align:right;">
0.9640032
</td>
</tr>
</tbody>
</table>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
11 genes with a sum of PIP&gt;0.8 in npglut
</caption>
<thead>
<tr>
<th style="text-align:left;">
entrez
</th>
<th style="text-align:left;">
gene_name
</th>
<th style="text-align:right;">
0hr
</th>
<th style="text-align:right;">
1hr
</th>
<th style="text-align:right;">
6hr
</th>
<th style="text-align:right;">
PIP_sum
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
147199
</td>
<td style="text-align:left;">
SCGB1C1
</td>
<td style="text-align:right;">
0.7717690
</td>
<td style="text-align:right;">
1.8012047
</td>
<td style="text-align:right;">
0.5578409
</td>
<td style="text-align:right;">
3.1308146
</td>
</tr>
<tr>
<td style="text-align:left;">
10752
</td>
<td style="text-align:left;">
CHL1
</td>
<td style="text-align:right;">
0.5807502
</td>
<td style="text-align:right;">
1.6709706
</td>
<td style="text-align:right;">
0.0632061
</td>
<td style="text-align:right;">
2.3149269
</td>
</tr>
<tr>
<td style="text-align:left;">
153478
</td>
<td style="text-align:left;">
PLEKHG4B
</td>
<td style="text-align:right;">
0.1767481
</td>
<td style="text-align:right;">
1.8801067
</td>
<td style="text-align:right;">
0.1273833
</td>
<td style="text-align:right;">
2.1842382
</td>
</tr>
<tr>
<td style="text-align:left;">
642273
</td>
<td style="text-align:left;">
FAM110C
</td>
<td style="text-align:right;">
0.0794568
</td>
<td style="text-align:right;">
2.0154440
</td>
<td style="text-align:right;">
0.0296686
</td>
<td style="text-align:right;">
2.1245694
</td>
</tr>
<tr>
<td style="text-align:left;">
79501
</td>
<td style="text-align:left;">
OR4F5
</td>
<td style="text-align:right;">
0.1222013
</td>
<td style="text-align:right;">
1.7474011
</td>
<td style="text-align:right;">
0.0606685
</td>
<td style="text-align:right;">
1.9302709
</td>
</tr>
<tr>
<td style="text-align:left;">
347688
</td>
<td style="text-align:left;">
TUBB8
</td>
<td style="text-align:right;">
0.1060814
</td>
<td style="text-align:right;">
1.3095797
</td>
<td style="text-align:right;">
0.1366785
</td>
<td style="text-align:right;">
1.5523396
</td>
</tr>
<tr>
<td style="text-align:left;">
6539
</td>
<td style="text-align:left;">
SLC6A12
</td>
<td style="text-align:right;">
0.1880736
</td>
<td style="text-align:right;">
1.0990499
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
1.2871235
</td>
</tr>
<tr>
<td style="text-align:left;">
441308
</td>
<td style="text-align:left;">
OR4F21
</td>
<td style="text-align:right;">
0.0748905
</td>
<td style="text-align:right;">
0.9931209
</td>
<td style="text-align:right;">
0.1160667
</td>
<td style="text-align:right;">
1.1840781
</td>
</tr>
<tr>
<td style="text-align:left;">
116033993
</td>
<td style="text-align:left;">
FOXL3
</td>
<td style="text-align:right;">
0.0998150
</td>
<td style="text-align:right;">
1.0728347
</td>
<td style="text-align:right;">
0.0107396
</td>
<td style="text-align:right;">
1.1833893
</td>
</tr>
<tr>
<td style="text-align:left;">
260334
</td>
<td style="text-align:left;">
TUBB8B
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
0.8932926
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
0.8932926
</td>
</tr>
<tr>
<td style="text-align:left;">
100287171
</td>
<td style="text-align:left;">
WASHC1
</td>
<td style="text-align:right;">
0.0901979
</td>
<td style="text-align:right;">
0.6749801
</td>
<td style="text-align:right;">
0.0720088
</td>
<td style="text-align:right;">
0.8371868
</td>
</tr>
</tbody>
</table>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
9 genes with a sum of PIP&gt;0.8 in nmglut
</caption>
<thead>
<tr>
<th style="text-align:left;">
entrez
</th>
<th style="text-align:left;">
gene_name
</th>
<th style="text-align:right;">
0hr
</th>
<th style="text-align:right;">
1hr
</th>
<th style="text-align:right;">
6hr
</th>
<th style="text-align:right;">
PIP_sum
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
56940
</td>
<td style="text-align:left;">
DUSP22
</td>
<td style="text-align:right;">
0.9678381
</td>
<td style="text-align:right;">
0.6161381
</td>
<td style="text-align:right;">
2.1732086
</td>
<td style="text-align:right;">
3.7571849
</td>
</tr>
<tr>
<td style="text-align:left;">
116033993
</td>
<td style="text-align:left;">
FOXL3
</td>
<td style="text-align:right;">
0.8991185
</td>
<td style="text-align:right;">
0.1075336
</td>
<td style="text-align:right;">
1.3344591
</td>
<td style="text-align:right;">
2.3411112
</td>
</tr>
<tr>
<td style="text-align:left;">
153478
</td>
<td style="text-align:left;">
PLEKHG4B
</td>
<td style="text-align:right;">
0.5104614
</td>
<td style="text-align:right;">
0.5066379
</td>
<td style="text-align:right;">
1.0463948
</td>
<td style="text-align:right;">
2.0634941
</td>
</tr>
<tr>
<td style="text-align:left;">
147199
</td>
<td style="text-align:left;">
SCGB1C1
</td>
<td style="text-align:right;">
0.3690119
</td>
<td style="text-align:right;">
0.4290185
</td>
<td style="text-align:right;">
0.8275264
</td>
<td style="text-align:right;">
1.6255569
</td>
</tr>
<tr>
<td style="text-align:left;">
642273
</td>
<td style="text-align:left;">
FAM110C
</td>
<td style="text-align:right;">
0.0377058
</td>
<td style="text-align:right;">
0.2367689
</td>
<td style="text-align:right;">
0.8159939
</td>
<td style="text-align:right;">
1.0904686
</td>
</tr>
<tr>
<td style="text-align:left;">
152687
</td>
<td style="text-align:left;">
ZNF595
</td>
<td style="text-align:right;">
0.0333377
</td>
<td style="text-align:right;">
0.0299158
</td>
<td style="text-align:right;">
0.9912925
</td>
<td style="text-align:right;">
1.0545460
</td>
</tr>
<tr>
<td style="text-align:left;">
6539
</td>
<td style="text-align:left;">
SLC6A12
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
0.1518917
</td>
<td style="text-align:right;">
0.8448997
</td>
<td style="text-align:right;">
0.9967914
</td>
</tr>
<tr>
<td style="text-align:left;">
441308
</td>
<td style="text-align:left;">
OR4F21
</td>
<td style="text-align:right;">
0.0873300
</td>
<td style="text-align:right;">
0.0756479
</td>
<td style="text-align:right;">
0.8194241
</td>
<td style="text-align:right;">
0.9824020
</td>
</tr>
<tr>
<td style="text-align:left;">
79850
</td>
<td style="text-align:left;">
TLCD3A
</td>
<td style="text-align:right;">
0.0925506
</td>
<td style="text-align:right;">
0.3125340
</td>
<td style="text-align:right;">
0.5108540
</td>
<td style="text-align:right;">
0.9159386
</td>
</tr>
</tbody>
</table>
</div>
<div id="locus-level" class="section level3">
<h3>Locus level</h3>
<div id="npglut" class="section level4">
<h4>NPglut</h4>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; ">
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
6 loci with a sum of PIP&gt;0.8 in npglut
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
locus
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
0hr
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
1hr
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
6hr
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
PIP_sum
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
12_35
</td>
<td style="text-align:right;">
0.1336861
</td>
<td style="text-align:right;">
0.8514970
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.9851832
</td>
</tr>
<tr>
<td style="text-align:left;">
2_104
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.9043174
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.9043174
</td>
</tr>
<tr>
<td style="text-align:left;">
10_55
</td>
<td style="text-align:right;">
0.1060814
</td>
<td style="text-align:right;">
0.7234694
</td>
<td style="text-align:right;">
0.0734215
</td>
<td style="text-align:right;">
0.9029723
</td>
</tr>
<tr>
<td style="text-align:left;">
22_17
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.8962075
</td>
<td style="text-align:right;">
0.8962075
</td>
</tr>
<tr>
<td style="text-align:left;">
18_16
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.8932926
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.8932926
</td>
</tr>
<tr>
<td style="text-align:left;">
11_36
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.7846408
</td>
<td style="text-align:right;">
0.0501284
</td>
<td style="text-align:right;">
0.8347692
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="gaba" class="section level4">
<h4>GABA</h4>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; ">
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
18 loci with a sum of PIP&gt;0.8 in GABA
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
locus
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
0hr
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
1hr
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
6hr
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
PIP_sum
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
11_67
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.5343491
</td>
<td style="text-align:right;">
0.5821666
</td>
<td style="text-align:right;">
1.1165158
</td>
</tr>
<tr>
<td style="text-align:left;">
12_35
</td>
<td style="text-align:right;">
0.4294953
</td>
<td style="text-align:right;">
0.1255602
</td>
<td style="text-align:right;">
0.4567561
</td>
<td style="text-align:right;">
1.0118116
</td>
</tr>
<tr>
<td style="text-align:left;">
13_13
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.9999771
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.9999771
</td>
</tr>
<tr>
<td style="text-align:left;">
7_4
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.9750582
</td>
<td style="text-align:right;">
0.9750582
</td>
</tr>
<tr>
<td style="text-align:left;">
17_27
</td>
<td style="text-align:right;">
0.4428002
</td>
<td style="text-align:right;">
0.0775082
</td>
<td style="text-align:right;">
0.4437574
</td>
<td style="text-align:right;">
0.9640657
</td>
</tr>
<tr>
<td style="text-align:left;">
18_16
</td>
<td style="text-align:right;">
0.4672065
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.4967968
</td>
<td style="text-align:right;">
0.9640032
</td>
</tr>
<tr>
<td style="text-align:left;">
1_71
</td>
<td style="text-align:right;">
0.4023598
</td>
<td style="text-align:right;">
0.1169965
</td>
<td style="text-align:right;">
0.4281076
</td>
<td style="text-align:right;">
0.9474639
</td>
</tr>
<tr>
<td style="text-align:left;">
8_3
</td>
<td style="text-align:right;">
0.2506064
</td>
<td style="text-align:right;">
0.0735730
</td>
<td style="text-align:right;">
0.6005350
</td>
<td style="text-align:right;">
0.9247145
</td>
</tr>
<tr>
<td style="text-align:left;">
7_98
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.9150728
</td>
<td style="text-align:right;">
0.9150728
</td>
</tr>
<tr>
<td style="text-align:left;">
19_6
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.9028704
</td>
<td style="text-align:right;">
0.9028704
</td>
</tr>
<tr>
<td style="text-align:left;">
3_108
</td>
<td style="text-align:right;">
0.1809504
</td>
<td style="text-align:right;">
0.1550705
</td>
<td style="text-align:right;">
0.5642721
</td>
<td style="text-align:right;">
0.9002929
</td>
</tr>
<tr>
<td style="text-align:left;">
5_53
</td>
<td style="text-align:right;">
0.3701993
</td>
<td style="text-align:right;">
0.1080968
</td>
<td style="text-align:right;">
0.3937391
</td>
<td style="text-align:right;">
0.8720353
</td>
</tr>
<tr>
<td style="text-align:left;">
5_92
</td>
<td style="text-align:right;">
0.1133957
</td>
<td style="text-align:right;">
0.1838282
</td>
<td style="text-align:right;">
0.5729803
</td>
<td style="text-align:right;">
0.8702042
</td>
</tr>
<tr>
<td style="text-align:left;">
2_42
</td>
<td style="text-align:right;">
0.3903023
</td>
<td style="text-align:right;">
0.0520022
</td>
<td style="text-align:right;">
0.4149691
</td>
<td style="text-align:right;">
0.8572736
</td>
</tr>
<tr>
<td style="text-align:left;">
11_36
</td>
<td style="text-align:right;">
0.6559164
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.1919371
</td>
<td style="text-align:right;">
0.8478535
</td>
</tr>
<tr>
<td style="text-align:left;">
6_21
</td>
<td style="text-align:right;">
0.7105724
</td>
<td style="text-align:right;">
0.0162137
</td>
<td style="text-align:right;">
0.1193322
</td>
<td style="text-align:right;">
0.8461183
</td>
</tr>
<tr>
<td style="text-align:left;">
5_52
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.8225518
</td>
<td style="text-align:right;">
0.8225518
</td>
</tr>
<tr>
<td style="text-align:left;">
22_20
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.2224478
</td>
<td style="text-align:right;">
0.5775814
</td>
<td style="text-align:right;">
0.8000292
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="nmglut" class="section level4">
<h4>NMglut</h4>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; ">
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
10 loci with a sum of PIP&gt;0.8 in nmglut
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
locus
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
0hr
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
1hr
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
6hr
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
PIP_sum
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
6_62
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.9751299
</td>
<td style="text-align:right;">
0.9751299
</td>
</tr>
<tr>
<td style="text-align:left;">
22_17
</td>
<td style="text-align:right;">
0.9685206
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.9685206
</td>
</tr>
<tr>
<td style="text-align:left;">
7_3
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.9660026
</td>
<td style="text-align:right;">
0.9660026
</td>
</tr>
<tr>
<td style="text-align:left;">
7_4
</td>
<td style="text-align:right;">
0.8991185
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.0525755
</td>
<td style="text-align:right;">
0.9516940
</td>
</tr>
<tr>
<td style="text-align:left;">
2_104
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.2367689
</td>
<td style="text-align:right;">
0.6986057
</td>
<td style="text-align:right;">
0.9353746
</td>
</tr>
<tr>
<td style="text-align:left;">
11_67
</td>
<td style="text-align:right;">
0.2033132
</td>
<td style="text-align:right;">
0.1971618
</td>
<td style="text-align:right;">
0.4737916
</td>
<td style="text-align:right;">
0.8742666
</td>
</tr>
<tr>
<td style="text-align:left;">
6_25
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.3974046
</td>
<td style="text-align:right;">
0.4439669
</td>
<td style="text-align:right;">
0.8413715
</td>
</tr>
<tr>
<td style="text-align:left;">
12_2
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.0335081
</td>
<td style="text-align:right;">
0.8026575
</td>
<td style="text-align:right;">
0.8361656
</td>
</tr>
<tr>
<td style="text-align:left;">
5_81
</td>
<td style="text-align:right;">
0.2480799
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
0.5741231
</td>
<td style="text-align:right;">
0.8222031
</td>
</tr>
<tr>
<td style="text-align:left;">
5_53
</td>
<td style="text-align:right;">
0.2237801
</td>
<td style="text-align:right;">
0.1759913
</td>
<td style="text-align:right;">
0.4158145
</td>
<td style="text-align:right;">
0.8155859
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="shared-signals-between-ctwas-and-asoc" class="section level2">
<h2>Shared signals between cTWAS and ASoC</h2>
<p>For each cell type, if an allelic specific open chromatin (ASoC) SNP
(FDR&lt;0.05) is within 500 bps upstream and downstream of a cPeak and
the matched cPeak has GWAS PIP &gt; 0.8, then they shared the same
signal.</p>
<div id="gaba-1" class="section level3">
<h3>GABA</h3>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
Significant cTWAS peaks overlapped with ASoC in GABA 0 hours
</caption>
<thead>
<tr>
<th style="text-align:left;">
peakrange
</th>
<th style="text-align:left;">
ASOC_SNP
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
chr11_113577991_113578491
</td>
<td style="text-align:left;">
rs7928017
</td>
</tr>
</tbody>
</table>
<p><img src="figure/caQTL_cTWAS.Rmd/unnamed-chunk-25-1.png" width="672" style="display: block; margin: auto;" /></p>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
Significant cTWAS peaks overlapped with ASoC in GABA 1 hours
</caption>
<thead>
<tr>
<th style="text-align:left;">
peakrange
</th>
<th style="text-align:left;">
ASOC_SNP
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
chr12_57096240_57096740
</td>
<td style="text-align:left;">
rs324015
</td>
</tr>
<tr>
<td style="text-align:left;">
chr7_2010548_2011048
</td>
<td style="text-align:left;">
rs11771625
</td>
</tr>
<tr>
<td style="text-align:left;">
chr3_175410533_175411033
</td>
<td style="text-align:left;">
rs2862015
</td>
</tr>
<tr>
<td style="text-align:left;">
chr5_90929346_90929846
</td>
<td style="text-align:left;">
rs10035662
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2_65539480_65539980
</td>
<td style="text-align:left;">
rs702906
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11_113577991_113578491
</td>
<td style="text-align:left;">
rs7928017
</td>
</tr>
</tbody>
</table>
<p><img src="figure/caQTL_cTWAS.Rmd/unnamed-chunk-27-1.png" width="672" style="display: block; margin: auto;" /></p>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
Significant cTWAS peaks overlapped with ASoC in GABA 6 hours
</caption>
<thead>
<tr>
<th style="text-align:left;">
peakrange
</th>
<th style="text-align:left;">
ASOC_SNP
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
chr12_57096240_57096740
</td>
<td style="text-align:left;">
rs324015
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1_115314324_115314824
</td>
<td style="text-align:left;">
rs12145726
</td>
</tr>
<tr>
<td style="text-align:left;">
chr7_2010548_2011048
</td>
<td style="text-align:left;">
rs11771625
</td>
</tr>
<tr>
<td style="text-align:left;">
chr5_90929346_90929846
</td>
<td style="text-align:left;">
rs10035662
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2_65539480_65539980
</td>
<td style="text-align:left;">
rs702906
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11_113577991_113578491
</td>
<td style="text-align:left;">
rs7928017
</td>
</tr>
</tbody>
</table>
<p><img src="figure/caQTL_cTWAS.Rmd/unnamed-chunk-29-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="npglut-1" class="section level3">
<h3>NPglut</h3>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
Significant cTWAS peaks overlapped with ASoC in NPglut 0 hours
</caption>
<thead>
<tr>
<th style="text-align:left;">
peakrange
</th>
<th style="text-align:left;">
ASOC_SNP
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
chr12_57096240_57096740
</td>
<td style="text-align:left;">
rs324015
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2_172098931_172099431
</td>
<td style="text-align:left;">
rs62183857
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2_172098931_172099431
</td>
<td style="text-align:left;">
rs62183858
</td>
</tr>
</tbody>
</table>
<p><img src="figure/caQTL_cTWAS.Rmd/unnamed-chunk-31-1.png" width="672" style="display: block; margin: auto;" /></p>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
Significant cTWAS peaks overlapped with ASoC in NPglut 1 hours
</caption>
<thead>
<tr>
<th style="text-align:left;">
peakrange
</th>
<th style="text-align:left;">
ASOC_SNP
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
chr2_172098931_172099431
</td>
<td style="text-align:left;">
rs62183857
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2_172098931_172099431
</td>
<td style="text-align:left;">
rs62183858
</td>
</tr>
<tr>
<td style="text-align:left;">
chr18_29920961_29921461
</td>
<td style="text-align:left;">
rs9964412
</td>
</tr>
<tr>
<td style="text-align:left;">
chr10_86244301_86244801
</td>
<td style="text-align:left;">
rs74716128
</td>
</tr>
</tbody>
</table>
<p><img src="figure/caQTL_cTWAS.Rmd/unnamed-chunk-33-1.png" width="672" style="display: block; margin: auto;" /></p>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
Significant cTWAS peaks overlapped with ASoC in NPglut 6 hours
</caption>
<thead>
<tr>
<th style="text-align:left;">
peakrange
</th>
<th style="text-align:left;">
ASOC_SNP
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
chr13_35764141_35764641
</td>
<td style="text-align:left;">
rs2282978
</td>
</tr>
<tr>
<td style="text-align:left;">
chr13_35764141_35764641
</td>
<td style="text-align:left;">
rs74008916
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1_115314324_115314824
</td>
<td style="text-align:left;">
rs1199604025
</td>
</tr>
</tbody>
</table>
<p><img src="figure/caQTL_cTWAS.Rmd/unnamed-chunk-35-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="nmglut-1" class="section level3">
<h3>NMglut</h3>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
Significant cTWAS peaks overlapped with ASoC in NMglut 0 hours
</caption>
<thead>
<tr>
<th style="text-align:left;">
peakrange
</th>
<th style="text-align:left;">
ASOC_SNP
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
chr11_113577991_113578491
</td>
<td style="text-align:left;">
rs78605981
</td>
</tr>
</tbody>
</table>
<p><img src="figure/caQTL_cTWAS.Rmd/unnamed-chunk-37-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-37-1">
Past versions of unnamed-chunk-37-1.png
</button>
</p>
<div id="fig-unnamed-chunk-37-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/ef4395e10a7a838a47f2ebe9b18e1b72d3e9b34a/docs/figure/caQTL_cTWAS.Rmd/unnamed-chunk-37-1.png" target="_blank">ef4395e</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-02-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/eedfee5819b640e6616ca4b8aea86b52983b5b9c/docs/figure/caQTL_cTWAS.Rmd/unnamed-chunk-37-1.png" target="_blank">eedfee5</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-02-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
Significant cTWAS peaks overlapped with ASoC in NMglut 1 hours
</caption>
<thead>
<tr>
<th style="text-align:left;">
peakrange
</th>
<th style="text-align:left;">
ASOC_SNP
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
chr2_172098931_172099431
</td>
<td style="text-align:left;">
rs62183857
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2_172098931_172099431
</td>
<td style="text-align:left;">
rs62183858
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2_172098931_172099431
</td>
<td style="text-align:left;">
rs79365605
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11_113577991_113578491
</td>
<td style="text-align:left;">
rs7928017
</td>
</tr>
</tbody>
</table>
<p><img src="figure/caQTL_cTWAS.Rmd/unnamed-chunk-39-1.png" width="672" style="display: block; margin: auto;" /></p>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
Significant cTWAS peaks overlapped with ASoC in NMglut 6 hours
</caption>
<thead>
<tr>
<th style="text-align:left;">
peakrange
</th>
<th style="text-align:left;">
ASOC_SNP
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
chr2_172098931_172099431
</td>
<td style="text-align:left;">
rs62183858
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2_172098931_172099431
</td>
<td style="text-align:left;">
rs62183857
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11_113577991_113578491
</td>
<td style="text-align:left;">
rs7928017
</td>
</tr>
</tbody>
</table>
<p><img src="figure/caQTL_cTWAS.Rmd/unnamed-chunk-41-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="rerunning-ctwas-merging-caqtl-with-the-same-snp"
class="section level3">
<h3>Rerunning cTWAS merging caQTL with the same SNP</h3>
<p>For each SNP, we only selected one cPeak related to it with
FDR&lt;0.05.</p>
<div id="gaba-parameters-1" class="section level4">
<h4>GABA parameters</h4>
<p>GABA has more cPeaks than those run with multiple cPeaks. But only 6
hours has more.</p>
<p><img src="figure/caQTL_cTWAS.Rmd/unnamed-chunk-42-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-42-1">
Past versions of unnamed-chunk-42-1.png
</button>
</p>
<div id="fig-unnamed-chunk-42-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/ef4395e10a7a838a47f2ebe9b18e1b72d3e9b34a/docs/figure/caQTL_cTWAS.Rmd/unnamed-chunk-42-1.png" target="_blank">ef4395e</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-02-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>library(RColorBrewer)
myPalette &lt;- brewer.pal(4, &quot;Set2&quot;) 
pie(GABA.params$group_pve, col=myPalette, border=&quot;white&quot;,
    main = paste0(sprintf(&quot;%.2f&quot;, 100-(GABA.params$group_pve[1]/GABA.params$total_pve)*100), &quot;% PVE for GABA OCR clusters&quot;))</code></pre>
<p><img src="figure/caQTL_cTWAS.Rmd/unnamed-chunk-43-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-43-1">
Past versions of unnamed-chunk-43-1.png
</button>
</p>
<div id="fig-unnamed-chunk-43-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/ef4395e10a7a838a47f2ebe9b18e1b72d3e9b34a/docs/figure/caQTL_cTWAS.Rmd/unnamed-chunk-43-1.png" target="_blank">ef4395e</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-02-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.1.2 (2021-11-01)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur 10.16

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] ggvenn_0.1.10        ggplot2_3.3.5        dplyr_1.0.7         
 [4] kableExtra_1.3.4     RColorBrewer_1.1-2   GenomicRanges_1.46.1
 [7] GenomeInfoDb_1.30.1  IRanges_2.28.0       S4Vectors_0.32.3    
[10] BiocGenerics_0.40.0  workflowr_1.7.0     

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.11            svglite_2.1.0          getPass_0.2-2         
 [4] ps_1.6.0               assertthat_0.2.1       rprojroot_2.0.2       
 [7] digest_0.6.29          utf8_1.2.2             R6_2.5.1              
[10] evaluate_0.14          httr_1.4.2             highr_0.9             
[13] pillar_1.7.0           zlibbioc_1.40.0        rlang_1.0.0           
[16] rstudioapi_0.13        whisker_0.4            callr_3.7.0           
[19] jquerylib_0.1.4        rmarkdown_2.11         labeling_0.4.2        
[22] webshot_0.5.5          stringr_1.4.0          RCurl_1.98-1.6        
[25] munsell_0.5.0          compiler_4.1.2         httpuv_1.6.5          
[28] xfun_0.29              systemfonts_1.0.4      pkgconfig_2.0.3       
[31] htmltools_0.5.2        tidyselect_1.1.1       tibble_3.1.6          
[34] GenomeInfoDbData_1.2.7 viridisLite_0.4.0      fansi_1.0.2           
[37] withr_2.4.3            crayon_1.4.2           later_1.3.0           
[40] bitops_1.0-7           jsonlite_1.7.3         gtable_0.3.0          
[43] lifecycle_1.0.1        DBI_1.1.2              git2r_0.29.0          
[46] magrittr_2.0.2         scales_1.1.1           cli_3.1.1             
[49] stringi_1.7.6          farver_2.1.0           XVector_0.34.0        
[52] fs_1.5.2               promises_1.2.0.1       xml2_1.3.3            
[55] bslib_0.3.1            ellipsis_0.3.2         vctrs_0.3.8           
[58] generics_0.1.2         cowplot_1.1.1          tools_4.1.2           
[61] glue_1.6.1             purrr_0.3.4            processx_3.5.2        
[64] fastmap_1.1.0          yaml_2.2.2             colorspace_2.0-2      
[67] rvest_1.0.2            knitr_1.37             sass_0.4.1            </code></pre>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
