<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Lifan Liang" />

<meta name="date" content="2024-05-23" />

<title>Interaction testing with mixed effect</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">neuron_stimulation_100line_eQTL</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/LifanLiang/neuron_stim_eQTL100">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Interaction testing with mixed effect</h1>
<h4 class="author">Lifan Liang</h4>
<h4 class="date">2024-05-23</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-06-06
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>neuron_stim_eQTL100/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges">
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted
changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges"
class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of
the R Markdown file created these results, you’ll want to first commit
it to the Git repo. If you’re still working on the analysis, you can
ignore this warning. When you’re finished, you can run
<code>wflow_publish</code> to commit the R Markdown file and build the
HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231109code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231109)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231109code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231109)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomLifanLiangneuronstimeQTL100tree90bd1d4b5d25293bdddd657d85a4201d601f507atargetblank90bd1d4a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/tree/90bd1d4b5d25293bdddd657d85a4201d601f507a" target="_blank">90bd1d4</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomLifanLiangneuronstimeQTL100tree90bd1d4b5d25293bdddd657d85a4201d601f507atargetblank90bd1d4a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/tree/90bd1d4b5d25293bdddd657d85a4201d601f507a" target="_blank">90bd1d4</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    data/.DS_Store
    Ignored:    data/single_group_ctwas/.DS_Store

Untracked files:
    Untracked:  data/SLC18A2_rs142285028_9cond.txt
    Untracked:  data/STXBP5_rs2818867_9cond.txt
    Untracked:  data/dynamic_eGene_beta_filtered.RDS
    Untracked:  data/neuroticism_ctwas_related_sumstats.RDS

Unstaged changes:
    Modified:   analysis/cellregmap.Rmd
    Modified:   analysis/eQTL_interaction.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/cellregmap.Rmd</code>) and HTML
(<code>docs/cellregmap.html</code>) files. If you’ve configured a remote
Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/90bd1d4b5d25293bdddd657d85a4201d601f507a/docs/cellregmap.html" target="_blank">90bd1d4</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-05-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/b79da9bd45ab7660ebb738b887a3f84ce1f31ced/analysis/cellregmap.Rmd" target="_blank">b79da9b</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-05-23
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/b79da9bd45ab7660ebb738b887a3f84ce1f31ced/docs/cellregmap.html" target="_blank">b79da9b</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-05-23
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="mixed-effect-model" class="section level2">
<h2>Mixed effect model</h2>
<p>CellRegMap was used for the purpose of interaction testing. Given
around 95 cell lines’ transcriptome measured for 3 cell types and 3 time
points, the model to perform interaction testing is:</p>
<p><span class="math display">\[
y = G\beta + G\gamma + W\psi + \mu + \epsilon
\]</span></p>
<p>where <span class="math inline">\(y\)</span> is gene expression
jointly normalized across 9 conditions, <span
class="math inline">\(G\)</span> is genotype, <span
class="math inline">\(W\)</span> are covariates, <span
class="math inline">\(\mu\)</span> is individual random effect, <span
class="math inline">\(\epsilon\)</span> is iid error, and <span
class="math inline">\(\beta\)</span> and <span
class="math inline">\(\psi\)</span> are corresponding fixed effects. The
focus of the model is <span class="math inline">\(\gamma\)</span>, the
random slope for genotypes.</p>
<p><span class="math display">\[
\gamma \sim MVN(0,\sigma Z)
\]</span></p>
<p>where <span class="math inline">\(Z\)</span> is the design matrix for
the 9 conditions. Elements within the same condition are 1, otherwise 0.
<span class="math inline">\(\sigma\)</span> dictates the effect size of
<span class="math inline">\(\gamma\)</span>. CellRegMap follows
StructLMM’s idea of using Rao’s score test to evaluate the null
hypothesis that <span class="math inline">\(\sigma\)</span> is 0.
According to CellRegMap, this is the highest power test when many
contexts were jointly tested.</p>
</div>
<div id="p-value-calibration." class="section level2">
<h2>P value calibration.</h2>
<p>We permuted the 9 conditions for each individual while preserving the
fixed effect of genotypes and conditions. Permutation P values show that
the calibration of P values are unstable, it can be inflated or deflated
depending on the gene tested.</p>
<p><img src="assets/ADAM10_interaction_permP.png" /> <img
src="assets/TSNK9_interaction_permP.png" /></p>
</div>
<div id="results" class="section level2">
<h2>Results</h2>
<p>We tested the dynamic effects on all the top eQTLs from <a
href="eQTL_glimix.html">mixed effect eQTL mapping</a>. And applied
Storey’s q value for multiple testing correction. There are 805
significant eQTLs.</p>
<p>By visualizing the eQTL, it seems most significant results are cell
type specific eQTLs. Some are dynamic eQTL shared across cell types. We
need to construct the context differently to detect cell type specific
dynamic eQTL.</p>
<div id="cell-type-specific-eqtl" class="section level3">
<h3>Cell type specific eQTL</h3>
<p><img src="assets/SYK_9cond.png" /></p>
<p><img src="assets/GNAT3_9cond.png" /> <img
src="assets/SMAD3_9cond.png" /></p>
</div>
<div id="dynamic-eqtl-shared-across-cell-types" class="section level3">
<h3>Dynamic eQTL shared across cell types</h3>
<p><img src="assets/ADAM10_9cond.png" /></p>
</div>
<div id="compared-with-heuristic-approach" class="section level3">
<h3>Compared with heuristic approach</h3>
<p>The heuristic approach identified dynamic eQTLs with FDR&lt;0.05 in
1/6 hours and nominal p value &gt; 0.1 in 0 hour. Somehow it identifies
more eQTLs than any interaction testing I tried. After checking the
interaction testing results of heuristic eQTL. The cause is two
folds:</p>
<div
id="testing-in-9-conditions-reduce-power-for-cell-type-specific-dynamic-eqtl"
class="section level4">
<h4>Testing in 9 conditions reduce power for cell type specific dynamic
eQTL</h4>
<p>This is one example of GABA specific dynamic eQTL.</p>
<p><img src="assets/example_celltype_specific_dynamic.png" /></p>
<table>
<colgroup>
<col width="13%" />
<col width="14%" />
<col width="9%" />
<col width="13%" />
<col width="12%" />
<col width="12%" />
<col width="13%" />
<col width="12%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"></th>
<th align="left">snps</th>
<th align="left">gene</th>
<th align="right">statistic</th>
<th align="right">pvalue</th>
<th align="right">FDR</th>
<th align="right">beta</th>
<th align="right">se</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">0hr_GABA</td>
<td align="left">rs142285028</td>
<td align="left">SLC18A2</td>
<td align="right">0.8605207</td>
<td align="right">0.3925710</td>
<td align="right">0.9029732</td>
<td align="right">0.1367555</td>
<td align="right">0.1589219</td>
</tr>
<tr class="even">
<td align="left">0hr_nmglut</td>
<td align="left">rs142285028</td>
<td align="left">SLC18A2</td>
<td align="right">0.7950990</td>
<td align="right">0.4299130</td>
<td align="right">0.9342332</td>
<td align="right">0.1271680</td>
<td align="right">0.1599398</td>
</tr>
<tr class="odd">
<td align="left">0hr_npglut</td>
<td align="left">rs142285028</td>
<td align="left">SLC18A2</td>
<td align="right">-1.8359811</td>
<td align="right">0.0706074</td>
<td align="right">0.6290750</td>
<td align="right">-0.3181558</td>
<td align="right">0.1732892</td>
</tr>
<tr class="even">
<td align="left">1hr_GABA</td>
<td align="left">rs142285028</td>
<td align="left">SLC18A2</td>
<td align="right">9.3387801</td>
<td align="right">0.0000000</td>
<td align="right">0.0000000</td>
<td align="right">1.0719865</td>
<td align="right">0.1147887</td>
</tr>
<tr class="odd">
<td align="left">1hr_nmglut</td>
<td align="left">rs142285028</td>
<td align="left">SLC18A2</td>
<td align="right">-0.4924553</td>
<td align="right">0.6240568</td>
<td align="right">0.9623638</td>
<td align="right">-0.0863948</td>
<td align="right">0.1754368</td>
</tr>
<tr class="even">
<td align="left">1hr_npglut</td>
<td align="left">rs142285028</td>
<td align="left">SLC18A2</td>
<td align="right">0.2922657</td>
<td align="right">0.7709240</td>
<td align="right">0.9725083</td>
<td align="right">0.0489167</td>
<td align="right">0.1673705</td>
</tr>
<tr class="odd">
<td align="left">6hr_GABA</td>
<td align="left">rs142285028</td>
<td align="left">SLC18A2</td>
<td align="right">6.9816324</td>
<td align="right">0.0000000</td>
<td align="right">0.0000016</td>
<td align="right">0.9048678</td>
<td align="right">0.1296069</td>
</tr>
<tr class="even">
<td align="left">6hr_nmglut</td>
<td align="left">rs142285028</td>
<td align="left">SLC18A2</td>
<td align="right">0.3181802</td>
<td align="right">0.7514531</td>
<td align="right">0.9800319</td>
<td align="right">0.0586216</td>
<td align="right">0.1842401</td>
</tr>
<tr class="odd">
<td align="left">6hr_npglut</td>
<td align="left">rs142285028</td>
<td align="left">SLC18A2</td>
<td align="right">1.5008975</td>
<td align="right">0.1376952</td>
<td align="right">0.7126045</td>
<td align="right">0.2606004</td>
<td align="right">0.1736297</td>
</tr>
</tbody>
</table>
<p>Changing the design matrix Z and input datawould help with the
testing.</p>
<table>
<thead>
<tr class="header">
<th align="left">Testing.approach</th>
<th align="left">Pval</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">time point testing across 9 conditions</td>
<td align="left">2.70e-03</td>
</tr>
<tr class="even">
<td align="left">GABA time point testing across 9 condition</td>
<td align="left">1.48e-09</td>
</tr>
<tr class="odd">
<td align="left">Time point testing within GABA samples</td>
<td align="left">2.33e-15</td>
</tr>
</tbody>
</table>
</div>
<div id="heuristic-dynamic-eqtl-may-have-similar-betas-but-different-se"
class="section level4">
<h4>Heuristic dynamic eQTL may have similar betas but different SE</h4>
<p>Some heuristic dynamic eQTLs are not significant in any interaction
testing. They usually have similar effect size. But one condition has a
smaller standard error, resulting in borderline significant P value and
FDR.</p>
<p><img src="assets/example_smallSE.png" /></p>
<table>
<colgroup>
<col width="13%" />
<col width="12%" />
<col width="8%" />
<col width="12%" />
<col width="12%" />
<col width="12%" />
<col width="13%" />
<col width="12%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"></th>
<th align="left">snps</th>
<th align="left">gene</th>
<th align="right">statistic</th>
<th align="right">pvalue</th>
<th align="right">FDR</th>
<th align="right">beta</th>
<th align="right">se</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">0hr_GABA</td>
<td align="left">rs2818867</td>
<td align="left">STXBP5</td>
<td align="right">-3.505089</td>
<td align="right">0.0008190</td>
<td align="right">0.0927614</td>
<td align="right">-0.2117779</td>
<td align="right">0.0604201</td>
</tr>
<tr class="even">
<td align="left">0hr_nmglut</td>
<td align="left">rs2818867</td>
<td align="left">STXBP5</td>
<td align="right">-1.437220</td>
<td align="right">0.1562196</td>
<td align="right">0.8318559</td>
<td align="right">-0.1086921</td>
<td align="right">0.0756266</td>
</tr>
<tr class="odd">
<td align="left">0hr_npglut</td>
<td align="left">rs2818867</td>
<td align="left">STXBP5</td>
<td align="right">-1.747244</td>
<td align="right">0.0849807</td>
<td align="right">0.6617738</td>
<td align="right">-0.1203133</td>
<td align="right">0.0688589</td>
</tr>
<tr class="even">
<td align="left">1hr_GABA</td>
<td align="left">rs2818867</td>
<td align="left">STXBP5</td>
<td align="right">-2.207051</td>
<td align="right">0.0305915</td>
<td align="right">0.5071340</td>
<td align="right">-0.1734916</td>
<td align="right">0.0786079</td>
</tr>
<tr class="odd">
<td align="left">1hr_nmglut</td>
<td align="left">rs2818867</td>
<td align="left">STXBP5</td>
<td align="right">-2.825466</td>
<td align="right">0.0062643</td>
<td align="right">0.3271402</td>
<td align="right">-0.2469860</td>
<td align="right">0.0874142</td>
</tr>
<tr class="even">
<td align="left">1hr_npglut</td>
<td align="left">rs2818867</td>
<td align="left">STXBP5</td>
<td align="right">-1.234892</td>
<td align="right">0.2208856</td>
<td align="right">0.7986773</td>
<td align="right">-0.0846395</td>
<td align="right">0.0685401</td>
</tr>
<tr class="odd">
<td align="left">6hr_GABA</td>
<td align="left">rs2818867</td>
<td align="left">STXBP5</td>
<td align="right">-1.258781</td>
<td align="right">0.2121757</td>
<td align="right">0.8078649</td>
<td align="right">-0.1120046</td>
<td align="right">0.0889786</td>
</tr>
<tr class="even">
<td align="left">6hr_nmglut</td>
<td align="left">rs2818867</td>
<td align="left">STXBP5</td>
<td align="right">-3.673081</td>
<td align="right">0.0005123</td>
<td align="right">0.0933896</td>
<td align="right">-0.3009610</td>
<td align="right">0.0819369</td>
</tr>
<tr class="odd">
<td align="left">6hr_npglut</td>
<td align="left">rs2818867</td>
<td align="left">STXBP5</td>
<td align="right">-4.009399</td>
<td align="right">0.0001454</td>
<td align="right">0.0180394</td>
<td align="right">-0.2001024</td>
<td align="right">0.0499083</td>
</tr>
</tbody>
</table>
</div>
<div id="fdr-distribution-of-top-dynamic-eqtl" class="section level4">
<h4>FDR distribution of top dynamic eQTL</h4>
<p><img src="assets/heuristic_dynamic_FDR_hist.png" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.1.2 (2021-11-01)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur 10.16

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] workflowr_1.7.0

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.11      highr_0.9        bslib_0.3.1      compiler_4.1.2  
 [5] pillar_1.7.0     later_1.3.0      git2r_0.29.0     jquerylib_0.1.4 
 [9] tools_4.1.2      getPass_0.2-2    digest_0.6.29    jsonlite_1.7.3  
[13] evaluate_0.14    tibble_3.1.6     lifecycle_1.0.1  pkgconfig_2.0.3 
[17] rlang_1.0.0      cli_3.1.1        rstudioapi_0.13  yaml_2.2.2      
[21] xfun_0.29        fastmap_1.1.0    httr_1.4.2       stringr_1.4.0   
[25] knitr_1.37       sass_0.4.1       fs_1.5.2         vctrs_0.3.8     
[29] rprojroot_2.0.2  glue_1.6.1       R6_2.5.1         processx_3.5.2  
[33] fansi_1.0.2      rmarkdown_2.11   callr_3.7.0      magrittr_2.0.2  
[37] whisker_0.4      ps_1.6.0         promises_1.2.0.1 htmltools_0.5.2 
[41] ellipsis_0.3.2   httpuv_1.6.5     utf8_1.2.2       stringi_1.7.6   
[45] crayon_1.4.2    </code></pre>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
