<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Lifan Liang" />

<meta name="date" content="2024-03-20" />

<title>cTWAS_all_cell</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.26/datatables.js"></script>
<link href="site_libs/dt-core-1.12.1/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.12.1/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.12.1/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">neuron_stimulation_100line_eQTL</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/LifanLiang/neuron_stim_eQTL100">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">cTWAS_all_cell</h1>
<h4 class="author">Lifan Liang</h4>
<h4 class="date">2024-03-20</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-03-28
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>neuron_stim_eQTL100/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231109code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231109)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231109code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231109)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomLifanLiangneuronstimeQTL100tree412ec7c9ea54927e6e815b4d924d6f46acc170a2targetblank412ec7ca">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/tree/412ec7c9ea54927e6e815b4d924d6f46acc170a2" target="_blank">412ec7c</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomLifanLiangneuronstimeQTL100tree412ec7c9ea54927e6e815b4d924d6f46acc170a2targetblank412ec7ca"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/tree/412ec7c9ea54927e6e815b4d924d6f46acc170a2" target="_blank">412ec7c</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    data/.DS_Store
    Ignored:    data/single_group_ctwas/.DS_Store

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/cTWAS_all_cell.Rmd</code>) and
HTML (<code>docs/cTWAS_all_cell.html</code>) files. If you’ve configured
a remote Git repository (see <code>?wflow_git_remote</code>), click on
the hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/412ec7c9ea54927e6e815b4d924d6f46acc170a2/analysis/cTWAS_all_cell.Rmd" target="_blank">412ec7c</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/412ec7c9ea54927e6e815b4d924d6f46acc170a2/docs/cTWAS_all_cell.html" target="_blank">412ec7c</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/4574572f5c0bab5da48bbda91b9b61e1ca9c0cb9/docs/cTWAS_all_cell.html" target="_blank">4574572</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/ee16cd002e6ddfe470811241dff7941f8f200894/analysis/cTWAS_all_cell.Rmd" target="_blank">ee16cd0</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/ee16cd002e6ddfe470811241dff7941f8f200894/docs/cTWAS_all_cell.html" target="_blank">ee16cd0</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/be0ac9a06b968c97d9fc4d667515f4dc02f91883/docs/cTWAS_all_cell.html" target="_blank">be0ac9a</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/a10668bc89bd0de675fe10aac40dfd62f94819d5/analysis/cTWAS_all_cell.Rmd" target="_blank">a10668b</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
<td>
wflow_publish("analysis")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/435d87d5d7a74d1891cb7c0362b33234bcfdc575/docs/cTWAS_all_cell.html" target="_blank">435d87d</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/268fb56e5367015a06f1fe705ab5411e3f266846/analysis/cTWAS_all_cell.Rmd" target="_blank">268fb56</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
<td>
wflow_publish("analysis")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/bee6c05ecbabca67f912ea1eb8cd1f2f2bce3697/docs/cTWAS_all_cell.html" target="_blank">bee6c05</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/0c544d0141dc0c075444f7bcb6a6cf77ac845d57/analysis/cTWAS_all_cell.Rmd" target="_blank">0c544d0</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-21
</td>
<td>
wflow_publish(c("data", "analysis"))
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="weights" class="section level2">
<h2>Weights</h2>
<p>Use the same set of eQTLs as cell type sperated cTWAS.These eQTLs are
the unions of top eQTL within the same cell type. But eQTLs with nominal
P values less than 0.1 in a certain condition was removed.</p>
<pre><code>       SNP   0hr_GABA   1hr_GABA   6hr_GABA 0hr_nmglut 1hr_nmglut 6hr_nmglut 
   6964380       3517       3509       3507       2153       2118       2157 
0hr_npglut 1hr_npglut 6hr_npglut 
      4389       4366       4405 </code></pre>
</div>
<div id="parameters" class="section level2">
<h2>Parameters</h2>
<p><img src="figure/cTWAS_all_cell.Rmd/unnamed-chunk-2-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-2-1">
Past versions of unnamed-chunk-2-1.png
</button>
</p>
<div id="fig-unnamed-chunk-2-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bee6c05ecbabca67f912ea1eb8cd1f2f2bce3697/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-2-1.png" target="_blank">bee6c05</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="prior-for-each-context" class="section level3">
<h3>Prior for each context</h3>
<p><img src="figure/cTWAS_all_cell.Rmd/unnamed-chunk-3-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-3-1">
Past versions of unnamed-chunk-3-1.png
</button>
</p>
<div id="fig-unnamed-chunk-3-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bee6c05ecbabca67f912ea1eb8cd1f2f2bce3697/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-3-1.png" target="_blank">bee6c05</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>NMglut has higher priors than the other cell types. However, this is
also the cell type with the least sample size for eQTL and hence the
smallest number of eGenes.</p>
<p><img src="figure/cTWAS_all_cell.Rmd/unnamed-chunk-4-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-4-1">
Past versions of unnamed-chunk-4-1.png
</button>
</p>
<div id="fig-unnamed-chunk-4-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bee6c05ecbabca67f912ea1eb8cd1f2f2bce3697/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-4-1.png" target="_blank">bee6c05</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="pve-for-each-context" class="section level3">
<h3>PVE for each context</h3>
<p>Total PVE from all eGenes in all contexts is 11.0%.</p>
<p><img src="figure/cTWAS_all_cell.Rmd/unnamed-chunk-5-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-5-1">
Past versions of unnamed-chunk-5-1.png
</button>
</p>
<div id="fig-unnamed-chunk-5-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bee6c05ecbabca67f912ea1eb8cd1f2f2bce3697/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-5-1.png" target="_blank">bee6c05</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>barplot(perc[c(2,5,8,3,6,9,4,7,10)], las=2, col=c(rep(&quot;red&quot;,3), rep(&quot;yellow&quot;,3), rep(&quot;blue&quot;,3)))</code></pre>
<p><img src="figure/cTWAS_all_cell.Rmd/unnamed-chunk-6-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-6-1">
Past versions of unnamed-chunk-6-1.png
</button>
</p>
<div id="fig-unnamed-chunk-6-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bee6c05ecbabca67f912ea1eb8cd1f2f2bce3697/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-6-1.png" target="_blank">bee6c05</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="tuning-coverage-in-fine-mapping" class="section level3">
<h3>Tuning coverage in fine-mapping</h3>
<p>“coverage” dictates how many SNPs are included into credible set.
Lowering coverage results in smaller and more coherent credible sets
that may pass the impurity threshold.</p>
<p><img src="figure/cTWAS_all_cell.Rmd/unnamed-chunk-7-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-7-1">
Past versions of unnamed-chunk-7-1.png
</button>
</p>
<div id="fig-unnamed-chunk-7-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/435d87d5d7a74d1891cb7c0362b33234bcfdc575/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-7-1.png" target="_blank">435d87d</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bee6c05ecbabca67f912ea1eb8cd1f2f2bce3697/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-7-1.png" target="_blank">bee6c05</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="results" class="section level2">
<h2>Results</h2>
<p>There are 60 egenes if the cutoff was set to PIP&gt;0.5.</p>
<p><img src="figure/cTWAS_all_cell.Rmd/unnamed-chunk-8-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-8-1">
Past versions of unnamed-chunk-8-1.png
</button>
</p>
<div id="fig-unnamed-chunk-8-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/be0ac9a06b968c97d9fc4d667515f4dc02f91883/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-8-1.png" target="_blank">be0ac9a</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/435d87d5d7a74d1891cb7c0362b33234bcfdc575/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-8-1.png" target="_blank">435d87d</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="comparing-with-gtex" class="section level3">
<h3>Comparing with GTEx</h3>
<p>Cerebellum hemisphere is the tissue with the highest power in GTEx
cTWAS. There are 28 genes with PIPs &gt; 0.5.</p>
<p><img src="assets/venn_GTEx.png" /></p>
<table>
<colgroup>
<col width="2%" />
<col width="5%" />
<col width="6%" />
<col width="8%" />
<col width="9%" />
<col width="9%" />
<col width="8%" />
<col width="9%" />
<col width="9%" />
<col width="8%" />
<col width="9%" />
<col width="9%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"></th>
<th align="left">genename</th>
<th align="right">PIP_sum</th>
<th align="right">PIP_0hr_GABA</th>
<th align="right">PIP_0hr_nmglut</th>
<th align="right">PIP_0hr_npglut</th>
<th align="right">PIP_1hr_GABA</th>
<th align="right">PIP_1hr_nmglut</th>
<th align="right">PIP_1hr_npglut</th>
<th align="right">PIP_6hr_GABA</th>
<th align="right">PIP_6hr_nmglut</th>
<th align="right">PIP_6hr_npglut</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">62</td>
<td align="left">FOXN2</td>
<td align="right">0.8830154</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0.000000</td>
<td align="right">0.0000000</td>
<td align="right">0.8830154</td>
<td align="right">0.00000</td>
<td align="right">0.0000000</td>
<td align="right">0</td>
<td align="right">0.0000000</td>
</tr>
<tr class="even">
<td align="left">130</td>
<td align="left">SERPINI1</td>
<td align="right">0.7920083</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0.000000</td>
<td align="right">0.2293343</td>
<td align="right">0.3876045</td>
<td align="right">0.00000</td>
<td align="right">0.1750695</td>
<td align="right">0</td>
<td align="right">0.0000000</td>
</tr>
<tr class="odd">
<td align="left">146</td>
<td align="left">TDRD6</td>
<td align="right">0.7867461</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0.217928</td>
<td align="right">0.3031301</td>
<td align="right">0.0000000</td>
<td align="right">0.00000</td>
<td align="right">0.0000000</td>
<td align="right">0</td>
<td align="right">0.2656881</td>
</tr>
<tr class="even">
<td align="left">161</td>
<td align="left">ZNF823</td>
<td align="right">0.9242300</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0.000000</td>
<td align="right">0.0000000</td>
<td align="right">0.0000000</td>
<td align="right">0.92423</td>
<td align="right">0.0000000</td>
<td align="right">0</td>
<td align="right">0.0000000</td>
</tr>
</tbody>
</table>
<p><img src="docs/assets/cTWAS_allcell_FOXN2.png" /></p>
</div>
<div id="cell-type-specific-egenes" class="section level3">
<h3>Cell type specific eGenes</h3>
<p>An eGene is assigned to a cell type if over 60% of the PIP sum are
from that cell type. Genes with higher sum of PIP are more likely to be
cell type specific.</p>
<pre><code>
      GABA     nmglut     npglut Unspecific 
        21         18         13          8 </code></pre>
<p><img src="figure/cTWAS_all_cell.Rmd/unnamed-chunk-12-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-12-1">
Past versions of unnamed-chunk-12-1.png
</button>
</p>
<div id="fig-unnamed-chunk-12-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/412ec7c9ea54927e6e815b4d924d6f46acc170a2/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-12-1.png" target="_blank">412ec7c</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="dynamic-egenes" class="section level3">
<h3>Dynamic eGenes</h3>
<p>For each cell type, we computed <code>PIP_diff</code> as in previous
cTWAS results. <code>PIP_diff</code> &gt; 0.5 in a certain cell type was
regarded as Dynamic eGenes. None of the unspecific eGenes are
dynamic.</p>
<div id="dynamic-egenes-in-gaba" class="section level4">
<h4>Dynamic eGenes in GABA</h4>
<p>GABA has 21 eGenes, 12 of them are dynamic.</p>
<pre><code>
Attaching package: &#39;dplyr&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:stats&#39;:

    filter, lag</code></pre>
<pre><code>The following objects are masked from &#39;package:base&#39;:

    intersect, setdiff, setequal, union</code></pre>
<div id="htmlwidget-7657cc3e42637b46d6e4" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-7657cc3e42637b46d6e4">{"x":{"filter":"none","vertical":false,"data":[["64","54","40","94","9","121","124","70","153","154","78","97","32","114","109","142","102","120","7","108","27"],["GALK2","EFHB","CROT","NDUFA6","ARL14EP","RBBP5","RCC1","GPR137","XKR4","XRN2","KCNA4","NLRC5","CHL1","PPFIA3","PCNX1","SYNGR1","NUDT22","RASA3","ANKRD39","PCDH7","CDPF1"],[0.937840919816047,0.885489351807218,0.868267869654942,0.840067687944772,0.813893454826733,0.764878853802945,0.735657608968323,0.710177091630636,0.612057718738372,0.572312290869186,0.548551293787507,0.525354230688015,0.669147051920772,0.732555730881537,0.506382495498012,0.527051722997696,0.603687005036302,0.714221526355868,0.758136462474273,0.8229272530088,0.924944260000973],[0,0,0,0,0,0,0,0,0,0,0,0,0.20331188644087,0.52827228979762,0.36324645386944,0.527051722997696,0.603687005036302,0.714221526355868,0.758136462474273,0.8229272530088,0.924944260000973],[0,0.885489351807218,0,0.840067687944772,0,0.433995316689152,0.735657608968323,0,0.612057718738372,0,0.548551293787507,0.525354230688015,0.264244209801506,0.204283441083917,0,0,0,0,0,0,0],[0.937840919816047,0,0.868267869654942,0,0.813893454826733,0.330883537113793,0,0.710177091630636,0,0.572312290869186,0,0,0.201590955678396,0,0,0,0,0,0,0,0],[0.937840919816047,0.885489351807218,0.868267869654942,0.840067687944772,0.813893454826733,0.764878853802945,0.735657608968323,0.710177091630636,0.612057718738372,0.572312290869186,0.548551293787507,0.525354230688015,0.669147051920772,0.732555730881537,0.36324645386944,0.527051722997696,0.603687005036302,0.714221526355868,0.758136462474273,0.8229272530088,0.924944260000973],[0.937840919816047,0.885489351807218,0.868267869654942,0.840067687944772,0.813893454826733,0.764878853802945,0.735657608968323,0.710177091630636,0.612057718738372,0.572312290869186,0.548551293787507,0.525354230688015,0.262523279039032,-0.323988848713703,-0.36324645386944,-0.527051722997696,-0.603687005036302,-0.714221526355868,-0.758136462474273,-0.8229272530088,-0.924944260000973]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>genename<\/th>\n      <th>PIP_sum<\/th>\n      <th>PIP_0hr_GABA<\/th>\n      <th>PIP_1hr_GABA<\/th>\n      <th>PIP_6hr_GABA<\/th>\n      <th>GABA_sum<\/th>\n      <th>GABA_PIP_diff<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":2,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":5,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":6,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":7,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[2,3,4,5,6,7]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render","options.columnDefs.2.render","options.columnDefs.3.render","options.columnDefs.4.render","options.columnDefs.5.render"],"jsHooks":[]}</script>
<div id="enrichment-analysis-for-gaba-specific-genes"
class="section level5">
<h5>Enrichment analysis for GABA specific genes</h5>
<pre><code>Uploading data to Enrichr... Done.
  Querying GO_Biological_Process_2018... Done.
Parsing results... Done.</code></pre>
<div id="htmlwidget-427ea3e24281aa554de8" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-427ea3e24281aa554de8">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10"],["regulation of short-term neuronal synaptic plasticity (GO:0048172)","regulation of neuronal synaptic plasticity (GO:0048168)","regulation of MHC class I biosynthetic process (GO:0045343)","negative regulation of type I interferon-mediated signaling pathway (GO:0060339)","positive regulation of type I interferon-mediated signaling pathway (GO:0060340)","alditol metabolic process (GO:0019400)","glycerol-3-phosphate metabolic process (GO:0006072)","regulated exocytosis (GO:0045055)","coenzyme A metabolic process (GO:0015936)","medium-chain fatty acid metabolic process (GO:0051791)"],["2/11","2/24","1/6","1/6","1/7","1/7","1/8","2/148","1/10","1/10"],[5.74219267049387e-05,0.000285795311196131,0.00628419133253944,0.00628419133253944,0.00732789827728551,0.00732789827728551,0.00837056168160556,0.0104161052146076,0.0104527615118677,0.0104527615118677],[0.00585703652390375,0.0145755608710027,0.0901703523007283,0.0901703523007283,0.0901703523007283,0.0901703523007283,0.0901703523007283,0.0901703523007283,0.0901703523007283,0.0901703523007283],["SYNGR1;PPFIA3","SYNGR1;PPFIA3","NLRC5","NLRC5","NLRC5","GALK2","GALK2","SYNGR1;PCDH7","CROT","CROT"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>P.value<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render"],"jsHooks":[]}</script>
</div>
<div id="enrichment-analysis-for-gaba-dynamic-genes"
class="section level5">
<h5>Enrichment analysis for GABA dynamic genes</h5>
<pre><code>Uploading data to Enrichr... Done.
  Querying GO_Biological_Process_2018... Done.
Parsing results... Done.</code></pre>
<div id="htmlwidget-a50efc78b4c04b4a25da" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-a50efc78b4c04b4a25da">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10"],["regulation of MHC class I biosynthetic process (GO:0045343)","negative regulation of type I interferon-mediated signaling pathway (GO:0060339)","alditol metabolic process (GO:0019400)","positive regulation of type I interferon-mediated signaling pathway (GO:0060340)","glycerol-3-phosphate metabolic process (GO:0006072)","coenzyme A metabolic process (GO:0015936)","medium-chain fatty acid metabolic process (GO:0051791)","carnitine metabolic process (GO:0009437)","glycerol metabolic process (GO:0006071)","regulation of transferase activity (GO:0051338)"],["1/6","1/6","1/7","1/7","1/8","1/10","1/10","1/11","1/11","1/14"],[0.00359499844647963,0.00359499844647963,0.00419301510496052,0.00419301510496052,0.00479070309341298,0.00598509351248777,0.00598509351248777,0.00658179619787483,0.00658179619787483,0.00836993582094223],[0.0658179619787483,0.0658179619787483,0.0658179619787483,0.0658179619787483,0.0658179619787483,0.0658179619787483,0.0658179619787483,0.0658179619787483,0.0658179619787483,0.068583369882975],["NLRC5","NLRC5","GALK2","NLRC5","GALK2","CROT","CROT","CROT","GALK2","NLRC5"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>P.value<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render"],"jsHooks":[]}</script>
<p><img src="assets/cTWAS_allcell_KCNA4.png" /></p>
<p><img src="assets/cTWAS_allcell_NLRC5.png" /></p>
</div>
</div>
<div id="dynamic-egenes-in-nmglut" class="section level4">
<h4>Dynamic eGenes in NMglut</h4>
<p>nmglut has 19 eGenes, 13 of them are dynamic.</p>
<div id="htmlwidget-1c14e9d616c779d1b646" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-1c14e9d616c779d1b646">{"x":{"filter":"none","vertical":false,"data":[["158","68","38","62","12","5","45","63","96","18","107","47","10","99","8","53","81","95"],["ZMYND10","GLIPR2","CPT1C","FOXN2","ATP5PO","ADAMTS2","CYP7B1","FZD4","NGEF","C17orf75","OXLD1","DCP1B","ATE1","NPIPB2","ANKRD45","DNPH1","KIAA0319","NDUFS5"],[0.940355591290139,0.938742625633472,0.905347681624163,0.88301539876483,0.78429283206186,0.7495226687979,0.742701508150125,0.704223815145326,0.698673284675076,0.650158345780552,0.640488813351533,0.596419033075071,0.707817613969744,0.702496106482498,0.566994321408849,0.558263998092711,0.511527827108036,0.577652913489344],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.577652913489344],[0,0.938742625633472,0.905347681624163,0.88301539876483,0.78429283206186,0.7495226687979,0,0.704223815145326,0.698673284675076,0,0,0.596419033075071,0.2647860752224,0.302850254309491,0.388638373605348,0.38558095805507,0.356365282890939,0],[0.940355591290139,0,0,0,0,0,0.742701508150125,0,0,0.650158345780552,0.640488813351533,0,0.185883537497403,0.126261128941732,0,0,0,0],[0.940355591290139,0.938742625633472,0.905347681624163,0.88301539876483,0.78429283206186,0.7495226687979,0.742701508150125,0.704223815145326,0.698673284675076,0.650158345780552,0.640488813351533,0.596419033075071,0.450669612719803,0.429111383251223,0.388638373605348,0.38558095805507,0.356365282890939,0.577652913489344],[0.940355591290139,0.938742625633472,0.905347681624163,0.88301539876483,0.78429283206186,0.7495226687979,0.742701508150125,0.704223815145326,0.698673284675076,0.650158345780552,0.640488813351533,0.596419033075071,0.450669612719803,0.429111383251223,0.388638373605348,0.38558095805507,0.356365282890939,-0.577652913489344]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>genename<\/th>\n      <th>PIP_sum<\/th>\n      <th>PIP_0hr_nmglut<\/th>\n      <th>PIP_1hr_nmglut<\/th>\n      <th>PIP_6hr_nmglut<\/th>\n      <th>nmglut_sum<\/th>\n      <th>nmglut_PIP_diff<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":2,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":5,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":6,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":7,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[2,3,4,5,6,7]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render","options.columnDefs.2.render","options.columnDefs.3.render","options.columnDefs.4.render","options.columnDefs.5.render"],"jsHooks":[]}</script>
<div id="enrichment-analysis-for-nmglut-specific-genes"
class="section level5">
<h5>Enrichment analysis for NMglut specific genes</h5>
<pre><code>Uploading data to Enrichr... Done.
  Querying GO_Biological_Process_2018... Done.
Parsing results... Done.</code></pre>
<div id="htmlwidget-771ad2536610c1179489" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-771ad2536610c1179489">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10"],["negative regulation of dendritic spine development (GO:0061000)","deadenylation-independent decapping of nuclear-transcribed mRNA (GO:0031087)","B cell chemotaxis (GO:0035754)","generation of neurons (GO:0048699)","negative regulation of dendrite development (GO:2000171)","negative regulation of dendrite morphogenesis (GO:0050774)","deadenylation-dependent decapping of nuclear-transcribed mRNA (GO:0000290)","nuclear-transcribed mRNA catabolic process, deadenylation-independent decay (GO:0031086)","motile cilium assembly (GO:0044458)","purine-containing compound catabolic process (GO:0072523)"],["1/6","1/6","1/6","2/130","1/7","1/7","1/8","1/8","1/13","1/14"],[0.00538846593898824,0.00538846593898824,0.00538846593898824,0.00599208526644116,0.0062838777740326,0.0062838777740326,0.0071785287204102,0.0071785287204102,0.0116403892865311,0.0125304865571367],[0.0834503963747686,0.0834503963747686,0.0834503963747686,0.0834503963747686,0.0834503963747686,0.0834503963747686,0.0834503963747686,0.0834503963747686,0.105939568164883,0.105939568164883],["NGEF","DCP1B","CYP7B1","FZD4;KIAA0319","KIAA0319","NGEF","DCP1B","DCP1B","ZMYND10","DNPH1"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>P.value<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render"],"jsHooks":[]}</script>
</div>
<div id="enrichment-analysis-for-nmglut-dynamic-genes"
class="section level5">
<h5>Enrichment analysis for NMglut dynamic genes</h5>
<pre><code>Uploading data to Enrichr... Done.
  Querying GO_Biological_Process_2018... Done.
Parsing results... Done.</code></pre>
<div id="htmlwidget-118ca324790aac282ebf" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-118ca324790aac282ebf">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10"],["negative regulation of dendritic spine development (GO:0061000)","deadenylation-independent decapping of nuclear-transcribed mRNA (GO:0031087)","B cell chemotaxis (GO:0035754)","negative regulation of dendrite morphogenesis (GO:0050774)","deadenylation-dependent decapping of nuclear-transcribed mRNA (GO:0000290)","nuclear-transcribed mRNA catabolic process, deadenylation-independent decay (GO:0031086)","motile cilium assembly (GO:0044458)","inner dynein arm assembly (GO:0036159)","outer dynein arm assembly (GO:0036158)","regulation of dendritic spine morphogenesis (GO:0061001)"],["1/6","1/6","1/6","1/7","1/8","1/8","1/13","1/16","1/18","1/23"],[0.00359499844647963,0.00359499844647963,0.00359499844647963,0.00419301510496052,0.00479070309341298,0.00479070309341298,0.00777421720760798,0.00956039001281181,0.0107495345067082,0.013716673976595],[0.0534961845431116,0.0534961845431116,0.0534961845431116,0.0534961845431116,0.0534961845431116,0.0534961845431116,0.0744103647013907,0.079629895627998,0.079629895627998,0.079629895627998],["NGEF","DCP1B","CYP7B1","NGEF","DCP1B","DCP1B","ZMYND10","ZMYND10","ZMYND10","NGEF"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>P.value<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render"],"jsHooks":[]}</script>
<p><img src="assets/cTWAS_allcell_NGEF.png" /></p>
<p><img src="assets/cTWAS_allcell_FZD4.png" /></p>
</div>
</div>
<div id="dynamic-egenes-in-npglut" class="section level4">
<h4>Dynamic eGenes in NPglut</h4>
<p>npglut has 13 eGenes, 9 of them are dynamic</p>
<div id="htmlwidget-7af0e23c13b7c4def9a9" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-7af0e23c13b7c4def9a9">{"x":{"filter":"none","vertical":false,"data":[["147","92","161","105","111","138","65","156","33","146","112","88","15"],["THBS3","NAGA","ZNF823","OLA1","PCYT1A","SMDT1","GCH1","ZCCHC4","CHRM3","TDRD6","PDXDC1","MOB3A","BMF"],[0.958523468139702,0.952637017566327,0.924230021097604,0.800427559876444,0.755054652974393,0.728947420080327,0.702388865128127,0.61685713169152,0.611909983807443,0.786746131007089,0.672211938974441,0.732585049667497,0.908743687922834],[0,0,0,0,0,0,0,0,0,0.217928009627839,0.45143416166357,0.732585049667497,0.908743687922834],[0,0,0.924230021097604,0,0,0.370719909931534,0,0,0,0,0.220777777310871,0,0],[0.958523468139702,0.952637017566327,0,0.800427559876444,0.755054652974393,0.358227510148793,0.702388865128127,0.61685713169152,0.611909983807443,0.265688052633165,0,0,0],[0.958523468139702,0.952637017566327,0.924230021097604,0.800427559876444,0.755054652974393,0.728947420080327,0.702388865128127,0.61685713169152,0.611909983807443,0.483616062261004,0.672211938974441,0.732585049667497,0.908743687922834],[0.958523468139702,0.952637017566327,0.924230021097604,0.800427559876444,0.755054652974393,0.728947420080327,0.702388865128127,0.61685713169152,0.611909983807443,0.047760043005326,-0.230656384352699,-0.732585049667497,-0.908743687922834]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>genename<\/th>\n      <th>PIP_sum<\/th>\n      <th>PIP_0hr_npglut<\/th>\n      <th>PIP_1hr_npglut<\/th>\n      <th>PIP_6hr_npglut<\/th>\n      <th>npglut_sum<\/th>\n      <th>npglut_PIP_diff<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":2,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":5,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":6,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":7,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[2,3,4,5,6,7]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render","options.columnDefs.2.render","options.columnDefs.3.render","options.columnDefs.4.render","options.columnDefs.5.render"],"jsHooks":[]}</script>
<div id="enrichment-analysis-for-npglut-specific-genes"
class="section level5">
<h5>Enrichment analysis for NPglut specific genes</h5>
<pre><code>Uploading data to Enrichr... Done.
  Querying GO_Biological_Process_2018... Done.
Parsing results... Done.</code></pre>
<div id="htmlwidget-3989c53c1df7f640b079" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-3989c53c1df7f640b079">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10"],["protein insertion into mitochondrial membrane involved in apoptotic signaling pathway (GO:0001844)","regulation of superoxide metabolic process (GO:0090322)","phospholipase C-activating G-protein coupled acetylcholine receptor signaling pathway (GO:0007207)","regulation of cellular response to oxidative stress (GO:1900407)","glycosyl compound catabolic process (GO:1901658)","adenylate cyclase-inhibiting G-protein coupled acetylcholine receptor signaling pathway (GO:0007197)","glycoside catabolic process (GO:0016139)","pteridine-containing compound biosynthetic process (GO:0042559)","glycosphingolipid catabolic process (GO:0046479)","glycosylceramide metabolic process (GO:0006677)"],["1/6","1/6","1/6","1/6","1/6","1/6","1/7","1/7","1/7","1/7"],[0.00389409648539669,0.00389409648539669,0.00389409648539669,0.00389409648539669,0.00389409648539669,0.00389409648539669,0.00454175360916762,0.00454175360916762,0.00454175360916762,0.00454175360916762],[0.0349839774342359,0.0349839774342359,0.0349839774342359,0.0349839774342359,0.0349839774342359,0.0349839774342359,0.0349839774342359,0.0349839774342359,0.0349839774342359,0.0349839774342359],["BMF","GCH1","CHRM3","GCH1","NAGA","CHRM3","NAGA","GCH1","NAGA","NAGA"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>P.value<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render"],"jsHooks":[]}</script>
</div>
<div id="enrichment-analysis-for-npglut-dynamic-genes"
class="section level5">
<h5>Enrichment analysis for NPglut dynamic genes</h5>
<pre><code>Uploading data to Enrichr... Done.
  Querying GO_Biological_Process_2018... Done.
Parsing results... Done.</code></pre>
<div id="htmlwidget-f8241748eb28a8a54961" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-f8241748eb28a8a54961">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10"],["phospholipase C-activating G-protein coupled acetylcholine receptor signaling pathway (GO:0007207)","regulation of cellular response to oxidative stress (GO:1900407)","glycosyl compound catabolic process (GO:1901658)","regulation of superoxide metabolic process (GO:0090322)","adenylate cyclase-inhibiting G-protein coupled acetylcholine receptor signaling pathway (GO:0007197)","organonitrogen compound biosynthetic process (GO:1901566)","glycosphingolipid catabolic process (GO:0046479)","glycosylceramide metabolic process (GO:0006677)","glycoside catabolic process (GO:0016139)","pteridine-containing compound biosynthetic process (GO:0042559)"],["1/6","1/6","1/6","1/6","1/6","2/181","1/7","1/7","1/7","1/7"],[0.00269725593591418,0.00269725593591418,0.00269725593591418,0.00269725593591418,0.00269725593591418,0.00281223611243233,0.00314617197276363,0.00314617197276363,0.00314617197276363,0.00314617197276363],[0.0213939694147927,0.0213939694147927,0.0213939694147927,0.0213939694147927,0.0213939694147927,0.0213939694147927,0.0213939694147927,0.0213939694147927,0.0213939694147927,0.0213939694147927],["CHRM3","GCH1","NAGA","GCH1","CHRM3","PCYT1A;GCH1","NAGA","NAGA","NAGA","GCH1"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>P.value<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render"],"jsHooks":[]}</script>
<p><img src="assets/cTWAS_allcell_NAGA.png" /></p>
<p><img src="assets/cTWAS_allcell_THBS3.png" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.1.2 (2021-11-01)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur 10.16

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] enrichR_3.2     dplyr_1.0.7     ggplot2_3.3.5   workflowr_1.7.0

loaded via a namespace (and not attached):
 [1] tidyselect_1.1.1  xfun_0.29         bslib_0.3.1       purrr_0.3.4      
 [5] colorspace_2.0-2  vctrs_0.3.8       generics_0.1.2    htmltools_0.5.2  
 [9] yaml_2.2.2        utf8_1.2.2        rlang_1.0.0       jquerylib_0.1.4  
[13] later_1.3.0       pillar_1.7.0      withr_2.4.3       glue_1.6.1       
[17] DBI_1.1.2         lifecycle_1.0.1   stringr_1.4.0     munsell_0.5.0    
[21] gtable_0.3.0      htmlwidgets_1.5.4 evaluate_0.14     WriteXLS_6.4.0   
[25] labeling_0.4.2    knitr_1.37        callr_3.7.0       fastmap_1.1.0    
[29] crosstalk_1.2.0   httpuv_1.6.5      ps_1.6.0          curl_4.3.2       
[33] fansi_1.0.2       highr_0.9         Rcpp_1.0.11       DT_0.26          
[37] promises_1.2.0.1  scales_1.1.1      jsonlite_1.7.3    farver_2.1.0     
[41] fs_1.5.2          rjson_0.2.21      digest_0.6.29     stringi_1.7.6    
[45] processx_3.5.2    getPass_0.2-2     rprojroot_2.0.2   grid_4.1.2       
[49] cowplot_1.1.1     cli_3.1.1         tools_4.1.2       magrittr_2.0.2   
[53] sass_0.4.1        tibble_3.1.6      crayon_1.4.2      whisker_0.4      
[57] pkgconfig_2.0.3   ellipsis_0.3.2    assertthat_0.2.1  rmarkdown_2.11   
[61] httr_1.4.2        rstudioapi_0.13   R6_2.5.1          git2r_0.29.0     
[65] compiler_4.1.2   </code></pre>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
