<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Lifan Liang" />

<meta name="date" content="2024-08-29" />

<title>ctwas analysis with caQTL and ASOC jointly</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.26/datatables.js"></script>
<link href="site_libs/dt-core-1.12.1/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.12.1/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.12.1/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">neuron_stimulation_100line_eQTL</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/LifanLiang/neuron_stim_eQTL100">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">ctwas analysis with caQTL and ASOC
jointly</h1>
<h4 class="author">Lifan Liang</h4>
<h4 class="date">2024-08-29</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-11-07
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>neuron_stim_eQTL100/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231109code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231109)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231109code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231109)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomLifanLiangneuronstimeQTL100treeaaa95524bc613e321e6567be327a75a9f503e936targetblankaaa9552a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/tree/aaa95524bc613e321e6567be327a75a9f503e936" target="_blank">aaa9552</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomLifanLiangneuronstimeQTL100treeaaa95524bc613e321e6567be327a75a9f503e936targetblankaaa9552a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/tree/aaa95524bc613e321e6567be327a75a9f503e936" target="_blank">aaa9552</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    data/.DS_Store
    Ignored:    data/single_group_ctwas/.DS_Store

Untracked files:
    Untracked:  GTEx_vs_SCZ.pdf.2024-08-26_21-04-15.log
    Untracked:  GTEx_vs_SCZ.png
    Untracked:  GTEx_vs_SCZ.png.2024-08-26_13-44-01.log
    Untracked:  GTEx_vs_SCZ.png.2024-08-26_20-30-40.log
    Untracked:  GTEx_vs_SCZ.png.2024-08-26_20-51-18.log
    Untracked:  GTEx_vs_SCZ.png.2024-08-26_20-59-28.log
    Untracked:  GTEx_vs_SCZ.png.2024-08-26_21-03-42.log
    Untracked:  GTEx_vs_SCZ.tiff
    Untracked:  GTEx_vs_SCZ.tiff.2024-08-26_21-04-43.log
    Untracked:  Rplot.png
    Untracked:  geneMapping.rds

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/ctwas_caQTL_ASOC.Rmd</code>) and
HTML (<code>docs/ctwas_caQTL_ASOC.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/aaa95524bc613e321e6567be327a75a9f503e936/analysis/ctwas_caQTL_ASOC.Rmd" target="_blank">aaa9552</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-11-07
</td>
<td>
wflow_publish(c("analysis", "docs", "data"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/607bd8b3bf2166aa2a193128ee89f83359a7fdb6/docs/ctwas_caQTL_ASOC.html" target="_blank">607bd8b</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-11-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/cebe60cfbbe71a9cdb2826f2adc05746104109af/analysis/ctwas_caQTL_ASOC.Rmd" target="_blank">cebe60c</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-11-06
</td>
<td>
wflow_publish(c("analysis", "data"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/79e4e57ea1e21aeac178ea0068f6468241f049bb/docs/ctwas_caQTL_ASOC.html" target="_blank">79e4e57</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-10-29
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/4a40cd92bbbeed3a55eb10ac0a71bf480f85c6a4/analysis/ctwas_caQTL_ASOC.Rmd" target="_blank">4a40cd9</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-10-29
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/4a40cd92bbbeed3a55eb10ac0a71bf480f85c6a4/docs/ctwas_caQTL_ASOC.html" target="_blank">4a40cd9</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-10-29
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/43696b6ff7a26e24a06018662ef26024b70e4564/docs/ctwas_caQTL_ASOC.html" target="_blank">43696b6</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-10-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/f69690566da4c8d9d822ca80316ac61e5011add5/analysis/ctwas_caQTL_ASOC.Rmd" target="_blank">f696905</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-10-16
</td>
<td>
wflow_publish(c("analysis", "data"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/a24afc5a140c8fdc268b4d00bca9dac9f4825727/docs/ctwas_caQTL_ASOC.html" target="_blank">a24afc5</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-10-10
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/93684496e9096c93e1eb0a20610b3041996548ca/analysis/ctwas_caQTL_ASOC.Rmd" target="_blank">9368449</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-10-10
</td>
<td>
wflow_publish(c("analysis", "data"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/063887648a6ca03a3d2ec10dd39d9441f5caf900/docs/ctwas_caQTL_ASOC.html" target="_blank">0638876</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-10-10
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/72fff519566d61034dae2156ffc61dc48465574c/analysis/ctwas_caQTL_ASOC.Rmd" target="_blank">72fff51</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-10-10
</td>
<td>
wflow_publish(c("analysis", "data"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/ef89bedf0b33c8ccac40a501984d4fb3a2d187c1/docs/ctwas_caQTL_ASOC.html" target="_blank">ef89bed</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-09-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/c693b4cae6b42a429e5f5f88059e9231760a890b/analysis/ctwas_caQTL_ASOC.Rmd" target="_blank">c693b4c</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-09-19
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/c693b4cae6b42a429e5f5f88059e9231760a890b/docs/ctwas_caQTL_ASOC.html" target="_blank">c693b4c</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-09-19
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/cd29898491e8efe18a5f89de4e4336a6425727c6/docs/ctwas_caQTL_ASOC.html" target="_blank">cd29898</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-09-10
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/e9d5e77500d023697c7e98da5d6f85caa04942b4/analysis/ctwas_caQTL_ASOC.Rmd" target="_blank">e9d5e77</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-09-10
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/e9d5e77500d023697c7e98da5d6f85caa04942b4/docs/ctwas_caQTL_ASOC.html" target="_blank">e9d5e77</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-09-10
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/a5a77fe7bd6dda2e04e5fbfdc807f0f31031d4af/docs/ctwas_caQTL_ASOC.html" target="_blank">a5a77fe</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-29
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/3dd8c95045af26bd6cf49083edf0dbaec57c1574/analysis/ctwas_caQTL_ASOC.Rmd" target="_blank">3dd8c95</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-29
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/3dd8c95045af26bd6cf49083edf0dbaec57c1574/docs/ctwas_caQTL_ASOC.html" target="_blank">3dd8c95</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-29
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="merging-caqtl-and-asoc" class="section level2">
<h2>Merging caQTL and ASOC</h2>
<p>I got top caQTL per peak per condition, first I took the union of
caQTLs FDR&lt;5% within the same cell type and removed caQTLs with
nominal P value &gt; 0.1. The allelic specific open chromatin (ASOC) was
processed the same way. Then these two sets were merged together. If an
ASOC was within a peak with a caQTL. The one with smaller nominal P
value would be included into the merged set.</p>
<table style="width:100%;">
<colgroup>
<col width="14%" />
<col width="8%" />
<col width="8%" />
<col width="8%" />
<col width="10%" />
<col width="10%" />
<col width="10%" />
<col width="10%" />
<col width="10%" />
<col width="10%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"></th>
<th align="right">0hr_GABA</th>
<th align="right">1hr_GABA</th>
<th align="right">6hr_GABA</th>
<th align="right">0hr_npglut</th>
<th align="right">1hr_npglut</th>
<th align="right">6hr_npglut</th>
<th align="right">0hr_nmglut</th>
<th align="right">1hr_nmglut</th>
<th align="right">6hr_nmglut</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">ASOC</td>
<td align="right">11297</td>
<td align="right">18252</td>
<td align="right">21118</td>
<td align="right">5573</td>
<td align="right">9259</td>
<td align="right">10116</td>
<td align="right">11370</td>
<td align="right">22207</td>
<td align="right">21846</td>
</tr>
<tr class="even">
<td align="left">processed_ASOC</td>
<td align="right">17428</td>
<td align="right">8860</td>
<td align="right">18466</td>
<td align="right">23678</td>
<td align="right">12376</td>
<td align="right">27961</td>
<td align="right">25789</td>
<td align="right">12892</td>
<td align="right">27897</td>
</tr>
<tr class="odd">
<td align="left">caQTL</td>
<td align="right">4318</td>
<td align="right">8954</td>
<td align="right">10504</td>
<td align="right">1802</td>
<td align="right">4435</td>
<td align="right">6017</td>
<td align="right">4957</td>
<td align="right">10497</td>
<td align="right">11622</td>
</tr>
<tr class="even">
<td align="left">processed_caQTL</td>
<td align="right">13520</td>
<td align="right">14082</td>
<td align="right">14164</td>
<td align="right">7475</td>
<td align="right">7757</td>
<td align="right">7835</td>
<td align="right">15352</td>
<td align="right">15895</td>
<td align="right">15918</td>
</tr>
<tr class="odd">
<td align="left">Union</td>
<td align="right">27730</td>
<td align="right">33110</td>
<td align="right">34944</td>
<td align="right">15186</td>
<td align="right">18380</td>
<td align="right">18824</td>
<td align="right">30126</td>
<td align="right">38137</td>
<td align="right">38078</td>
</tr>
</tbody>
</table>
</div>
<div id="parameter-extimation" class="section level2">
<h2>Parameter extimation</h2>
<p><img src="assets/joint_caqtl_asoc_params.png" /> ### PVE</p>
<p>Overall, total PVE across all condition was 28.6%.</p>
<p><img src="figure/ctwas_caQTL_ASOC.Rmd/unnamed-chunk-3-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-3-1">
Past versions of unnamed-chunk-3-1.png
</button>
</p>
<div id="fig-unnamed-chunk-3-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/3dd8c95045af26bd6cf49083edf0dbaec57c1574/docs/figure/ctwas_caQTL_ASOC.Rmd/unnamed-chunk-3-1.png" target="_blank">3dd8c95</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-29
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/ctwas_caQTL_ASOC.Rmd/unnamed-chunk-4-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-4-1">
Past versions of unnamed-chunk-4-1.png
</button>
</p>
<div id="fig-unnamed-chunk-4-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/3dd8c95045af26bd6cf49083edf0dbaec57c1574/docs/figure/ctwas_caQTL_ASOC.Rmd/unnamed-chunk-4-1.png" target="_blank">3dd8c95</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-29
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="enrichment" class="section level3">
<h3>Enrichment</h3>
<p><img src="figure/ctwas_caQTL_ASOC.Rmd/unnamed-chunk-5-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-5-1">
Past versions of unnamed-chunk-5-1.png
</button>
</p>
<div id="fig-unnamed-chunk-5-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/3dd8c95045af26bd6cf49083edf0dbaec57c1574/docs/figure/ctwas_caQTL_ASOC.Rmd/unnamed-chunk-5-1.png" target="_blank">3dd8c95</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-29
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="significant-peaks" class="section level2">
<h2>Significant peaks</h2>
<p>There are 59 peaks/ASOCs if the cutoff was set to PIP&gt;0.5.</p>
<p><img src="figure/ctwas_caQTL_ASOC.Rmd/unnamed-chunk-6-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-6-1">
Past versions of unnamed-chunk-6-1.png
</button>
</p>
<div id="fig-unnamed-chunk-6-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/3dd8c95045af26bd6cf49083edf0dbaec57c1574/docs/figure/ctwas_caQTL_ASOC.Rmd/unnamed-chunk-6-1.png" target="_blank">3dd8c95</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-29
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="pip-distribution-across-conditions" class="section level3">
<h3>PIP distribution across conditions</h3>
<p>For these 19 peaks/ASOCs with PIP&gt;0.8,</p>
<p><img src="figure/ctwas_caQTL_ASOC.Rmd/unnamed-chunk-7-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-7-1">
Past versions of unnamed-chunk-7-1.png
</button>
</p>
<div id="fig-unnamed-chunk-7-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/c693b4cae6b42a429e5f5f88059e9231760a890b/docs/figure/ctwas_caQTL_ASOC.Rmd/unnamed-chunk-7-1.png" target="_blank">c693b4c</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-09-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/3dd8c95045af26bd6cf49083edf0dbaec57c1574/docs/figure/ctwas_caQTL_ASOC.Rmd/unnamed-chunk-7-1.png" target="_blank">3dd8c95</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-29
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="likely-target-genes" class="section level3">
<h3>Likely target genes</h3>
<p>We haven’t computed the ABC score. But I still tried three approaches
to identify target genes: (1) check the eQTL with FDR&lt;20%. Possible
candidate genes are shown in the column “eqtl.gene”; (2) check the
co-activation between peaks accessibility and gene expression computed
by Zicheng. Cut off is FDR&lt;20%. The column “p2g.gene” is showing the
results; (3) Check which genes have overlapped with the peak, shown in
the column “overlap.gene”.</p>
<p>Co-activation and eQTL agreed with each other on two ASoCs. One is
rs12741781 targeting CEP170 and SDCCAG8. Another is rs215759 targeting
TCF20.</p>
<p>Please note that I don’t have peak information for ASoCs. So I simply
extended ASoCs by 250bps downstream and upstream as the peak.</p>
<div id="htmlwidget-7657cc3e42637b46d6e4" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-7657cc3e42637b46d6e4">{"x":{"filter":"none","vertical":false,"data":[["281","348","59","451","354","211","96","288","352","401","425","454","439","455","271","394","218","389","129","230","390","39","438","223","89","358","262","387","231","204","212","136","229","174","397","430","13","123","308","364","285","156","198","287","146","206","470","256","2","413","462","222","404","227","213","346","293","139"],["cPeak_chr12_57096240_57096740","cPeak_chr18_29920961_29921461","ASOC_rs146351281","cPeak_chr6_92378607_92379107","cPeak_chr1_115314324_115314824","ASOC_rs72723227","ASOC_rs2157591","cPeak_chr13_34165196_34165696","cPeak_chr19_5977436_5977936","cPeak_chr3_175410533_175411033","cPeak_chr5_135670508_135671008","cPeak_chr7_157665058_157665558","cPeak_chr5_90929346_90929846","cPeak_chr7_1986364_1986864","cPeak_chr11_64044962_64045462","cPeak_chr2_65539480_65539980","ASOC_rs74716128","cPeak_chr2_50016402_50016902","ASOC_rs3194383","ASOC_rs7928017","cPeak_chr2_50516287_50516787","ASOC_rs12741781","cPeak_chr5_89432966_89433466","ASOC_rs76403597","ASOC_rs2007044","cPeak_chr1_175822509_175823009","cPeak_chr11_113032951_113033451","cPeak_chr2_37004928_37005428","ASOC_rs79520962","ASOC_rs709210","ASOC_rs72986630","ASOC_rs3736166","ASOC_rs78838815","ASOC_rs61897848","cPeak_chr2_76019114_76019614","cPeak_chr5_156356081_156356581","ASOC_rs10822099","ASOC_rs301807","cPeak_chr17_32496237_32496737","cPeak_chr22_41362794_41363294","cPeak_chr13_109043532_109044032","ASOC_rs4719431","ASOC_rs6701496","cPeak_chr13_114179012_114179512","ASOC_rs4292","ASOC_rs7107939","cPeak_chr9_98306783_98307283","cPeak_chr10_102555979_102556479","ASOC_rs10127957","cPeak_chr3_71533041_71533541","cPeak_chr8_129857300_129857800","ASOC_rs7626087","cPeak_chr3_37941713_37942213","ASOC_rs7824786","ASOC_rs73396802","cPeak_chr17_67481828_67482328","cPeak_chr14_29707667_29708167","ASOC_rs3768886"],["rs324015","rs10502536","rs146351281","rs634940","rs4454548","rs72723227","rs2157591","rs9539890","rs150292328","rs2862015","rs17679727","rs17731604","rs7723305","rs58120505","rs478294","rs6705904","rs74716128","rs6746395","rs3194383","rs7928017","rs7598402","rs12741781","rs62378245","rs76403597","rs2007044","rs7516044","rs12277922","rs62133105","rs79520962","rs709210","rs72986630","rs3736166","rs78838815","rs61897848","rs17012055","rs10037772","rs10822099","rs301807","rs8069868","rs11090045","rs2150597","rs4719431","rs6701496","rs428637","rs4292","rs7107939","rs7869257","rs10883731","rs10127957","rs1432611","rs423915","rs7626087","rs71323644","rs7824786","rs73396802","rs8077271","rs2225899","rs3768886"],["chr12:57096240-57096740","chr18:29920961-29921461","chr15:88221646-88222146","chr6:92378607-92379107","chr1:115314324-115314824","chr5:7245301-7245801","chr22:41951840-41952340","chr13:34165196-34165696","chr19:5977436-5977936","chr3:175410533-175411033","chr5:135670508-135671008","chr7:157665058-157665558","chr5:90929346-90929846","chr7:1986364-1986864","chr11:64044962-64045462","chr2:65539480-65539980","chr10:86244297-86244797","chr2:50016402-50016902","chr4:54100250-54100750","chr11:113577790-113578290","chr2:50516287-50516787","chr1:243264600-243265100","chr5:89432966-89433466","chr17:13526896-13527396","chr12:2235544-2236044","chr1:175822509-175823009","chr11:113032951-113033451","chr2:37004928-37005428","chr7:128031208-128031708","chr3:50320188-50320688","chr19:11738671-11739171","chr17:36544737-36545237","chr4:31170699-31171199","chr11:47176837-47177337","chr2:76019114-76019614","chr5:156356081-156356581","chr10:62962022-62962522","chr1:8424513-8425013","chr17:32496237-32496737","chr22:41362794-41363294","chr13:109043532-109044032","chr7:2100427-2100927","chr1:201038450-201038950","chr13:114179012-114179512","chr17:63476730-63477230","chr11:132535824-132536324","chr9:98306783-98307283","chr10:102555979-102556479","chr1:65844756-65845256","chr3:71533041-71533541","chr8:129857300-129857800","chr3:2517517-2518017","chr3:37941713-37942213","chr8:143169382-143169882","chr6:32947770-32948270","chr17:67481828-67482328","chr14:29707667-29708167","chr2:224585194-224585694"],[1.00128364564131,0.955016809403755,0.935967528082685,0.926186294474483,0.92560085424797,0.902640951915856,0.891536256989942,0.890347006220991,0.887786204938828,0.88289767140028,0.88055756934831,0.87982843660989,0.878869937986801,0.858232528152708,0.840471878389686,0.801673523743285,0.797994062668209,0.792948144733943,0.791100511611807,0.770041216527018,0.748730178977026,0.74367827410658,0.738998236830677,0.736005928263763,0.732923971283406,0.701764116543406,0.699173785506241,0.678347185525669,0.654241801826239,0.65392595109455,0.652486601681515,0.651165994384468,0.638629578178004,0.631087238064105,0.626664770762676,0.615854168236515,0.586298607369774,0.581436024613071,0.580274163745853,0.579661541108369,0.578839527008372,0.569038988897952,0.567284033440001,0.55635290802638,0.551865834162528,0.549425446727721,0.547460122850862,0.544767589968057,0.532529314236902,0.53133141115095,0.524709939388683,0.523931988543103,0.523548971195521,0.522655220554002,0.520190713743276,0.517150199037993,0.514664097928086,0.50657372602868],["STAT6","","","","LINC01765","","TCF20","","","","","","ADGRV1","MAD1L1","GPR137,RCOR2","","","","","","","SDCCAG8,CEP170","","","","TNR","NCAM1","","","","ZNF823","ZNHIT3,DHRS11,GGNBP2","","","","","","","","POLR3H,TEF,PHF5A","","MAD1L1","","RASA3,CDC16","","","TBC1D2","","","","","","","LY6H","HLA-DMA,HLA-DPA1","","",""],["","","","","","","SNU13,NDUFA6-DT,TCF20,NDUFA6,SEPTIN3","","MICOS13,RPL36","","","","","","","","GRID1","","","DRD2","","CEP170,SDCCAG8","","","","","","","","","","","","ARHGAP1,C11orf49","","","","RERE","","TOB2,XRCC6,SNU13,SREBF2","","","","","AC015923.1","","","CNNM2","","","","","","","","","",""],["NAB2","LOC105372045","NTRK3-AS1","LINC02531","NGF","LOC105374643","SMIM45","LINC02343","RANBP3","NAALADL2-AS2","SLC25A48-AS1","LOC105375614","LOC107986433","MIR4655","FLRT1","LOC105374780","MIR346","LOC107985810","GSX2","DRD2","LOC101927089","MIR4677","LINC02161","MIR548H3","LINC02371","LINC01657","LOC101928847","STRN","LRRC4","HYAL2","ZNF823","LOC105371749","LINC02497","ARFGAP2","GCFC2","LOC105377673","LOC107984012","RERE-AS1","CDK5R1","TEF","MYO16-AS2","MRM2","LOC101929305","CFAP97D2","ACE","NTM-IT","TBC1D2","SUFU","PDE4B","MIR1284","MIR5194","CNTN4-AS2","MIR26A1","GPIHBP1","LOC100294145","MIR548D2","LOC105370429","CUL3"],[7196,291118,30583,344721,23945,26990,0,182289,205,139436,16572,51034,315156,142110,9030,82908,19990,564505,726,102098,104175,81075,147742,16249,504002,54333,69490,38391,398,7468,0,61608,0,0,308128,383230,42114,0,9243,4038,57886,141277,14500,24320,0,251521,51133,52006,52241,8547,150652,373275,27190,43335,53009,10242,220277,0],["","DSC1,SLC25A52,PCAT18","","","NGF","LINC02123,TAS2R1","PDGFB,FAM83F,L3MBTL2,PMM1,DESI1,XRCC6,SNU13,MEI1,SHISA8,TNFRSF13C,CENPM,SEPT3,WBP2NL,NAGA,FAM109B,NDUFA6-AS1,NDUFA6,OGFRP1,NFAM1,RRP7A,LINC01656,SMC1B,RIBC2,C22orf46,CCDC134,SREBF2,SMDT1,MCAT,CYP2D6,APOBEC3G,SYNGR1,SLC25A17,POLR3H,TCF20,SERHL2,POLDIP3,CYTH4,CYB5R3","","RANBP3","","","","MEF2C,LINC02060","","","","","","GSX2","PLET1,TMPRSS5,ANKK1,ZW10,HTR3B,UBE4A,APOC3,BCO2,DRD2,TMPRSS13,TEX12,ARHGAP20,PTS,TTC12,HSPB2,HSPB2-C11orf52,CRYAB,CLDN25,NNMT,APOA5","","PLD5,CEP170,SDCCAG8,KMO,BECN2,MAP1LC3C,FH,OR6F1","","DHRS7C,CDRT7,CDRT8,TBC1D26,LINC00670,CDRT4","","","","","LRRC4","","ZNF823","GGNBP2","","ARFGAP2","","SGCD","","","","TAB1,ZC3H7B,APOBEC3A","","","KIF21B,KIF14,GPR25,CACNA1S,C1orf106,CRB1","","METTL2A,CYB561,KCNH6,DCAF7,TACO1,ACE,LIMD2,TANC2","","","FBXW4,PPRC1,ELOVL3,ACTR1A,SUFU","","","GSDMC","IL5RA,CNTN4,CNTN6,CNTN4-AS2,SETMAR,CHL1-AS2,CHL1,CRBN","","LY6H","HLA-DMB,HLA-DMA,HLA-DRB1","","PRKD1","CUL3"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>genename<\/th>\n      <th>snp<\/th>\n      <th>range<\/th>\n      <th>PIP_sum<\/th>\n      <th>eqtl.gene<\/th>\n      <th>p2g.gene<\/th>\n      <th>tss.gene<\/th>\n      <th>tss.dist<\/th>\n      <th>ABC.gene<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[4,8]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render"],"jsHooks":[]}</script>
</div>
<div id="connection-to-eqtl-ctwas" class="section level3">
<h3>Connection to eQTL cTWAS</h3>
<p>I tried to make Miami plot to visualize regional overlap between
caQTL cTWAS and the eQTL cTWAS. But the correlation is not obvious.The
bottom plot is setting all PIPs to 0 if their credible set index is 0
and genes/peaks with PIP&gt;50% were colored as red.</p>
<p><img src="assets/ctwas_SCZ_eqtl_vs_caqtl.png" /></p>
<p><br></p>
<p>To align the significant peaks to the significant genes in the cTWAS
results, we check QTL distance between the two. Given that each
peak/gene has 9 QTLs included at maximum, I used the QTL with the
highest PIP for the corresponding peak/gene. There 17 peaks close to a
gene (distance&lt;500K bps) in the cTWAS results. One SNP is shared by
an ASOC and the gene ZNF823.</p>
<div id="htmlwidget-427ea3e24281aa554de8" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-427ea3e24281aa554de8">{"x":{"filter":"none","vertical":false,"data":[["18","19","3","17","12","15","1","10","9","2","NA","NA.1","NA.2","NA.3","NA.4","NA.5","NA.6","NA.7"],["ASOC_rs72986630","ASOC_rs2157591","cPeak_chr2_50016402_50016902","cPeak_chr19_5977436_5977936","cPeak_chr11_64044962_64045462","ASOC_rs146351281","cPeak_chr1_115314324_115314824","cPeak_chr7_157665058_157665558","cPeak_chr7_1986364_1986864","ASOC_rs12741781",null,null,null,null,null,null,null,null],["ZNF823","NAGA","FOXN2","ZNF823","ARL14EP","GALK2","THBS3","CROT","CROT","THBS3",null,null,null,null,null,null,null,null],[0,332104,1688824,5740689,33857943,39220898,39805551,70056677,85625411,88149914,null,null,null,null,null,null,null,null],[0.924230021097604,0.952637017566327,0.88301539876483,0.924230021097604,0.813893454826733,0.937840919816047,0.958523468139702,0.868267869654942,0.868267869654942,0.958523468139702,null,null,null,null,null,null,null,null]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>peak<\/th>\n      <th>gene<\/th>\n      <th>distance<\/th>\n      <th>gene_PIP<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render"],"jsHooks":[]}</script>
<p>Venn diagram below showed the overlap between cPeaks and genes with
PIP&gt;50%.</p>
<pre><code>[1] 1</code></pre>
<p><img src="assets/ctwas_peaks_vs_gene.png" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.1.2 (2021-11-01)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur 10.16

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] RColorBrewer_1.1-2   VennDiagram_1.7.1    futile.logger_1.4.3 
 [4] GenomicRanges_1.46.1 GenomeInfoDb_1.30.1  IRanges_2.28.0      
 [7] S4Vectors_0.32.3     BiocGenerics_0.40.0  dplyr_1.0.7         
[10] pheatmap_1.0.12      ggplot2_3.3.5        workflowr_1.7.0     

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.11            getPass_0.2-2          ps_1.6.0              
 [4] assertthat_0.2.1       rprojroot_2.0.2        digest_0.6.29         
 [7] utf8_1.2.2             R6_2.5.1               futile.options_1.0.1  
[10] evaluate_0.14          httr_1.4.2             highr_0.9             
[13] pillar_1.7.0           zlibbioc_1.40.0        rlang_1.0.0           
[16] rstudioapi_0.13        whisker_0.4            callr_3.7.0           
[19] jquerylib_0.1.4        DT_0.26                rmarkdown_2.11        
[22] stringr_1.4.0          htmlwidgets_1.5.4      RCurl_1.98-1.6        
[25] munsell_0.5.0          compiler_4.1.2         httpuv_1.6.5          
[28] xfun_0.29              pkgconfig_2.0.3        htmltools_0.5.2       
[31] tidyselect_1.1.1       tibble_3.1.6           GenomeInfoDbData_1.2.7
[34] fansi_1.0.2            crayon_1.4.2           withr_2.4.3           
[37] later_1.3.0            bitops_1.0-7           jsonlite_1.7.3        
[40] gtable_0.3.0           lifecycle_1.0.1        DBI_1.1.2             
[43] formatR_1.11           git2r_0.29.0           magrittr_2.0.2        
[46] scales_1.1.1           cli_3.1.1              stringi_1.7.6         
[49] XVector_0.34.0         fs_1.5.2               promises_1.2.0.1      
[52] bslib_0.3.1            ellipsis_0.3.2         generics_0.1.2        
[55] vctrs_0.3.8            cowplot_1.1.1          lambda.r_1.2.4        
[58] tools_4.1.2            glue_1.6.1             purrr_0.3.4           
[61] crosstalk_1.2.0        processx_3.5.2         fastmap_1.1.0         
[64] yaml_2.2.2             colorspace_2.0-2       knitr_1.37            
[67] sass_0.4.1            </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
