<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Lifan Liang" />

<meta name="date" content="2024-03-20" />

<title>cTWAS results: neuron stimulation eQTLs for Schizophrenia</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.26/datatables.js"></script>
<link href="site_libs/dt-core-1.12.1/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.12.1/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.12.1/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">neuron_stimulation_100line_eQTL</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/LifanLiang/neuron_stim_eQTL100">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">cTWAS results: neuron stimulation eQTLs for
Schizophrenia</h1>
<h4 class="author">Lifan Liang</h4>
<h4 class="date">2024-03-20</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-06-28
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>neuron_stim_eQTL100/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231109code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231109)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231109code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231109)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomLifanLiangneuronstimeQTL100tree81ffbde0d68da376bd68f72f7906077018675eb2targetblank81ffbdea">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/tree/81ffbde0d68da376bd68f72f7906077018675eb2" target="_blank">81ffbde</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomLifanLiangneuronstimeQTL100tree81ffbde0d68da376bd68f72f7906077018675eb2targetblank81ffbdea"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/tree/81ffbde0d68da376bd68f72f7906077018675eb2" target="_blank">81ffbde</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    data/.DS_Store
    Ignored:    data/single_group_ctwas/.DS_Store

Untracked files:
    Untracked:  Fig15SE_dynamic_count_eqtl_ctwas.pdf
    Untracked:  GTEx_vs_SCZ.pdf.2024-08-26_21-04-15.log
    Untracked:  GTEx_vs_SCZ.png
    Untracked:  GTEx_vs_SCZ.tiff
    Untracked:  GTEx_vs_SCZ.tiff.2024-08-26_21-04-43.log
    Untracked:  Rplot.png
    Untracked:  VennDiagram.2025-06-27_16-07-46.log
    Untracked:  geneMapping.rds
    Untracked:  plot4pdf.R

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/cTWAS_all_cell.Rmd</code>) and
HTML (<code>docs/cTWAS_all_cell.html</code>) files. If you’ve configured
a remote Git repository (see <code>?wflow_git_remote</code>), click on
the hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/81ffbde0d68da376bd68f72f7906077018675eb2/analysis/cTWAS_all_cell.Rmd" target="_blank">81ffbde</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
<td>
wflow_publish(c("analysis", "data"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/091002db3e225d86fad4fefe5c1b7fe8eaeed45b/docs/cTWAS_all_cell.html" target="_blank">091002d</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-12-10
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/b94f2a13b7778a72727ea5c98604e2dbb2387ca5/analysis/cTWAS_all_cell.Rmd" target="_blank">b94f2a1</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-12-10
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/ba6ddfe48e75e6a2b3d9f36ef005cd3bb8ab982b/docs/cTWAS_all_cell.html" target="_blank">ba6ddfe</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-12-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/8d6a3836c8157e699adab62beada134ca7638e59/analysis/cTWAS_all_cell.Rmd" target="_blank">8d6a383</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-12-09
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/a4e0ba51139ff121a443e23d111f7c74bb0cd0fd/docs/cTWAS_all_cell.html" target="_blank">a4e0ba5</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-12-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/a04679de525106b13e233b6a7a3127d5df87ed56/analysis/cTWAS_all_cell.Rmd" target="_blank">a04679d</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-12-05
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/93fda5cd9a4ec04dc65bc3813ba90cba7e713fa8/docs/cTWAS_all_cell.html" target="_blank">93fda5c</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-12-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/8b6a42541343ef50f737f1c40ae22ffa1a21fcb0/analysis/cTWAS_all_cell.Rmd" target="_blank">8b6a425</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-12-02
</td>
<td>
wflow_publish(c("analysis", "docs", "data"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/ef89bedf0b33c8ccac40a501984d4fb3a2d187c1/docs/cTWAS_all_cell.html" target="_blank">ef89bed</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-09-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/c693b4cae6b42a429e5f5f88059e9231760a890b/analysis/cTWAS_all_cell.Rmd" target="_blank">c693b4c</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-09-19
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/c693b4cae6b42a429e5f5f88059e9231760a890b/docs/cTWAS_all_cell.html" target="_blank">c693b4c</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-09-19
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/a5a77fe7bd6dda2e04e5fbfdc807f0f31031d4af/docs/cTWAS_all_cell.html" target="_blank">a5a77fe</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-29
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/3dd8c95045af26bd6cf49083edf0dbaec57c1574/analysis/cTWAS_all_cell.Rmd" target="_blank">3dd8c95</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-29
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/3dd8c95045af26bd6cf49083edf0dbaec57c1574/docs/cTWAS_all_cell.html" target="_blank">3dd8c95</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-29
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/c255b081903658d8cfe9666eba4458b290d79939/docs/cTWAS_all_cell.html" target="_blank">c255b08</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/7d70d1a246cfdd0f983b820d2eff381cf7ecdd8e/analysis/cTWAS_all_cell.Rmd" target="_blank">7d70d1a</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-05
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/7d70d1a246cfdd0f983b820d2eff381cf7ecdd8e/docs/cTWAS_all_cell.html" target="_blank">7d70d1a</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-05
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/3242ba61bacd56b9877476b60953ef2ed8055ef0/docs/cTWAS_all_cell.html" target="_blank">3242ba6</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-07-26
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/0e6b329808a511fe04b48403d8411059366b4852/analysis/cTWAS_all_cell.Rmd" target="_blank">0e6b329</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-07-26
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/e321e9e35509b3492cc8220b74379c9f132bfa07/docs/cTWAS_all_cell.html" target="_blank">e321e9e</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-07-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/af945140e0908343089287046f6bf7b725961213/analysis/cTWAS_all_cell.Rmd" target="_blank">af94514</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-07-23
</td>
<td>
wflow_publish(c("analysis/", "docs", "data"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/af945140e0908343089287046f6bf7b725961213/docs/cTWAS_all_cell.html" target="_blank">af94514</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-07-23
</td>
<td>
wflow_publish(c("analysis/", "docs", "data"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/b20c5c056d548faef9e4c4db521a550f0faa2a76/docs/cTWAS_all_cell.html" target="_blank">b20c5c0</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-24
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/44b4264e6a5a0b19e87587fc22652ea17883608c/analysis/cTWAS_all_cell.Rmd" target="_blank">44b4264</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-24
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/44b4264e6a5a0b19e87587fc22652ea17883608c/docs/cTWAS_all_cell.html" target="_blank">44b4264</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-24
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/5530b53f8c72c7406f8b8753830c68ab3fe79e26/docs/cTWAS_all_cell.html" target="_blank">5530b53</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/1655cdd0d6c6e48267022579b86b5f96418e3f6e/analysis/cTWAS_all_cell.Rmd" target="_blank">1655cdd</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-16
</td>
<td>
wflow_publish(c("analysis", "docs", "data"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/1655cdd0d6c6e48267022579b86b5f96418e3f6e/docs/cTWAS_all_cell.html" target="_blank">1655cdd</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-16
</td>
<td>
wflow_publish(c("analysis", "docs", "data"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/6d64fd37636e5c33f662874c872ad70be72b733f/docs/cTWAS_all_cell.html" target="_blank">6d64fd3</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/6e61fb8dd16291844c4e378728d8495edffac318/analysis/cTWAS_all_cell.Rmd" target="_blank">6e61fb8</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-11
</td>
<td>
wflow_publish(c("analysis"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/ac0bb63a17c3c144cead716a1d8cf790f2d166c7/docs/cTWAS_all_cell.html" target="_blank">ac0bb63</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-10
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/dfe2bf5f4c931827df1fcce1597e5053eb40cfa8/analysis/cTWAS_all_cell.Rmd" target="_blank">dfe2bf5</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-10
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/dfe2bf5f4c931827df1fcce1597e5053eb40cfa8/docs/cTWAS_all_cell.html" target="_blank">dfe2bf5</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-10
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/1b438e1d554ef95f811eea1359fcb80f409691b3/docs/cTWAS_all_cell.html" target="_blank">1b438e1</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/25c360f208ab0fa33479a5a80542e0ff721e5504/analysis/cTWAS_all_cell.Rmd" target="_blank">25c360f</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-04
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/25c360f208ab0fa33479a5a80542e0ff721e5504/docs/cTWAS_all_cell.html" target="_blank">25c360f</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-04
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/62d4f251e7602cff1e4e6852986f31a8e3e50fd8/docs/cTWAS_all_cell.html" target="_blank">62d4f25</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/5efb2b47489dfe23f00aca1a0e7e21a456602c42/analysis/cTWAS_all_cell.Rmd" target="_blank">5efb2b4</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-03
</td>
<td>
wflow_publish("analysis")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/7257ddf4c8e1d50162496c033e6cb66eedc9b9e5/docs/cTWAS_all_cell.html" target="_blank">7257ddf</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/81c058e1e5857df7fad891b968872766ba8df262/analysis/cTWAS_all_cell.Rmd" target="_blank">81c058e</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-03
</td>
<td>
wflow_publish("analysis")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/4ae5cadc48ac80c3bbfd7b74cd768fe8da31166f/docs/cTWAS_all_cell.html" target="_blank">4ae5cad</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/600a88fc701b52faa92fe43aade58146d058f3fc/analysis/cTWAS_all_cell.Rmd" target="_blank">600a88f</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-03
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/600a88fc701b52faa92fe43aade58146d058f3fc/docs/cTWAS_all_cell.html" target="_blank">600a88f</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-03
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/9eec00fdbd6966b47a5796595de668e0637c4fee/docs/cTWAS_all_cell.html" target="_blank">9eec00f</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/412ec7c9ea54927e6e815b4d924d6f46acc170a2/analysis/cTWAS_all_cell.Rmd" target="_blank">412ec7c</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/412ec7c9ea54927e6e815b4d924d6f46acc170a2/docs/cTWAS_all_cell.html" target="_blank">412ec7c</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/4574572f5c0bab5da48bbda91b9b61e1ca9c0cb9/docs/cTWAS_all_cell.html" target="_blank">4574572</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/ee16cd002e6ddfe470811241dff7941f8f200894/analysis/cTWAS_all_cell.Rmd" target="_blank">ee16cd0</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/ee16cd002e6ddfe470811241dff7941f8f200894/docs/cTWAS_all_cell.html" target="_blank">ee16cd0</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/be0ac9a06b968c97d9fc4d667515f4dc02f91883/docs/cTWAS_all_cell.html" target="_blank">be0ac9a</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/a10668bc89bd0de675fe10aac40dfd62f94819d5/analysis/cTWAS_all_cell.Rmd" target="_blank">a10668b</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
<td>
wflow_publish("analysis")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/435d87d5d7a74d1891cb7c0362b33234bcfdc575/docs/cTWAS_all_cell.html" target="_blank">435d87d</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/268fb56e5367015a06f1fe705ab5411e3f266846/analysis/cTWAS_all_cell.Rmd" target="_blank">268fb56</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
<td>
wflow_publish("analysis")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/bee6c05ecbabca67f912ea1eb8cd1f2f2bce3697/docs/cTWAS_all_cell.html" target="_blank">bee6c05</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/0c544d0141dc0c075444f7bcb6a6cf77ac845d57/analysis/cTWAS_all_cell.Rmd" target="_blank">0c544d0</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-21
</td>
<td>
wflow_publish(c("data", "analysis"))
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="weights" class="section level2">
<h2>Weights</h2>
<p>Use the same set of eQTLs as cell type sperated cTWAS.These eQTLs are
the unions of top eQTL within the same cell type. But eQTLs with nominal
P values less than 0.1 in a certain condition was removed.</p>
<pre><code>
Attaching package: &#39;dplyr&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:stats&#39;:

    filter, lag</code></pre>
<pre><code>The following objects are masked from &#39;package:base&#39;:

    intersect, setdiff, setequal, union</code></pre>
<pre><code>       SNP   0hr_GABA   1hr_GABA   6hr_GABA 0hr_nmglut 1hr_nmglut 6hr_nmglut 
   6964380       3517       3509       3507       2153       2118       2157 
0hr_npglut 1hr_npglut 6hr_npglut 
      4389       4366       4405 </code></pre>
</div>
<div id="parameters" class="section level2">
<h2>Parameters</h2>
<p><img src="figure/cTWAS_all_cell.Rmd/unnamed-chunk-2-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-2-1">
Past versions of unnamed-chunk-2-1.png
</button>
</p>
<div id="fig-unnamed-chunk-2-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bee6c05ecbabca67f912ea1eb8cd1f2f2bce3697/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-2-1.png" target="_blank">bee6c05</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="prior-for-each-context" class="section level3">
<h3>Prior for each context</h3>
<p><img src="figure/cTWAS_all_cell.Rmd/unnamed-chunk-3-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-3-1">
Past versions of unnamed-chunk-3-1.png
</button>
</p>
<div id="fig-unnamed-chunk-3-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bee6c05ecbabca67f912ea1eb8cd1f2f2bce3697/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-3-1.png" target="_blank">bee6c05</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>NMglut has higher priors than the other cell types. However, this is
also the cell type with the least sample size for eQTL and hence the
smallest number of eGenes.</p>
<p><img src="figure/cTWAS_all_cell.Rmd/unnamed-chunk-4-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-4-1">
Past versions of unnamed-chunk-4-1.png
</button>
</p>
<div id="fig-unnamed-chunk-4-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bee6c05ecbabca67f912ea1eb8cd1f2f2bce3697/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-4-1.png" target="_blank">bee6c05</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="enrichment" class="section level3">
<h3>Enrichment</h3>
<pre class="r"><code>library(ggplot2)
dat &lt;- data.frame(enrichment=ctwas.params$enrichment,
                  pve=ctwas.params$group_pve[-1]/ctwas.params$total_pve)
temp &lt;- strsplit(rownames(dat),&quot;_&quot;)
dat$celltype &lt;- sapply(temp,function(x){x[2]})
dat$timepoint &lt;- sapply(temp,function(x){x[1]})
ggplot(dat,aes(x=celltype,y=timepoint)) + 
  geom_count(aes(size=pve, color=enrichment)) + 
  theme_classic()</code></pre>
<p><img src="figure/cTWAS_all_cell.Rmd/unnamed-chunk-5-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-5-1">
Past versions of unnamed-chunk-5-1.png
</button>
</p>
<div id="fig-unnamed-chunk-5-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/c255b081903658d8cfe9666eba4458b290d79939/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-5-1.png" target="_blank">c255b08</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/af945140e0908343089287046f6bf7b725961213/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-5-1.png" target="_blank">af94514</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-07-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bee6c05ecbabca67f912ea1eb8cd1f2f2bce3697/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-5-1.png" target="_blank">bee6c05</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="pve-for-each-context" class="section level3">
<h3>PVE for each context</h3>
<p>Total PVE from all eGenes in all contexts is 11.0%.</p>
<p><img src="figure/cTWAS_all_cell.Rmd/unnamed-chunk-6-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-6-1">
Past versions of unnamed-chunk-6-1.png
</button>
</p>
<div id="fig-unnamed-chunk-6-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/c255b081903658d8cfe9666eba4458b290d79939/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-6-1.png" target="_blank">c255b08</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bee6c05ecbabca67f912ea1eb8cd1f2f2bce3697/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-6-1.png" target="_blank">bee6c05</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/cTWAS_all_cell.Rmd/unnamed-chunk-7-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-7-1">
Past versions of unnamed-chunk-7-1.png
</button>
</p>
<div id="fig-unnamed-chunk-7-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/c255b081903658d8cfe9666eba4458b290d79939/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-7-1.png" target="_blank">c255b08</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/435d87d5d7a74d1891cb7c0362b33234bcfdc575/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-7-1.png" target="_blank">435d87d</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bee6c05ecbabca67f912ea1eb8cd1f2f2bce3697/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-7-1.png" target="_blank">bee6c05</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="tuning-coverage-in-fine-mapping" class="section level3">
<h3>Tuning coverage in fine-mapping</h3>
<p>“coverage” dictates how many SNPs are included into credible set.
Lowering coverage results in smaller and more coherent credible sets
that may pass the impurity threshold.</p>
<p><img src="figure/cTWAS_all_cell.Rmd/unnamed-chunk-8-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-8-1">
Past versions of unnamed-chunk-8-1.png
</button>
</p>
<div id="fig-unnamed-chunk-8-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/c255b081903658d8cfe9666eba4458b290d79939/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-8-1.png" target="_blank">c255b08</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/af945140e0908343089287046f6bf7b725961213/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-8-1.png" target="_blank">af94514</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-07-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/be0ac9a06b968c97d9fc4d667515f4dc02f91883/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-8-1.png" target="_blank">be0ac9a</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/435d87d5d7a74d1891cb7c0362b33234bcfdc575/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-8-1.png" target="_blank">435d87d</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="results" class="section level2">
<h2>Results</h2>
<p>There are 44 protein coding ]egenes if the cutoff was set to
PIP&gt;0.5.</p>
<p><img src="figure/cTWAS_all_cell.Rmd/unnamed-chunk-9-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-9-1">
Past versions of unnamed-chunk-9-1.png
</button>
</p>
<div id="fig-unnamed-chunk-9-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/c255b081903658d8cfe9666eba4458b290d79939/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-9-1.png" target="_blank">c255b08</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/af945140e0908343089287046f6bf7b725961213/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-9-1.png" target="_blank">af94514</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-07-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/dfe2bf5f4c931827df1fcce1597e5053eb40cfa8/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-9-1.png" target="_blank">dfe2bf5</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-10
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/435d87d5d7a74d1891cb7c0362b33234bcfdc575/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-9-1.png" target="_blank">435d87d</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bee6c05ecbabca67f912ea1eb8cd1f2f2bce3697/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-9-1.png" target="_blank">bee6c05</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="comparing-with-gtex" class="section level3">
<h3>Comparing with GTEx</h3>
<p>Compare the cTWAS results of 13 GTEx tissues with our results. Each
GTEx tissue was analyzed with single group cTWAS ones. GTEx genes were
selected with PIP &gt; 0.5. Histogram below showed the number of
overlapped genes with each GTEx brain tissue.</p>
<div id="shared-with-each-gtex-brain-tissue" class="section level4">
<h4>Shared with each GTEx brain tissue</h4>
<pre><code>Loading required package: grid</code></pre>
<pre><code>Loading required package: futile.logger</code></pre>
<p><img src="figure/cTWAS_all_cell.Rmd/unnamed-chunk-10-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-10-1">
Past versions of unnamed-chunk-10-1.png
</button>
</p>
<div id="fig-unnamed-chunk-10-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/c255b081903658d8cfe9666eba4458b290d79939/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-10-1.png" target="_blank">c255b08</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/af945140e0908343089287046f6bf7b725961213/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-10-1.png" target="_blank">af94514</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-07-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/dfe2bf5f4c931827df1fcce1597e5053eb40cfa8/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-10-1.png" target="_blank">dfe2bf5</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-10
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/25c360f208ab0fa33479a5a80542e0ff721e5504/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-10-1.png" target="_blank">25c360f</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-04
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/9eec00fdbd6966b47a5796595de668e0637c4fee/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-10-1.png" target="_blank">9eec00f</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/435d87d5d7a74d1891cb7c0362b33234bcfdc575/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-10-1.png" target="_blank">435d87d</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="total-ctwas-significant-genes-within-each-gtex-brain-tissue"
class="section level4">
<h4>Total cTWAS significant genes within each GTEx brain tissue</h4>
<p><img src="figure/cTWAS_all_cell.Rmd/unnamed-chunk-11-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-11-1">
Past versions of unnamed-chunk-11-1.png
</button>
</p>
<div id="fig-unnamed-chunk-11-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/c255b081903658d8cfe9666eba4458b290d79939/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-11-1.png" target="_blank">c255b08</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/1b438e1d554ef95f811eea1359fcb80f409691b3/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-11-1.png" target="_blank">1b438e1</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-04
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/600a88fc701b52faa92fe43aade58146d058f3fc/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-11-1.png" target="_blank">600a88f</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-03
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/9eec00fdbd6966b47a5796595de668e0637c4fee/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-11-1.png" target="_blank">9eec00f</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/412ec7c9ea54927e6e815b4d924d6f46acc170a2/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-11-1.png" target="_blank">412ec7c</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="venn-diagram-between-our-ctwas-genes-and-gtex-brain-tissues"
class="section level4">
<h4>Venn diagram between our cTWAS genes and GTEx brain tissues</h4>
<p>Venn diagram showed the intersection of our results and the union of
all GTEx brain genes.</p>
<p><img src="assets/GTEx_vs_SCZ.tiff" /></p>
</div>
<div id="the-shared-8-genes-pip-in-our-ctwas-results"
class="section level4">
<h4>The shared 8 genes PIP in our cTWAS results</h4>
<table style="width:100%;">
<colgroup>
<col width="1%" />
<col width="4%" />
<col width="5%" />
<col width="6%" />
<col width="8%" />
<col width="8%" />
<col width="6%" />
<col width="8%" />
<col width="8%" />
<col width="6%" />
<col width="8%" />
<col width="8%" />
<col width="8%" />
<col width="5%" />
<col width="5%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"></th>
<th align="left">genename</th>
<th align="right">PIP_sum</th>
<th align="right">PIP_0hr_GABA</th>
<th align="right">PIP_0hr_nmglut</th>
<th align="right">PIP_0hr_npglut</th>
<th align="right">PIP_1hr_GABA</th>
<th align="right">PIP_1hr_nmglut</th>
<th align="right">PIP_1hr_npglut</th>
<th align="right">PIP_6hr_GABA</th>
<th align="right">PIP_6hr_nmglut</th>
<th align="right">PIP_6hr_npglut</th>
<th align="right">GWAS_Pval_loci</th>
<th align="right">eQTL_Pval</th>
<th align="right">Zgene_Pval</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">15</td>
<td align="left">FOXN2</td>
<td align="right">0.8830154</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0.000000</td>
<td align="right">0.0000000</td>
<td align="right">0.8830154</td>
<td align="right">0.00000</td>
<td align="right">0.0000000</td>
<td align="right">0.0000000</td>
<td align="right">0.0000000</td>
<td align="right">1.00e-07</td>
<td align="right">0.0000000</td>
<td align="right">0.0000003</td>
</tr>
<tr class="even">
<td align="left">36</td>
<td align="left">SERPINI1</td>
<td align="right">0.9263201</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0.000000</td>
<td align="right">0.2293343</td>
<td align="right">0.3876045</td>
<td align="right">0.00000</td>
<td align="right">0.1750695</td>
<td align="right">0.1343118</td>
<td align="right">0.0000000</td>
<td align="right">1.29e-05</td>
<td align="right">0.0000000</td>
<td align="right">0.0000129</td>
</tr>
<tr class="odd">
<td align="left">39</td>
<td align="left">TDRD6</td>
<td align="right">0.7867461</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0.217928</td>
<td align="right">0.3031301</td>
<td align="right">0.0000000</td>
<td align="right">0.00000</td>
<td align="right">0.0000000</td>
<td align="right">0.0000000</td>
<td align="right">0.2656881</td>
<td align="right">8.21e-05</td>
<td align="right">0.0000000</td>
<td align="right">0.0001497</td>
</tr>
<tr class="even">
<td align="left">44</td>
<td align="left">ZNF823</td>
<td align="right">0.9242300</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0.000000</td>
<td align="right">0.0000000</td>
<td align="right">0.0000000</td>
<td align="right">0.92423</td>
<td align="right">0.0000000</td>
<td align="right">0.0000000</td>
<td align="right">0.0000000</td>
<td align="right">0.00e+00</td>
<td align="right">0.0003992</td>
<td align="right">0.0000000</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="enrichment-analysis-regardless-of-cell-types"
class="section level3">
<h3>Enrichment analysis regardless of cell types</h3>
<div id="enrichment-analysis-of-all-genes-together"
class="section level4">
<h4>Enrichment analysis of all genes together</h4>
<pre><code>Welcome to enrichR
Checking connection ... </code></pre>
<pre><code>Enrichr ... Connection is Live!
FlyEnrichr ... Connection is Live!
WormEnrichr ... Connection is Live!
YeastEnrichr ... Connection is Live!
FishEnrichr ... Connection is Live!
OxEnrichr ... Connection is Live!</code></pre>
<pre><code>Uploading data to Enrichr... Done.
  Querying GO_Biological_Process_2023... Done.
Parsing results... Done.</code></pre>
<div id="htmlwidget-7657cc3e42637b46d6e4" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-7657cc3e42637b46d6e4">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10"],["Amino-Acid Betaine Metabolic Process (GO:0006577)","Carnitine Metabolic Process (GO:0009437)","Regulation Of Short-Term Neuronal Synaptic Plasticity (GO:0048172)","ATP Metabolic Process (GO:0046034)","Regulation Of Neuronal Synaptic Plasticity (GO:0048168)","Proton Motive Force-Driven Mitochondrial ATP Synthesis (GO:0042776)","Proton Motive Force-Driven ATP Synthesis (GO:0015986)","Oxidative Phosphorylation (GO:0006119)","B Cell Chemotaxis (GO:0035754)","Regulation Of Superoxide Metabolic Process (GO:0090322)"],["2/6","2/9","2/13","2/34","2/34","2/53","2/60","2/63","1/5","1/5"],[7.05539937632524e-05,0.000168621075335605,0.000363308377221552,0.00253751053903283,0.00253751053903283,0.00607016855958655,0.00772143402844758,0.00848431365219572,0.0109526884693577,0.0109526884693577],[0.0202489962100534,0.0241971243106593,0.0347565014208618,0.145653104940485,0.145653104940485,0.163598902422409,0.163598902422409,0.163598902422409,0.163598902422409,0.163598902422409],["CROT;CPT1C","CROT;CPT1C","SYNGR1;PPFIA3","OLA1;ATP5PO","SYNGR1;PPFIA3","NDUFA6;ATP5PO","NDUFA6;ATP5PO","NDUFA6;ATP5PO","CYP7B1","GCH1"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>P.value<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render"],"jsHooks":[]}</script>
</div>
<div id="enrichment-analysis-of-dynamic-genes" class="section level4">
<h4>Enrichment analysis of dynamic genes</h4>
<p>PIPs in time points <code>6hr</code> and <code>1hr</code> are summed
up and then subtracted the time point <code>0hr</code>. If the
difference is larger than 0.5, genes are called dynamic.</p>
<pre><code>Uploading data to Enrichr... Done.
  Querying GO_Biological_Process_2018... Done.
Parsing results... Done.</code></pre>
<div id="htmlwidget-427ea3e24281aa554de8" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-427ea3e24281aa554de8">{"x":{"filter":"none","vertical":false,"data":[[],[],[],[],[],[]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>P.value<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render"],"jsHooks":[]}</script>
</div>
<div id="enrichment-analysis-of-static-genes" class="section level4">
<h4>Enrichment analysis of static genes</h4>
<p>Static genes are high confidence risk genes that are not dynamic
genes.</p>
<pre><code>Uploading data to Enrichr... Done.
  Querying GO_Biological_Process_2018... Done.
Parsing results... Done.</code></pre>
<div id="htmlwidget-a50efc78b4c04b4a25da" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-a50efc78b4c04b4a25da">{"x":{"filter":"none","vertical":false,"data":[[],[],[],[],[],[]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>P.value<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render"],"jsHooks":[]}</script>
</div>
<div id="overlap-of-dynamic-genes-vs-static-genes-in-gtex"
class="section level4">
<h4>Overlap of dynamic genes vs static genes in GTEx</h4>
<p><img src="figure/cTWAS_all_cell.Rmd/unnamed-chunk-17-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-17-1">
Past versions of unnamed-chunk-17-1.png
</button>
</p>
<div id="fig-unnamed-chunk-17-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/c693b4cae6b42a429e5f5f88059e9231760a890b/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-17-1.png" target="_blank">c693b4c</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-09-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/c255b081903658d8cfe9666eba4458b290d79939/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-17-1.png" target="_blank">c255b08</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/af945140e0908343089287046f6bf7b725961213/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-17-1.png" target="_blank">af94514</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-07-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/44b4264e6a5a0b19e87587fc22652ea17883608c/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-17-1.png" target="_blank">44b4264</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/dfe2bf5f4c931827df1fcce1597e5053eb40cfa8/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-17-1.png" target="_blank">dfe2bf5</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-10
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="cell-type-specific-egenes" class="section level3">
<h3>Cell type specific eGenes</h3>
<p>An eGene is assigned to a cell type if over 60% of the PIP sum are
from that cell type. Genes with higher sum of PIP are more likely to be
cell type specific.</p>
<pre><code>
      GABA     nmglut     npglut Unspecific 
        18         11         10          5 </code></pre>
<p><img src="figure/cTWAS_all_cell.Rmd/unnamed-chunk-19-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-19-1">
Past versions of unnamed-chunk-19-1.png
</button>
</p>
<div id="fig-unnamed-chunk-19-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/c693b4cae6b42a429e5f5f88059e9231760a890b/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-19-1.png" target="_blank">c693b4c</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-09-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/3dd8c95045af26bd6cf49083edf0dbaec57c1574/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-19-1.png" target="_blank">3dd8c95</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/c255b081903658d8cfe9666eba4458b290d79939/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-19-1.png" target="_blank">c255b08</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/af945140e0908343089287046f6bf7b725961213/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-19-1.png" target="_blank">af94514</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-07-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Heatmap shows most the significant genes are both cell type specific
and time point specific.</p>
<pre><code>pdf 
  3 </code></pre>
<div id="cell-type-composition-across-pip-distribution"
class="section level4">
<h4>Cell type composition across PIP distribution</h4>
<p><img src="figure/cTWAS_all_cell.Rmd/unnamed-chunk-21-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-21-1">
Past versions of unnamed-chunk-21-1.png
</button>
</p>
<div id="fig-unnamed-chunk-21-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/c693b4cae6b42a429e5f5f88059e9231760a890b/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-21-1.png" target="_blank">c693b4c</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-09-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/c255b081903658d8cfe9666eba4458b290d79939/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-21-1.png" target="_blank">c255b08</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="cell-type-composition-of-gtex-shared-genes"
class="section level4">
<h4>Cell type composition of GTEx shared genes</h4>
<p>Since GTEx is bulk eQTL, the cell type contribution of the shared
genes for SCZ can only be elucidated by our joint condition cTWAS
results.</p>
<p><img src="figure/cTWAS_all_cell.Rmd/unnamed-chunk-22-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-22-1">
Past versions of unnamed-chunk-22-1.png
</button>
</p>
<div id="fig-unnamed-chunk-22-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/c693b4cae6b42a429e5f5f88059e9231760a890b/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-22-1.png" target="_blank">c693b4c</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-09-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="distribution-of-shared-genes-across-cell-types"
class="section level5">
<h5>Distribution of shared genes across cell types</h5>
<pre><code>
    nmglut     npglut Unspecific 
         1          2          1 </code></pre>
</div>
<div id="distribution-of-unique-genes-across-cell-types"
class="section level5">
<h5>Distribution of unique genes across cell types</h5>
<pre><code>
      GABA     nmglut     npglut Unspecific 
        18         10          8          4 </code></pre>
</div>
</div>
</div>
<div id="time-point-specific-genes" class="section level3">
<h3>Time point specific genes</h3>
<pre><code>
      PIP_     PIP_1h     PIP_6h Unspecific 
        18         11         10          5 </code></pre>
<pre><code>
dynamic    PIP_  PIP_1h  PIP_6h  Static 
      4      18      11      10       1 </code></pre>
<div id="time-point-composition-of-gene-pips" class="section level4">
<h4>Time point composition of gene PIPs</h4>
<p>Since GTEx is bulk eQTL, the cell type contribution of the shared
genes for SCZ can only be elucidated by our joint condition cTWAS
results.</p>
<p><img src="figure/cTWAS_all_cell.Rmd/unnamed-chunk-27-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-27-1">
Past versions of unnamed-chunk-27-1.png
</button>
</p>
<div id="fig-unnamed-chunk-27-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/93fda5cd9a4ec04dc65bc3813ba90cba7e713fa8/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-27-1.png" target="_blank">93fda5c</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-12-02
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/c693b4cae6b42a429e5f5f88059e9231760a890b/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-27-1.png" target="_blank">c693b4c</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-09-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/cTWAS_all_cell.Rmd/unnamed-chunk-28-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-28-1">
Past versions of unnamed-chunk-28-1.png
</button>
</p>
<div id="fig-unnamed-chunk-28-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/c693b4cae6b42a429e5f5f88059e9231760a890b/docs/figure/cTWAS_all_cell.Rmd/unnamed-chunk-28-1.png" target="_blank">c693b4c</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-09-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="dynamic-egenes" class="section level3">
<h3>Dynamic eGenes</h3>
<p>For each cell type, we computed <code>PIP_diff</code> as in previous
cTWAS results. <code>PIP_diff</code> &gt; 0.5 in a certain cell type was
regarded as Dynamic eGenes. None of the unspecific eGenes are
dynamic.</p>
<div id="dynamic-egenes-in-gaba" class="section level4">
<h4>Dynamic eGenes in GABA</h4>
<p>GABA has 7 eGenes, 5 of them are dynamic.</p>
<div id="htmlwidget-1c14e9d616c779d1b646" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-1c14e9d616c779d1b646">{"x":{"filter":"none","vertical":false,"data":[["GALK2","EFHB","CROT","NDUFA6","ARL14EP","RBBP5","RCC1","GPR137","XRN2","CHL1","PPFIA3","PCNX1","SYNGR1","NUDT22","RASA3","ANKRD39","PCDH7","CDPF1"],["GALK2","EFHB","CROT","NDUFA6","ARL14EP","RBBP5","RCC1","GPR137","XRN2","CHL1","PPFIA3","PCNX1","SYNGR1","NUDT22","RASA3","ANKRD39","PCDH7","CDPF1"],[0.937840919816047,0.885489351807218,0.868267869654942,0.840067687944772,0.813893454826733,0.764878853802945,0.735657608968323,0.710177091630636,0.628404020472196,0.669147051920772,0.732555730881537,0.506382495498012,0.527051722997696,0.603687005036302,0.714221526355868,0.758136462474273,0.8229272530088,0.924944260000973],[0,0,0,0,0,0,0,0,0.0205816114999036,0.20331188644087,0.52827228979762,0.36324645386944,0.527051722997696,0.603687005036302,0.714221526355868,0.758136462474273,0.8229272530088,0.924944260000973],[0,0.885489351807218,0,0.840067687944772,0,0.433995316689152,0.735657608968323,0,0.0355101181031068,0.264244209801506,0.204283441083917,0,0,0,0,0,0,0],[0.937840919816047,0,0.868267869654942,0,0.813893454826733,0.330883537113793,0,0.710177091630636,0.572312290869186,0.201590955678396,0,0,0,0,0,0,0,0],[1.50995031331211e-06,1.39604693761089e-06,1.69321949491931e-15,1.95464853405833e-06,1.2842249307321e-06,4.95508583243399e-06,8.25302700640605e-09,1.25335534596867e-07,1.10504086302653e-05,3.39305828320367e-05,9.15204100639211e-12,3.30756047464246e-14,2.48096147902836e-11,2.01230075497059e-07,4.00418639677817e-07,2.15750902923861e-08,2.89080267604468e-08,7.45162866379611e-06],[8.68988488778213e-05,1.40712937340677e-06,3.13847980892573e-05,1.8136283926884e-06,3.18119534046593e-05,8.17211749182273e-07,0.000309329393999677,0.000212560516466419,9.02690566887615e-06,0.000296194045402909,1.10371965272517e-05,8.14067845802323e-08,1.0834066724405e-06,8.79796620559762e-05,8.92673333267425e-07,7.87595361536556e-05,8.60693580507827e-07,0.000104262176224763],[1.50995031332779e-06,2.19562967019419e-06,1.48487205866537e-08,1.95464853407277e-06,1.28422493072232e-06,6.6262123263284e-06,9.05979653321994e-08,2.67831407628121e-06,1.1861420731115e-05,0.000180983707823933,8.59152260180926e-06,2.76279577317246e-06,4.25403370003189e-09,6.83359473011684e-07,4.00418639673664e-07,3.14769548316395e-08,2.89080267606224e-08,7.45162866378538e-06],[0.937840919816047,0.885489351807218,0.868267869654942,0.840067687944772,0.813893454826733,0.764878853802945,0.735657608968323,0.710177091630636,0.628404020472196,0.669147051920772,0.732555730881537,0.36324645386944,0.527051722997696,0.603687005036302,0.714221526355868,0.758136462474273,0.8229272530088,0.924944260000973],[0.937840919816047,0.885489351807218,0.868267869654942,0.840067687944772,0.813893454826733,0.764878853802945,0.735657608968323,0.710177091630636,0.587240797472389,0.262523279039032,-0.323988848713703,-0.36324645386944,-0.527051722997696,-0.603687005036302,-0.714221526355868,-0.758136462474273,-0.8229272530088,-0.924944260000973]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>genename<\/th>\n      <th>PIP_sum<\/th>\n      <th>PIP_0hr_GABA<\/th>\n      <th>PIP_1hr_GABA<\/th>\n      <th>PIP_6hr_GABA<\/th>\n      <th>GWAS_Pval_loci<\/th>\n      <th>eQTL_Pval<\/th>\n      <th>Zgene_Pval<\/th>\n      <th>GABA_sum<\/th>\n      <th>GABA_PIP_diff<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":2,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":5,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":6,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":7,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":8,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":9,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":10,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[2,3,4,5,6,7,8,9,10]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render","options.columnDefs.2.render","options.columnDefs.3.render","options.columnDefs.4.render","options.columnDefs.5.render","options.columnDefs.6.render","options.columnDefs.7.render","options.columnDefs.8.render"],"jsHooks":[]}</script>
<div id="enrichment-analysis-for-gaba-specific-genes"
class="section level5">
<h5>Enrichment analysis for GABA specific genes</h5>
<pre><code>Uploading data to Enrichr... Done.
  Querying GO_Biological_Process_2023... Done.
Parsing results... Done.</code></pre>
<div id="htmlwidget-771ad2536610c1179489" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-771ad2536610c1179489">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10"],["Regulation Of Short-Term Neuronal Synaptic Plasticity (GO:0048172)","Regulation Of Neuronal Synaptic Plasticity (GO:0048168)","Amino-Acid Betaine Metabolic Process (GO:0006577)","Galactose Metabolic Process (GO:0006012)","Alditol Metabolic Process (GO:0019400)","Termination Of RNA Polymerase II Transcription (GO:0006369)","Carnitine Metabolic Process (GO:0009437)","Regulation Of Store-Operated Calcium Entry (GO:2001256)","Negative Regulation Of Bone Resorption (GO:0045779)","Mitotic Nuclear Membrane Organization (GO:0101024)"],["2/13","2/34","1/6","1/6","1/7","1/7","1/9","1/9","1/9","1/9"],[5.93211570172091e-05,0.000421912124260292,0.00538846593898824,0.00538846593898824,0.0062838777740326,0.0062838777740326,0.00807241931005015,0.00807241931005015,0.00807241931005015,0.00807241931005015],[0.00605075801575533,0.0215175183372749,0.0670338671167522,0.0670338671167522,0.0670338671167522,0.0670338671167522,0.0670338671167522,0.0670338671167522,0.0670338671167522,0.0670338671167522],["SYNGR1;PPFIA3","SYNGR1;PPFIA3","CROT","GALK2","GALK2","XRN2","CROT","EFHB","GPR137","RCC1"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>P.value<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render"],"jsHooks":[]}</script>
</div>
<div id="enrichment-analysis-for-gaba-dynamic-genes"
class="section level5">
<h5>Enrichment analysis for GABA dynamic genes</h5>
<pre><code>Uploading data to Enrichr... Done.
  Querying GO_Biological_Process_2023... Done.
Parsing results... Done.</code></pre>
<div id="htmlwidget-118ca324790aac282ebf" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-118ca324790aac282ebf">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10"],["Regulation Of Short-Term Neuronal Synaptic Plasticity (GO:0048172)","Regulation Of Neuronal Synaptic Plasticity (GO:0048168)","Amino-Acid Betaine Metabolic Process (GO:0006577)","Galactose Metabolic Process (GO:0006012)","Alditol Metabolic Process (GO:0019400)","Termination Of RNA Polymerase II Transcription (GO:0006369)","Carnitine Metabolic Process (GO:0009437)","Regulation Of Store-Operated Calcium Entry (GO:2001256)","Negative Regulation Of Bone Resorption (GO:0045779)","Mitotic Nuclear Membrane Organization (GO:0101024)"],["2/13","2/34","1/6","1/6","1/7","1/7","1/9","1/9","1/9","1/9"],[5.93211570172091e-05,0.000421912124260292,0.00538846593898824,0.00538846593898824,0.0062838777740326,0.0062838777740326,0.00807241931005015,0.00807241931005015,0.00807241931005015,0.00807241931005015],[0.00605075801575533,0.0215175183372749,0.0670338671167522,0.0670338671167522,0.0670338671167522,0.0670338671167522,0.0670338671167522,0.0670338671167522,0.0670338671167522,0.0670338671167522],["SYNGR1;PPFIA3","SYNGR1;PPFIA3","CROT","GALK2","GALK2","XRN2","CROT","EFHB","GPR137","RCC1"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>P.value<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render"],"jsHooks":[]}</script>
<div class="figure">
<img src="assets/cTWAS_allcell_KCNA4.png" alt="" />
<p class="caption">TWAS locus plot for KCNA4</p>
</div>
<div class="figure">
<img src="assets/cTWAS_allcell_NLRC5.png" alt="" />
<p class="caption">TWAS locus plot for NLRC5</p>
</div>
</div>
</div>
<div id="dynamic-egenes-in-nmglut" class="section level4">
<h4>Dynamic eGenes in NMglut</h4>
<p>nmglut has 6 eGenes, all of them are dynamic.</p>
<div id="htmlwidget-7af0e23c13b7c4def9a9" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-7af0e23c13b7c4def9a9">{"x":{"filter":"none","vertical":false,"data":[["ZMYND10","GLIPR2","CPT1C","FOXN2","ADAMTS2","CYP7B1","FZD4","NGEF","ATP5PO","C17orf75","DNPH1"],["ZMYND10","GLIPR2","CPT1C","FOXN2","ADAMTS2","CYP7B1","FZD4","NGEF","ATP5PO","C17orf75","DNPH1"],[0.940355591290139,0.938742625633472,0.905347681624163,0.88301539876483,0.7495226687979,0.742701508150125,0.704223815145326,0.803313540051484,0.910519394634601,0.650158345780552,0.637174050392357],[0,0,0,0,0,0,0,0,0.126226562572741,0,0],[0,0.938742625633472,0.905347681624163,0.88301539876483,0.7495226687979,0,0.704223815145326,0.698673284675076,0.78429283206186,0,0.38558095805507],[0.940355591290139,0,0,0,0,0.742701508150125,0,0,0,0.650158345780552,0],[1.34576949226334e-06,1.02126388188242e-08,9.15204100639211e-12,1.39068311186213e-07,8.39077564402231e-05,3.12490334282285e-08,0.00032874625920371,2.16618855619588e-17,7.29879300814998e-05,3.82254675206968e-06,5.08431134370757e-08],[2.9873517516195e-05,1.27794732164595e-05,2.25970911292455e-10,3.8437362539544e-09,3.30004206047013e-06,1.0428437016814e-05,0.000176440859348177,1.53553028974885e-05,1.88617439430284e-05,2.53016626711147e-05,3.55955348814735e-10],[5.49651800554382e-05,5.7995577776577e-08,2.04958193869384e-11,3.25871821343548e-07,9.10799429516561e-05,9.64260673029828e-08,0.000328746259203471,3.09714353595671e-13,7.29879300815934e-05,1.18614207310325e-05,2.35849164195923e-07],[0.940355591290139,0.938742625633472,0.905347681624163,0.88301539876483,0.7495226687979,0.742701508150125,0.704223815145326,0.698673284675076,0.910519394634601,0.650158345780552,0.38558095805507],[0.940355591290139,0.938742625633472,0.905347681624163,0.88301539876483,0.7495226687979,0.742701508150125,0.704223815145326,0.698673284675076,0.658066269489119,0.650158345780552,0.38558095805507]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>genename<\/th>\n      <th>PIP_sum<\/th>\n      <th>PIP_0hr_nmglut<\/th>\n      <th>PIP_1hr_nmglut<\/th>\n      <th>PIP_6hr_nmglut<\/th>\n      <th>GWAS_Pval_loci<\/th>\n      <th>eQTL_Pval<\/th>\n      <th>Zgene_Pval<\/th>\n      <th>nmglut_sum<\/th>\n      <th>nmglut_PIP_diff<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":2,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":5,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":6,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":7,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":8,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":9,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":10,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[2,3,4,5,6,7,8,9,10]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render","options.columnDefs.2.render","options.columnDefs.3.render","options.columnDefs.4.render","options.columnDefs.5.render","options.columnDefs.6.render","options.columnDefs.7.render","options.columnDefs.8.render"],"jsHooks":[]}</script>
<div id="enrichment-analysis-for-nmglut-specific-genes"
class="section level5">
<h5>Enrichment analysis for NMglut specific genes</h5>
<pre><code>Uploading data to Enrichr... Done.
  Querying GO_Biological_Process_2018... Done.
Parsing results... Done.</code></pre>
<div id="htmlwidget-3989c53c1df7f640b079" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-3989c53c1df7f640b079">{"x":{"filter":"none","vertical":false,"data":[[],[],[],[],[],[]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>P.value<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render"],"jsHooks":[]}</script>
</div>
<div id="enrichment-analysis-for-nmglut-dynamic-genes"
class="section level5">
<h5>Enrichment analysis for NMglut dynamic genes</h5>
<pre><code>Uploading data to Enrichr... Done.
  Querying GO_Biological_Process_2018... Done.
Parsing results... Done.</code></pre>
<div id="htmlwidget-f8241748eb28a8a54961" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-f8241748eb28a8a54961">{"x":{"filter":"none","vertical":false,"data":[[],[],[],[],[],[]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>P.value<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render"],"jsHooks":[]}</script>
<div class="figure">
<img src="assets/cTWAS_allcell_NGEF.png" alt="" />
<p class="caption">TWAS locus plot for NGEF</p>
</div>
<div class="figure">
<img src="assets/cTWAS_allcell_FZD4.png" alt="" />
<p class="caption">TWAS locus plot for FZD4</p>
</div>
</div>
</div>
<div id="dynamic-egenes-in-npglut" class="section level4">
<h4>Dynamic eGenes in NPglut</h4>
<p>npglut has 5 eGenes, 4 of them are dynamic</p>
<div id="htmlwidget-224ae002e7acfef1f0c4" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-224ae002e7acfef1f0c4">{"x":{"filter":"none","vertical":false,"data":[["THBS3","NAGA","ZNF823","OLA1","PCYT1A","SMDT1","GCH1","TDRD6","MOB3A","BMF"],["THBS3","NAGA","ZNF823","OLA1","PCYT1A","SMDT1","GCH1","TDRD6","MOB3A","BMF"],[0.958523468139702,0.952637017566327,0.924230021097604,0.800427559876444,0.755054652974393,0.728947420080327,0.702388865128127,0.786746131007089,0.732585049667497,0.908743687922834],[0,0,0,0,0,0,0,0.217928009627839,0.732585049667497,0.908743687922834],[0,0,0.924230021097604,0,0,0.370719909931534,0,0,0,0],[0.958523468139702,0.952637017566327,0,0.800427559876444,0.755054652974393,0.358227510148793,0.702388865128127,0.265688052633165,0,0],[1.43372935401717e-07,2.39652139016889e-14,3.52976576422236e-10,1.46071910096446e-07,3.97506217345987e-05,2.39652139016889e-14,1.4249724226616e-06,8.20553715310247e-05,2.03366359412943e-06,8.29084111686257e-11],[0.000291784456214919,1.30951538053316e-05,0.000399224210219279,6.76461395411707e-05,1.03911916083748e-05,0.00024832146323647,6.55564350242665e-05,3.10872060031092e-08,1.61527461333437e-05,4.25286848935953e-06],[1.43372935402604e-07,1.0408285816579e-10,3.52976576424559e-10,6.80122025500247e-06,3.97506217344012e-05,7.22475225053278e-12,2.32167902669992e-06,0.000149677344158645,9.69988605710638e-06,1.04020873926883e-10],[0.958523468139702,0.952637017566327,0.924230021097604,0.800427559876444,0.755054652974393,0.728947420080327,0.702388865128127,0.483616062261004,0.732585049667497,0.908743687922834],[0.958523468139702,0.952637017566327,0.924230021097604,0.800427559876444,0.755054652974393,0.728947420080327,0.702388865128127,0.047760043005326,-0.732585049667497,-0.908743687922834]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>genename<\/th>\n      <th>PIP_sum<\/th>\n      <th>PIP_0hr_npglut<\/th>\n      <th>PIP_1hr_npglut<\/th>\n      <th>PIP_6hr_npglut<\/th>\n      <th>GWAS_Pval_loci<\/th>\n      <th>eQTL_Pval<\/th>\n      <th>Zgene_Pval<\/th>\n      <th>npglut_sum<\/th>\n      <th>npglut_PIP_diff<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":2,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":5,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":6,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":7,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":8,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":9,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":10,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[2,3,4,5,6,7,8,9,10]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render","options.columnDefs.2.render","options.columnDefs.3.render","options.columnDefs.4.render","options.columnDefs.5.render","options.columnDefs.6.render","options.columnDefs.7.render","options.columnDefs.8.render"],"jsHooks":[]}</script>
<div id="enrichment-analysis-for-npglut-specific-genes"
class="section level5">
<h5>Enrichment analysis for NPglut specific genes</h5>
<pre><code>Uploading data to Enrichr... Done.
  Querying GO_Biological_Process_2018... Done.
Parsing results... Done.</code></pre>
<div id="htmlwidget-8a77ddace3de46aaf004" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-8a77ddace3de46aaf004">{"x":{"filter":"none","vertical":false,"data":[[],[],[],[],[],[]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>P.value<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render"],"jsHooks":[]}</script>
</div>
<div id="enrichment-analysis-for-npglut-dynamic-genes"
class="section level5">
<h5>Enrichment analysis for NPglut dynamic genes</h5>
<pre><code>Uploading data to Enrichr... Done.
  Querying GO_Biological_Process_2018... Done.
Parsing results... Done.</code></pre>
<div id="htmlwidget-69fb037def67fa5c79bd" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-69fb037def67fa5c79bd">{"x":{"filter":"none","vertical":false,"data":[[],[],[],[],[],[]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>P.value<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render"],"jsHooks":[]}</script>
<div class="figure">
<img src="assets/cTWAS_allcell_NAGA.png" alt="" />
<p class="caption">cTWAS locus plot for NAGA</p>
</div>
<div class="figure">
<img src="assets/cTWAS_allcell_THBS3.png" alt="" />
<p class="caption">TWAS locus plot for THBS3</p>
</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.1.2 (2021-11-01)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur 10.16

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] grid      stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
[1] dichromat_2.0-0     pheatmap_1.0.12     enrichR_3.2        
[4] VennDiagram_1.7.1   futile.logger_1.4.3 ggplot2_3.3.5      
[7] dplyr_1.0.7         workflowr_1.7.0    

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.11          getPass_0.2-2        ps_1.6.0            
 [4] assertthat_0.2.1     rprojroot_2.0.2      digest_0.6.29       
 [7] utf8_1.2.2           R6_2.5.1             futile.options_1.0.1
[10] WriteXLS_6.4.0       evaluate_0.14        httr_1.4.2          
[13] highr_0.9            pillar_1.7.0         rlang_1.1.6         
[16] curl_6.2.2           rstudioapi_0.13      whisker_0.4         
[19] callr_3.7.0          jquerylib_0.1.4      DT_0.26             
[22] rmarkdown_2.11       labeling_0.4.2       stringr_1.4.0       
[25] htmlwidgets_1.5.4    munsell_0.5.0        compiler_4.1.2      
[28] httpuv_1.6.5         xfun_0.29            pkgconfig_2.0.3     
[31] htmltools_0.5.2      tidyselect_1.1.1     tibble_3.1.6        
[34] fansi_1.0.2          crayon_1.4.2         withr_2.4.3         
[37] later_1.3.0          jsonlite_1.7.3       gtable_0.3.0        
[40] lifecycle_1.0.4      DBI_1.1.2            git2r_0.29.0        
[43] magrittr_2.0.2       formatR_1.11         scales_1.1.1        
[46] cli_3.6.5            stringi_1.7.6        farver_2.1.0        
[49] fs_1.5.2             promises_1.2.0.1     bslib_0.3.1         
[52] ellipsis_0.3.2       generics_0.1.2       vctrs_0.6.5         
[55] cowplot_1.1.1        RColorBrewer_1.1-2   rjson_0.2.21        
[58] lambda.r_1.2.4       tools_4.1.2          glue_1.6.1          
[61] purrr_0.3.4          crosstalk_1.2.0      processx_3.5.2      
[64] fastmap_1.1.0        yaml_2.2.2           colorspace_2.0-2    
[67] knitr_1.37           sass_0.4.1          </code></pre>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
