<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Lifan Liang" />

<meta name="date" content="2024-06-27" />

<title>eqtl_cluster</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.26/datatables.js"></script>
<link href="site_libs/dt-core-1.12.1/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.12.1/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.12.1/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">neuron_stimulation_100line_eQTL</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/LifanLiang/neuron_stim_eQTL100">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">eqtl_cluster</h1>
<h4 class="author">Lifan Liang</h4>
<h4 class="date">2024-06-27</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-07-10
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>neuron_stim_eQTL100/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges">
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted
changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges"
class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of
the R Markdown file created these results, you’ll want to first commit
it to the Git repo. If you’re still working on the analysis, you can
ignore this warning. When you’re finished, you can run
<code>wflow_publish</code> to commit the R Markdown file and build the
HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231109code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231109)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231109code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231109)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomLifanLiangneuronstimeQTL100treebd24196f3d1095878c2e49beb0eee805677ad0eftargetblankbd24196a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/tree/bd24196f3d1095878c2e49beb0eee805677ad0ef" target="_blank">bd24196</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomLifanLiangneuronstimeQTL100treebd24196f3d1095878c2e49beb0eee805677ad0eftargetblankbd24196a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/tree/bd24196f3d1095878c2e49beb0eee805677ad0ef" target="_blank">bd24196</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    data/.DS_Store
    Ignored:    data/single_group_ctwas/.DS_Store

Untracked files:
    Untracked:  data/ISLRv2_corrected_June_2023.pdf

Unstaged changes:
    Modified:   analysis/eqtl_cluster.Rmd
    Modified:   data/dynami_beta_cluster.RDS
    Modified:   data/dynamic_beta_cluster_enrichment_GO.RDS

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/eqtl_cluster.Rmd</code>) and HTML
(<code>docs/eqtl_cluster.html</code>) files. If you’ve configured a
remote Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/bd24196f3d1095878c2e49beb0eee805677ad0ef/docs/eqtl_cluster.html" target="_blank">bd24196</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-07-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/9ef346f850e489b6fa1cade218b8b0f76e8fcbf7/analysis/eqtl_cluster.Rmd" target="_blank">9ef346f</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-07-02
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/9ef346f850e489b6fa1cade218b8b0f76e8fcbf7/docs/eqtl_cluster.html" target="_blank">9ef346f</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-07-02
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/d87a4745f6b512ab3f8e55ce06d4b048297dcd68/docs/eqtl_cluster.html" target="_blank">d87a474</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-06-27
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/f7d6213edf9dbe9213a1d439835e43115c39db84/analysis/eqtl_cluster.Rmd" target="_blank">f7d6213</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-06-27
</td>
<td>
wflow_publish(c("analysis", "docs", "data"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/f7d6213edf9dbe9213a1d439835e43115c39db84/docs/eqtl_cluster.html" target="_blank">f7d6213</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-06-27
</td>
<td>
wflow_publish(c("analysis", "docs", "data"))
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="introduction" class="section level2">
<h2>Introduction</h2>
<p>This section describes the procedure to collect the set of dynamic
eQTL and cluster them to identify patterns of dynamic genetic effects
related stimulation response.</p>
<div id="candidate-gene-snp-pairs-for-dynamic-eqtl-testing"
class="section level3">
<h3>Candidate gene-SNP pairs for dynamic eQTL testing</h3>
<p>The candidate gene-SNP pairs for testing come from the union of top
eQTLs from fixed effect eQTL mapping for each condition. The union of
top eQTLs were further LD pruned. For each gene, pairwise <span
class="math inline">\(R^2\)</span> among the set of top eQTLs (9 at
maximum) based on the 100 cell lines were computed. When <span
class="math inline">\(R^2\)</span> is above 0.1, the SNP with higher
nominal P values in <a href="eQTL_glimix.html">main effect eQTLs</a>
were removed.</p>
<p>It results in 15247 gene-SNP pairs with 9880 unique genes.</p>
</div>
<div id="hypothesis-testing" class="section level3">
<h3>Hypothesis testing</h3>
<p>We used <a href="cellregmap.html">CellRegMap</a> to test six
different hypothesis with the candidate set. The first three utilized
samples across 9 conditions altogether, while hypothesis 4-6 were tested
with 3 conditions respectively. The major difference among these
hypothesis is the design matrix for the random slope.</p>
<ol style="list-style-type: decimal">
<li>Condition specific genetic effects across 9 conditions. This model
includes all 9 conditions. In the design matrix, elements (samples)
within the same condition have 1, otherwise 0.</li>
<li>Cell type specific genetic effects across 9 conditions. This model
includes all 9 conditions. In the design matrix, elements within the
same cell type have 1, otherwise 0.</li>
<li>Shared time point specific genetic effects across 9 conditions. This
model includes all 9 conditions. In the design matrix, elements within
the same time point have 1, otherwise 0.</li>
<li>Time point specific effects within GABA cells. This model includes
the three conditions within the GABA cell type. In the design matrix,
elements within the same time point have 1, otherwise 0.</li>
<li>Time point specific effects within NMglut cells; This model includes
the three conditions within the NMglut cell type. In the design matrix,
elements within the same time point have 1, otherwise 0.</li>
<li>Time point specific effects within NPglut cells. This model includes
the three conditions within the NPglut cell type. In the design matrix,
elements within the same time point have 1, otherwise 0.</li>
</ol>
<p>After obtaining nominal P values from CellRegMap, we used Storey’s Q
value for FDR correction across the cadidates, ignoring weak LD within
genes. Q value &lt; 0.05 is the significant threshold.</p>
<pre><code> condition  cell_type time_point       GABA     NPglut     NMglut 
      2434       2082        822        530        660        227 </code></pre>
<p>The Venn diagram for eQTLs discovered from hypothesis 4-6 suggest
higher cell type specificity than previous results in fixed effect
interaction. It could be due to the LD pruning for the current
results.</p>
<p><img src="assets/cellregmap_celltype_venn.png" /></p>
</div>
</div>
<div id="clustering-dynamic-eqtls" class="section level2">
<h2>Clustering dynamic eQTLs</h2>
<p>We used the union of significant gene-SNP pairs described above for
hierarchical clustering.There are 3928 pairs in total. Due to the
residual correlation caused by repeated measurement / individual random
effects, we used effect size and standard error from the interaction
only mixed linear model described in <a href="mashr_analysis.html">MASH
analysis</a>. Z scores are consistent with condition by condition (CxC)
eQTL in general.</p>
<p><img src="assets/interaction_vs_CXC.png" /></p>
<p>The positive effect direction is the most common direction for the
gene-SNP pairs across 9 conditions. Effect size would be set to zero if
the absolute Z score is less than 2. The whole gene-SNP pair was removed
if the maximum absolute Z score across 9 conditions is less than 1.
After filtering, there are 3452 gene-SNP pairs for clustering. Dynamic
eQTL patterns are quite clear for any one time point.</p>
<p><img src="assets/Dynamic_eQTL_clustering.png" /></p>
<div id="enrichment-analysis" class="section level3">
<h3>Enrichment analysis</h3>
<p>Enrichment analysis with 2023 GO biological process. Each row is the
top enrichment term corresponding to each cluster.</p>
<div id="htmlwidget-7657cc3e42637b46d6e4" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-7657cc3e42637b46d6e4">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15"],["Antigen Processing And Presentation Of Endogenous Peptide Antigen (GO:0002483)","Small GTPase Mediated Signal Transduction (GO:0007264)","Positive Regulation Of Vascular Endothelial Growth Factor Signaling Pathway (GO:1900748)","Negative Regulation Of Binding (GO:0051100)","rRNA Processing (GO:0006364)","Positive Regulation Of Cell Projection Organization (GO:0031346)","Positive Regulation Of Small GTPase Mediated Signal Transduction (GO:0051057)","Positive Regulation Of Lipopolysaccharide-Mediated Signaling Pathway (GO:0031666)","Regulation Of Plasma Membrane Bounded Cell Projection Assembly (GO:0120032)","Regulation Of Peptidyl-Cysteine S-nitrosylation (GO:2000169)","DNA Damage Response (GO:0006974)","Chemical Synaptic Transmission (GO:0007268)","Neurotrophin Signaling Pathway (GO:0038179)","Regulation Of Adenylate Cyclase Activity (GO:0045761)","Regulation Of cAMP-mediated Signaling (GO:0043949)"],["6/21","8/112","2/7","6/86","9/101","8/118","4/17","2/6","5/75","2/5","7/384","9/273","3/15","3/17","3/18"],[0.000100309836312528,9.27386064013692e-06,0.00146665597181593,0.000518428812721563,2.97037834095938e-05,2.07132793903162e-05,7.75046127682903e-05,0.00106581848444788,0.000328657573492068,0.000500539984616602,0.00236642327031853,0.00128351707345751,0.000301220179291295,0.000275902173828237,0.00052104852825584],[0.253783885870695,0.0106742135967976,0.415191049884098,0.462783561621038,0.0472884231880734,0.0256016133264309,0.12966521716135,0.373908755791144,0.122814311912314,0.202564991085168,0.29815473972833,0.351004089698251,0.172296793469256,0.270108228177844,0.362662133033816],["ERAP1;HLA-B;HLA-C;HLA-A;HLA-DRB1;ULBP3","TIAM2;DIRAS3;RASGEF1B;DOK6;RASGRF2;RASGRF1;RAB38;RIN2","SMOC2;ROBO1","LRPAP1;IFI16;CAV1;SUMO4;EFHB;AURKA","EBNA1BP2;EXOSC10;DDX47;NVL;MAK16;NOLC1;RIOK2;RIOK1;DDX52","CHODL;PTPRD;NLGN1;TENM3;FBXW8;FUT9;PLXNC1;SCN1B","NTRK1;NOTCH1;SRC;ITGAV","PRKCA;SASH1","P2RX4;PODXL;TBC1D14;RAC1;TBC1D22A","ATP2B4;TXN","MSH6;RFC2;XRCC2;NFATC2;KMT5B;HDGFL2;REXO4","CNP;SYT1;RPS6KA2;CHRNA7;GAD2;LIN7A;SEPTIN9;CPLX1;GABRG1","NTRK1;DDIT4;RAPGEF1","STIM1;NF1;CRHR1","PEX5L;PDE3B;RAPGEF2"],[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>P.value<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n      <th>cluster<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[3,4,6]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.1.2 (2021-11-01)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur 10.16

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] workflowr_1.7.0

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.11       bslib_0.3.1       compiler_4.1.2    pillar_1.7.0     
 [5] later_1.3.0       git2r_0.29.0      jquerylib_0.1.4   tools_4.1.2      
 [9] getPass_0.2-2     digest_0.6.29     jsonlite_1.7.3    evaluate_0.14    
[13] tibble_3.1.6      lifecycle_1.0.1   pkgconfig_2.0.3   rlang_1.0.0      
[17] cli_3.1.1         rstudioapi_0.13   crosstalk_1.2.0   yaml_2.2.2       
[21] xfun_0.29         fastmap_1.1.0     httr_1.4.2        stringr_1.4.0    
[25] knitr_1.37        htmlwidgets_1.5.4 sass_0.4.1        fs_1.5.2         
[29] vctrs_0.3.8       DT_0.26           rprojroot_2.0.2   glue_1.6.1       
[33] R6_2.5.1          processx_3.5.2    fansi_1.0.2       rmarkdown_2.11   
[37] callr_3.7.0       magrittr_2.0.2    whisker_0.4       ps_1.6.0         
[41] promises_1.2.0.1  htmltools_0.5.2   ellipsis_0.3.2    httpuv_1.6.5     
[45] utf8_1.2.2        stringi_1.7.6     crayon_1.4.2     </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
