<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Lifan Liang" />

<meta name="date" content="2024-03-21" />

<title>eQTL_glimix</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">neuron_stimulation_100line_eQTL</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/LifanLiang/neuron_stim_eQTL100">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">eQTL_glimix</h1>
<h4 class="author">Lifan Liang</h4>
<h4 class="date">2024-03-21</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-06-27
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 2
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>neuron_stim_eQTL100/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomLifanLiangneuronstimeQTL100treef7d6213edf9dbe9213a1d439835e43115c39db84targetblankf7d6213a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/tree/f7d6213edf9dbe9213a1d439835e43115c39db84" target="_blank">f7d6213</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomLifanLiangneuronstimeQTL100treef7d6213edf9dbe9213a1d439835e43115c39db84targetblankf7d6213a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/tree/f7d6213edf9dbe9213a1d439835e43115c39db84" target="_blank">f7d6213</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    data/.DS_Store
    Ignored:    data/single_group_ctwas/.DS_Store

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/eQTL_glimix.Rmd</code>) and HTML
(<code>docs/eQTL_glimix.html</code>) files. If you’ve configured a
remote Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/f7d6213edf9dbe9213a1d439835e43115c39db84/analysis/eQTL_glimix.Rmd" target="_blank">f7d6213</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-06-27
</td>
<td>
wflow_publish(c("analysis", "docs", "data"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/f7d6213edf9dbe9213a1d439835e43115c39db84/docs/eQTL_glimix.html" target="_blank">f7d6213</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-06-27
</td>
<td>
wflow_publish(c("analysis", "docs", "data"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/abd45a30a1f6e211574ed9920e8622dd79db539c/docs/eQTL_glimix.html" target="_blank">abd45a3</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-05-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/d8f18556a075dc85dabe8636f459eda77c11bf39/analysis/eQTL_glimix.Rmd" target="_blank">d8f1855</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-05-16
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/d8f18556a075dc85dabe8636f459eda77c11bf39/docs/eQTL_glimix.html" target="_blank">d8f1855</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-05-16
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/f3ff2d007584ecec6a62c3f4750d5f4601455301/docs/eQTL_glimix.html" target="_blank">f3ff2d0</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-29
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/63ff50e78e00a1204c9861de585e0e5d4c4ef387/analysis/eQTL_glimix.Rmd" target="_blank">63ff50e</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-29
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/63ff50e78e00a1204c9861de585e0e5d4c4ef387/docs/eQTL_glimix.html" target="_blank">63ff50e</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-29
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/7eef9271b6b07d247ea4ecbe5d1746e513de5b4b/docs/eQTL_glimix.html" target="_blank">7eef927</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/f1b144a2e5c67f466d7cc680a62804af5eb0028a/analysis/eQTL_glimix.Rmd" target="_blank">f1b144a</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-25
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/929b1705a3840c8e39b94e517ed06b3c4b32d2c4/docs/eQTL_glimix.html" target="_blank">929b170</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/01ad2dabea989bbe6012ad76cda92637955fd694/analysis/eQTL_glimix.Rmd" target="_blank">01ad2da</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-25
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/1e7c84dfbc8dd4097779c29e81c2f5e7b6f5433f/docs/eQTL_glimix.html" target="_blank">1e7c84d</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/6904664ed75c09b14e38f1c4b5f61c0bc33e3284/analysis/eQTL_glimix.Rmd" target="_blank">6904664</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-25
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/01fabbcc312bbae1a90ea499ff6e6dbc616d4f09/docs/eQTL_glimix.html" target="_blank">01fabbc</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/df91c6d6cd71cc9a7fedb0976fac0d8e93089b94/analysis/eQTL_glimix.Rmd" target="_blank">df91c6d</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-25
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/c57a15cc3f72ca9ac721de045c6393a9208d91a4/docs/eQTL_glimix.html" target="_blank">c57a15c</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/55ff8e46b8e34e0af4dd176483eb7560ca4a49ac/analysis/eQTL_glimix.Rmd" target="_blank">55ff8e4</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-17
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/55ff8e46b8e34e0af4dd176483eb7560ca4a49ac/docs/eQTL_glimix.html" target="_blank">55ff8e4</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-17
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/1b438e1d554ef95f811eea1359fcb80f409691b3/docs/eQTL_glimix.html" target="_blank">1b438e1</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/25c360f208ab0fa33479a5a80542e0ff721e5504/analysis/eQTL_glimix.Rmd" target="_blank">25c360f</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-04
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/25c360f208ab0fa33479a5a80542e0ff721e5504/docs/eQTL_glimix.html" target="_blank">25c360f</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-04
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/4ae5cadc48ac80c3bbfd7b74cd768fe8da31166f/docs/eQTL_glimix.html" target="_blank">4ae5cad</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/600a88fc701b52faa92fe43aade58146d058f3fc/analysis/eQTL_glimix.Rmd" target="_blank">600a88f</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-03
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/600a88fc701b52faa92fe43aade58146d058f3fc/docs/eQTL_glimix.html" target="_blank">600a88f</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-03
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/16cc513c896261c89a38fa217ba9a5b53f228cd8/docs/eQTL_glimix.html" target="_blank">16cc513</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/586008310eb8456d7d545b51b0f4847a5d6e1e84/analysis/eQTL_glimix.Rmd" target="_blank">5860083</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-03-21
</td>
<td>
wflow_publish("analysis")
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="introduction" class="section level3">
<h3>Introduction</h3>
<p>This page introduces how to adapt a python package, “glimix_core”, to
fit the individual random effect shared across 9 conditions (3
timepoints by 3 cell types).</p>
</div>
<div id="individual-random-effect-shared-across-all-contexts"
class="section level3">
<h3>Individual random effect shared across all contexts</h3>
<div id="null-model" class="section level4">
<h4>Null model</h4>
<p>First, gene expression PC is computed from residual expression after
regressing out other covariates. Then gene expression was regressed out
of all covariates (e.g. sex, age, disease status, 5 genotype PCs, 15
expression PCs) jointly across conditions. The coefficients of
covariates were estimated together with the random effects. In this way,
using residual expression to estimate model likelihood and individual
random effects from the null model is identical to using all covariates.
Regressing out all covariates speeds up the genotype scanning by around
100 times. However, using residual gene expression may underestimate
genetic effects in the alternative model.</p>
<p><span class="math display">\[
y = C\mu + \epsilon
\]</span></p>
<p>where <span class="math inline">\(X\)</span> is a vector of 1 since
all other covariates have been regressed out beforehand, <span
class="math inline">\(\mu\)</span> is the intercept term, and <span
class="math inline">\(\epsilon\)</span> was sampled from a multivariate
Normal with forms below:</p>
<p><span class="math display">\[
\epsilon \sim MVN(0, S(v_0 \cdot Z + v_1 \cdot I))
\]</span></p>
<p>where <span class="math inline">\(Z\)</span> is the design matrix
such that samples within the same individual have correlation of 1 and
samples for different individuals are 0, <span
class="math inline">\(I\)</span> is the identity matrix to capture i.i.d
noise, <span class="math inline">\(S\)</span> is the overall scaling
coefficient, <span class="math inline">\(v_0\)</span> and <span
class="math inline">\(v_1\)</span> captures the relative weights of
individual random effects and noise. There are 6822 genes with
individual variation more than 1% of overall error.</p>
<p>The null model provides baseline model likelihood and an estimate of
<span class="math inline">\(v_0\)</span> and <span
class="math inline">\(v_1\)</span>.</p>
<div class="figure">
<img src="assets/log_glimix_random_effect.png" alt="" />
<p class="caption">distribution of <span
class="math inline">\(log10(\dfrac{v_0}{v_0+v_1})\)</span></p>
</div>
<div class="figure">
<img src="assets/glimix_random_effect.png" alt="" />
<p class="caption">distribution of <span
class="math inline">\(\dfrac{v_0}{v_0+v_1}\)</span></p>
</div>
</div>
<div id="alternative-model" class="section level4">
<h4>Alternative model</h4>
<p>We estimate the persistent genetic effects with following model. All
parameters will be re-estimated except <span
class="math inline">\(v_0\)</span> and <span
class="math inline">\(v_1\)</span> in the distribution of <span
class="math inline">\(\epsilon\)</span>.</p>
<p><span class="math display">\[
y = G\beta + X\mu+ \epsilon
\]</span></p>
</div>
<div id="likelihood-ratio-testing" class="section level4">
<h4>Likelihood ratio testing</h4>
<p>The alternative model provides the alternative model likelihood and
an estimate of <span class="math inline">\(\beta\)</span> and its
standard error. Since the log likelihood ratio follows Chi-square
distribution asymptotically, we can compute the P values for genetic
effects:</p>
<p><span class="math display">\[
\lambda_{LR} = 2(\ell_{alt} - \ell_{null}) \sim Chisq(1)
\]</span></p>
</div>
<div id="p-value-calibration" class="section level4">
<h4>P value calibration</h4>
<p>We permuted the genotypes such that gene expression in different
conditions belonging to the same individual still belong to the same
individual. For example, during permutation, gene expression for CD02 in
all time points and cell tyeps will be linked to the genotypes of
CD04.</p>
<p>We permuted genotyeps for 500 genes with lowest individual random
effects, genotypes were permuted 100 times for each gene, P values are
well calibrated:</p>
<div class="figure">
<img src="assets/P_calibration_low_randeff.png" alt="" />
<p class="caption">P values with permuted genotyeps for genes with low
individual random effects</p>
</div>
<p>We also performed the permutation for 1000 genes with highest
individual random effects, P values seems slightly inflated.</p>
<div class="figure">
<img src="assets/P_calibration_high_randeff.png" alt="" />
<p class="caption">P values with permuted genotyeps for genes with high
individual random effects</p>
</div>
</div>
<div id="gene-level-fdr-control" class="section level4">
<h4>Gene level FDR control</h4>
<p>We followed the the procedure described in FastQTL. Using the same
permutation above, we obtain 1000 mininum P values from 1000 permutation
with each gene. One thing to be aware is that all cis genotypes were
shuffled together so that the LD structure remains. And then we fit a
Beta distribution to the 1000 minimum P values with MLE. The adjusted P
value would be the cumulative probability of the empirical minimum P
value in the Beta distribution.</p>
<p><img src="assets/glimix_beta_fitting.png"
alt="Example of Beta distribution fitting" /> The number of eGenes
idenitified by mixed effect is much smaller than that by aggregating
conditions together and run fixed effects.</p>
<div class="figure">
<img src="assets/venn_limix_vs_fixed.png" alt="" />
<p class="caption">eGene overlap between mixed effect eQTL and fixed
effect eQTL.</p>
</div>
</div>
</div>
<div id="estimating-individual-random-effects-with-lead-variant"
class="section level3">
<h3>Estimating individual random effects with lead variant</h3>
<p>Given the lead variants for each gene from the eQTL results above, we
first estimated fixed effects of covariates and individual random
effects with the following model:</p>
<p><span class="math display">\[
y = G\beta + X\mu+ \epsilon
\]</span> where <span class="math inline">\(G\)</span> is the genotype
of the corresponding lead variant. <span
class="math inline">\(X\)</span> includes all the covariates and the
offset, <span class="math inline">\(\mu\)</span> is the fixed effects,
and <span class="math inline">\(\epsilon\)</span> follows the same
definition as in the section above.</p>
<p>Then we regressed out covariates based on the fixed effect estimated
and the model and fixed <span class="math inline">\(v_0\)</span> and
<span class="math inline">\(v_1\)</span> estimated from the model. This
time, we observed individual random effect reduced a lot compared to
noise.</p>
<p><img src="assets/indiv_randeff_leadSNP.png" /></p>
<div id="association-testing" class="section level4">
<h4>Association testing</h4>
<p>Null model, alternative model, and likelihood ratio testing are
identical to previous precedure except the the ratio of the random
effects are fixed beforehand. We continue to use Beta approximation to
obtain adjusted P values for each gene.</p>
<p>Below is the scatter plot of the adjusted P values with Beta
approximation (Y axis) and empirical P values (X axis) on the scale of
%-log10$. Although a little too stringent, Beta approximation is quite
accurate at the tail.</p>
<p><img src="assets/beta_calibration.png" /></p>
<p>Instead of Storey’s Pi in the previous approach, we performed
Benjamini-Hochberg for FDR control. Genes with FDR &lt; 0.05 is
considered eGenes. Overall we identified 8462 eGenes, around 700 eGenes
more than the bulk approach.</p>
<p><img src="assets/venn_limix_vs_fixed%20(1).png" /></p>
<p>The scatter plot of P values shows linear mixed model has higher
power than bulk.</p>
<p><img src="assets/bulk_vs_leadSNP_scatter.png" /></p>
</div>
</div>
<div id="comparing-with-fixed-effect-model" class="section level3">
<h3>Comparing with fixed effect model</h3>
<p>We used the same normalized expression and covariates as input for
matrixeQTL, so as to investigate the role of individual random effects.
It seems individual random effect is competing with genetic effects. The
increased power compared to bulk is due to the increased sample size by
stacking all conditions.</p>
<p><img src="assets/IRE_egenes_venn.png" /></p>
<p>For genes with less than 1% individual random effects. Mixed effect
model and fixed effect model have highly correlated nominal P
values.</p>
<p><img src="assets/noIRE_leadSNP_vs_mateqtl.png" /></p>
<p>For genes with more than 1% individual random effects. Mixed effect
model has lower nominal P values than fixed effect model. It seems by
pretending we have 824 independent samples, power would be greatly
increased. However, correlattion caused by repeated measurement of the
same individuals would confound genetic effects. Although most repeated
measurement correlation are already captured by genotype PCs and
expression PCs.</p>
<p><img src="assets/IRE_leadSNP_vs_mateqtl.png" /></p>
<p>Another minor issue could be the permutation in matrixeQTL did not
preserve individual structrure. Hence FDR is not stringent enough.</p>
<p><img src="assets/noIRE_FDR_leadSNP_vs_mateqtl.png" /></p>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
