<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Lifan Liang" />

<meta name="date" content="2024-04-11" />

<title>cTWAS Results: eQTL results for Bipolar</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.26/datatables.js"></script>
<link href="site_libs/dt-core-1.12.1/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.12.1/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.12.1/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">neuron_stimulation_100line_eQTL</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/LifanLiang/neuron_stim_eQTL100">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">cTWAS Results: eQTL results for
Bipolar</h1>
<h4 class="author">Lifan Liang</h4>
<h4 class="date">2024-04-11</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-04-16
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>neuron_stim_eQTL100/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges">
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted
changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges"
class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of
the R Markdown file created these results, you’ll want to first commit
it to the Git repo. If you’re still working on the analysis, you can
ignore this warning. When you’re finished, you can run
<code>wflow_publish</code> to commit the R Markdown file and build the
HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231109code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231109)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231109code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231109)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomLifanLiangneuronstimeQTL100tree5530b53f8c72c7406f8b8753830c68ab3fe79e26targetblank5530b53a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/tree/5530b53f8c72c7406f8b8753830c68ab3fe79e26" target="_blank">5530b53</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomLifanLiangneuronstimeQTL100tree5530b53f8c72c7406f8b8753830c68ab3fe79e26targetblank5530b53a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/tree/5530b53f8c72c7406f8b8753830c68ab3fe79e26" target="_blank">5530b53</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    data/.DS_Store
    Ignored:    data/single_group_ctwas/.DS_Store

Unstaged changes:
    Modified:   analysis/eQTL_cTWAS_bipolar.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/eQTL_cTWAS_bipolar.Rmd</code>)
and HTML (<code>docs/eQTL_cTWAS_bipolar.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/5530b53f8c72c7406f8b8753830c68ab3fe79e26/docs/eQTL_cTWAS_bipolar.html" target="_blank">5530b53</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/1655cdd0d6c6e48267022579b86b5f96418e3f6e/analysis/eQTL_cTWAS_bipolar.Rmd" target="_blank">1655cdd</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-16
</td>
<td>
wflow_publish(c("analysis", "docs", "data"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/1655cdd0d6c6e48267022579b86b5f96418e3f6e/docs/eQTL_cTWAS_bipolar.html" target="_blank">1655cdd</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-16
</td>
<td>
wflow_publish(c("analysis", "docs", "data"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/6d64fd37636e5c33f662874c872ad70be72b733f/docs/eQTL_cTWAS_bipolar.html" target="_blank">6d64fd3</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/6e61fb8dd16291844c4e378728d8495edffac318/analysis/eQTL_cTWAS_bipolar.Rmd" target="_blank">6e61fb8</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-11
</td>
<td>
wflow_publish(c("analysis"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/ed904886dce4cfb993e1fb1812f5d51a863b4456/docs/eQTL_cTWAS_bipolar.html" target="_blank">ed90488</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bb0d1d36b058717a68e94e56a209219706f7f7f4/analysis/eQTL_cTWAS_bipolar.Rmd" target="_blank">bb0d1d3</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-11
</td>
<td>
wflow_publish(c("analysis", "docs", "data"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/bb0d1d36b058717a68e94e56a209219706f7f7f4/docs/eQTL_cTWAS_bipolar.html" target="_blank">bb0d1d3</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-11
</td>
<td>
wflow_publish(c("analysis", "docs", "data"))
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="weights" class="section level2">
<h2>Weights</h2>
<p>Use the same set of eGenes as previous restuls.These eGenes are the
unions of eGenes across all cell types. CONTENT will produce
tissue-specific weights for that gene if it has nonzeros weights in that
certain tissue.</p>
<pre><code>
Attaching package: &#39;dplyr&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:stats&#39;:

    filter, lag</code></pre>
<pre><code>The following objects are masked from &#39;package:base&#39;:

    intersect, setdiff, setequal, union</code></pre>
<pre><code>       SNP   0hr_GABA 0hr_nmglut 0hr_npglut   1hr_GABA 1hr_nmglut 1hr_npglut 
   6934250       4157       2553       5151       4183       2516       5112 
  6hr_GABA 6hr_nmglut 6hr_npglut 
      4127       2562       5170 </code></pre>
</div>
<div id="parameters" class="section level2">
<h2>Parameters</h2>
<p><img src="figure/eQTL_cTWAS_bipolar.Rmd/unnamed-chunk-2-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-2-1">
Past versions of unnamed-chunk-2-1.png
</button>
</p>
<div id="fig-unnamed-chunk-2-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/1655cdd0d6c6e48267022579b86b5f96418e3f6e/docs/figure/eQTL_cTWAS_bipolar.Rmd/unnamed-chunk-2-1.png" target="_blank">1655cdd</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bb0d1d36b058717a68e94e56a209219706f7f7f4/docs/figure/eQTL_cTWAS_bipolar.Rmd/unnamed-chunk-2-1.png" target="_blank">bb0d1d3</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="prior-for-each-context" class="section level3">
<h3>Prior for each context</h3>
<p><img src="figure/eQTL_cTWAS_bipolar.Rmd/unnamed-chunk-3-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-3-1">
Past versions of unnamed-chunk-3-1.png
</button>
</p>
<div id="fig-unnamed-chunk-3-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/1655cdd0d6c6e48267022579b86b5f96418e3f6e/docs/figure/eQTL_cTWAS_bipolar.Rmd/unnamed-chunk-3-1.png" target="_blank">1655cdd</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bb0d1d36b058717a68e94e56a209219706f7f7f4/docs/figure/eQTL_cTWAS_bipolar.Rmd/unnamed-chunk-3-1.png" target="_blank">bb0d1d3</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="pve-for-each-context" class="section level3">
<h3>PVE for each context</h3>
<p>Total PVE from all eGenes in all contexts is 13.5%.</p>
<p><img src="figure/eQTL_cTWAS_bipolar.Rmd/unnamed-chunk-4-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-4-1">
Past versions of unnamed-chunk-4-1.png
</button>
</p>
<div id="fig-unnamed-chunk-4-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/1655cdd0d6c6e48267022579b86b5f96418e3f6e/docs/figure/eQTL_cTWAS_bipolar.Rmd/unnamed-chunk-4-1.png" target="_blank">1655cdd</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bb0d1d36b058717a68e94e56a209219706f7f7f4/docs/figure/eQTL_cTWAS_bipolar.Rmd/unnamed-chunk-4-1.png" target="_blank">bb0d1d3</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/eQTL_cTWAS_bipolar.Rmd/unnamed-chunk-5-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-5-1">
Past versions of unnamed-chunk-5-1.png
</button>
</p>
<div id="fig-unnamed-chunk-5-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/1655cdd0d6c6e48267022579b86b5f96418e3f6e/docs/figure/eQTL_cTWAS_bipolar.Rmd/unnamed-chunk-5-1.png" target="_blank">1655cdd</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bb0d1d36b058717a68e94e56a209219706f7f7f4/docs/figure/eQTL_cTWAS_bipolar.Rmd/unnamed-chunk-5-1.png" target="_blank">bb0d1d3</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="results" class="section level2">
<h2>Results</h2>
<p>There are 55 egenes if the cutoff was set to PIP&gt;0.5.</p>
<p><img src="figure/eQTL_cTWAS_bipolar.Rmd/unnamed-chunk-6-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-6-1">
Past versions of unnamed-chunk-6-1.png
</button>
</p>
<div id="fig-unnamed-chunk-6-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/1655cdd0d6c6e48267022579b86b5f96418e3f6e/docs/figure/eQTL_cTWAS_bipolar.Rmd/unnamed-chunk-6-1.png" target="_blank">1655cdd</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bb0d1d36b058717a68e94e56a209219706f7f7f4/docs/figure/eQTL_cTWAS_bipolar.Rmd/unnamed-chunk-6-1.png" target="_blank">bb0d1d3</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="comparison-with-schizophrenia-ctwas-results"
class="section level3">
<h3>Comparison with Schizophrenia cTWAS results</h3>
<pre><code>Loading required package: grid</code></pre>
<pre><code>Loading required package: futile.logger</code></pre>
<pre><code>[1] 1</code></pre>
<p><img src="docs/assets/venn_SCZ_vs_bipolar.png" /></p>
</div>
<div id="enrichment-analysis-regardless-of-cell-types"
class="section level3">
<h3>Enrichment analysis regardless of cell types</h3>
<div id="enrichment-analysis-of-all-genes-together"
class="section level4">
<h4>Enrichment analysis of all genes together</h4>
<pre><code>Welcome to enrichR
Checking connection ... </code></pre>
<pre><code>Enrichr ... Connection is Live!
FlyEnrichr ... Connection is Live!
WormEnrichr ... Connection is Live!
YeastEnrichr ... Connection is Live!
FishEnrichr ... Connection is Live!
OxEnrichr ... Connection is Live!</code></pre>
<pre><code>Uploading data to Enrichr... Done.
  Querying GO_Biological_Process_2018... Done.
Parsing results... Done.</code></pre>
<div id="htmlwidget-7657cc3e42637b46d6e4" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-7657cc3e42637b46d6e4">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10"],["positive regulation of endothelial cell migration (GO:0010595)","positive regulation of blood vessel endothelial cell migration (GO:0043536)","regulation of blood vessel endothelial cell migration (GO:0043535)","positive regulation of epithelial cell migration (GO:0010634)","positive regulation of JNK cascade (GO:0046330)","positive regulation of stress-activated MAPK cascade (GO:0032874)","retrograde transport, endosome to Golgi (GO:0042147)","NADH metabolic process (GO:0006734)","protein repair (GO:0030091)","regulation of protein K63-linked ubiquitination (GO:1900044)"],["3/70","2/45","2/50","2/74","2/76","2/80","2/82","1/6","1/6","1/6"],[0.000113071615530326,0.00167639746200947,0.00206572935455435,0.00446486536638843,0.00470335896749297,0.00519768076905213,0.00545344788828848,0.00807362689081275,0.00807362689081275,0.00807362689081275],[0.0307554794242486,0.0895544953924226,0.0895544953924226,0.0895544953924226,0.0895544953924226,0.0895544953924226,0.0895544953924226,0.0895544953924226,0.0895544953924226,0.0895544953924226],["HMGB1;PLCG1;SASH1","HMGB1;PLCG1","HMGB1;PLCG1","PLCG1;SASH1","HMGB1;SASH1","HMGB1;SASH1","ARFRP1;DENND2A","MDH1","MSRA","SASH1"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>P.value<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render"],"jsHooks":[]}</script>
</div>
<div id="enrichment-analysis-of-dynamic-genes" class="section level4">
<h4>Enrichment analysis of dynamic genes</h4>
<p>PIPs in time points <code>6hr</code> and <code>1hr</code> are summed
up and then subtracted the time point <code>0hr</code>. If the
difference is larger than 0.5, genes are called dynamic.</p>
<pre><code>Uploading data to Enrichr... Done.
  Querying GO_Biological_Process_2018... Done.
Parsing results... Done.</code></pre>
<div id="htmlwidget-427ea3e24281aa554de8" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-427ea3e24281aa554de8">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10"],["regulation of protein K63-linked ubiquitination (GO:1900044)","regulation of protein localization to cell cortex (GO:1904776)","NADH metabolic process (GO:0006734)","oxaloacetate metabolic process (GO:0006107)","malate metabolic process (GO:0006108)","cellular response to histamine (GO:0071420)","cytoskeletal anchoring at nuclear membrane (GO:0090286)","regulation of morphogenesis of an epithelium (GO:1905330)","regulation of organ morphogenesis (GO:2000027)","response to histamine (GO:0034776)"],["1/6","1/6","1/6","1/7","1/7","1/7","1/8","1/9","1/9","1/10"],[0.00359499844647963,0.00359499844647963,0.00359499844647963,0.00419301510496052,0.00419301510496052,0.00419301510496052,0.00479070309341298,0.00538806252238183,0.00538806252238183,0.00598509351248777],[0.0453744640101918,0.0453744640101918,0.0453744640101918,0.0453744640101918,0.0453744640101918,0.0453744640101918,0.0453744640101918,0.0453744640101918,0.0453744640101918,0.0453744640101918],["SASH1","SAPCD2","MDH1","MDH1","MDH1","GABRB2","SYNE1","SAPCD2","SAPCD2","GABRB2"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>P.value<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render"],"jsHooks":[]}</script>
</div>
<div id="enrichment-analysis-of-static-genes" class="section level4">
<h4>Enrichment analysis of static genes</h4>
<p>Static genes are significant genes that are not dynamic genes.</p>
<pre><code>Uploading data to Enrichr... Done.
  Querying GO_Biological_Process_2018... Done.
Parsing results... Done.</code></pre>
<div id="htmlwidget-a50efc78b4c04b4a25da" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-a50efc78b4c04b4a25da">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10"],["positive regulation of blood vessel endothelial cell migration (GO:0043536)","regulation of blood vessel endothelial cell migration (GO:0043535)","positive regulation of endothelial cell migration (GO:0010595)","plasma membrane bounded cell projection organization (GO:0120036)","protein repair (GO:0030091)","neuron development (GO:0048666)","T-helper 1 type immune response (GO:0042088)","regulation of toll-like receptor 9 signaling pathway (GO:0034163)","V(D)J recombination (GO:0033151)","regulation of dendritic cell differentiation (GO:2001198)"],["2/45","2/50","2/70","2/118","1/6","2/144","1/7","1/7","1/7","1/7"],[0.0005101666330676,0.000629902065454966,0.00123110357510169,0.0034466401799523,0.00449206839321453,0.00508327752447252,0.00523891689414434,0.00523891689414434,0.00523891689414434,0.00523891689414434],[0.0636884033875521,0.0636884033875521,0.0636884033875521,0.0636884033875521,0.0636884033875521,0.0636884033875521,0.0636884033875521,0.0636884033875521,0.0636884033875521,0.0636884033875521],["HMGB1;PLCG1","HMGB1;PLCG1","HMGB1;PLCG1","HMGB1;FRY","MSRA","HMGB1;FRY","HMGB1","HMGB1","HMGB1","HMGB1"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>P.value<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render"],"jsHooks":[]}</script>
</div>
</div>
<div id="cell-type-specific-egenes" class="section level3">
<h3>Cell type specific eGenes</h3>
<p>An eGene is assigned to a cell type if over 60% of the PIP sum are
from that cell type. Genes with higher sum of PIP are more likely to be
cell type specific.</p>
<pre><code>
      GABA     nmglut     npglut Unspecific 
         4          8         11          4 </code></pre>
<div id="cell-type-composition-across-pip-distribution"
class="section level4">
<h4>Cell type composition across PIP distribution</h4>
<p><img src="figure/eQTL_cTWAS_bipolar.Rmd/unnamed-chunk-12-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-12-1">
Past versions of unnamed-chunk-12-1.png
</button>
</p>
<div id="fig-unnamed-chunk-12-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/1655cdd0d6c6e48267022579b86b5f96418e3f6e/docs/figure/eQTL_cTWAS_bipolar.Rmd/unnamed-chunk-12-1.png" target="_blank">1655cdd</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bb0d1d36b058717a68e94e56a209219706f7f7f4/docs/figure/eQTL_cTWAS_bipolar.Rmd/unnamed-chunk-12-1.png" target="_blank">bb0d1d3</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-04-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="dynamic-egenes-in-gaba" class="section level4">
<h4>Dynamic eGenes in GABA</h4>
<p>GABA has 4 eGenes, 0 of them are dynamic.</p>
<div id="htmlwidget-1c14e9d616c779d1b646" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-1c14e9d616c779d1b646">{"x":{"filter":"none","vertical":false,"data":[["73","46","55","7"],["SYNRG","MSRA","ONECUT3","ARFRP1"],[0.575939045152933,0.704123223377119,0.947472580342486,0.955488960056224],[0.575939045152933,0.704123223377119,0.947472580342486,0.955488960056224],[0,0,0,0],[0,0,0,0],[0.575939045152933,0.704123223377119,0.947472580342486,0.955488960056224],[-0.575939045152933,-0.704123223377119,-0.947472580342486,-0.955488960056224]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>genename<\/th>\n      <th>PIP_sum<\/th>\n      <th>PIP_0hr_GABA<\/th>\n      <th>PIP_1hr_GABA<\/th>\n      <th>PIP_6hr_GABA<\/th>\n      <th>GABA_sum<\/th>\n      <th>GABA_PIP_diff<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":2,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":5,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":6,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":7,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[2,3,4,5,6,7]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render","options.columnDefs.2.render","options.columnDefs.3.render","options.columnDefs.4.render","options.columnDefs.5.render"],"jsHooks":[]}</script>
</div>
<div id="dynamic-egenes-in-nmglut" class="section level4">
<h4>Dynamic eGenes in NMglut</h4>
<p>nmglut has 8 eGenes, 5 of them are dynamic.</p>
<div id="htmlwidget-771ad2536610c1179489" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-771ad2536610c1179489">{"x":{"filter":"none","vertical":false,"data":[["76","67","14","26","68","40","53","71"],["TMED8","SAPCD2","CAMK2N2","EFHB","SASH1","HLA-DMA","NT5DC1","SUMO2"],[0.999950603781023,0.66656631624976,0.736457818043706,0.642515402315624,0.547256034484603,0.76727025973925,0.549033900462452,0.566869524100816],[0,0,0,0,0,0.0970977244423139,0.309777599493364,0.566869524100816],[0.999950603781023,0.66656631624976,0.662926388409223,0.642515402315624,0.547256034484603,0.44790061754008,0.11415932697426,0],[0,0,0,0,0,0,0,0],[0.999950603781023,0.66656631624976,0.662926388409223,0.642515402315624,0.547256034484603,0.544998341982394,0.423936926467624,0.566869524100816],[0.999950603781023,0.66656631624976,0.662926388409223,0.642515402315624,0.547256034484603,0.350802893097766,-0.195618272519104,-0.566869524100816]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>genename<\/th>\n      <th>PIP_sum<\/th>\n      <th>PIP_0hr_nmglut<\/th>\n      <th>PIP_1hr_nmglut<\/th>\n      <th>PIP_6hr_nmglut<\/th>\n      <th>nmglut_sum<\/th>\n      <th>nmglut_PIP_diff<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":2,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":5,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":6,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":7,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[2,3,4,5,6,7]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render","options.columnDefs.2.render","options.columnDefs.3.render","options.columnDefs.4.render","options.columnDefs.5.render"],"jsHooks":[]}</script>
</div>
<div id="dynamic-egenes-in-npglut" class="section level4">
<h4>Dynamic eGenes in NPglut</h4>
<p>npglut has 11 eGenes, 5 of them are dynamic.</p>
<div id="htmlwidget-118ca324790aac282ebf" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-118ca324790aac282ebf">{"x":{"filter":"none","vertical":false,"data":[["45","56","63","22","34","8","33","60","16","42","9"],["MDH1","PACS1","PRB2","DENND2A","GABRB2","ASIC2","FRY","PLCG1","CDAN1","HMGB1","ATF6B"],[0.847304518449159,0.856971490250861,0.735317660609107,0.69923465019585,0.695901667448489,0.805413290250384,0.551806437187215,0.768378514770342,0.801020280845438,0.839008109305283,0.843673627906923],[0,0,0,0,0.0630865859039228,0.309888484179369,0.551806437187215,0.768378514770342,0.801020280845438,0.839008109305283,0.843673627906923],[0,0.821854822585901,0.735317660609107,0.69923465019585,0.448626151371271,0.495524806071015,0,0,0,0,0],[0.847304518449159,0,0,0,0.184188930173295,0,0,0,0,0,0],[0.847304518449159,0.821854822585901,0.735317660609107,0.69923465019585,0.695901667448489,0.805413290250384,0.551806437187215,0.768378514770342,0.801020280845438,0.839008109305283,0.843673627906923],[0.847304518449159,0.821854822585901,0.735317660609107,0.69923465019585,0.569728495640643,0.185636321891646,-0.551806437187215,-0.768378514770342,-0.801020280845438,-0.839008109305283,-0.843673627906923]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>genename<\/th>\n      <th>PIP_sum<\/th>\n      <th>PIP_0hr_npglut<\/th>\n      <th>PIP_1hr_npglut<\/th>\n      <th>PIP_6hr_npglut<\/th>\n      <th>npglut_sum<\/th>\n      <th>npglut_PIP_diff<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":2,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":5,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":6,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"targets":7,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[2,3,4,5,6,7]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render","options.columnDefs.2.render","options.columnDefs.3.render","options.columnDefs.4.render","options.columnDefs.5.render"],"jsHooks":[]}</script>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.1.2 (2021-11-01)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur 10.16

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] grid      stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
[1] ggplot2_3.3.5       enrichR_3.2         VennDiagram_1.7.1  
[4] futile.logger_1.4.3 dplyr_1.0.7         workflowr_1.7.0    

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.11          getPass_0.2-2        ps_1.6.0            
 [4] assertthat_0.2.1     rprojroot_2.0.2      digest_0.6.29       
 [7] utf8_1.2.2           R6_2.5.1             futile.options_1.0.1
[10] WriteXLS_6.4.0       evaluate_0.14        httr_1.4.2          
[13] highr_0.9            pillar_1.7.0         rlang_1.0.0         
[16] curl_4.3.2           rstudioapi_0.13      whisker_0.4         
[19] callr_3.7.0          jquerylib_0.1.4      DT_0.26             
[22] rmarkdown_2.11       labeling_0.4.2       stringr_1.4.0       
[25] htmlwidgets_1.5.4    munsell_0.5.0        compiler_4.1.2      
[28] httpuv_1.6.5         xfun_0.29            pkgconfig_2.0.3     
[31] htmltools_0.5.2      tidyselect_1.1.1     tibble_3.1.6        
[34] fansi_1.0.2          crayon_1.4.2         withr_2.4.3         
[37] later_1.3.0          jsonlite_1.7.3       gtable_0.3.0        
[40] lifecycle_1.0.1      DBI_1.1.2            git2r_0.29.0        
[43] magrittr_2.0.2       formatR_1.11         scales_1.1.1        
[46] cli_3.1.1            stringi_1.7.6        farver_2.1.0        
[49] fs_1.5.2             promises_1.2.0.1     bslib_0.3.1         
[52] ellipsis_0.3.2       generics_0.1.2       vctrs_0.3.8         
[55] cowplot_1.1.1        rjson_0.2.21         lambda.r_1.2.4      
[58] tools_4.1.2          glue_1.6.1           purrr_0.3.4         
[61] crosstalk_1.2.0      processx_3.5.2       fastmap_1.1.0       
[64] yaml_2.2.2           colorspace_2.0-2     knitr_1.37          
[67] sass_0.4.1          </code></pre>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
