<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Lifan Liang" />

<meta name="date" content="2024-05-22" />

<title>Genetic effect sharing by MASH</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">neuron_stimulation_100line_eQTL</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/LifanLiang/neuron_stim_eQTL100">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Genetic effect sharing by MASH</h1>
<h4 class="author">Lifan Liang</h4>
<h4 class="date">2024-05-22</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-08-05
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 2
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>neuron_stim_eQTL100/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomLifanLiangneuronstimeQTL100tree7d70d1a246cfdd0f983b820d2eff381cf7ecdd8etargetblank7d70d1aa">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/tree/7d70d1a246cfdd0f983b820d2eff381cf7ecdd8e" target="_blank">7d70d1a</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomLifanLiangneuronstimeQTL100tree7d70d1a246cfdd0f983b820d2eff381cf7ecdd8etargetblank7d70d1aa"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/tree/7d70d1a246cfdd0f983b820d2eff381cf7ecdd8e" target="_blank">7d70d1a</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    data/.DS_Store
    Ignored:    data/single_group_ctwas/.DS_Store

Untracked files:
    Untracked:  GTEx_vs_SCZ.png
    Untracked:  GTEx_vs_SCZ.png.2024-08-05_13-02-45.log
    Untracked:  GTEx_vs_SCZ.png.2024-08-05_13-21-32.log
    Untracked:  GTEx_vs_SCZ.png.2024-08-05_13-23-14.log
    Untracked:  GTEx_vs_SCZ.png.2024-08-05_13-30-00.log
    Untracked:  data/processed_neuroticism_res4.rds

Unstaged changes:
    Modified:   data/processed_SCZ_res4.rds
    Modified:   data/processed_adhd_res4.rds

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/mashr_analysis.Rmd</code>) and
HTML (<code>docs/mashr_analysis.html</code>) files. If you’ve configured
a remote Git repository (see <code>?wflow_git_remote</code>), click on
the hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/7d70d1a246cfdd0f983b820d2eff381cf7ecdd8e/analysis/mashr_analysis.Rmd" target="_blank">7d70d1a</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-08-05
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/ec426279fdc06572fd82e4697ef37e4c4c472527/docs/mashr_analysis.html" target="_blank">ec42627</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-05-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/05be4447b1a46f5e02dedb8e8510a17bfc7997ab/analysis/mashr_analysis.Rmd" target="_blank">05be444</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-05-22
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/05be4447b1a46f5e02dedb8e8510a17bfc7997ab/docs/mashr_analysis.html" target="_blank">05be444</a>
</td>
<td>
Lifan Liang
</td>
<td>
2024-05-22
</td>
<td>
wflow_publish(c("analysis", "docs"))
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div
id="condition-specific-eqtl-with-shared-individual-random-effects-and-covariates-fixed-effects"
class="section level2">
<h2>Condition-specific eQTL with shared individual random effects and
covariates fixed effects</h2>
<p>We previously attempted to use MASH to analyze effect sharing pattern
from condition by condition eQTLs. It results in highly inflated
posterior estimates due to the correlation of individual effect. To
address this issue, we pooled the 9 conditions together to estimate
individual variation as an individual random effect (IRE). For details
of the analysis, please visit <a href="assets/eQTL_glimix.html">this
page</a>.</p>
<p>With the estimation of IRE controlling for the persistent effect of
lead variant, we run a model with 9 condition-specific genetic
effect:</p>
<p><span class="math display">\[
Y= (G\times C)\beta + X\mu + \epsilon
\]</span></p>
<p>where <span class="math inline">\(G\times C\)</span> is the
interaction term between genotypes and the 9 conditions, while other
terms are the same as the model described in <a
href="eQTL_glimix.html">the mixed effect model</a>. Please note that we
reuse the estimation of <span class="math inline">\(v0\)</span> and
<span class="math inline">\(v1\)</span> from previous results. For
convienient, eQTLs computed from this method is called interaction eQTL
(ieQTL).</p>
</div>
<div id="comparison-with-seperately-estimated-eqtls"
class="section level2">
<h2>Comparison with seperately estimated eQTLs</h2>
<p>We compared the Z scores of ieQTLs with those of eQTLs computed
condition by condition. They are highly concordant. But ieQTLs have
higher z scores in all conditions. Note that we only compared the 12272
gene-SNP pairs selected as the strong subset in the next section.</p>
<p><img src="assets/eQTL_zscore_shared_vs_seperate.png" /></p>
</div>
<div id="mash-analysis-with-lead-eqtls-in-persistent-effect-model"
class="section level2">
<h2>MASH analysis with lead eQTLs in persistent effect model</h2>
<p>We basically followed the <a
href="https://stephenslab.github.io/mashr/articles/eQTL_outline.html">tutorial</a>
of MASH for eQTL.</p>
<ol style="list-style-type: decimal">
<li>Randomly sampled 4 SNPs per gene as the null to learn correlation
structure.</li>
<li>Selected lead variants for each gene with FDR&gt;0.2 in the
persistent effect across all conditions. There are 12272 lead eQTLs in
total within this strong subset.</li>
<li>Learned data-driven covariance matrices using the strong
subset.</li>
<li>Fit all sharing patterns to the random subsets to learn mixture
weights and scaling coefficients.</li>
<li>Compute posterior for the strong subset with the fitted model.</li>
</ol>
<p>Distribution of the posterior weights are shown below:</p>
<p><img src="assets/dist_sharing_strong.png" /></p>
<p>PCA_1 and tPCA are very similar to equal effects. Below is the
heatmap for PCA1 covariance matrix.</p>
<p><img src="assets/PCA1_strong.png" /></p>
<p>The first eigen vector (&gt;99% PVE) of this covariance matrix also
indicates equal sharing.</p>
<p><img src="assets/PCA1_eigen_strong.png" /></p>
<p>Therefore, 90% of genetic effects are shared across all conditions.
At first, I suspected the individual effect is still strong. If that’s
the case, the random subset should exhibit similar effect sharing.
However, the posterior distribution of the random subset only has around
45% effects shared across conditions.</p>
<p><img src="assets/dist_sharing_random.png" /></p>
</div>
<div id="persistent-effect-is-biased-towards-universal-sharing"
class="section level2">
<h2>Persistent effect is biased towards universal sharing</h2>
<p>Persistent effects are strong when the genetic effect is consistent
across contexts. Hence lead variants in persistent effects tend to be
shared across conditions. To verify this hypothesis, I divided the lead
eQTLs into two subsets: one with &gt;80% posterior weights in universal
sharing (including PCA1, tPCA, and equal_effects) and the rest. As shown
in the box plot below, eQTLs with high universal sharing have much
smaller FDR than those with lower sharing.</p>
<p><img src="assets/main_effect_bias.png" /></p>
</div>
<div id="extract-snp-gene-pairs-from-condition-by-condition-eqtls"
class="section level2">
<h2>Extract SNP-gene pairs from condition by condition eQTLs</h2>
<p>To avoid the bias of persistent effects, we selected the union of
lead eQTLs per gene per context as the cadidate for the strong subset.
FDR threshold is 0.2. However, lead eQTLs for the same gene may be in
high LD and over-represent certain sharing patterns.</p>
<p>Therefore, for each gene, we computed <span
class="math inline">\(R^2\)</span> among the lead variants based on
genotypes of the 100 cell lines. SNP pairs with <span
class="math inline">\(R^2&gt;0.1\)</span> were pruned. After pruning, we
obtained 57398 SNP-gene pairs.</p>
<p>These SNP-gene pairs were utilized as the strong subset for MASH
analysis and we rerun the previous procedure. But ED was replaced by
ultimate deconvolution (UD). And we included FLASH results to the
starting point.</p>
<p>We removed gene-SNP pairs with posterior of being NULL (point mass at
zero) above 80%. As shown in the figure below, most eQTLs are shared
across all conditions (equal effects and FLASH_nonneg_2). Around 17% of
eQTLs were outside the all sharing pattern.</p>
<p><img src="assets/MASH/posterior_weight.png" /></p>
<p>We also computed pairwise sharing among tissues with the posterior
effect sizes. Correlation was mostly above 0.95.</p>
<p><img src="assets/MASH/all_sharing.png" /></p>
<p>One of the patterns, PCA_5, shows shared dynamic effects although it
only has 1% posterior weight.</p>
<p><img src="assets/MASH/PCA5_cov.png" /></p>
<p>Its first Eigen vector is:</p>
<p><img src="assets/MASH/PCA5_eigen1.png" /></p>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
