<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Lifan Liang" />

<meta name="date" content="2025-10-25" />

<title>Power analysis for ASoC</title>

<script src="site_libs/header-attrs-2.30/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/htmltools-fill-0.5.9/fill.css" rel="stylesheet" />
<script src="site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.34.0/datatables.js"></script>
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.2.2/css/crosstalk.min.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.2.2/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-6.5.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">neuron_stimulation_100line_eQTL</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/LifanLiang/neuron_stim_eQTL100">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Power analysis for ASoC</h1>
<h4 class="author">Lifan Liang</h4>
<h4 class="date">2025-10-25</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-12-09
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>neuron_stim_eQTL100/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.2). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231109code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231109)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231109code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231109)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomLifanLiangneuronstimeQTL100tree10addbfe5e25eca0f031c74767c6ad4a00ee95ddtargetblank10addbfa">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/tree/10addbfe5e25eca0f031c74767c6ad4a00ee95dd" target="_blank">10addbf</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomLifanLiangneuronstimeQTL100tree10addbfe5e25eca0f031c74767c6ad4a00ee95ddtargetblank10addbfa"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/tree/10addbfe5e25eca0f031c74767c6ad4a00ee95dd" target="_blank">10addbf</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    data/.DS_Store
    Ignored:    data/single_group_ctwas/.DS_Store

Untracked files:
    Untracked:  Fig15SE_dynamic_count_eqtl_ctwas.pdf
    Untracked:  GTEx_vs_SCZ.pdf.2024-08-26_21-04-15.log
    Untracked:  GTEx_vs_SCZ.png
    Untracked:  GTEx_vs_SCZ.tiff
    Untracked:  GTEx_vs_SCZ.tiff.2024-08-26_21-04-43.log
    Untracked:  Rplot.png
    Untracked:  geneMapping.rds
    Untracked:  mouse_sesenory_experince_inh.csv
    Untracked:  mouse_sesenory_experince_inh.csv.numbers
    Untracked:  plot4pdf.R

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/power_analysis.Rmd</code>) and
HTML (<code>docs/power_analysis.html</code>) files. If you’ve configured
a remote Git repository (see <code>?wflow_git_remote</code>), click on
the hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/10addbfe5e25eca0f031c74767c6ad4a00ee95dd/analysis/power_analysis.Rmd" target="_blank">10addbf</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-12-09
</td>
<td>
wflow_publish(c("analysis", "data"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/5bc7f0f38eeaf9295f5215a7822f29fb76b2ff6a/docs/power_analysis.html" target="_blank">5bc7f0f</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-10-29
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/75ad8dd06b99635c0de322e57de0fdaef0674150/analysis/power_analysis.Rmd" target="_blank">75ad8dd</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-10-29
</td>
<td>
wflow_publish("analysis")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/c7a8b261209df2f915844793e56a261244ce3a92/docs/power_analysis.html" target="_blank">c7a8b26</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-10-28
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/b53736d686d74e2e370e1f1dbbdcc41d27397b97/analysis/power_analysis.Rmd" target="_blank">b53736d</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-10-28
</td>
<td>
wflow_publish("analysis")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/81156aaf3a1be7bbea4fa1d12197f0bba5ec6fcc/docs/power_analysis.html" target="_blank">81156aa</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-10-28
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/dbd32fcd72ad10e4b8d2728302aa55033ab54887/analysis/power_analysis.Rmd" target="_blank">dbd32fc</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-10-28
</td>
<td>
wflow_publish("analysis")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/0b76d483533029f6c7f2467344dc2aca74a1aca8/docs/power_analysis.html" target="_blank">0b76d48</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-10-27
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/c6d665509231edc81b34f6407d2796df5a46211e/analysis/power_analysis.Rmd" target="_blank">c6d6655</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-10-27
</td>
<td>
wflow_publish("analysis")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/9dd2b709ee39922e7af9ecca3fc7d17ec142c0eb/docs/power_analysis.html" target="_blank">9dd2b70</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-10-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/6d924eac2d736054d5a740118e5ece748d684508/analysis/power_analysis.Rmd" target="_blank">6d924ea</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-10-25
</td>
<td>
wflow_publish("analysis")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/72829ddb321b0a9a5c785a250a59ac41e382aa7f/docs/power_analysis.html" target="_blank">72829dd</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-10-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/9591c0d8fcaebf8855cbf2aaf7318602282dfe65/analysis/power_analysis.Rmd" target="_blank">9591c0d</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-10-25
</td>
<td>
wflow_publish("analysis")
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="introduction" class="section level2">
<h2>Introduction</h2>
<p>Here, we investigate the power of allele-specific analysis with
two-sided binomial tests on read counts between two alleles by pooling
all the reads from the heterozygote individuals.</p>
<div id="median-of-reads-per-cell-line" class="section level3">
<h3>Median of reads per cell line</h3>
<p>From the previous analysis results on allele-specific open chromatin
(ASoC), each heterozygote sample added 10 reads to the total coverage on
median. This is the same across all the context for ASoC results.</p>
<pre><code>0hr_GABA 0hr_nmgl 0hr_npgl 1hr_GABA 1hr_nmgl 1hr_npgl 6hr_GABA 6hr_nmgl 
10.66667 10.16667 10.16667 10.66667 10.30000 10.42105 10.42857 10.00000 
6hr_npgl 
10.00000 </code></pre>
</div>
<div id="distribution-of-reference-ratio." class="section level3">
<h3>Distribution of reference ratio.</h3>
<p>The distribution of reference ratio at FDR&lt;5% is bimodal, with the
mode around 0.35 and 0.6.</p>
<p><img src="figure/power_analysis.Rmd/unnamed-chunk-3-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-3-1">
Past versions of unnamed-chunk-3-1.png
</button>
</p>
<div id="fig-unnamed-chunk-3-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/72829ddb321b0a9a5c785a250a59ac41e382aa7f/docs/figure/power_analysis.Rmd/unnamed-chunk-3-1.png" target="_blank">72829dd</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-10-25
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>We “folded” the reference ratio so only <span
class="math inline">\([0.5,1.0]\)</span> is shown. And then we computed
the 10% and 90% quantile of the distribution. The ratio varies from 0.55
to 0.75. This range of ratio will be used in simulation experiments.</p>
<pre><code>      10%       90% 
0.5567349 0.7642049 </code></pre>
<div class="datatables html-widget html-fill-item" id="htmlwidget-7657cc3e42637b46d6e4" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-7657cc3e42637b46d6e4">{"x":{"filter":"none","vertical":false,"data":[["0hr_GABA","0hr_nmgl","0hr_npgl","1hr_GABA","1hr_nmgl","1hr_npgl","6hr_GABA","6hr_nmgl","6hr_npgl"],[0.5567348608837969,0.5622204169382314,0.5695938843764934,0.5582924380734476,0.5650477056406594,0.5687483232729711,0.5597062545068637,0.568181818181818,0.569750090388176],[0.7642048517520212,0.771193409610984,0.7692307692307691,0.757575757575758,0.7675079907417612,0.7675685693106007,0.76,0.764705882352941,0.764705882352941]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>10%<\/th>\n      <th>90%<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"10%","targets":1},{"name":"90%","targets":2}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
<div id="the-nominal-p-value-cutoff-for-significant-asoc"
class="section level3">
<h3>The nominal P value cutoff for significant ASoC</h3>
<p>The table below shows the largest P values for significant ASoC with
FDR&lt;5% as cutoff. We will use P=0.001 as the significance threshold
for the simulation.</p>
<pre><code>   0hr_GABA    0hr_nmgl    0hr_npgl    1hr_GABA    1hr_nmgl    1hr_npgl 
0.003808623 0.002602171 0.003291092 0.005047105 0.003266516 0.004740406 
   6hr_GABA    6hr_nmgl    6hr_npgl 
0.005110180 0.003326893 0.004572156 </code></pre>
</div>
</div>
<div id="simulation" class="section level2">
<h2>Simulation</h2>
<p>Read count on two alleles was generated with a Binomial distribution.
The synthetic experiment only has two parameters. One is the number of
trails. This represents the total coverage that increases with sample
size (<span class="math inline">\(N_{read}=N_{sample} \times
10\)</span>). The other is the success probability, which represents the
allele-specific effect size.</p>
<div id="comparing-two-different-effective-sample-sizes"
class="section level3">
<h3>Comparing two different effective sample sizes</h3>
<p>Given that there are two cohorts, one has 38 heterozygote individuals
and the other 143. The success probability takes a value in the range
of<span class="math inline">\([0.55,0.75]\)</span> with the step of
0.025. For each combination of sample size and effect size, we sampled
200 “loci” from the The distribution and run the Binomial test. The
statistical power of the binomial test is calculated as the proportion
of loci with P value &lt; 0.001 in each setting.</p>
<pre class="r"><code># Generate the data and 
ratio &lt;- 22:30 * 0.025
N_sample &lt;- c(38,143)
N_trial &lt;- 100
power &lt;- matrix(nrow=length(N_sample), ncol=length(ratio))
colnames(power) &lt;- ratio
rownames(power) &lt;- N_sample
for(i in ratio) {
  for(j in N_sample){
    temp &lt;- rep(0,N_trial)
    x &lt;- rbinom(N_trial,j*10,i)
    for(k in 1:N_trial) {
      temp[k] &lt;- binom.test(x[k],j*10,0.5,&quot;two.sided&quot;)$p.value&lt;0.001
    }
    power[as.character(j),as.character(i)] &lt;- mean(temp)
  }
}

# Plot legend in first row
layout(matrix(c(1, 2), nrow = 2), heights = c(1.2, 5))
par(mar = c(0, 0, 0, 0))  # No margins
plot.new()
legend(&quot;center&quot;, legend = N_sample, col = 1:2, lty = 1, lwd=3, horiz = T,
       bty=&quot;n&quot;, title = &quot;Sample size&quot;)
par(mar = c(5, 4, 0, 2))
matplot(t(power), xaxt=&#39;n&#39;, xlab=&quot;allelic bias&quot;, type = &quot;l&quot;, lty = 1, col = 1:2,
        lwd=3, ylab=&quot;statistical power&quot;)
axis(side=1, at=1:ncol(power), labels=colnames(power))</code></pre>
<p><img src="figure/power_analysis.Rmd/unnamed-chunk-6-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-6-1">
Past versions of unnamed-chunk-6-1.png
</button>
</p>
<div id="fig-unnamed-chunk-6-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/81156aaf3a1be7bbea4fa1d12197f0bba5ec6fcc/docs/figure/power_analysis.Rmd/unnamed-chunk-6-1.png" target="_blank">81156aa</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-10-28
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/0b76d483533029f6c7f2467344dc2aca74a1aca8/docs/figure/power_analysis.Rmd/unnamed-chunk-6-1.png" target="_blank">0b76d48</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-10-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div
id="comparing-different-total-sample-sizes-with-different-minor-allele-frequence-maf"
class="section level3">
<h3>Comparing different total sample sizes with different minor allele
frequence (MAF)</h3>
<p>Given the total sample size of 38 and 143 and the MAF, we sampled the
number of heterozygote individuals for each loci and then sampled the
read counts given various allelic bias.</p>
<div id="maf10" class="section level4">
<h4>MAF=10%</h4>
<p><img src="figure/power_analysis.Rmd/unnamed-chunk-7-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-7-1">
Past versions of unnamed-chunk-7-1.png
</button>
</p>
<div id="fig-unnamed-chunk-7-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/5bc7f0f38eeaf9295f5215a7822f29fb76b2ff6a/docs/figure/power_analysis.Rmd/unnamed-chunk-7-1.png" target="_blank">5bc7f0f</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-10-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/81156aaf3a1be7bbea4fa1d12197f0bba5ec6fcc/docs/figure/power_analysis.Rmd/unnamed-chunk-7-1.png" target="_blank">81156aa</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-10-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="maf20" class="section level4">
<h4>MAF=20%</h4>
<p><img src="figure/power_analysis.Rmd/unnamed-chunk-8-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-8-1">
Past versions of unnamed-chunk-8-1.png
</button>
</p>
<div id="fig-unnamed-chunk-8-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/5bc7f0f38eeaf9295f5215a7822f29fb76b2ff6a/docs/figure/power_analysis.Rmd/unnamed-chunk-8-1.png" target="_blank">5bc7f0f</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-10-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/81156aaf3a1be7bbea4fa1d12197f0bba5ec6fcc/docs/figure/power_analysis.Rmd/unnamed-chunk-8-1.png" target="_blank">81156aa</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-10-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="maf30" class="section level4">
<h4>MAF=30%</h4>
<p><img src="figure/power_analysis.Rmd/unnamed-chunk-9-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-9-1">
Past versions of unnamed-chunk-9-1.png
</button>
</p>
<div id="fig-unnamed-chunk-9-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/5bc7f0f38eeaf9295f5215a7822f29fb76b2ff6a/docs/figure/power_analysis.Rmd/unnamed-chunk-9-1.png" target="_blank">5bc7f0f</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-10-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/81156aaf3a1be7bbea4fa1d12197f0bba5ec6fcc/docs/figure/power_analysis.Rmd/unnamed-chunk-9-1.png" target="_blank">81156aa</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-10-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="maf40" class="section level4">
<h4>MAF=40%</h4>
<p><img src="figure/power_analysis.Rmd/unnamed-chunk-10-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-10-1">
Past versions of unnamed-chunk-10-1.png
</button>
</p>
<div id="fig-unnamed-chunk-10-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/5bc7f0f38eeaf9295f5215a7822f29fb76b2ff6a/docs/figure/power_analysis.Rmd/unnamed-chunk-10-1.png" target="_blank">5bc7f0f</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-10-29
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="comparison-across-maf-by-averaging-power-among-allele-bias"
class="section level4">
<h4>Comparison across MAF by averaging power among allele bias</h4>
<p><img src="figure/power_analysis.Rmd/unnamed-chunk-11-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-11-1">
Past versions of unnamed-chunk-11-1.png
</button>
</p>
<div id="fig-unnamed-chunk-11-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/5bc7f0f38eeaf9295f5215a7822f29fb76b2ff6a/docs/figure/power_analysis.Rmd/unnamed-chunk-11-1.png" target="_blank">5bc7f0f</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-10-29
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>library(GenomicRanges)
# Compute the number of significant ASoC in each context
sapply(asocs, function(x){sum(x$FDR&lt;0.05)})
# Convert significant SNPs into GenomicRnages
asoc.range &lt;- lapply(asocs,function(x){temp&lt;-makeGRangesFromDataFrame(x[x$FDR&lt;0.05,], seqnames.field = &quot;CHROM&quot;, start.field = &quot;POS&quot;, end.field = &quot;POS&quot;);start(temp)&lt;-start(temp)-500;end(temp)&lt;-end(temp)+500;temp})

caqtl &lt;- readRDS(&quot;/Users/lifanliang/Documents/neuron_stim/caQTLs_25kb_fdr0.05.rds&quot;)
caqtl1 &lt;- lapply(caqtl, function(x){unique(x$phenotype_id)})
caqtl2 &lt;- lapply(caqtl1, function(x){temp&lt;-do.call(rbind,strsplit(x,&quot;-&quot;)); colnames(temp)&lt;-c(&quot;chr&quot;,&quot;start&quot;,&quot;end&quot;);temp})
caqtl.range &lt;- lapply(caqtl2, function(x){makeGRangesFromDataFrame(as.data.frame(x))})

length(reduce(c(caqtl.range[[3]],asoc.range[[7]])))/length(reduce(asoc.range[[7]]))
length(reduce(c(caqtl.range[[6]],asoc.range[[8]])))/length(reduce(asoc.range[[8]]))
length(reduce(c(caqtl.range[[9]],asoc.range[[9]])))/length(reduce(asoc.range[[9]]))


sum(!overlapsAny(caqtl.range[[3]], asoc.range[[7]])) / length(reduce(asoc.range[[7]]))
sum(!overlapsAny(caqtl.range[[6]], asoc.range[[7]])) / length(reduce(asoc.range[[8]]))
sum(!overlapsAny(caqtl.range[[9]], asoc.range[[7]])) / length(reduce(asoc.range[[9]]))</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.1.2 (2021-11-01)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur 10.16

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] tidyr_1.3.0       scales_1.4.0      ggplot2_4.0.1     data.table_1.17.8
[5] workflowr_1.7.2  

loaded via a namespace (and not attached):
 [1] tidyselect_1.2.1   xfun_0.54          bslib_0.3.1        purrr_1.2.0       
 [5] vctrs_0.6.5        generics_0.1.4     htmltools_0.5.9    yaml_2.3.11       
 [9] rlang_1.1.6        jquerylib_0.1.4    later_1.3.1        pillar_1.11.1     
[13] glue_1.8.0         withr_3.0.2        RColorBrewer_1.1-3 S7_0.2.1          
[17] lifecycle_1.0.4    stringr_1.6.0      gtable_0.3.6       ragg_1.2.5        
[21] htmlwidgets_1.6.4  evaluate_1.0.5     labeling_0.4.3     knitr_1.50        
[25] callr_3.7.6        fastmap_1.1.1      httpuv_1.6.10      ps_1.9.1          
[29] crosstalk_1.2.2    Rcpp_1.0.10        promises_1.5.0     DT_0.34.0         
[33] jsonlite_2.0.0     systemfonts_1.0.4  farver_2.1.1       otel_0.2.0        
[37] fs_1.6.2           textshaping_0.3.6  digest_0.6.31      stringi_1.7.12    
[41] processx_3.8.6     dplyr_1.1.2        getPass_0.2-4      rprojroot_2.1.1   
[45] grid_4.1.2         cli_3.6.5          tools_4.1.2        magrittr_2.0.4    
[49] sass_0.4.6         tibble_3.3.0       dichromat_2.0-0.1  whisker_0.4.1     
[53] pkgconfig_2.0.3    rmarkdown_2.30     httr_1.4.7         rstudioapi_0.17.1 
[57] R6_2.6.1           git2r_0.32.0       compiler_4.1.2    </code></pre>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
