<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Lifan Liang" />

<meta name="date" content="2025-06-19" />

<title>revision</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet" />
<script src="site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.33/datatables.js"></script>
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-6.5.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">neuron_stimulation_100line_eQTL</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/LifanLiang/neuron_stim_eQTL100">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">revision</h1>
<h4 class="author">Lifan Liang</h4>
<h4 class="date">2025-06-19</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-10-25
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>neuron_stim_eQTL100/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231109code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231109)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231109code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231109)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomLifanLiangneuronstimeQTL100tree9591c0d8fcaebf8855cbf2aaf7318602282dfe65targetblank9591c0da">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/tree/9591c0d8fcaebf8855cbf2aaf7318602282dfe65" target="_blank">9591c0d</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomLifanLiangneuronstimeQTL100tree9591c0d8fcaebf8855cbf2aaf7318602282dfe65targetblank9591c0da"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/tree/9591c0d8fcaebf8855cbf2aaf7318602282dfe65" target="_blank">9591c0d</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    data/.DS_Store
    Ignored:    data/single_group_ctwas/.DS_Store

Untracked files:
    Untracked:  Fig15SE_dynamic_count_eqtl_ctwas.pdf
    Untracked:  GTEx_vs_SCZ.pdf.2024-08-26_21-04-15.log
    Untracked:  GTEx_vs_SCZ.png
    Untracked:  GTEx_vs_SCZ.tiff
    Untracked:  GTEx_vs_SCZ.tiff.2024-08-26_21-04-43.log
    Untracked:  Rplot.png
    Untracked:  VennDiagram.2025-06-27_16-07-46.log
    Untracked:  VennDiagram.2025-06-28_19-58-17.log
    Untracked:  VennDiagram.2025-07-14_11-29-03.log
    Untracked:  VennDiagram.2025-07-14_11-29-11.log
    Untracked:  VennDiagram.2025-07-14_11-33-41.log
    Untracked:  VennDiagram.2025-07-14_11-34-04.log
    Untracked:  VennDiagram.2025-07-14_11-34-15.log
    Untracked:  VennDiagram.2025-07-14_11-34-30.log
    Untracked:  VennDiagram.2025-07-14_11-34-36.log
    Untracked:  VennDiagram.2025-07-14_11-34-47.log
    Untracked:  VennDiagram.2025-07-14_11-35-18.log
    Untracked:  VennDiagram.2025-07-14_11-35-51.log
    Untracked:  VennDiagram.2025-07-14_11-36-00.log
    Untracked:  VennDiagram.2025-07-14_11-36-09.log
    Untracked:  VennDiagram.2025-07-14_11-36-19.log
    Untracked:  VennDiagram.2025-07-14_11-36-55.log
    Untracked:  VennDiagram.2025-07-14_12-38-13.log
    Untracked:  VennDiagram.2025-07-14_14-55-21.log
    Untracked:  VennDiagram.2025-10-25_21-43-03.log
    Untracked:  data/gProfiler_mmusculus_hsapiens_ext.csv
    Untracked:  data/gProfiler_mmusculus_hsapiens_inh.csv
    Untracked:  data/gene_fdr_results.RDS
    Untracked:  data/jubao_mice_eqtl_ext.csv
    Untracked:  data/jubao_mice_eqtl_inh.csv
    Untracked:  data/mouse_sesenory_experince_ext.csv
    Untracked:  data/mouse_sesenory_experince_inh.csv
    Untracked:  geneMapping.rds
    Untracked:  mouse_sesenory_experince_inh.csv
    Untracked:  mouse_sesenory_experince_inh.csv.numbers
    Untracked:  plot4pdf.R

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/revision.Rmd</code>) and HTML
(<code>docs/revision.html</code>) files. If you’ve configured a remote
Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/9591c0d8fcaebf8855cbf2aaf7318602282dfe65/analysis/revision.Rmd" target="_blank">9591c0d</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-10-25
</td>
<td>
wflow_publish("analysis")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/82738f57649ef8f75bbfcca3bc7389c43641740e/docs/revision.html" target="_blank">82738f5</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-07-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/8cc320f1858d165d03eb3f43becb758f7a2f747f/analysis/revision.Rmd" target="_blank">8cc320f</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-07-14
</td>
<td>
wflow_publish(c("analysis", "data"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/3a1b03675882e70df32f3f1084d872b95b236fe1/docs/revision.html" target="_blank">3a1b036</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-07-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/c14d3c06b155436e4d020741d035936a5142eb3c/analysis/revision.Rmd" target="_blank">c14d3c0</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-07-14
</td>
<td>
wflow_publish(c("analysis", "data"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/01204609ff977dc80978f77ef2a4574df045dfc8/docs/revision.html" target="_blank">0120460</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-07-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/d636d7868d1fa713f77107fb7a5bf0b9d86e4200/analysis/revision.Rmd" target="_blank">d636d78</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-07-14
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/d636d7868d1fa713f77107fb7a5bf0b9d86e4200/docs/revision.html" target="_blank">d636d78</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-07-14
</td>
<td>
wflow_publish(c("analysis", "data", "docs"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LifanLiang/neuron_stim_eQTL100/bcec9455b663a80918225173ddf1b04aaca549c2/docs/revision.html" target="_blank">bcec945</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/81ffbde0d68da376bd68f72f7906077018675eb2/analysis/revision.Rmd" target="_blank">81ffbde</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
<td>
wflow_publish(c("analysis", "data"))
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="rb-estimate" class="section level2">
<h2>rb estimate</h2>
<p>rb is essentially estimating pairwise correlation of genetic effect
between two conditions after correcting residual correlation. We
implemented the procedure following the description by Qi, et al (PMID:
29891976). Number in the bracket is standard error.</p>
<div id="eqtl-correlation-within-our-gene-expression"
class="section level3">
<h3>eQTL correlation within our gene expression</h3>
<p><img src="figure/revision.Rmd/unnamed-chunk-1-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-1">
Past versions of unnamed-chunk-1-1.png
</button>
</p>
<div id="fig-unnamed-chunk-1-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bcec9455b663a80918225173ddf1b04aaca549c2/docs/figure/revision.Rmd/unnamed-chunk-1-1.png" target="_blank">bcec945</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="eqtl-correlation-with-gtex-brain-tissues"
class="section level3">
<h3>eQTL correlation with GTEx brain tissues</h3>
<p><img src="figure/revision.Rmd/unnamed-chunk-2-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-2-1">
Past versions of unnamed-chunk-2-1.png
</button>
</p>
<div id="fig-unnamed-chunk-2-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bcec9455b663a80918225173ddf1b04aaca549c2/docs/figure/revision.Rmd/unnamed-chunk-2-1.png" target="_blank">bcec945</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="eqtl-correlation-with-psychencode2-prenatal-brain"
class="section level3">
<h3>eQTL correlation with psychENCODE2 prenatal brain</h3>
<p><img src="figure/revision.Rmd/unnamed-chunk-3-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-3-1">
Past versions of unnamed-chunk-3-1.png
</button>
</p>
<div id="fig-unnamed-chunk-3-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bcec9455b663a80918225173ddf1b04aaca549c2/docs/figure/revision.Rmd/unnamed-chunk-3-1.png" target="_blank">bcec945</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="eqtl-correlation-with-doparminergic-neurons"
class="section level3">
<h3>eQTL correlation with doparminergic neurons</h3>
<p><img src="figure/revision.Rmd/unnamed-chunk-4-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-4-1">
Past versions of unnamed-chunk-4-1.png
</button>
</p>
<div id="fig-unnamed-chunk-4-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bcec9455b663a80918225173ddf1b04aaca549c2/docs/figure/revision.Rmd/unnamed-chunk-4-1.png" target="_blank">bcec945</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="mash-results" class="section level2">
<h2>MASH results</h2>
<p>We extracted top QTLs for each molecular trait (i.e. gene or peak)
with FDR &lt; 20% for each condition and took a union. And then we did
LD clumping with the union of eQTL within a molecular trait. QTLs are
ranked by nominal P values in 6hr NPGlut for both caQTL and eQTL. <span
class="math inline">\(R^2\)</span> cutoff is 0.1. After LD clumping, we
have 75149 eQTLs for 14235 genes and</p>
<div id="eqtl-results" class="section level3">
<h3>eQTL results</h3>
<pre><code>Loading required package: ashr</code></pre>
<div id="pairwise-sharing" class="section level4">
<h4>Pairwise sharing</h4>
<p>We used the default method in MASH. For each pair of tissues, first
identify the effects that are significant (by lfsr&lt;lfsr_thresh) in at
least one of the two tissues. Then compute what fraction of these have
an estimated (posterior mean) effect size within a factor
<code>factor</code> of one another. Given the sharing matrix, we perform
hierarchical clustering with Ward criteria.</p>
<pre class="r"><code>sharing.e &lt;- get_pairwise_sharing(res.e)
values &lt;- as.data.frame(sharing.e)
values[] &lt;- lapply(values, sprintf, fmt=&quot;%.3f&quot;)
pheatmap(sharing.e, display_numbers = values,
         color=colorRampPalette(c(&quot;white&quot;, &quot;red&quot;))(100),
         clustering_method = &quot;ward.D2&quot;)</code></pre>
<p><img src="figure/revision.Rmd/unnamed-chunk-6-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-6-1">
Past versions of unnamed-chunk-6-1.png
</button>
</p>
<div id="fig-unnamed-chunk-6-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bcec9455b663a80918225173ddf1b04aaca549c2/docs/figure/revision.Rmd/unnamed-chunk-6-1.png" target="_blank">bcec945</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Most eQTLs are condition specific. eQTLs shared by two conditions and
all conditions are the second highest.</p>
<p><img src="figure/revision.Rmd/unnamed-chunk-7-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-7-1">
Past versions of unnamed-chunk-7-1.png
</button>
</p>
<div id="fig-unnamed-chunk-7-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bcec9455b663a80918225173ddf1b04aaca549c2/docs/figure/revision.Rmd/unnamed-chunk-7-1.png" target="_blank">bcec945</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>The upset plot showed similar information. eQTLs are distributed into
these catetegories in order: (1) Shared by all; (2) condition specific;
(3) GABA-specific eQTL; (4) NPglut stimulation specific; (5) GABA
stimulation specific; (6) Glut specific.</p>
<pre><code>Warning in plot_theme(plot): The `legend.text.align` theme element is not
defined in the element hierarchy.</code></pre>
<pre><code>Warning in plot_theme(plot): The `legend.title.align` theme element is not
defined in the element hierarchy.</code></pre>
<pre><code>Warning in plot_theme(plot): The `legend.text.align` theme element is not
defined in the element hierarchy.</code></pre>
<pre><code>Warning in plot_theme(plot): The `legend.title.align` theme element is not
defined in the element hierarchy.</code></pre>
<pre><code>Warning in plot_theme(plot): The `legend.text.align` theme element is not
defined in the element hierarchy.</code></pre>
<pre><code>Warning in plot_theme(plot): The `legend.title.align` theme element is not
defined in the element hierarchy.</code></pre>
<p><img src="figure/revision.Rmd/unnamed-chunk-8-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-8-1">
Past versions of unnamed-chunk-8-1.png
</button>
</p>
<div id="fig-unnamed-chunk-8-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bcec9455b663a80918225173ddf1b04aaca549c2/docs/figure/revision.Rmd/unnamed-chunk-8-1.png" target="_blank">bcec945</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>We also obtained the mixture proportions of all covariance patterns.
Below is a barplot showing all the patterns with proportion &gt; 1%.</p>
<p><img src="figure/revision.Rmd/unnamed-chunk-9-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-9-1">
Past versions of unnamed-chunk-9-1.png
</button>
</p>
<div id="fig-unnamed-chunk-9-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bcec9455b663a80918225173ddf1b04aaca549c2/docs/figure/revision.Rmd/unnamed-chunk-9-1.png" target="_blank">bcec945</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>The pattern that explains 22.5% of mixture was <code>PCA_1</code>.
The covariance matrix was shown as below:</p>
<p><img src="figure/revision.Rmd/unnamed-chunk-10-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-10-1">
Past versions of unnamed-chunk-10-1.png
</button>
</p>
<div id="fig-unnamed-chunk-10-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bcec9455b663a80918225173ddf1b04aaca549c2/docs/figure/revision.Rmd/unnamed-chunk-10-1.png" target="_blank">bcec945</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>And the corresponding 1st eigen vector (52.7% PVE) is:</p>
<pre class="r"><code>eigen1 &lt;- eigen(res.e$fitted_g$Ulist$PCA_1)$vectors[,1]
names(eigen1) &lt;- colnames(dat)
barplot(abs(eigen1)[c(1,4,7,2,5,8,3,6,9)], las=2)</code></pre>
<p><img src="figure/revision.Rmd/unnamed-chunk-11-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-11-1">
Past versions of unnamed-chunk-11-1.png
</button>
</p>
<div id="fig-unnamed-chunk-11-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bcec9455b663a80918225173ddf1b04aaca549c2/docs/figure/revision.Rmd/unnamed-chunk-11-1.png" target="_blank">bcec945</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="caqtl-results" class="section level3">
<h3>caQTL results</h3>
<p>Pairwise sharing among conditions:</p>
<p><img src="figure/revision.Rmd/unnamed-chunk-12-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-12-1">
Past versions of unnamed-chunk-12-1.png
</button>
</p>
<div id="fig-unnamed-chunk-12-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bcec9455b663a80918225173ddf1b04aaca549c2/docs/figure/revision.Rmd/unnamed-chunk-12-1.png" target="_blank">bcec945</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Distribution of caQTL across the number of conditions they are
significant in. The distribution is similar to the one in eQTL.</p>
<p><img src="figure/revision.Rmd/unnamed-chunk-13-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-13-1">
Past versions of unnamed-chunk-13-1.png
</button>
</p>
<div id="fig-unnamed-chunk-13-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bcec9455b663a80918225173ddf1b04aaca549c2/docs/figure/revision.Rmd/unnamed-chunk-13-1.png" target="_blank">bcec945</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Distribution of caQTLs across different intersections, consisting of
condition-specific caQTLs, shared by all, and cell type-specifc
caQTLs.</p>
<pre><code>Warning in plot_theme(plot): The `legend.text.align` theme element is not
defined in the element hierarchy.</code></pre>
<pre><code>Warning in plot_theme(plot): The `legend.title.align` theme element is not
defined in the element hierarchy.</code></pre>
<pre><code>Warning in plot_theme(plot): The `legend.text.align` theme element is not
defined in the element hierarchy.</code></pre>
<pre><code>Warning in plot_theme(plot): The `legend.title.align` theme element is not
defined in the element hierarchy.</code></pre>
<pre><code>Warning in plot_theme(plot): The `legend.text.align` theme element is not
defined in the element hierarchy.</code></pre>
<pre><code>Warning in plot_theme(plot): The `legend.title.align` theme element is not
defined in the element hierarchy.</code></pre>
<p><img src="figure/revision.Rmd/unnamed-chunk-14-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-14-1">
Past versions of unnamed-chunk-14-1.png
</button>
</p>
<div id="fig-unnamed-chunk-14-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bcec9455b663a80918225173ddf1b04aaca549c2/docs/figure/revision.Rmd/unnamed-chunk-14-1.png" target="_blank">bcec945</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/revision.Rmd/unnamed-chunk-15-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-15-1">
Past versions of unnamed-chunk-15-1.png
</button>
</p>
<div id="fig-unnamed-chunk-15-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bcec9455b663a80918225173ddf1b04aaca549c2/docs/figure/revision.Rmd/unnamed-chunk-15-1.png" target="_blank">bcec945</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><code>tPCA</code> covariance matrix has the highest mixture
proportion (16.8%).</p>
<p><img src="figure/revision.Rmd/unnamed-chunk-16-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-16-1">
Past versions of unnamed-chunk-16-1.png
</button>
</p>
<div id="fig-unnamed-chunk-16-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bcec9455b663a80918225173ddf1b04aaca549c2/docs/figure/revision.Rmd/unnamed-chunk-16-1.png" target="_blank">bcec945</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>The eigen vector for this covariance matrix (72.5% PVE).</p>
<p><img src="figure/revision.Rmd/unnamed-chunk-17-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-17-1">
Past versions of unnamed-chunk-17-1.png
</button>
</p>
<div id="fig-unnamed-chunk-17-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bcec9455b663a80918225173ddf1b04aaca549c2/docs/figure/revision.Rmd/unnamed-chunk-17-1.png" target="_blank">bcec945</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/revision.Rmd/unnamed-chunk-18-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-18-1">
Past versions of unnamed-chunk-18-1.png
</button>
</p>
<div id="fig-unnamed-chunk-18-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/bcec9455b663a80918225173ddf1b04aaca549c2/docs/figure/revision.Rmd/unnamed-chunk-18-1.png" target="_blank">bcec945</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-06-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="dynamic-eqtl-testing" class="section level2">
<h2>Dynamic eQTL testing</h2>
<div id="nominal-p-values-for-highlighted-dynamic-eqtl"
class="section level3">
<h3>Nominal P values for highlighted dynamic eQTL</h3>
<ul>
<li>NPglut: ADAM10 | rs6494026 P value: 4.139992e-13</li>
<li>GABA: CROT | rs13233308 P value: 0.295749</li>
<li>NMglut: CPT1C | rs12104272 P value: 0.2200463</li>
</ul>
<p>P values for CROT and CPT1C (nominated by cTWAS) do not have
significant P values. But there is a SNP in weak LD that is significant
for nominal values for GABA CROT (rs6946379) and CPT1C</p>
</div>
<div id="dynamic-eqtl-expanded-to-fdr20-in-marginal-association-testing"
class="section level3">
<h3>Dynamic eQTL expanded to FDR&lt;20% in marginal association
testing</h3>
<p>We expanded the scope of eQTL testing to FDR&lt;20% with LD pruning
(<span class="math inline">\(R^2&lt;0.1\)</span>). So that multiple
testing burden is reduced. P values of eQTLs within a gene were
corrected with Bonferroni FWER. Benjamini Hochberg FDR was applied
across genes. Overall, we found 630, 953, and 1110 dynamic eQTLs in
nmglut, GABA, and npglut respectively. Dynamic eQTLs are less enriched
at TSS compared to eQTLs not significant for dynamic testing.</p>
<p><img src="figure/revision.Rmd/unnamed-chunk-19-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-19-1">
Past versions of unnamed-chunk-19-1.png
</button>
</p>
<div id="fig-unnamed-chunk-19-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/d636d7868d1fa713f77107fb7a5bf0b9d86e4200/docs/figure/revision.Rmd/unnamed-chunk-19-1.png" target="_blank">d636d78</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-07-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/revision.Rmd/unnamed-chunk-20-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-20-1">
Past versions of unnamed-chunk-20-1.png
</button>
</p>
<div id="fig-unnamed-chunk-20-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/d636d7868d1fa713f77107fb7a5bf0b9d86e4200/docs/figure/revision.Rmd/unnamed-chunk-20-1.png" target="_blank">d636d78</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-07-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/revision.Rmd/unnamed-chunk-21-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-21-1">
Past versions of unnamed-chunk-21-1.png
</button>
</p>
<div id="fig-unnamed-chunk-21-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/d636d7868d1fa713f77107fb7a5bf0b9d86e4200/docs/figure/revision.Rmd/unnamed-chunk-21-1.png" target="_blank">d636d78</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-07-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="updated-ctwas-results" class="section level2">
<h2>Updated cTWAS results</h2>
<p>We tried to rerun the analysis with updated version of cTWAS. So that
enrichment may have standard error. However, percentage of h2g decreased
from 11% to 6%. Sheng reported that old results were inflated because of
sampling 10% of GWAS SNPs to estimate prior parameters
(<code>thin=0.1</code>). Updated results do not require sampling any
more. We have tried different thinning parameters (i.e. 10%, 20%, 50%,
100%). Results show consistent decrease of h2g</p>
<div id="h2g-decrease-as-sampling-ratio-of-gwas-snps-increased"
class="section level4">
<h4>h2g decrease as sampling ratio of GWAS SNPs increased</h4>
<p><img src="figure/revision.Rmd/unnamed-chunk-22-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-22-1">
Past versions of unnamed-chunk-22-1.png
</button>
</p>
<div id="fig-unnamed-chunk-22-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/d636d7868d1fa713f77107fb7a5bf0b9d86e4200/docs/figure/revision.Rmd/unnamed-chunk-22-1.png" target="_blank">d636d78</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-07-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Moreover, distribution of h2g across conditions changed a lot
compared to original results. NMglut has much less heritability, and
conditions of early response have higher h2g in general.</p>
</div>
<div id="enrichment-of-ctwas-results-without-thinning"
class="section level4">
<h4>Enrichment of cTWAS results without thinning</h4>
<p><img src="figure/revision.Rmd/unnamed-chunk-23-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-23-1">
Past versions of unnamed-chunk-23-1.png
</button>
</p>
<div id="fig-unnamed-chunk-23-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/d636d7868d1fa713f77107fb7a5bf0b9d86e4200/docs/figure/revision.Rmd/unnamed-chunk-23-1.png" target="_blank">d636d78</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-07-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/revision.Rmd/unnamed-chunk-24-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-24-1">
Past versions of unnamed-chunk-24-1.png
</button>
</p>
<div id="fig-unnamed-chunk-24-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/d636d7868d1fa713f77107fb7a5bf0b9d86e4200/docs/figure/revision.Rmd/unnamed-chunk-24-1.png" target="_blank">d636d78</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-07-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/revision.Rmd/unnamed-chunk-25-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-25-1">
Past versions of unnamed-chunk-25-1.png
</button>
</p>
<div id="fig-unnamed-chunk-25-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/d636d7868d1fa713f77107fb7a5bf0b9d86e4200/docs/figure/revision.Rmd/unnamed-chunk-25-1.png" target="_blank">d636d78</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-07-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="putative-risk-genes-from-different-thinning"
class="section level4">
<h4>Putative risk genes from different thinning</h4>
<p>For the gene with PIP &gt; 50% in previous results, we compared their
PIP in the updated results.</p>
<p><img src="figure/revision.Rmd/unnamed-chunk-26-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-26-1">
Past versions of unnamed-chunk-26-1.png
</button>
</p>
<div id="fig-unnamed-chunk-26-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/d636d7868d1fa713f77107fb7a5bf0b9d86e4200/docs/figure/revision.Rmd/unnamed-chunk-26-1.png" target="_blank">d636d78</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-07-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Below is the heatmap of 28 putative risk gene with PIP&gt;50% in
updated cTWAS results without thinning. Highlighted genes such as CPT1C,
CROT, and NAGA are still there. NGEF nominated in NMglut in previous
results have PIP&lt;50%.</p>
<pre><code>2025-10-25 21:44:21 INFO::Annotating susie alpha result...
2025-10-25 21:44:21 INFO::Map molecular traits to genes.</code></pre>
<pre><code>2025-10-25 21:44:21 INFO::Compute combined PIPs...</code></pre>
<p><img src="figure/revision.Rmd/unnamed-chunk-27-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-27-1">
Past versions of unnamed-chunk-27-1.png
</button>
</p>
<div id="fig-unnamed-chunk-27-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/3a1b03675882e70df32f3f1084d872b95b236fe1/docs/figure/revision.Rmd/unnamed-chunk-27-1.png" target="_blank">3a1b036</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-07-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>The updated results shared 3 genes with Cerebellar (i.e. SERPINI1,
FOXN2, ZNF823). ZNF823 is the only gene shared by all GTEx brain
tissues.</p>
<pre class="r"><code>library(VennDiagram)</code></pre>
<pre><code>Loading required package: grid</code></pre>
<pre><code>Loading required package: futile.logger</code></pre>
<pre class="r"><code>p1 &lt;- venn.diagram(list(neuron_stim=pip1$gene_name, Cerebellar=GTEx.gene$Brain_Cerebellar_Hemisphere),
            filename=NULL, cat.pos=c(-20,20), alpha=0.4, fill=c(&quot;red&quot;,&quot;blue&quot;),fontface = &quot;bold&quot;, cex=2)
grid.newpage()
grid.draw(p1)</code></pre>
<p><img src="figure/revision.Rmd/unnamed-chunk-29-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-29-1">
Past versions of unnamed-chunk-29-1.png
</button>
</p>
<div id="fig-unnamed-chunk-29-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/LifanLiang/neuron_stim_eQTL100/blob/3a1b03675882e70df32f3f1084d872b95b236fe1/docs/figure/revision.Rmd/unnamed-chunk-29-1.png" target="_blank">3a1b036</a>
</td>
<td>
Lifan Liang
</td>
<td>
2025-07-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="egenes-with-other-stimulus" class="section level3">
<h3>eGenes with other stimulus</h3>
<p>Compared with mouse sensory experience DEGs from Jubao:</p>
<ul>
<li>a Early response genes (|log2(FC)|&gt;1 , FDR &lt;0.05) whose
expression increases</li>
<li>b Early response genes (|log2(FC)|&gt;1 , FDR &lt;0.05) whose
expression decreases</li>
<li>c Late response genes (|log2(FC)|&gt;1 , FDR &lt;0.05) whose
expression increases</li>
<li>d Late response genes (|log2(FC)|&gt;1 , FDR &lt;0.05) whose
expression decreases</li>
</ul>
<div id="degs-with-excitatory-neuron-cells" class="section level4">
<h4>DEGs with excitatory neuron cells?</h4>
<p>Out of 108 DEGs (log fold change &gt; 1 and FDR&lt;5%), 61 of them
was an eQTL in at least one condition.</p>
<div class="datatables html-widget html-fill-item" id="htmlwidget-7657cc3e42637b46d6e4" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-7657cc3e42637b46d6e4">{"x":{"filter":"none","vertical":false,"data":[["2","3","5","6","8","9","10","12","14","16","17","19","20","21","23","24","26","27","30","33","34","36","38","39","40","45","47","50","51","52","57","58","59","60","62","63","69","72","73","74","79","80","81","82","83","84","85","86","88","89","91","95","96","99","100","101","103","105","106","107","108"],["Ankrd33b","Arc","Arl5b","Atxn1","Bdnf","Bhlhe40","Camk1g","Cbln2","Cdc42ep3","Chuk","Cited2","Crh","Cyp51","Dnajb1","Dot1l","Dusp1","Dusp6","Egln1","Egr4","Fbxo33","Fmnl1","Fosb","Frmd6","Gadd45a","Gadd45b","Hivep3","Hsd17b12","Ier5","Ifrd1","Ina","Kdm7a","Klf10","Lrrk2","Mapk4","Mest","Midn","Nptx2","Nr4a3","Nrn1","Nrp1","Penk","Per1","Phf21b","Phlda1","Pim1","Pim3","Ppme1","Prkg2","R3hdm1","Rcan2","Rnd3","Sgk1","Sgsm1","Stk40","Tfrc","Tiparp","Tpbg","Trib1","Tsnax","Vgf","Zdbf2"],["","a","a","","c","","","c","a","","a","c","","a","c","a","a","c","a","a","c","a","a","","a","","c","","a","","a","","c","c","a","a","c","a","","","c","a","c","a","c","a","c","","c","","","","c","a","a","a","","","c","c","a"],["","a","a","","c","","a","c","","","","","","","","","","","a","","c","a","a","","a","","","","a","","","a","","","","a","c","a","","","","","a","","a","","","c","","","c","","c","a","","a","","","","c","a"],["","a","","","c","a","c","","","","","","","","","","a","","a","","","a","","","a","","","c","","","","","","","","a","c","a","c","","","","c","","","","","","","","","","c","","","","","","","c",""],["","a","","c","c","","","","","a","","","a","","","a","","","a","","","a","","","a","","","","","","","","a","","","a","","a","","","","","","","","","","","","","","","c","","","a","","a","","","c"],["","a","a","","c","a","","c","","","","","a","","","","a","","a","a","c","a","","","a","a","","","a","","a","","","","a","a","c","a","c","c","c","a","a","","","a","","","","","","a","","a","","a","","","","","a"],["a","a","a","","","","","","","","","","","","","","","","a","","c","a","","a","a","","","","","c","","","","","","a","c","a","c","","","a","","","","","","","","c","","","c","a","","a","c","","","c","a"],["ERG","ERG","ERG","LRG","LRG","ERG","Both","LRG","ERG","ERG","ERG","LRG","ERG","ERG","LRG","ERG","ERG","LRG","ERG","ERG","LRG","ERG","ERG","ERG","ERG","ERG","LRG","LRG","ERG","LRG","ERG","ERG","Both","LRG","ERG","ERG","LRG","ERG","LRG","LRG","LRG","ERG","Both","ERG","Both","ERG","LRG","LRG","LRG","LRG","LRG","ERG","LRG","ERG","ERG","ERG","LRG","ERG","LRG","LRG","Both"],["ANKRD33B","ARC","ARL5B","ATXN1","BDNF","BHLHE40","CAMK1G","CBLN2","CDC42EP3","CHUK","CITED2","CRH","CYP51A1","DNAJB1","DOT1L","DUSP1","DUSP6","EGLN1","EGR4","FBXO33","FMNL1","FOSB","FRMD6","GADD45A","GADD45B","HIVEP3","HSD17B12","IER5","IFRD1","INA","KDM7A","KLF10","LRRK2","MAPK4","MEST","MIDN","NPTX2","NR4A3","NRN1","NRP1","PENK","PER1","PHF21B","PHLDA1","PIM1","PIM3","PPME1","PRKG2","R3HDM1","RCAN2","RND3","SGK1","SGSM1","STK40","TFRC","TIPARP","TPBG","TRIB1","TSNAX","VGF","ZDBF2"],[false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,true,false,false,false,true,false,true,false,true,false,false,false,false,false,false,false,false,true,true,true,false,false,true,false,true,false,true,false,false,false,true,false,false,false,false,true],[false,false,false,false,false,false,true,false,false,false,false,false,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,false,false,false,false,true,false,false,true,false,false,false,false,false,false,true,false,false,false,false,false,false,false],[true,false,true,false,false,false,false,false,true,false,true,false,false,false,false,true,false,true,false,false,true,false,true,false,false,false,true,false,false,false,false,false,true,false,true,false,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,true,false,false,false,false,false,true,false,false],[false,false,true,false,false,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,false,false,false,true,false,true,false,false,false,false,false,false],[false,false,false,false,false,true,false,true,false,false,false,false,false,true,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,true,true,false,false,false,true,true,false,false,true,false,false,true,false,false,false,true,false,true,false,false,false,false,false,false,false,false,false,false],[false,false,false,true,true,false,false,true,false,false,true,false,false,false,false,true,false,false,false,false,true,false,false,false,false,false,true,true,true,false,false,true,false,false,false,true,true,false,false,false,true,true,false,false,false,true,false,true,false,false,false,false,false,false,true,true,false,false,false,true,false],[false,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,true,false,true,false,false,true,false,false,true,false,false,true,true,false,false,true,false,false,true,false,false,true,false,false,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,false,false,false,false],[false,false,false,false,false,false,false,true,true,true,false,true,false,false,false,false,false,true,false,false,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,true,false,false],[false,true,false,true,true,false,false,false,false,false,false,false,false,false,false,true,false,true,false,false,false,false,true,true,false,false,true,false,true,false,false,false,false,true,true,true,true,false,false,false,false,false,false,false,false,false,false,true,false,true,false,false,true,false,false,false,false,true,false,true,false]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>GeneName<\/th>\n      <th>ExcL23<\/th>\n      <th>ExcL4<\/th>\n      <th>ExcL5_3<\/th>\n      <th>ExcL5_2<\/th>\n      <th>ExcL5_1<\/th>\n      <th>ExcL6<\/th>\n      <th>logFC.1<\/th>\n      <th>ortholog_name<\/th>\n      <th>eqtl_in_0hr_GABA<\/th>\n      <th>eqtl_in_0hr_nmglut<\/th>\n      <th>eqtl_in_0hr_npglut<\/th>\n      <th>eqtl_in_1hr_GABA<\/th>\n      <th>eqtl_in_1hr_nmglut<\/th>\n      <th>eqtl_in_1hr_npglut<\/th>\n      <th>eqtl_in_6hr_GABA<\/th>\n      <th>eqtl_in_6hr_nmglut<\/th>\n      <th>eqtl_in_6hr_npglut<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"GeneName","targets":1},{"name":"ExcL23","targets":2},{"name":"ExcL4","targets":3},{"name":"ExcL5_3","targets":4},{"name":"ExcL5_2","targets":5},{"name":"ExcL5_1","targets":6},{"name":"ExcL6","targets":7},{"name":"logFC.1","targets":8},{"name":"ortholog_name","targets":9},{"name":"eqtl_in_0hr_GABA","targets":10},{"name":"eqtl_in_0hr_nmglut","targets":11},{"name":"eqtl_in_0hr_npglut","targets":12},{"name":"eqtl_in_1hr_GABA","targets":13},{"name":"eqtl_in_1hr_nmglut","targets":14},{"name":"eqtl_in_1hr_npglut","targets":15},{"name":"eqtl_in_6hr_GABA","targets":16},{"name":"eqtl_in_6hr_nmglut","targets":17},{"name":"eqtl_in_6hr_npglut","targets":18}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
<div id="degs-with-inhibitory-neuron-cells" class="section level4">
<h4>DEGs with inhibitory neuron cells?</h4>
<p>Out of 102 DEGs (log fold change &gt; 1 and FDR&lt;5%), 52 of them
was an eQTL in at least one condition.</p>
<div class="datatables html-widget html-fill-item" id="htmlwidget-427ea3e24281aa554de8" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-427ea3e24281aa554de8">{"x":{"filter":"none","vertical":false,"data":[["4","6","9","11","14","16","17","18","19","20","23","24","25","26","27","32","34","39","44","46","50","51","53","55","59","60","61","62","63","65","66","67","70","72","73","74","75","76","77","78","80","82","83","85","88","89","90","93","96","97","98","101"],["Arc","Baiap2","Ccnt1","Cenpa","Cpeb1","Crh","Crhbp","Ctsb","Cxcl14","Dusp1","Egr4","Elavl4","Elovl4","Eno1","Eno1","Fosb","Gabra4","Glul","Gtf3c1","Hcn1","Hunk","Ifrd1","Igfbp5","Itgav","Lgi1","Llph","Mbp","Mdm2","Midn","Ngef","Noct","Npy","Nr4a3","Ntrk3","Osbpl3","Pam","Paqr8","Pcdh17","Pcnp","Per1","Plppr4","Prickle2","Prrc2a","Sccpdh","Serpine2","Snx19","Socs2","Syn2","Timp2","Tiparp","Ube2g1","Vgf"],["","","","","","","","","","","","","","c","c","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","a",""],["","","","","","c","","","","","","","","","","","","","a","","","","c","","","","","","","","","","a","","","c","","","","","","","","","c","","a","c","","","","c"],["","","","","","a","","","","","","","","","","","","","","","","","","","","","c","","","","","","","","","","","","","","","","a","","","","","","","","",""],["a","c","a","c","","","c","","","a","a","","c","","","a","c","a","","c","c","a","","c","","","","c","a","","a","c","a","a","c","","a","c","a","a","","a","","c","","","","","","a","","c"],["","","","","a","","","","c","","","","","","","","","","","","","","","","","","c","","","","","","","","","","","","","","","","","","","","","","a","","",""],["","","","","","","","a","","","","a","","","","","","","","","","","","","c","c","","","","c","","","","","","","","","","","c","","","","","a","","","","","",""],["ERG","LRG","ERG","LRG","ERG","Both","LRG","ERG","LRG","ERG","ERG","ERG","LRG","LRG","LRG","ERG","LRG","ERG","ERG","LRG","LRG","ERG","LRG","LRG","LRG","LRG","LRG","LRG","ERG","LRG","ERG","LRG","ERG","ERG","LRG","LRG","ERG","LRG","ERG","ERG","LRG","ERG","ERG","LRG","LRG","ERG","ERG","LRG","ERG","ERG","ERG","LRG"],["ARC","BAIAP2","CCNT1","CENPA","CPEB1","CRH","CRHBP","CTSB","CXCL14","DUSP1","EGR4","ELAVL4","ELOVL4","ENO1","ENO1","FOSB","GABRA4","GLUL","GTF3C1","HCN1","HUNK","IFRD1","IGFBP5","ITGAV","LGI1","LLPH","MBP","MDM2","MIDN","NGEF","NOCT","NPY","NR4A3","NTRK3","OSBPL3","PAM","PAQR8","PCDH17","PCNP","PER1","PLPPR4","PRICKLE2","PRRC2A","SCCPDH","SERPINE2","SNX19","SOCS2","SYN2","TIMP2","TIPARP","UBE2G1","VGF"],[false,false,false,false,false,false,true,true,false,false,false,false,false,true,true,false,true,false,false,false,true,false,true,true,true,false,true,false,false,false,false,true,false,false,false,true,false,false,false,false,true,true,true,true,false,true,false,true,true,true,false,false],[false,false,false,false,false,false,false,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,true,false,false,false,false,false,false,false,false,false,true,false,false,false,true,false,true,false,false,false,false],[false,false,false,false,true,false,false,true,false,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,false,false,false,false,true,false,true,false,true,false,true,false,false,false,true,false,true,true,true,false,true,false,false,false,false],[false,false,true,false,false,false,false,true,false,false,false,false,true,true,true,true,true,false,false,false,true,false,true,false,true,false,false,false,false,true,false,false,false,false,true,true,true,false,false,false,false,true,false,true,false,true,false,true,false,false,false,false],[false,false,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,false,false,false,false,false,false,true,false,true,false,false,false,true,false,true,false,false,false,false],[false,false,false,false,false,false,true,false,false,true,false,false,false,false,false,false,false,false,true,true,true,true,false,false,false,false,true,true,true,true,true,true,false,true,true,true,false,true,false,true,false,true,false,true,true,true,true,true,true,true,true,true],[true,true,false,false,false,false,false,true,false,false,true,false,true,false,false,false,true,true,false,false,false,false,false,false,true,false,true,false,false,true,false,false,false,true,true,true,false,false,true,false,false,true,true,true,false,true,false,true,false,false,false,false],[false,false,true,true,false,true,false,false,true,false,false,false,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,false,true,false,true,false,false,false,false],[true,true,false,false,false,false,true,true,true,true,false,false,false,true,true,false,false,false,false,true,true,true,false,true,false,true,true,false,true,true,false,false,false,true,false,true,false,false,false,false,false,false,false,true,true,true,false,true,false,false,true,true]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>GeneName<\/th>\n      <th>Int_Pv<\/th>\n      <th>Int_Vip<\/th>\n      <th>Int_Sst_1<\/th>\n      <th>Int_Sst_2<\/th>\n      <th>Int_Npy<\/th>\n      <th>Int_Cck<\/th>\n      <th>logFC.1<\/th>\n      <th>ortholog_name<\/th>\n      <th>eqtl_in_0hr_GABA<\/th>\n      <th>eqtl_in_0hr_nmglut<\/th>\n      <th>eqtl_in_0hr_npglut<\/th>\n      <th>eqtl_in_1hr_GABA<\/th>\n      <th>eqtl_in_1hr_nmglut<\/th>\n      <th>eqtl_in_1hr_npglut<\/th>\n      <th>eqtl_in_6hr_GABA<\/th>\n      <th>eqtl_in_6hr_nmglut<\/th>\n      <th>eqtl_in_6hr_npglut<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"GeneName","targets":1},{"name":"Int_Pv","targets":2},{"name":"Int_Vip","targets":3},{"name":"Int_Sst_1","targets":4},{"name":"Int_Sst_2","targets":5},{"name":"Int_Npy","targets":6},{"name":"Int_Cck","targets":7},{"name":"logFC.1","targets":8},{"name":"ortholog_name","targets":9},{"name":"eqtl_in_0hr_GABA","targets":10},{"name":"eqtl_in_0hr_nmglut","targets":11},{"name":"eqtl_in_0hr_npglut","targets":12},{"name":"eqtl_in_1hr_GABA","targets":13},{"name":"eqtl_in_1hr_nmglut","targets":14},{"name":"eqtl_in_1hr_npglut","targets":15},{"name":"eqtl_in_6hr_GABA","targets":16},{"name":"eqtl_in_6hr_nmglut","targets":17},{"name":"eqtl_in_6hr_npglut","targets":18}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.1.2 (2021-11-01)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur 10.16

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] grid      stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
 [1] DT_0.33             biomaRt_2.50.3      VennDiagram_1.7.3  
 [4] futile.logger_1.4.3 ctwas_0.5.34        ComplexUpset_1.3.3 
 [7] ggplot2_3.5.2       mashr_0.2.69        ashr_2.2-54        
[10] pheatmap_1.0.13     workflowr_1.7.1    

loaded via a namespace (and not attached):
  [1] BiocFileCache_2.2.1         plyr_1.8.6                 
  [3] lazyeval_0.2.2              crosstalk_1.2.1            
  [5] BiocParallel_1.28.3         GenomeInfoDb_1.30.1        
  [7] LDlinkR_1.4.0               digest_0.6.29              
  [9] invgamma_1.2                ensembldb_2.18.4           
 [11] htmltools_0.5.8.1           SQUAREM_2021.1             
 [13] magrittr_2.0.3              memoise_2.0.1              
 [15] tzdb_0.2.0                  Biostrings_2.62.0          
 [17] readr_2.1.2                 AMR_3.0.0                  
 [19] matrixStats_1.5.0           locuszoomr_0.3.8           
 [21] prettyunits_1.2.0           colorspace_2.1-1           
 [23] blob_1.2.4                  rappdirs_0.3.3             
 [25] ggrepel_0.9.1               xfun_0.52                  
 [27] dplyr_1.0.7                 callr_3.7.6                
 [29] crayon_1.5.3                RCurl_1.98-1.17            
 [31] jsonlite_2.0.0              zoo_1.8-14                 
 [33] glue_1.8.0                  gtable_0.3.6               
 [35] zlibbioc_1.40.0             XVector_0.34.0             
 [37] DelayedArray_0.20.0         BiocGenerics_0.40.0        
 [39] abind_1.4-8                 scales_1.4.0               
 [41] futile.options_1.0.1        mvtnorm_1.1-3              
 [43] DBI_1.2.3                   Rcpp_1.0.11                
 [45] viridisLite_0.4.2           progress_1.2.3             
 [47] bit_4.6.0                   stats4_4.1.2               
 [49] truncnorm_1.0-9             htmlwidgets_1.6.4          
 [51] httr_1.4.7                  RColorBrewer_1.1-3         
 [53] pkgconfig_2.0.3             XML_3.99-0.18              
 [55] farver_2.1.0                sass_0.4.1                 
 [57] dbplyr_2.1.1                tidyselect_1.2.1           
 [59] labeling_0.4.3              rlang_1.1.6                
 [61] later_1.3.0                 AnnotationDbi_1.56.2       
 [63] pgenlibr_0.3.6              tools_4.1.2                
 [65] cachem_1.0.6                cli_3.6.5                  
 [67] generics_0.1.4              RSQLite_2.2.9              
 [69] evaluate_1.0.4              stringr_1.5.1              
 [71] fastmap_1.1.0               yaml_2.3.10                
 [73] processx_3.8.6              knitr_1.50                 
 [75] bit64_4.6.0-1               fs_1.5.2                   
 [77] purrr_1.1.0                 KEGGREST_1.34.0            
 [79] AnnotationFilter_1.18.0     whisker_0.4.1              
 [81] formatR_1.14                xml2_1.3.3                 
 [83] compiler_4.1.2              rstudioapi_0.17.1          
 [85] plotly_4.11.0               filelock_1.0.3             
 [87] curl_6.4.0                  png_0.1-8                  
 [89] tibble_3.3.0                bslib_0.3.1                
 [91] stringi_1.7.6               ps_1.9.1                   
 [93] GenomicFeatures_1.46.5      lattice_0.22-7             
 [95] ProtGenerics_1.26.0         Matrix_1.3-4               
 [97] vctrs_0.6.5                 pillar_1.11.0              
 [99] lifecycle_1.0.4             jquerylib_0.1.4            
[101] data.table_1.17.8           cowplot_1.2.0              
[103] bitops_1.0-9                irlba_2.3.5.1              
[105] httpuv_1.6.5                patchwork_1.3.1            
[107] rtracklayer_1.54.0          GenomicRanges_1.46.1       
[109] R6_2.6.1                    BiocIO_1.4.0               
[111] promises_1.2.0.1            IRanges_2.28.0             
[113] lambda.r_1.2.4              dichromat_2.0-0.1          
[115] assertthat_0.2.1            SummarizedExperiment_1.24.0
[117] rprojroot_2.1.0             rjson_0.2.21               
[119] withr_3.0.2                 GenomicAlignments_1.30.0   
[121] Rsamtools_2.10.0            S4Vectors_0.32.4           
[123] GenomeInfoDbData_1.2.7      parallel_4.1.2             
[125] hms_1.1.3                   tidyr_1.2.0                
[127] gggrid_0.2-0                rmarkdown_2.29             
[129] MatrixGenerics_1.6.0        logging_0.10-108           
[131] git2r_0.29.0                mixsqp_0.3-43              
[133] getPass_0.2-4               Biobase_2.54.0             
[135] rmeta_3.0                   restfulr_0.0.16            </code></pre>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
