---
title: "Aggregation of cTWAS results"
author: "Lifan Liang"
date: "2024-07-24"
output: workflowr::wflow_html
editor_options:
  chunk_output_type: console
---

## Introduction

Aggregate cTWAS results from NPD traits and investigate sharing

```{r,echo=F}
fs <- c("SCZ","Bipolar","Depression","Neuroticism","ADHD")
ress <- list()
for(f in fs) {
  temp <- readRDS(paste0("data/processed_",f,"_res4.rds"))
  rownames(temp) <- temp$genename
  ress[[f]] <- temp#[temp$PIP_sum>0.8,]
}
```


```{r,echo=F}
fs1 <- c("joint_celltype","bipolar","Depression","neuroticisum","ADHD")
params <- list()
for(i in seq_along(fs1)) {
  params[[fs[i]]] <- readRDS(paste0("data/",fs1[i],"_cTWAS_params.rds"))
}
```

```{r,echo=F}
library(ggplot2)
pves <- lapply(params, function(x) {x$group_pve[-1]/x$total_pve})
enrichment <- lapply(params, function(x) {x$enrichment})
dat <- as.data.frame(cbind(do.call(c,pves),do.call(c,enrichment)))
colnames(dat) <- c("PVE", "enrichment")
dat$condition <- sapply(strsplit(rownames(dat),"\\."), function(x){x[2]})
dat$condition <- factor(dat$condition, levels = c(paste0(c(0,1,6),"hr_GABA"),
                                          paste0(c(0,1,6),"hr_nmglut"),
                                          paste0(c(0,1,6),"hr_npglut")))
dat$trait <- sapply(strsplit(rownames(dat),"\\."), function(x){x[1]})

ggplot(dat,aes(x=condition,y=trait)) + geom_count(aes(color = PVE, size=enrichment)) +guides(color = 'legend') + theme_minimal() 
```


## Cell type distribution of significant genes

```{r, echo=F}
library(ggplot2)
count1 <- list()
for(n in names(ress)) {
  count1[[n]] <- data.frame(table(ress[[n]]$celltype))
  colnames(count1[[n]]) <- c("cell_type","gene_count")
  count1[[n]]$disease <- n
}
count1 <- do.call(rbind,count1)

ggplot(count1,aes(x=disease, y=gene_count, fill=cell_type)) + 
  geom_bar(position="stack", stat="identity") + 
  theme(axis.text.x = element_text(angle = 45, vjust = 1, hjust = 1))
```

## Dynamic gene counts across cell types

```{r, echo=F}
ress[[3]]$PIP_diff <- rowSums(ress[[3]][,6:11]) - rowSums(ress[[3]][,3:5])
ress[[4]]$PIP_diff <- rowSums(ress[[4]][,6:10]) - rowSums(ress[[4]][,3:5])
ress[[5]]$PIP_diff <- rowSums(ress[[5]][,3:7])

dymcount = sapply(ress, function(x){sum(x$PIP_diff>0.5)})
statcount = sapply(ress, function(x){sum(x$PIP_diff<0.5)})
count2 <- data.frame(gene_count=c(dymcount,statcount), 
           disease=c(names(dymcount),names(statcount)),
           type=c(rep("dynamic",length(dymcount)), rep("static",length(statcount))))

ggplot(count2,aes(x=disease, y=gene_count, fill=type)) + 
  geom_bar(position="stack", stat="identity") + 
  theme(axis.text.x = element_text(angle = 45, vjust = 1, hjust = 1))
```

```{r, echo=F}
ress1 <- lapply(ress, function(x) {
  c(rowSums(x[,grepl("0hr",colnames(x))]),
  rowSums(x[,grepl("1hr",colnames(x))]),
  rowSums(x[,grepl("6hr",colnames(x))]))
})

dymcount = sapply(ress, function(x){sum(x$PIP_diff>0.5)})
statcount = sapply(ress, function(x){sum(x$PIP_diff<0.5)})
count2 <- data.frame(gene_count=c(dymcount,statcount), 
           disease=c(names(dymcount),names(statcount)),
           type=c(rep("dynamic",length(dymcount)), rep("static",length(statcount))))

ggplot(count2,aes(x=disease, y=gene_count, fill=type)) + 
  geom_bar(position="stack", stat="identity") + 
  theme(axis.text.x = element_text(angle = 45, vjust = 1, hjust = 1)) + 
  theme_classic()
```

## Significant gene distribution across the five traits

Not much sharing among traits.

```{r, echo=F}
library(pheatmap)

gene.univ <- unique(do.call(c, lapply(ress, function(x){x$genename})))
dat1 <- matrix(0,nrow=length(gene.univ),ncol=length(ress))
rownames(dat1)=gene.univ
colnames(dat1)=names(ress)
for(n in names(ress)) {
  dat1[rownames(ress[[n]]),n] <- ress[[n]]$PIP_sum
}
pheatmap(dat1, cluster_rows = F, cluster_cols = F)
```

## Enrichment analysis

### Three traits with PIP>0.8
```{r, echo=F}
library(enrichR)
library(dplyr)
gene.univ <- unique(do.call(c, lapply(ress[1:3], function(x){x$genename[x$PIP_sum>0.8]})))
enrich.res <- enrichr(gene.univ, "GO_Biological_Process_2023")
head(enrich.res$GO_Biological_Process_2023[-(5:8)],10) %>%
  DT::datatable() %>%
  DT::formatRound(columns=c('P.value', 'Adjusted.P.value'), digits=3)
```

### Three traits with PIP>0.5
```{r, echo=F}
library(enrichR)
library(dplyr)
gene.univ <- unique(do.call(c, lapply(ress[1:3], function(x){x$genename[x$PIP_sum>0.5]})))
enrich.res <- enrichr(gene.univ, "GO_Biological_Process_2023")
head(enrich.res$GO_Biological_Process_2023[-(5:8)],10) %>%
  DT::datatable() %>%
  DT::formatRound(columns=c('P.value', 'Adjusted.P.value'), digits=3)
```

### Five traits with PIP>0.8
```{r, echo=F}
library(enrichR)
library(dplyr)
gene.univ <- unique(do.call(c, lapply(ress, function(x){x$genename[x$PIP_sum>0.8]})))
enrich.res <- enrichr(gene.univ, "GO_Biological_Process_2023")
head(enrich.res$GO_Biological_Process_2023[-(5:8)],10) %>%
  DT::datatable() %>%
  DT::formatRound(columns=c('P.value', 'Adjusted.P.value'), digits=3)
```

### Five traits with PIP>0.5
```{r, echo=F}
library(enrichR)
library(dplyr)
gene.univ <- unique(do.call(c, lapply(ress, function(x){x$genename[x$PIP_sum>0.5]})))
enrich.res <- enrichr(gene.univ, "GO_Biological_Process_2023")
head(enrich.res$GO_Biological_Process_2023[-(5:8)],10) %>%
  DT::datatable() %>%
  DT::formatRound(columns=c('P.value', 'Adjusted.P.value'), digits=3)
```

## Analysis of three genes enriched in lipid metabolism

### CPT1C

CPT1C was a top gene in SCZ with a high PIP in 1hr_NMglut.

```{r,echo=F}
CPT1C.summary <- readRDS("data/CPT1C_investigation.RDS")
DT::datatable(CPT1C.summary$gene_PIP)
```

The SNP related to CPT1C in 1 hour NMglut was 50K bps away from the other two SNPs. Looking into this gene-SNP pair, 1hr_nmglut is the only condition with FDR < 0.1.

```{r,echo=F}
plot(x=-log10(CPT1C.summary$CPT1C_eqtl$pvalue), y=abs(CPT1C.summary$CPT1C_eqtl$beta),
     xlab="-log10 nominal P value", ylab="absolute effect size (beta)", 
     ylim=c(0,0.6), xlim=c(0,3.5))
text(-log10(CPT1C.summary$CPT1C_eqtl$pvalue), abs(CPT1C.summary$CPT1C_eqtl$beta),
     labels=rownames(CPT1C.summary$CPT1C_eqtl), cex= 0.7, pos=3)
```

The SNP, rs12104272, has a nominal P value around $10^{-8}$ in SCZ GWAS. It was mapped to SCAF1 in (another study)[https://pubmed.ncbi.nlm.nih.gov/31740837/]. However, in our dataset, this gene-SNP pair was not significant in any conditions.

```{r,echo=F}
DT::datatable(CPT1C.summary$SCAF1_eqtl)
```

### CROT

CROT has PIP=0.87 in SCZ.The corresponding SNP was rs13233308

```{r,echo=F}
CROT.summary <- readRDS("data/CROT_investigation.RDS")
DT::datatable(CROT.summary$gene_PIP)
```

```{r,echo=F}
plot(x=-log10(CROT.summary$CROT_eqtl$pvalue), y=abs(CROT.summary$CROT_eqtl$beta),
     xlab="-log10 nominal P value", ylab="absolute effect size (beta)",
     ylim=c(0,0.6),xlim=c(0,5))
text(-log10(CROT.summary$CROT_eqtl$pvalue), abs(CROT.summary$CROT_eqtl$beta),
     labels=rownames(CROT.summary$CROT_eqtl), cex= 0.7, pos=3)
```

### ACADM

ACADM has PIP=0.61 in 6 hour NMglut for the trait 

```{r,echo=F}
ACADM.summary <- readRDS("data/ACADM_investigation.RDS")
DT::datatable(ACADM.summary$gene_PIP)
```

```{r,echo=F}
plot(x=-log10(ACADM.summary$ACADM_eqtl$pvalue), y=abs(ACADM.summary$ACADM_eqtl$beta),
     xlab="-log10 nominal P value", ylab="absolute effect size (beta)", 
     ylim=c(0,0.7), xlim=c(0,6))
text(-log10(ACADM.summary$ACADM_eqtl$pvalue), abs(ACADM.summary$ACADM_eqtl$beta),
     labels=rownames(ACADM.summary$ACADM_eqtl), cex= 0.7, pos=3)
```

