---
title: "caQTL_cTWAS"
author: "Lifan Liang"
date: "2023-12-08"
output: workflowr::wflow_html
editor_options:
  chunk_output_type: console
---

## Peak selection

cPeaks was defined as FDR < 0.05. The inclusion of peaks for each condition (a unique combination of time points and cell types) is: (1) a caPeak in any time point in the same cell type; (2) nominal P value < 0.1 in the current condition. The number of peaks was shown in the table below

```{r, echo=F}
library(GenomicRanges)
GABA1 <- readRDS("data/caQTL_GABA_res.RDS")
nmglut1 <- readRDS("data/caQTL_nmglut_res.RDS")
npglut1 <- readRDS("data/caQTL_npglut_res.RDS")

npeaks <- rbind(table(GABA1$timepoint), table(nmglut1$timepoint), table(npglut1$timepoint))
colnames(npeaks) <- paste0(c(0,1,6),"hour")
rownames(npeaks) <- c("GABA", "nmglut", "npglut")
knitr::kable(npeaks, caption="Number of peaks included in cTWAS")
```

## cTWAS parameter estimation

For each peak selected, the lead caQTL was used to build the prediction model. cTWAS first performed parameter estimation without fine mapping.

### GABA parameters
```{r, echo=F}
GABA.params <- readRDS("data/GABA_params.rds")
nmglut.params <- readRDS("data/nmglut_params.rds")
npglut.params <- readRDS("data/npglut_params.rds")

GABA.params$convergence_plot
```

```{r}
library(RColorBrewer)
myPalette <- brewer.pal(4, "Set2") 
pie(GABA.params$group_pve, col=myPalette, border="white",
    main = paste0(sprintf("%.2f", 100-(GABA.params$group_pve[1]/GABA.params$total_pve)*100), "% PVE for GABA peaks"))
```

### NPglut parameters
```{r, echo=F}
npglut.params$convergence_plot
```

```{r, echo=F}
pie(npglut.params$group_pve, col=myPalette, border="white",
    main = paste0(sprintf("%.2f", 100-(npglut.params$group_pve[1]/npglut.params$total_pve)*100), "% PVE for NPglut peaks"))
```

### NMglut parameters

```{r, echo=F}
nmglut.params$convergence_plot
```

```{r, echo=F}
pie(nmglut.params$group_pve, col=myPalette, border="white",
    main = paste0(sprintf("%.2f", 100-(nmglut.params$group_pve[1]/nmglut.params$total_pve)*100), "% PVE for NMglut peaks"))
```


## Significant peaks

A peak is significant for the cell type if its sum of PIP across time points is more than 0.8. There are 22 peaks in GABA, 6 peaks in NPglut, and 11 peaks in NMglut.

### GABA significant peaks

```{r, echo=F}
library(kableExtra)
options(knitr.kable.NA = '')
GABA2 <- aggregate(GABA1$susie_pip, list(GABA1$genename), sum)
GABA3 <- merge(GABA2, GABA1[GABA1$timepoint=="0hr_GABA",c("genename","susie_pip")], 
                 by.x="Group.1", by.y="genename",all.x = T)
colnames(GABA3) <- c("genename","PIP_sum","PIP_0hr_GABA")
GABA4 <- merge(GABA3, GABA1[GABA1$timepoint=="1hr_GABA",c("genename","susie_pip")], 
                 by="genename",all.x = T)
colnames(GABA4)[4] <- "PIP_1hr_GABA"
GABA5 <- merge(GABA4, GABA1[GABA1$timepoint=="6hr_GABA",c("genename","susie_pip")], 
                 by="genename",all.x = T)
colnames(GABA5)[5] <- "PIP_6hr_GABA"
GABA5[is.na(GABA5)] <- 0
GABA5 <- GABA5[order(GABA5$PIP_sum, decreasing = T),]
GABA5$PIP_diff <- GABA5$PIP_sum - 2 * GABA5$PIP_0hr_GABA
colnames(GABA5)[1] <- "PeakRange"
GABA5 <- GABA5[order(GABA5$PIP_sum,decreasing = T),]
GABA5[GABA5$PIP_sum>0.8,] %>%
  kable(caption = "Significant peaks for the GABA cell type", digits=3, row.names = F) %>%
  scroll_box(height="400px")
```

### npglut significant peaks

```{r, echo=F}
options(knitr.kable.NA = '')
npglut2 <- aggregate(npglut1$susie_pip, list(npglut1$genename), sum)
npglut3 <- merge(npglut2, npglut1[npglut1$timepoint=="0hr_npglut",c("genename","susie_pip")], 
                 by.x="Group.1", by.y="genename",all.x = T)
colnames(npglut3) <- c("genename","PIP_sum","PIP_0hr_npglut")
npglut4 <- merge(npglut3, npglut1[npglut1$timepoint=="1hr_npglut",c("genename","susie_pip")], 
                 by="genename",all.x = T)
colnames(npglut4)[4] <- "PIP_1hr_npglut"
npglut5 <- merge(npglut4, npglut1[npglut1$timepoint=="6hr_npglut",c("genename","susie_pip")], 
                 by="genename",all.x = T)
colnames(npglut5)[5] <- "PIP_6hr_npglut"
npglut5[is.na(npglut5)] <- 0
npglut5 <- npglut5[order(npglut5$PIP_sum, decreasing = T),]
npglut5$PIP_diff <- npglut5$PIP_sum - 2 * npglut5$PIP_0hr_npglut
npglut5 <- npglut5[order(npglut5$PIP_sum,decreasing = T),]
colnames(npglut5)[1] <- "PeakRange"
npglut5[npglut5$PIP_sum>0.8,] %>%
  kable(caption = "Significant peaks for the npglut cell type", digits=3, row.names = F) %>%
  scroll_box(height="400px")
```

### nmglut significant peaks

```{r, echo=F}
nmglut2 <- aggregate(nmglut1$susie_pip, list(nmglut1$genename), sum)
nmglut3 <- merge(nmglut2, nmglut1[nmglut1$timepoint=="0hr_nmglut",c("genename","susie_pip")], 
               by.x="Group.1", by.y="genename",all.x = T)
colnames(nmglut3) <- c("genename","PIP_sum","PIP_0hr_nmglut")
nmglut4 <- merge(nmglut3, nmglut1[nmglut1$timepoint=="1hr_nmglut",c("genename","susie_pip")], 
               by="genename",all.x = T)
colnames(nmglut4)[4] <- "PIP_1hr_nmglut"
nmglut5 <- merge(nmglut4, nmglut1[nmglut1$timepoint=="6hr_nmglut",c("genename","susie_pip")], 
               by="genename",all.x = T)
colnames(nmglut5)[5] <- "PIP_6hr_nmglut"
nmglut5[is.na(nmglut5)] <- 0
nmglut5$PIP_diff <- nmglut5$PIP_sum - 2 * nmglut5$PIP_0hr_nmglut
nmglut5 <- nmglut5[order(nmglut5$PIP_sum,decreasing = T),]
colnames(nmglut5)[1] <- "PeakRange"
nmglut5[nmglut5$PIP_sum>0.8,] %>%
  kable(caption = "Significant peaks for the nmglut cell type", digits=3, row.names = F) %>%
  scroll_box(height="400px")
```

## Cell type overlap

### Overlap of the cTWAS input

```{r, echo=F, message=F}
library(ggvenn)

ggvenn(list(GABA=GABA5$PeakRange, nmglut=nmglut5$PeakRange, npglut=npglut5$PeakRange),
       c("GABA","nmglut","npglut"))
```

### Overlap of the cTWAS significant peaks

```{r, echo=F, message=F}
library(ggvenn)

ggvenn(list(GABA=GABA5[GABA5$PIP_sum>0.8,"PeakRange"], nmglut=nmglut5[nmglut5$PIP_sum>0.8,"PeakRange"], npglut=npglut5[npglut5$PIP_sum>0.8,"PeakRange"]), c("GABA","nmglut","npglut"))
```

## Dynamic cPeaks

In addition to PIP sum > 0.8, dynamic cpeaks must satisfy that:

$$
PIP_{GABA6} + PIP_{GABA1} - PIP_{GABA0} \geq 0.5
$$
for the GABA cell type. The same condition applies to the other cell types .

Peaks not included in one time point are regarded as $PIP=0$.

```{r,echo=F,message=F}
library(ggplot2)
Negenes <- c(sum((GABA5$PIP_diff>=0.5) & (GABA5$PIP_sum>0.8)),
             sum((nmglut5$PIP_diff>=0.5) & (nmglut5$PIP_sum>0.8)),
             sum((npglut5$PIP_diff>=0.5) & (npglut5$PIP_sum>0.8)),
             sum((GABA5$PIP_diff<0.5) & (GABA5$PIP_sum>0.8)),
             sum((nmglut5$PIP_diff<0.5) & (nmglut5$PIP_sum>0.8)),
             sum((npglut5$PIP_diff<0.5) & (npglut5$PIP_sum>0.8)))
dat <- data.frame(Ngenes=Negenes, celltype=rep(c("GABA","nmglut","npglut"),2),
           dynamic=c(rep(T,3), rep(F,3)))
ggplot(dat, aes(x=celltype, fill=dynamic, y=Ngenes, label=Ngenes)) + 
  geom_bar(position="stack", stat="identity") +
  geom_text(size = 6, position = position_stack(vjust = 0.5)) + 
  xlab("Number of Top genes") + ylab("Cell type")
```

## Cell type correlation

#### Comparing top genes across cell types nominated in the same locus

This table shows how different cell types may nominate different genes in the same locus. 

All shared peaks are consistent except locus 3&4 in chr7.

```{r, echo=F}
tpeak <- read.table("data/multigroup_cTWAS_toppeak_by_locus.txt", header=T, sep=" ")
tpeak$locus_tag <- rownames(tpeak)
rownames(tpeak) <- NULL
tpeak[,c(4,1,2,3)] %>%
  mutate_if(is.numeric, format, digits=3,nsmall = 0) %>%
  kable(caption="Dynamic genes by locus") %>%
  scroll_box(height="400px")
#knitr::kable(npglut.pip1, caption = "Top genes for the npGlut cell type")
```

## Weak signal aggregation

### SNP level

Aggregate PIPs of peaks by their shared lead variants.

```{r, echo=F}
npglut.aggrPIP <- readRDS("data/caQTL_NPglut_ctwas_aggr.RDS")
nmglut.aggrPIP <- readRDS("data/caQTL_NMglut_ctwas_aggr.RDS")
GABA.aggrPIP <- readRDS("data/caQTL_GABA_ctwas_aggr.RDS")

npglut.table <- npglut.aggrPIP$aggrbysnp[npglut.aggrPIP$aggrbysnp$PIP>0.8,]
nmglut.table <- nmglut.aggrPIP$aggrbysnp[nmglut.aggrPIP$aggrbysnp$PIP>0.8,]
GABA.table <- GABA.aggrPIP$aggrbysnp[GABA.aggrPIP$aggrbysnp$PIP>0.8,]

knitr::kable(npglut.table, caption="SNPs with a sum of PIP>0.8 in npglut")
```

```{r, echo=F}
knitr::kable(GABA.table, caption="SNPs with a sum of PIP>0.8 in GABA")
```

```{r, echo=F}
knitr::kable(nmglut.table, caption="SNPs with a sum of PIP>0.8 in nmglut")
```

### Gene level
After aggregating PIP to SNP, we assign each SNP to its nearest gene by TSS. SNP-gene pair with distance > 250L were removed. Then the PIP of SNPs were attributed its nearest genes.

```{r,echo=F}
npglut.table <- npglut.aggrPIP$aggrbygene[npglut.aggrPIP$aggrbygene$aggregate_PIP>0.8,]
nmglut.table <- nmglut.aggrPIP$aggrbygene[nmglut.aggrPIP$aggrbygene$aggregate_PIP>0.8,]
GABA.table <- GABA.aggrPIP$aggrbygene[GABA.aggrPIP$aggrbygene$aggregate_PIP>0.8,]

knitr::kable(npglut.table, caption="Genes with sum of PIP>0.8 in npglut")
```

```{r, echo=F}
knitr::kable(GABA.table, caption="Genes with sum of PIP>0.8 in GABA")
```

```{r, echo=F}
knitr::kable(nmglut.table, caption="Genes with sum of PIP>0.8 in nmglut")
```


### Locus level

Aggregate PIPs of peaks by locus. 

```{r, echo=F}
GABA.locusPIP <- aggregate(GABA1$susie_pip, list(GABA1$region_tag1, GABA1$region_tag2, GABA1$timepoint),sum)
colnames(GABA.locusPIP) <- c("chr","locus","timepoint","PIP_sum")
hist(GABA.locusPIP$PIP_sum, breaks = 30, main="GABA PIP aggregated on locus level")
lines(c(0.8,0.8),c(0,300), col="red ")
text(1.1,200,paste0(sum(GABA.locusPIP$PIP_sum>0.8)," locus (", sprintf("%.1f",mean(GABA.locusPIP$PIP_sum>0.8)*100),"%)"), col="blue")
```

```{r,echo=F}
npglut.table <- npglut.aggrPIP$aggrbylocus[npglut.aggrPIP$aggrbylocus$PIP>0.8,]
nmglut.table <- nmglut.aggrPIP$aggrbylocus[nmglut.aggrPIP$aggrbylocus$PIP>0.8,]
GABA.table <- GABA.aggrPIP$aggrbylocus[GABA.aggrPIP$aggrbylocus$PIP>0.8,]

knitr::kable(npglut.table[order(npglut.table$PIP, decreasing = T),], caption="Locus with sum of PIP>0.8 in npglut")
```

```{r,echo=F}
knitr::kable(GABA.table[order(GABA.table$PIP, decreasing = T),], caption="Locus with sum of PIP>0.8 in GABA")
```

```{r,echo=F}
knitr::kable(nmglut.table[order(nmglut.table$PIP, decreasing = T),], caption="Locus with sum of PIP>0.8 in nmglut")
```

## Shared signals between cTWAS and ASoC

For each cell type, if an allelic specific open chromatin (ASoC) SNP (FDR<0.05) is within 500 bps upstream and downstream of a cPeak and the matched cPeak has GWAS PIP > 0.8, then they shared the same signal.

### GABA

```{r, echo=F}
GABA0.asoc <- read.table("/Users/lifanliang/Documents/neuron_stim/ASoC_100_lines_22Dec2023/0hr_GABA_SNP_2_30_full.tsv",
                         header=T)
GABA1.asoc <- read.table("/Users/lifanliang/Documents/neuron_stim/ASoC_100_lines_22Dec2023/1hr_GABA_SNP_2_30_full.tsv",
                         header=T)
GABA6.asoc <- read.table("/Users/lifanliang/Documents/neuron_stim/ASoC_100_lines_22Dec2023/6hr_GABA_SNP_2_30_full.tsv",
                         header=T)
GABA0.asoc1 <- makeGRangesFromDataFrame(GABA0.asoc[GABA0.asoc$FDR<0.05,], seqnames.field = "CHROM", end.field = "POS", start.field = "POS", keep.extra.columns = T)
GABA1.asoc1 <- makeGRangesFromDataFrame(GABA1.asoc[GABA1.asoc$FDR<0.05,], seqnames.field = "CHROM", end.field = "POS", start.field = "POS", keep.extra.columns = T)
GABA6.asoc1 <- makeGRangesFromDataFrame(GABA6.asoc[GABA6.asoc$FDR<0.05,], seqnames.field = "CHROM", end.field = "POS", start.field = "POS", keep.extra.columns = T)
GABA6 <- GABA5[GABA5$PIP_sum>0.8,]
GABA6$chr <- sapply(strsplit(GABA6$PeakRange,"_"), function(x) {x[1]})
GABA6$start <- sapply(strsplit(GABA6$PeakRange,"_"), function(x) {x[2]})
GABA6$end <- sapply(strsplit(GABA6$PeakRange,"_"), function(x) {x[3]})
GABA7 <- makeGRangesFromDataFrame(GABA6)
start(GABA7) <- start(GABA7) - 500
end(GABA7) <- end(GABA7) + 500
asoc0 <- findOverlaps(GABA7,GABA0.asoc1)
asoc1 <- findOverlaps(GABA7,GABA1.asoc1)
asoc6 <- findOverlaps(GABA7,GABA6.asoc1)


peak2asoc0 <- data.frame(peakrange=GABA6$PeakRange[asoc0@from],ASOC_SNP=GABA0.asoc1$ID[asoc0@to])
knitr::kable(peak2asoc0, caption="Significant cTWAS peaks overlapped with ASoC in GABA 0 hours")
```

```{r,echo=F}
library(ggvenn)
temp <- GABA0.asoc1$ID
temp[asoc0@to] <- GABA6$PeakRange[asoc0@from]
ggvenn(list(GABA_caQTL_cTWAS=GABA6$PeakRange, ASoC_GABA_0hr=temp),
       c("GABA_caQTL_cTWAS","ASoC_GABA_0hr"))
```


```{r,echo=F}
peak2asoc1 <- data.frame(peakrange=GABA6$PeakRange[asoc1@from],ASOC_SNP=GABA1.asoc1$ID[asoc1@to])
knitr::kable(peak2asoc1, caption="Significant cTWAS peaks overlapped with ASoC in GABA 1 hours")
```

```{r,echo=F}
library(ggvenn)
temp <- GABA1.asoc1$ID
temp[asoc1@to] <- GABA6$PeakRange[asoc1@from]
ggvenn(list(GABA_caQTL_cTWAS=GABA6$PeakRange, ASoC_GABA_1hr=temp),
       c("GABA_caQTL_cTWAS","ASoC_GABA_1hr"))
```

```{r,echo=F}
peak2asoc6 <- data.frame(peakrange=GABA6$PeakRange[asoc6@from],ASOC_SNP=GABA6.asoc1$ID[asoc6@to])
knitr::kable(peak2asoc6, caption="Significant cTWAS peaks overlapped with ASoC in GABA 6 hours")
```

```{r,echo=F}
library(ggvenn)
temp <- GABA6.asoc1$ID
temp[asoc6@to] <- GABA6$PeakRange[asoc6@from]
ggvenn(list(GABA_caQTL_cTWAS=GABA6$PeakRange, ASoC_GABA_6hr=temp),
       c("GABA_caQTL_cTWAS","ASoC_GABA_6hr"))
```

### NPglut

```{r, echo=F}
npglut0.asoc <- read.table("/Users/lifanliang/Documents/neuron_stim/ASoC_100_lines_22Dec2023/0hr_npglut_SNP_2_30_full.tsv", header=T)
npglut1.asoc <- read.table("/Users/lifanliang/Documents/neuron_stim/ASoC_100_lines_22Dec2023/1hr_npglut_SNP_2_30_full.tsv", header=T)
npglut6.asoc <- read.table("/Users/lifanliang/Documents/neuron_stim/ASoC_100_lines_22Dec2023/6hr_npglut_SNP_2_30_full.tsv", header=T)
npglut0.asoc1 <- makeGRangesFromDataFrame(npglut0.asoc[npglut0.asoc$FDR<0.05,], seqnames.field = "CHROM", end.field = "POS", start.field = "POS", keep.extra.columns = T)
npglut1.asoc1 <- makeGRangesFromDataFrame(npglut1.asoc[npglut1.asoc$FDR<0.05,], seqnames.field = "CHROM", end.field = "POS", start.field = "POS", keep.extra.columns = T)
npglut6.asoc1 <- makeGRangesFromDataFrame(npglut6.asoc[npglut6.asoc$FDR<0.05,], seqnames.field = "CHROM", end.field = "POS", start.field = "POS", keep.extra.columns = T)
npglut6 <- npglut5[npglut5$PIP_sum>0.8,]
npglut6$chr <- sapply(strsplit(npglut6$PeakRange,"_"), function(x) {x[1]})
npglut6$start <- sapply(strsplit(npglut6$PeakRange,"_"), function(x) {x[2]})
npglut6$end <- sapply(strsplit(npglut6$PeakRange,"_"), function(x) {x[3]})
npglut7 <- makeGRangesFromDataFrame(npglut6)
start(npglut7) <- start(npglut7) - 500
end(npglut7) <- end(npglut7) + 500
asoc0 <- findOverlaps(npglut7,npglut0.asoc1)
asoc1 <- findOverlaps(npglut7,npglut1.asoc1)
asoc6 <- findOverlaps(npglut7,npglut6.asoc1)

peak2asoc0 <- data.frame(peakrange=npglut6$PeakRange[asoc0@from],ASOC_SNP=npglut0.asoc1$ID[asoc0@to])
knitr::kable(peak2asoc0, caption="Significant cTWAS peaks overlapped with ASoC in NPglut 0 hours")
```

```{r,echo=F}
temp <- npglut0.asoc1$ID
temp[asoc0@to] <- npglut6$PeakRange[asoc0@from]
ggvenn(list(npglut_caQTL_cTWAS=npglut6$PeakRange, ASoC_npglut_0hr=temp),
       c("npglut_caQTL_cTWAS","ASoC_npglut_0hr"))
```

```{r, echo=F}
peak2asoc1 <- data.frame(peakrange=npglut6$PeakRange[asoc1@from],ASOC_SNP=npglut1.asoc1$ID[asoc1@to])
knitr::kable(peak2asoc1, caption="Significant cTWAS peaks overlapped with ASoC in NPglut 1 hours")
```

```{r,echo=F}
temp <- npglut1.asoc1$ID
temp[asoc1@to] <- npglut6$PeakRange[asoc1@from]
ggvenn(list(npglut_caQTL_cTWAS=npglut6$PeakRange, ASoC_npglut_1hr=temp),
       c("npglut_caQTL_cTWAS","ASoC_npglut_1hr"))
```

```{r, echo=F}
peak2asoc6 <- data.frame(peakrange=GABA6$PeakRange[asoc6@from],ASOC_SNP=GABA6.asoc1$ID[asoc6@to])
knitr::kable(peak2asoc6, caption="Significant cTWAS peaks overlapped with ASoC in NPglut 6 hours")
```

```{r,echo=F}
temp <- npglut6.asoc1$ID
temp[asoc6@to] <- npglut6$PeakRange[asoc6@from]
ggvenn(list(npglut_caQTL_cTWAS=npglut6$PeakRange, ASoC_npglut_6hr=temp),
       c("npglut_caQTL_cTWAS","ASoC_npglut_6hr"))
```


### NMglut
```{r, echo=F}
nmglut0.asoc <- read.table("/Users/lifanliang/Documents/neuron_stim/ASoC_100_lines_22Dec2023/0hr_nmglut_SNP_2_30_full.tsv", header=T)
nmglut1.asoc <- read.table("/Users/lifanliang/Documents/neuron_stim/ASoC_100_lines_22Dec2023/1hr_nmglut_SNP_2_30_full.tsv", header=T)
nmglut6.asoc <- read.table("/Users/lifanliang/Documents/neuron_stim/ASoC_100_lines_22Dec2023/6hr_nmglut_SNP_2_30_full.tsv", header=T)
nmglut0.asoc1 <- makeGRangesFromDataFrame(nmglut0.asoc[nmglut0.asoc$FDR<0.05,], seqnames.field = "CHROM", end.field = "POS", start.field = "POS", keep.extra.columns = T)
nmglut1.asoc1 <- makeGRangesFromDataFrame(nmglut1.asoc[nmglut1.asoc$FDR<0.05,], seqnames.field = "CHROM", end.field = "POS", start.field = "POS", keep.extra.columns = T)
nmglut6.asoc1 <- makeGRangesFromDataFrame(nmglut6.asoc[nmglut6.asoc$FDR<0.05,], seqnames.field = "CHROM", end.field = "POS", start.field = "POS", keep.extra.columns = T)
nmglut6 <- nmglut5[nmglut5$PIP_sum>0.8,]
nmglut6$chr <- sapply(strsplit(nmglut6$PeakRange,"_"), function(x) {x[1]})
nmglut6$start <- sapply(strsplit(nmglut6$PeakRange,"_"), function(x) {x[2]})
nmglut6$end <- sapply(strsplit(nmglut6$PeakRange,"_"), function(x) {x[3]})
nmglut7 <- makeGRangesFromDataFrame(nmglut6)
start(nmglut7) <- start(nmglut7) - 500
end(nmglut7) <- end(nmglut7) + 500
asoc0 <- findOverlaps(nmglut7,nmglut0.asoc1)
asoc1 <- findOverlaps(nmglut7,nmglut1.asoc1)
asoc6 <- findOverlaps(nmglut7,nmglut6.asoc1)

peak2asoc0 <- data.frame(peakrange=nmglut6$PeakRange[asoc0@from],ASOC_SNP=nmglut0.asoc1$ID[asoc0@to])
knitr::kable(peak2asoc0, caption="Significant cTWAS peaks overlapped with ASoC in NMglut 0 hours")
```

```{r,echo=F}
temp <- nmglut0.asoc1$ID
temp[asoc0@to] <- nmglut6$PeakRange[asoc0@from]
ggvenn(list(nmglut_caQTL_cTWAS=nmglut6$PeakRange, ASoC_nmglut_0hr=temp),
       c("nmglut_caQTL_cTWAS","ASoC_nmglut_0hr"))
```

```{r, echo=F}
peak2asoc1 <- data.frame(peakrange=nmglut6$PeakRange[asoc1@from],ASOC_SNP=nmglut1.asoc1$ID[asoc1@to])
knitr::kable(peak2asoc1, caption="Significant cTWAS peaks overlapped with ASoC in NMglut 1 hours")
```

```{r,echo=F}
temp <- nmglut1.asoc1$ID
temp[asoc1@to] <- nmglut6$PeakRange[asoc1@from]
ggvenn(list(nmglut_caQTL_cTWAS=nmglut6$PeakRange, ASoC_nmglut_1hr=temp),
       c("nmglut_caQTL_cTWAS","ASoC_nmglut_1hr"))
```

```{r, echo=F}
peak2asoc6 <- data.frame(peakrange=nmglut6$PeakRange[asoc6@from],ASOC_SNP=nmglut6.asoc1$ID[asoc6@to])
knitr::kable(peak2asoc6, caption="Significant cTWAS peaks overlapped with ASoC in NMglut 6 hours")
```

```{r,echo=F}
temp <- nmglut6.asoc1$ID
temp[asoc6@to] <- nmglut6$PeakRange[asoc6@from]
ggvenn(list(nmglut_caQTL_cTWAS=nmglut6$PeakRange, ASoC_nmglut_6hr=temp),
       c("nmglut_caQTL_cTWAS","ASoC_nmglut_6hr"))
```

### Rerunning cTWAS merging caQTL with the same SNP

For each SNP, we only selected one cPeak related to it with FDR<0.05. 

#### GABA parameters

GABA has more cPeaks than those run with multiple cPeaks. But only 6 hours has more.

```{r, echo=F}
GABA.params <- readRDS("data/uniqueSNP_GABA_params.rds")
GABA.params$convergence_plot
```

```{r}
library(RColorBrewer)
myPalette <- brewer.pal(4, "Set2") 
pie(GABA.params$group_pve, col=myPalette, border="white",
    main = paste0(sprintf("%.2f", 100-(GABA.params$group_pve[1]/GABA.params$total_pve)*100), "% PVE for GABA OCR clusters"))
```


