---
title: "ctwas analysis with caQTL and ASOC jointly"
author: "Lifan Liang"
date: "2024-08-29"
output: workflowr::wflow_html
editor_options:
  chunk_output_type: console
---

## Merging caQTL and ASOC

I got top caQTL per peak per condition, first I took the union of caQTLs FDR<5% within the same cell type and removed caQTLs with nominal P value > 0.1. The allelic specific open chromatin (ASOC) was processed the same way. Then these two sets were merged together. If an ASOC was within a peak with a caQTL. The one with smaller nominal P value would be included into the merged set.

```{r, echo=F}
conds <- paste0(rep(c(0,1,6),each=3),"hr_",c("GABA","npglut","nmglut"))
conds1 <- paste0(c(0,1,6),"hr_",rep(c("GABA","npglut","nmglut"),each=3))

N_asoc <- c(11297,5573,11370,18252,9259,22207,21118,10116,21846)
names(N_asoc) <- conds

N_asoc1 <- c(17428,23678,25789,8860,12376,12892,18466,27961,27897)
names(N_asoc1) <- conds

N_caqtl <- c(4318,8954,10504,1802,4435,6017,4957,10497,11622)
names(N_caqtl) <- conds1

N_caqtl1 <- c(13520,14082,14164,7475,7757,7835,15352,15895,15918)
names(N_caqtl1) <- conds1

uni <- c(27730,33110,34944,15186,18380,18824,30126,38137,38078)
names(uni) <- conds1

dat <- rbind(N_asoc[conds1],N_asoc1[conds1],N_caqtl,N_caqtl1,uni)
rownames(dat) <- c("ASOC","processed_ASOC",
                   "caQTL","processed_caQTL",
                   "Union")
knitr::kable(dat)
```

## Parameter extimation

```{r,echo=F}
library(ggplot2)
params <- readRDS("data/joint_caQTL_ASOC_SCZ_params.RDS")
```

![](assets/joint_caqtl_asoc_params.png)
### PVE

Overall, total PVE across all condition was 28.6%.

```{r, echo=F}
perc <- params$group_pve / params$total_pve
pie(perc)
```


```{r, echo=F}
barplot(perc[c(1,4,7,2,5,8,3,6,9)+1], las=2, col=c(rep("red",3), rep("yellow",3), rep("blue",3)))
```

### Enrichment

```{r,echo=F}
enri <- params$enrichment[c(1,4,7,2,5,8,3,6,9)]
barplot(enri, las=2, col=c(rep("red",3), rep("yellow",3), rep("blue",3)))
```

## Significant peaks

There are 59 peaks/ASOCs if the cutoff was set to PIP>0.5.

```{r, echo=F}
res <- readRDS("data/caQTL_ASOC_SCZ_noSNP_res.RDS")
res <- res[res$cs_index>0,]
cond <- names(table(res$type))
res$genename <- sapply(strsplit(res$id, "\\|"), function(x) {x[1]})
res2 <- aggregate(res$susie_pip, list(res$genename), sum)
res3 <- merge(res2, res[res$type==cond[1],c("genename","susie_pip")], 
                 by.x="Group.1", by.y="genename",all.x = T)
colnames(res3) <- c("genename","PIP_sum",paste0("PIP_",cond[1]))
for(i in 2:length(cond)) {
  res3 <- merge(res3, res[res$type==cond[i],c("genename","susie_pip")], 
                    by="genename",all.x = T)
  colnames(res3)[i+2] <- paste0("PIP_",cond[i])
}
res3[is.na(res3)] <- 0

hist(res3$PIP_sum[res3$PIP_sum>0.5], breaks=5:11/10, labels=T, ylim = c(0, 27),
     xlab="sum of PIP", main="# of peaks/ASOCs in different PIP intervals")
```

### PIP distribution across conditions

For these 19 peaks/ASOCs with PIP>0.8,

```{r,echo=F}
library(pheatmap)

res4 <- res3[res3$PIP_sum>0.8,]
rownames(res4) <- res4$genename
res4 <- res4[,c(3,6,9,4,7,10,5,8,11)]
colnames(res4) <- substr(colnames(res4),5,nchar(res4))
pheatmap(res4,cluster_rows = F,cluster_cols = F, color = colorRampPalette(c("white","red"))(100))
```

### Likely target genes

We haven't computed the ABC score. But I still tried three approaches to identify target genes: (1) check the eQTL with FDR<20%. Possible candidate genes are shown in the column "eqtl.gene"; (2) check the co-activation between peaks accessibility and gene expression computed by Zicheng. Cut off is FDR<20%. The column "p2g.gene" is showing the results; (3) Check which genes have overlapped with the peak, shown in the column "overlap.gene".

Co-activation and eQTL agreed with each other on two ASoCs. One is rs12741781 targeting CEP170 and SDCCAG8. Another is rs215759 targeting TCF20.

Please note that I don't have peak information for ASoCs. So I simply extended ASoCs by 250bps downstream and upstream as the peak.

```{r,echo=F,message=F}
library(dplyr)
res5 <- readRDS("data/SCZ_cTWAS_peak_target_gene_withABC_filterTSSgenes.rds")
res5[res5=="NA"] <- ""
res5 <- res5[res5$PIP_sum>0.5,]
res5[,c(1,13,15,2,14,16,18,19,20)] %>%
  DT::datatable() %>%
  DT::formatRound(columns=c('PIP_sum'), digits=3)
```

### Connection to eQTL cTWAS

I tried to make Miami plot to visualize regional overlap between caQTL cTWAS and the eQTL cTWAS. But the correlation is not obvious.The bottom plot is setting all PIPs to 0 if their credible set index is 0 and genes/peaks with PIP>50% were colored as red.

![](assets/ctwas_SCZ_eqtl_vs_caqtl.png)

<br>

To align the significant peaks to the significant genes in the cTWAS results, we check QTL distance between the two. Given that each peak/gene has 9 QTLs included at maximum, I used the QTL with the highest PIP for the corresponding peak/gene. There are 2 peaks close to a gene (distance<500K bps) in the cTWAS results. One SNP is shared by an ASOC and the gene ZNF823.

```{r,echo=F,message=F,warning=F}
library(GenomicRanges)
library(dplyr)

eres <- readRDS("data/SCZ_cover8_noSNP_ctwas.RDS")
eres <- eres[eres$cs_index>0,]
cond <- names(table(eres$type))
eres$genename <- sapply(strsplit(eres$id, "\\|"), function(x) {x[1]})
eres2 <- aggregate(eres$susie_pip, list(eres$genename), sum)
eres3 <- merge(eres2, eres[eres$type==cond[1],c("genename","susie_pip")], 
                 by.x="Group.1", by.y="genename",all.x = T)
colnames(eres3) <- c("genename","PIP_sum",paste0("PIP_",cond[1]))
for(i in 2:length(cond)) {
  eres3 <- merge(eres3, eres[eres$type==cond[i],c("genename","susie_pip")], 
                    by="genename",all.x = T)
  colnames(eres3)[i+2] <- paste0("PIP_",cond[i])
}
eres3[is.na(eres3)] <- 0
eres4 <- eres3[eres3$PIP_sum>0.8,]


max.cond <- colnames(eres4[3:11])[apply(as.matrix(eres4[,3:11]),1,which.max)]
max.cond <- sapply(strsplit(max.cond,"_"), function(x){paste0(x[2],"_",x[3])})
cand <- paste0(eres4$genename,"|",max.cond)
eqtl.range <- makeGRangesFromDataFrame(eres[eres$id %in% cand,c(1:3,8,10)], 
                                       seqnames.field = "chrom",
                                       start.field = "pos",
                                       end.field = "pos",
                                       keep.extra.columns = T)


max.cond <- colnames(res4)[apply(as.matrix(res4),1,which.max)]
cand <- paste0(rownames(res4),"|",max.cond)
peak.range <- makeGRangesFromDataFrame(res[res$id %in% cand,c(1:3,8,10)], 
                                       seqnames.field = "chrom",
                                       start.field = "pos",
                                       end.field = "pos",
                                       keep.extra.columns = T)

dist2eqtl <- distanceToNearest(peak.range, eqtl.range)
ctwas.match <- data.frame(peak=res[names(peak.range),"genename"], gene="---", distance=Inf)
ctwas.match[dist2eqtl@from,"gene"] <- eres[names(eqtl.range),"genename"][dist2eqtl@to]
ctwas.match[dist2eqtl@from,"gene_PIP"] <- eqtl.range$susie_pip[dist2eqtl@to]
ctwas.match[dist2eqtl@from,"distance"] <- dist2eqtl@elementMetadata$distance
dat1 <- ctwas.match[ctwas.match$gene_PIP>0.8,]
dat1[order(dat1$distance),] %>% 
  DT::datatable() %>%
  DT::formatRound(columns=c("gene_PIP"), digits=3)
```

Venn diagram below showed the overlap between cPeaks and genes with PIP>50%.

```{r,echo=F,message=F}
library(VennDiagram)
library(RColorBrewer)
myCol <- brewer.pal(3, "Pastel2")[c(1,2)]
peaks <- ctwas.match$peak
peaks[ctwas.match$distance<5e5] <- ctwas.match$gene[ctwas.match$distance<5e5]
venn.diagram(list(cPeaks=peaks,eQTL=eres4$genename),fill=myCol,alpha=0.8,
             filename="docs/assets/ctwas_peaks_vs_gene.png",
             imagetype = "png",lty="blank",cex=3.5)
```

![](assets/ctwas_peaks_vs_gene.png)