---
title: "cTWAS results: neuron stimulation eQTLs for Schizophrenia"
author: "Lifan Liang"
date: "2024-03-20"
output: workflowr::wflow_html
editor_options:
  chunk_output_type: console
---

## Weights

Use the same set of eQTLs as cell type sperated cTWAS.These eQTLs are the unions of top eQTL within the same cell type. But eQTLs with nominal P values less than 0.1 in a certain condition was removed.

```{r, echo=F}
library(dplyr)
ctwas.params <- readRDS("data/joint_celltype_cTWAS_params.rds")
ctwas.params$group_size[c(1,2,5,8,3,6,9,4,7,10)]
```

## Parameters

```{r, echo=F}
ctwas.params$convergence_plot
```

### Prior for each context

```{r, echo=F}
barplot(ctwas.params$group_prior[c(1,2,5,8,3,6,9,4,7,10)],las=2)
```

NMglut has higher priors than the other cell types. However, this is also the cell type with the least sample size for eQTL and hence the smallest number of eGenes.

```{r, echo=F}
samples.size <- c(94,97,99,83,92,87,97,99,100)
names(samples.size) <- names(ctwas.params$group_prior)[c(2,5,8,3,6,9,4,7,10)]
barplot(samples.size,las=2,main="sample size for eQTL")
```

### Enrichment 

```{r}
library(ggplot2)
dat <- data.frame(enrichment=ctwas.params$enrichment,
                  pve=ctwas.params$group_pve[-1]/ctwas.params$total_pve)
temp <- strsplit(rownames(dat),"_")
dat$celltype <- sapply(temp,function(x){x[2]})
dat$timepoint <- sapply(temp,function(x){x[1]})
ggplot(dat,aes(x=celltype,y=timepoint)) + 
  geom_count(aes(size=pve, color=enrichment)) + 
  theme_classic()
```


### PVE for each context

Total PVE from all eGenes in all contexts is 11.0%.

```{r, echo=F}
perc <- ctwas.params$group_pve / ctwas.params$total_pve
perc1 <- c(perc[1], sum(perc[c(2,5,8)]), sum(perc[c(3,6,9)]),
           sum(perc[c(4,7,10)]))
names(perc1) <- c("SNP","GABA","NMglut","NPglut")
pie(perc1)
```

```{r, echo=F}
barplot(perc[c(2,5,8,3,6,9,4,7,10)], las=2, col=c(rep("red",3), rep("yellow",3), rep("blue",3)))
```

### Tuning coverage in fine-mapping

"coverage" dictates how many SNPs are included into credible set. Lowering coverage results in smaller and more coherent credible sets that may pass the impurity threshold.

```{r, echo=F}
egenes <- c(25,45,60,60,89)
names(egenes) <- c("95%", "80%", "60%", "40%", "Ignore")
p <- barplot(egenes,xlab="coverage", ylab="# of eGenes", ylim=c(0,max(egenes)+5))
text(x = p, y = egenes + 2.5, labels = egenes)
```

## Results

There are 45 \]egenes if the cutoff was set to PIP>0.5.

```{r, echo=F}
res <- readRDS("data/SCZ_cover8_noSNP_ctwas.RDS")
res <- res[res$cs_index>0,]
cond <- names(table(res$type))
res$genename <- sapply(strsplit(res$id, "\\|"), function(x) {x[1]})
res2 <- aggregate(res$susie_pip, list(res$genename), sum)
res3 <- merge(res2, res[res$type==cond[1],c("genename","susie_pip")], 
                 by.x="Group.1", by.y="genename",all.x = T)
colnames(res3) <- c("genename","PIP_sum",paste0("PIP_",cond[1]))
for(i in 2:length(cond)) {
  res3 <- merge(res3, res[res$type==cond[i],c("genename","susie_pip")], 
                    by="genename",all.x = T)
  colnames(res3)[i+2] <- paste0("PIP_",cond[i])
}
res3[is.na(res3)] <- 0

hist(res3$PIP_sum[res3$PIP_sum>0.5], breaks=5:10/10, labels=T, ylim = c(0, 21),
     xlab="sum of PIP", main="# of eGenes in different PIP intervals")
```


### Comparing with GTEx

Compare the cTWAS results of 13 GTEx tissues with our results. Each GTEx tissue was analyzed with single group cTWAS ones. GTEx genes were selected with PIP > 0.5. Histogram below showed the number of overlapped genes with each GTEx brain tissue.

#### Shared with each GTEx brain tissue

```{r, echo=F}
library(VennDiagram)
#library(RColorBrewer)
sumstats <- readRDS("data/SCZ_ctwas_related_sumstats.RDS")
res$tag <- paste(res$region_tag1, res$region_tag2, sep="_")
res4 <- res3[res3$PIP_sum>0.5,]
Pgene <- merge(res[c("id","genename","tag")], sumstats$gwas,
               by.x="tag",by.y="Group.1",all.x=F)
Pgene <- aggregate(Pgene$p, list(Pgene$genename), min)
res4 <- merge(res4, Pgene, by.x="genename", by.y="Group.1", all.x=T)
colnames(res4)[ncol(res4)] <- "GWAS_Pval_loci"

maxqtl <- aggregate(sumstats$eqtl$pvalue,list(sumstats$eqtl$gene),min)
res4 <- merge(res4, maxqtl, by.x="genename", by.y="Group.1")
colnames(res4)[ncol(res4)] <- "eQTL_Pval"

maxzgene <- aggregate(sumstats$zgene$p, list(sumstats$zgene$genename), min)
res4 <- merge(res4, maxzgene, by.x="genename", by.y="Group.1")
colnames(res4)[ncol(res4)] <- "Zgene_Pval"

GTEx.res <- readRDS("data/SCZ_GTEx_brain_cTWAS_geneSymbol_res.rds")
GTEx.gene <- lapply(GTEx.res, function(x){x$hgnc_symbol[x$susie_pip>0.5]})
gtex.union <- unique(unlist(GTEx.gene))
GTEx.gene$Brain_union <- gtex.union
shared <- sapply(GTEx.gene, function(x) {sum(x %in% res4$genename)})
names(shared) <- substr(names(shared), 7, nchar(names(shared)))
shared <- shared[order(shared, decreasing = T)]
x <- barplot(shared, las=2, xaxt="n")
text(x,par("usr")[3],names(shared), xpd=TRUE, srt=45,adj = c(1.1,1.1), xpd = TRUE, cex=0.6)
```

#### Total cTWAS significant genes within each GTEx brain tissue
```{r,echo=F}
gtex.total <- sapply(GTEx.gene, length)
names(gtex.total) <- substr(names(gtex.total), 7, nchar(names(gtex.total)))
x <- barplot(gtex.total, las=2, xaxt="n")
text(x,par("usr")[3],names(gtex.total), xpd=TRUE, srt=45,adj = c(1.1,1.1), xpd = TRUE, cex=0.6)
```


#### Venn diagram between our cTWAS genes and GTEx brain tissues
Venn diagram showed the intersection of our results and the union of all GTEx brain genes.

```{r,echo=F}
library(VennDiagram)
library(RColorBrewer)
myCol <- brewer.pal(3, "Pastel2")
venn.diagram(list(GTEX_union=gtex.union,SCZ=res4$genename[res4$PIP_sum>0.8],
                  GTEX_prefrontal_cortex=GTEx.gene[["Brain_Frontal_Cortex_BA9"]]),
             lwd = 2, fill = myCol,
             cex = 2, fontface = "bold",fontfamily = "sans",
             imagetype = "tiff", filename = "docs/assets/GTEx_vs_SCZ.tiff")

```


![](assets/GTEx_vs_SCZ.tiff)

#### The shared 8 genes PIP in our cTWAS results

```{r,echo=F}
knitr::kable(res4[res4$genename %in% gtex.union,])
```



### Enrichment analysis regardless of cell types

#### Enrichment analysis of all genes together

```{r, echo=F}
library(enrichR)

enrich.total <- enrichr(res4$genename, "GO_Biological_Process_2023")
head(enrich.total$GO_Biological_Process_2023[-(5:8)],10) %>%
  DT::datatable() %>%
  DT::formatRound(columns=c('P.value', 'Adjusted.P.value'), digits=3)
```


#### Enrichment analysis of dynamic genes

PIPs in time points `6hr` and `1hr` are summed up and then subtracted the time point `0hr`. If the difference is larger than 0.5, genes are called dynamic.

```{r, echo=F}
res4$PIP_diff <- rowSums(res4[,grepl("1hr", colnames(res4)) | grepl("6hr", colnames(res4))]) - 
  rowSums(res4[,grepl("0hr", colnames(res4))])
enrich.dynamic <- enrichr(res4$genename[res4$PIP_diff>0.5], "GO_Biological_Process_2018")
head(enrich.dynamic$GO_Biological_Process_2018[-(5:8)],10) %>%
  DT::datatable() %>%
  DT::formatRound(columns=c('P.value', 'Adjusted.P.value'), digits=3)
```

#### Enrichment analysis of static genes

Static genes are significant genes that are not dynamic genes.

```{r, echo=F}
enrich.static <- enrichr(res4$genename[res4$PIP_diff<=0.5], "GO_Biological_Process_2018")
head(enrich.static$GO_Biological_Process_2018[-(5:8)],10) %>%
  DT::datatable() %>%
  DT::formatRound(columns=c('P.value', 'Adjusted.P.value'), digits=3)
```


### Cell type specific eGenes

An eGene is assigned to a cell type if over 60% of the PIP sum are from that cell type. Genes with higher sum of PIP are more likely to be cell type specific.

```{r, echo=F}
res4$GABA_sum <- rowSums(res4[,c(3,6,9)])
res4$nmglut_sum <- rowSums(res4[,c(4,7,10)])
res4$npglut_sum <- rowSums(res4[,c(5,8,11)])
ctperc <- res4[c("GABA_sum","nmglut_sum","npglut_sum")] / res4$PIP_sum
res4$celltype <- "Unspecific"
ind <- which(ctperc>0.6, arr.ind=1)
res4[ind[,1],"celltype"] <- substr(colnames(ctperc),1,nchar(colnames(ctperc))-4)[ind[,2]]
table(res4$celltype)
```

```{r,echo=F}
celldist <- table(res4$celltype)
names(celldist) <- paste(names(celldist),":",celldist)
pie(celldist)
```


Heatmap shows most the significant genes are both cell type specific and time point specific.

```{r,echo=F}
library(pheatmap)
library(dichromat)


rownames(res4) <- res4$genename
res40 <- res4
temp <- strsplit(colnames(res40)[3:11],"_")
colnames(res40)[3:11] <- paste0(sapply(temp,function(x){x[3]}), "_",sapply(temp,function(x){x[2]}))
pheatmap(res40[res40$PIP_sum>0.8,c(3,6,9,4,7,10,5,8,11)],cluster_rows=F, cluster_cols=F,
         color = colorRampPalette(c("white","red"))(100))
```



#### Cell type composition across PIP distribution

```{r, echo=F}
library(ggplot2)

ggplot(res4[c("PIP_sum","celltype")], aes(PIP_sum, fill=celltype)) + geom_histogram(binwidth = 0.1, center=0.55) + theme_bw()
```

#### Cell type composition of GTEx shared genes

Since GTEx is bulk eQTL, the cell type contribution of the shared genes for SCZ can only be elucidated by our joint condition cTWAS results.

```{r,echo=F}
par(mfrow=c(1,2))
shared <- res4[res4$genename %in% gtex.union,]
notshared <- res4[!(res4$genename %in% gtex.union),]
pie(table(shared$celltype), main="GTEx shared genes",col = c("white","blue","red"))
pie(table(notshared$celltype), main="Genes not shared with GTEx", col=c("green","white","blue","red"))
```

##### Distribution of shared genes across cell types
```{r,echo=F}
table(shared$celltype)
```

##### Distribution of unique genes across cell types
```{r,echo=F}
table(notshared$celltype)
```

### Time point specific genes

```{r,echo=F}
res4$PIP_0hr <- rowSums(res4[,c(3,6,9)])
res4$PIP_1hr <- rowSums(res4[,c(4,7,10)])
res4$PIP_6hr <- rowSums(res4[,c(5,8,11)])
tpperc <- res4[c("PIP_0hr","PIP_1hr","PIP_6hr")] / res4$PIP_sum
res4$timepoint <- "Unspecific"
ind <- which(tpperc>0.6, arr.ind=1)
res4[ind[,1],"timepoint"] <- substr(colnames(tpperc),1,nchar(colnames(ctperc))-4)[ind[,2]]
table(res4$timepoint)
```

#### Time point composition of gene PIPs

Since GTEx is bulk eQTL, the cell type contribution of the shared genes for SCZ can only be elucidated by our joint condition cTWAS results.

```{r, echo=F}
library(ggplot2)

ggplot(res4[c("PIP_sum","timepoint")], aes(PIP_sum, fill=timepoint)) + geom_histogram(binwidth = 0.1, center=0.55) + theme_bw()
```

### Dynamic eGenes

For each cell type, we computed `PIP_diff` as in previous cTWAS results. `PIP_diff` > 0.5 in a certain cell type was regarded as Dynamic eGenes. None of the unspecific eGenes are dynamic.

#### Dynamic eGenes in GABA

GABA has 7 eGenes, 5 of them are dynamic.

```{r, echo=F}
library(dplyr)
res4[c("GABA_PIP_diff","nmglut_PIP_diff","npglut_PIP_diff")] <- res4[,6:8] + res4[9:11] - res4[3:5]
#saveRDS(res4,"data/processed_SCZ_res4.rds")
res4.GABA <- res4[res4$celltype=="GABA",c(1,2,3,6,9,12:14,16,24)]
#res4.GABA$GABA_PIP_diff <- res4.GABA$PIP_6hr_GABA + res4.GABA$PIP_1hr_GABA - res4.GABA$PIP_0hr_GABA
res4.GABA[order(res4.GABA$GABA_PIP_diff, decreasing = T),] %>%
  DT::datatable() %>%
  DT::formatRound(columns=colnames(res4.GABA)[-1], digits=3)
```

##### Enrichment analysis for GABA specific genes

```{r, message=F, echo=F}
library(enrichR)

enrich.GABA <- enrichr(res4.GABA$genename, "GO_Biological_Process_2023")
head(enrich.GABA$GO_Biological_Process_2023[-(5:8)],10) %>%
  DT::datatable() %>%
  DT::formatRound(columns=c('P.value', 'Adjusted.P.value'), digits=3)
```

##### Enrichment analysis for GABA dynamic genes

```{r, message=F, echo=F}
enrich.GABA1 <- enrichr(res4.GABA$genename[res4.GABA$GABA_PIP_diff>0.5], "GO_Biological_Process_2023")
head(enrich.GABA1$GO_Biological_Process_2023[-(5:8)],10) %>%
  DT::datatable() %>%
  DT::formatRound(columns=c('P.value', 'Adjusted.P.value'), digits=3)
```

![TWAS locus plot for KCNA4](assets/cTWAS_allcell_KCNA4.png)


![TWAS locus plot for NLRC5](assets/cTWAS_allcell_NLRC5.png)


#### Dynamic eGenes in NMglut

nmglut has 6 eGenes, all of them are dynamic.

```{r, echo=F}
res4.nmglut <- res4[res4$celltype=="nmglut",c(1,2,4,7,10,12:14,17,25)]
res4.nmglut[order(res4.nmglut$nmglut_PIP_diff, decreasing = T),] %>%
  DT::datatable() %>%
  DT::formatRound(columns=colnames(res4.nmglut)[-1], digits=3)
```

##### Enrichment analysis for NMglut specific genes

```{r, message=F, echo=F}
enrich.nmglut <- enrichr(res4.nmglut$genename, "GO_Biological_Process_2018")
head(enrich.nmglut$GO_Biological_Process_2018[-(5:8)],10) %>%
  DT::datatable() %>%
  DT::formatRound(columns=c('P.value', 'Adjusted.P.value'), digits=3)
```

##### Enrichment analysis for NMglut dynamic genes

```{r, message=F, echo=F}
enrich.nmglut1 <- enrichr(res4.nmglut$genename[res4.nmglut$nmglut_PIP_diff>0.5], "GO_Biological_Process_2018")
head(enrich.nmglut1$GO_Biological_Process_2018[-(5:8)],10) %>%
  DT::datatable() %>%
  DT::formatRound(columns=c('P.value', 'Adjusted.P.value'), digits=3)
```

![TWAS locus plot for NGEF](assets/cTWAS_allcell_NGEF.png)

![TWAS locus plot for FZD4](assets/cTWAS_allcell_FZD4.png)

#### Dynamic eGenes in NPglut

npglut has 5 eGenes, 4 of them are dynamic

```{r, echo=F}
res4.npglut <- res4[res4$celltype=="npglut",c(1,2,5,8,11,12:14,18,26)]
res4.npglut[order(res4.npglut$npglut_PIP_diff, decreasing = T),] %>%
  DT::datatable() %>%
  DT::formatRound(columns=colnames(res4.npglut)[-1], digits=3)
```

##### Enrichment analysis for NPglut specific genes

```{r, message=F, echo=F}
enrich.npglut <- enrichr(res4.npglut$genename, "GO_Biological_Process_2018")
head(enrich.npglut$GO_Biological_Process_2018[-(5:8)],10) %>%
  DT::datatable() %>%
  DT::formatRound(columns=c('P.value', 'Adjusted.P.value'), digits=3)
```

##### Enrichment analysis for NPglut dynamic genes

```{r, message=F, echo=F}
enrich.npglut1 <- enrichr(res4.npglut$genename[res4.npglut$npglut_PIP_diff>0.5], "GO_Biological_Process_2018")
head(enrich.npglut1$GO_Biological_Process_2018[-(5:8)],10) %>%
  DT::datatable() %>%
  DT::formatRound(columns=c('P.value', 'Adjusted.P.value'), digits=3)
```

![cTWAS locus plot for NAGA](assets/cTWAS_allcell_NAGA.png)

![TWAS locus plot for THBS3](assets/cTWAS_allcell_THBS3.png)


