---
title: "eQTL_glimix"
author: "Lifan Liang"
date: "2024-03-21"
output: workflowr::wflow_html
editor_options:
  chunk_output_type: console
---

### Introduction

This page introduces how to adapt a python package, "glimix_core", to fit the individual random effect shared across 9 conditions (3 timepoints by 3 cell types).

### Individual random effect shared across all contexts

#### Null model

First, gene expression PC is computed from residual expression after regressing out other covariates. Then gene expression was regressed out of all covariates (e.g. sex, age, disease status, 5 genotype PCs, 15 expression PCs) jointly across conditions. The coefficients of covariates were estimated together with the random effects. In this way, using residual expression to estimate model likelihood and individual random effects from the null model is identical to using all covariates. Regressing out all covariates speeds up the genotype scanning by around 100 times. However, using residual gene expression may underestimate genetic effects in the alternative model.

$$
y = C\mu + \epsilon 
$$

where $X$ is a vector of 1 since all other covariates have been regressed out beforehand, $\mu$ is the intercept term, and $\epsilon$ was sampled from a multivariate Normal with forms below:

$$
\epsilon \sim MVN(0, S(v_0 \cdot Z + v_1 \cdot I))
$$

where $Z$ is the design matrix such that samples within the same individual have correlation of 1 and samples for different individuals are 0, $I$ is the identity matrix to capture i.i.d noise, $S$ is the overall scaling coefficient, $v_0$ and $v_1$ captures the relative weights of individual random effects and noise. There are 6822 genes with individual variation more than 1% of overall error.

The null model provides baseline model likelihood and an estimate of $v_0$ and $v_1$.

![distribution of $log10(\dfrac{v_0}{v_0+v_1})$](assets/log_glimix_random_effect.png)

![distribution of $\dfrac{v_0}{v_0+v_1}$](assets/glimix_random_effect.png)

#### Alternative model

We estimate the persistent genetic effects with following model. All parameters will be re-estimated except $v_0$ and $v_1$ in the distribution of $\epsilon$.

$$
y = G\beta + X\mu+ \epsilon
$$

#### Likelihood ratio testing

The alternative model provides the alternative model likelihood and an estimate of $\beta$ and its standard error. Since the log likelihood ratio follows Chi-square distribution asymptotically, we can compute the P values for genetic effects:

$$
\lambda_{LR} = 2(\ell_{alt} - \ell_{null}) \sim Chisq(1)
$$

#### P value calibration

We permuted the genotypes such that gene expression in different conditions belonging to the same individual still belong to the same individual. For example, during permutation, gene expression for CD02 in all time points and cell tyeps will be linked to the genotypes of CD04.

We permuted genotyeps for 500 genes with lowest individual random effects, genotypes were permuted 100 times for each gene, P values are well calibrated:

![P values with permuted genotyeps for genes with low individual random effects](assets/P_calibration_low_randeff.png)

We also performed the permutation for 1000 genes with highest individual random effects, P values seems slightly inflated.

![P values with permuted genotyeps for genes with high individual random effects](assets/P_calibration_high_randeff.png)

#### Gene level FDR control

We followed the the procedure described in FastQTL. Using the same permutation above, we obtain 1000 mininum P values from 1000 permutation with each gene. One thing to be aware is that all cis genotypes were shuffled together so that the LD structure remains. And then we fit a Beta distribution to the 1000 minimum P values with MLE. The adjusted P value would be the cumulative probability of the empirical minimum P value in the Beta distribution.

![Example of Beta distribution fitting](assets/glimix_beta_fitting.png)
The number of eGenes idenitified by mixed effect is much smaller than that by aggregating conditions together and run fixed effects.

![eGene overlap between mixed effect eQTL and fixed effect eQTL.](assets/venn_limix_vs_fixed.png)

### Estimating individual random effects with lead variant

Given the lead variants for each gene from the eQTL results above, we first estimated fixed effects of covariates and individual random effects with the following model:

$$
y = G\beta + X\mu+ \epsilon
$$
where $G$ is the genotype of the corresponding lead variant. $X$ includes all the covariates and the offset, $\mu$ is the fixed effects, and $\epsilon$ follows the same definition as in the section above.

Then we regressed out covariates based on the fixed effect estimated and the model and fixed $v_0$ and $v_1$ estimated from the model. This time, we observed individual random effect reduced a lot compared to noise.

![](assets/indiv_randeff_leadSNP.png)

#### Association testing

Null model, alternative model, and likelihood ratio testing are identical to previous precedure except the the ratio of the random effects are fixed beforehand. We continue to use Beta approximation to obtain adjusted P values for each gene.

Below is the scatter plot of the adjusted P values with Beta approximation (Y axis) and empirical P values (X axis) on the scale of %-log10$. Although a little too stringent, Beta approximation is quite accurate at the tail.

![](assets/beta_calibration.png)

Instead of Storey's Pi in the previous approach, we performed Benjamini-Hochberg for FDR control. Genes with FDR < 0.05 is considered eGenes. Overall we identified 8462 eGenes, around 700 eGenes more than the bulk approach.

![](assets/venn_limix_vs_fixed (1).png)

The scatter plot of P values shows linear mixed model has higher power than bulk.

![](assets/bulk_vs_leadSNP_scatter.png)

### Comparing with fixed effect model

We used the same normalized expression and covariates as input for matrixeQTL, so as to investigate the role of individual random effects. It seems individual random effect is competing with genetic effects. The increased power compared to bulk is due to the increased sample size by stacking all conditions.

![](assets/IRE_egenes_venn.png)

For genes with less than 1% individual random effects. Mixed effect model and fixed effect model have highly correlated nominal P values. 

![](assets/noIRE_leadSNP_vs_mateqtl.png)

For genes with more than 1% individual random effects. Mixed effect model has lower nominal P values than fixed effect model. It seems by pretending we have 824 independent samples, power would be greatly increased. However, correlattion caused by repeated measurement of the same individuals would confound genetic effects. Although most repeated measurement correlation are already captured by genotype PCs and expression PCs.

![](assets/IRE_leadSNP_vs_mateqtl.png)

Another minor issue could be that permutation in matrixeQTL did not preserve individual structrure. Hence FDR is not stringent enough.

![](assets/noIRE_FDR_leadSNP_vs_mateqtl.png)

