---
title: "eQTL_glimix"
author: "Lifan Liang"
date: "2024-03-21"
output: workflowr::wflow_html
editor_options:
  chunk_output_type: console
---

### Introduction

This page introduces how to adapt a python package, "glimix_core", to fit the individual random effect shared across 9 conditions (3 timepoints by 3 cell types).

### Individual random effect shared across all contexts

#### Null model

First, gene expression PC is computed from residual expression after regressing out other covariates. Then gene expression was regressed out of all covariates (e.g. sex, age, disease status, 5 genotype PCs, 15 expression PCs) jointly across conditions. The coefficients of covariates were estimated together with the random effects. In this way, using residual expression to estimate model likelihood and individual random effects from the null model is identical to using all covariates. Regressing out all covariates speeds up the genotype scanning by around 100 times. However, using residual gene expression may underestimate genetic effects in the alternative model.

$$
y = X\mu + \epsilon 
$$

where $X$ is 15 expression PCs and a vector of 1 since all other covariates have been regressed out beforehand, $\mu$ indicates coefficients for the expression PCs plus the intercept term, and $\epsilon$ was sampled from a multivariate Normal with forms below:

$$
\epsilon \sim MVN(0, S(v_0 \cdot Z + v_1 \cdot I))
$$

where $Z$ is the design matrix such that samples within the same individual have correlation of 1 and samples for different individuals are 0, $I$ is the identity matrix to capture i.i.d noise, $S$ is the overall scaling coefficient, $v_0$ and $v_1$ captures the relative weights of individual random effects and noise. There are 6822 genes with individual variation more than 1% of overall error.

The null model provides baseline model likelihood and an estimate of $v_0$ and $v_1$.

![distribution of $log10(\dfrac{v_0}{v_0+v_1})$](assets/log_glimix_random_effect.png)

![distribution of $\dfrac{v_0}{v_0+v_1}$](assets/glimix_random_effect.png)

#### Alternative model

We estimate the persistent genetic effects with following model. All parameters will be re-estimated except $v_0$ and $v_1$ in the distribution of $\epsilon$.

$$
y = G\beta + X\mu+ \epsilon
$$

#### Likelihood ratio testing

The alternative model provides the alternative model likelihood and an estimate of $\beta$ and its standard error. Since the log likelihood ratio follows Chi-square distribution asymptotically, we can compute the P values for genetic effects:

$$
\lambda_{LR} = 2(\ell_{alt} - \ell_{null}) \sim Chisq(1)
$$

#### P value calibration

We permuted the genotypes such that gene expression in different conditions belonging to the same individual still belong to the same individual. For example, during permutation, gene expression for CD02 in all time points and cell tyeps will be linked to the genotypes of CD04.

We permuted genotyeps for 500 genes with lowest individual random effects, genotypes were permuted 100 times for each gene, P values are well calibrated:

![P values with permuted genotyeps for genes with low individual random effects](assets/P_calibration_low_randeff.png)

We also performed the permutation for 1000 genes with highest individual random effects, P values seems slightly inflated.

![P values with permuted genotyeps for genes with high individual random effects](assets/P_calibration_high_randeff.png)

#### Gene level FDR control

We followed the the procedure described in FastQTL. Using the same permutation above, we obtain 1000 mininum P values from 1000 permutation with each gene. One thing to be aware is that all cis genotypes were shuffled together so that the LD structure remains. And then we fit a Beta distribution to the 1000 minimum P values with MLE. The adjusted P value would be the cumulative probability of the empirical minimum P value in the Beta distribution.

![Example of Beta distribution fitting](assets/glimix_beta_fitting.png)
The number of eGenes idenitified by mixed effect is much smaller than that by aggregating conditions together and run fixed effects.

![eGene overlap between mixed effect eQTL and fixed effect eQTL.](assets/venn_limix_vs_fixed.png)


