---
title: "revision"
author: "Lifan Liang"
date: "2025-06-19"
output: workflowr::wflow_html
editor_options:
  chunk_output_type: console
---

## Power comparison between resting states and stimulating states

We took 79 cell lines that have relatively high number of cells across all 9 contexts to rerun eQTL mapping (MatrixeQTL). Below is the count of eGenes for the three cell types in GABA.

```{r,echo=F}
# Optional: names for the bars
values <- c(749, 776, 786)
names <- c("0 hr", "1 hr", "6 hr")

# Create the bar plot
bar_positions <- barplot(values,
        names.arg = names,
        col = "skyblue",
        main = "# of eGenes in GABA",
        xlab = "Time points",
        ylab = "# of eGenes",
        ylim = c(0, max(values) * 1.1))

# Add text labels on top of bars
text(x = bar_positions, y = values, labels = values, pos = 3, cex = 1, font = 2)
```



## rb estimate

rb is essentially estimating pairwise correlation of genetic effect between two conditions after correcting residual correlation. We implemented the procedure following the description by Qi, et al (PMID: 29891976). Number in the bracket is standard error.

### eQTL correlation within our gene expression

```{r, echo=F}
library(pheatmap)

est <- read.table("data/rb_est/neuron_stim_rb_ref_cortex.txt")
se <- read.table("data/rb_est/neuron_stim_rb_se_ref_cortex.txt")
est1 <- est
est1[] <- lapply(est, sprintf, fmt="%.3f")
se1 <- se
se1[] <- lapply(se, sprintf, fmt="%.3f")
values <- data.frame(mapply(paste0, est1, "\n (",se1, ")"))
idx <- c(1,4,7,2,5,8,3,6,9)
est1 <- est[idx,idx]
pheatmap(est1, cluster_rows = F, cluster_cols = F, angle_col = 315, color=colorRampPalette(c("white", "red"))(100), display_numbers = values)
```


### eQTL correlation with GTEx brain tissues

```{r, echo=F}
est <- readRDS("data/rb_est/rb_ns_vs_gtex.rds")
idx <- order(colMeans(est$rb),decreasing = T)
row_idx <- c(1,4,7,2,5,8,3,6,9)
est1 <- as.data.frame(est$rb[row_idx,idx])
est1[] <- lapply(est1, sprintf, fmt="%.3f")
se1 <- as.data.frame(est$rb.se[row_idx,idx])
se1[] <- lapply(se1, sprintf, fmt="%.3f")
values <- data.frame(mapply(paste0, est1, "\n (",se1, ")"))
pheatmap(est$rb[row_idx,idx], cluster_rows = F, cluster_cols = F, angle_col = 315, color=colorRampPalette(c("white", "red"))(100), display_numbers = values)
```


### eQTL correlation with psychENCODE2 prenatal brain

```{r, echo=F}
est <- readRDS("data/rb_est/rb_ns_vs_pec2.RDS")
est1 <- as.data.frame(est$rb)
est1[] <- lapply(est1, sprintf, fmt="%.3f")
se1 <- as.data.frame(est$se)
se1[] <- lapply(se1, sprintf, fmt="%.3f")
values <- data.frame(mapply(paste0, est1, "\n (",se1, ")"))
pheatmap(est$rb, cluster_rows = F, cluster_cols = F, angle_col = 315, color=colorRampPalette(c("white", "red"))(100), display_numbers = values)
```


### eQTL correlation with doparminergic neurons

```{r,echo=F}
est <- read.table("data/rb_est/neuron_stim_vs_jerber_dopar.txt")
se <- read.table("data/rb_est/neuron_stim_vs_jerber_dopar_se.txt")
est1 <- est
est1[] <- lapply(est, sprintf, fmt="%.3f")
se1 <- se
se1[] <- lapply(se, sprintf, fmt="%.3f")
values <- data.frame(mapply(paste0, est1, "\n (",se1, ")"))
idx <- c(1,4,7,2,5,8,3,6,9)
pheatmap(est, cluster_rows = F, cluster_cols = F, angle_col = 315, color=colorRampPalette(c("white", "red"))(100), display_numbers = values)
```

## rb estimate and Pi1 estimate with Bryois, et al.

Bryois, et al has combined the Roche datasets and the ROSMAP dataset to map single cell eQTL for nine cell types.

Pericytes has less than 30 cis-eQTLs and Endothelial cells only have 100 cis-eQTLs. rb estimate is probably not reliable. We removed these two cell types.

```{r,echo=F}
library(pheatmap)
est <- read.table("data/rb_est_bryois.txt")
est <- est[c(1,4,7,2,5,8,3,6,9),c(4,3,8,7,6,1,5)] # remove Peri and endo and sort according to mean rb
est1 <- est
est1[] <- lapply(est, sprintf, fmt="%.3f")
pheatmap(est, cluster_rows = F, cluster_cols = F, angle_col = 315, display_numbers = est1)
```

We have ascertained the nominal P values of our eQTLs (FDR<5%) in Bryois eQTL for each cell type. The non-null proportion of the P values (Pi1) were shown as below. 

```{r,echo=F}
pi1 <- read.table("data/pi1_ns_bryois.txt")
pi1 <- pi1[c(1,4,7,2,5,8,3,6,9),c(4,3,8,7,6,1,5)] # remove Peri and endo and sort according to mean rb
pi2 <- pi1
pi2[] <- lapply(pi1, sprintf, fmt="%.3f")
pheatmap(pi1, cluster_rows = F, cluster_cols = F, angle_col = 315, display_numbers = pi2)
```



## MASH results

We extracted top QTLs for each molecular trait (i.e. gene or peak) with FDR < 20% for each condition and took a union. And then we did LD clumping with the union of eQTL within a molecular trait. QTLs are ranked by nominal P values in 6hr NPGlut for both caQTL and eQTL. $R^2$ cutoff is 0.1. After LD clumping, we have 75149 eQTLs for 14235 genes and 

### eQTL results

```{r,echo=F}
library(mashr)

res.e <- readRDS("../Documents/neuron_stim/eqtl_mash_results.rds")
```

#### Pairwise sharing

We used the default method in MASH. For each pair of tissues, first identify the effects that are significant (by lfsr<lfsr_thresh) in at least one of the two tissues. Then compute what fraction of these have an estimated (posterior mean) effect size within a factor `factor` of one another. Given the sharing matrix, we perform hierarchical clustering with Ward criteria. 

```{r}
sharing.e <- get_pairwise_sharing(res.e)
values <- as.data.frame(sharing.e)
values[] <- lapply(values, sprintf, fmt="%.3f")
pheatmap(sharing.e, display_numbers = values,
         color=colorRampPalette(c("white", "red"))(100),
         clustering_method = "ward.D2")
```


Most eQTLs are condition specific. eQTLs shared by two conditions and all conditions are the second highest.

```{r, echo=F, caption="Histogram showing the number eQTLs with lfsr<5% across conditions"}
library(ggplot2)

freqs <- as.data.frame(table(rowSums(res.e$result$lfsr<0.05)))
ggplot(freqs, aes(x=Var1, y=Freq)) + geom_bar(stat="identity") + theme_classic() + 
  geom_text(aes(label = Freq), vjust = -0.5) + ylab("# of eQTL") + xlab("# of conditions")

```

The upset plot showed similar information. eQTLs are distributed into these catetegories in order: (1) Shared by all; (2) condition specific; (3) GABA-specific eQTL; (4) NPglut stimulation specific; (5) GABA stimulation specific; (6) Glut specific.

```{r, echo=F}
library(ComplexUpset)

dat <- as.data.frame(res.e$result$lfsr < 0.05)
upset(dat, colnames(dat), name="condition", width_ratio = 0.1, min_size=600)

```

We also obtained the mixture proportions of all covariance patterns. Below is a barplot showing all the patterns with proportion > 1%.

```{r, echo=F}
pi.e <- sort(get_estimated_pi(res.e),decreasing=F)
pi.e <- pi.e[pi.e>0.01]
barplot(pi.e, horiz = T, las=2)
```

The pattern that explains 22.5% of mixture was `PCA_1`. The covariance matrix was shown as below:

```{r, echo=F}
pheatmap(res.e$fitted_g$Ulist$PCA_1)
```

And the corresponding 1st eigen vector (52.7% PVE) is:

```{r, echo=F}
eigen1 <- eigen(res.e$fitted_g$Ulist$PCA_1)$vectors[,1]
names(eigen1) <- colnames(dat)
barplot(abs(eigen1)[c(1,4,7,2,5,8,3,6,9)], las=2)
```

#### Number of eGenes from MASH

```{r, echo=F}
library(dplyr)
lfsr <- as.data.frame(res.e$result$lfsr)
lfsr$gene <- sapply(strsplit(rownames(lfsr),"\\|"), function(x){x[1]})

result <- lfsr %>%
  group_by(gene) %>%
  summarise(across(starts_with(c("0hr", "1hr", "6hr")), ~ sum(.x < 0.05), .names = "count_{.col}"))

barplot(colSums(result[c(2,5,8,3,6,9,4,7,10)]>0),las=2,col=rep(1:3,each=3))
```



### caQTL results

Pairwise sharing among conditions:

```{r,echo=F}
res.c <- readRDS("../Documents/neuron_stim/caqtl_mash_results.rds")

sharing.c <- get_pairwise_sharing(res.c)
values <- as.data.frame(sharing.c)
values[] <- lapply(values, sprintf, fmt="%.3f")
pheatmap(sharing.c, display_numbers = values,
         color=colorRampPalette(c("white", "red"))(100),
         clustering_method = "ward.D2")
```

Distribution of caQTL across the number of conditions they are significant in. The distribution is similar to the one in eQTL.

```{r, echo=F}
freqs <- as.data.frame(table(rowSums(res.e$result$lfsr<0.05)))
ggplot(freqs, aes(x=Var1, y=Freq)) + geom_bar(stat="identity") + theme_classic() + 
  geom_text(aes(label = Freq), vjust = -0.5) + ylab("# of eQTL") + xlab("# of conditions")
```

Distribution of caQTLs across different intersections, consisting of condition-specific caQTLs, shared by all, and cell type-specifc caQTLs.

```{r, echo=F}
library(ComplexUpset)

dat <- as.data.frame(res.c$result$lfsr < 0.05)
upset(dat, colnames(dat), name="condition", width_ratio = 0.1, min_size=500)

```


```{r, echo=F}
pi.c <- sort(get_estimated_pi(res.c),decreasing=F)
pi.c <- pi.c[pi.c>0.01]
barplot(pi.c, horiz = T, las=2)
```

`tPCA` covariance matrix has the highest mixture proportion (16.8%).

```{r, echo=F}
pheatmap(res.c$fitted_g$Ulist$tPCA)
```

The eigen vector for this covariance matrix (72.5% PVE).

```{r, echo=F}
eigen.c <- eigen(res.c$fitted_g$Ulist$tPCA)$vectors[,1]
names(eigen.c) <- colnames(dat)
barplot(abs(eigen.c)[c(1,4,7,2,5,8,3,6,9)], las=2)
```


```{r, echo=F}
eigen.c <- eigen(res.c$fitted_g$Ulist$PCA_1)$vectors[,1]
names(eigen.c) <- colnames(dat)
barplot(abs(eigen.c)[c(1,4,7,2,5,8,3,6,9)], las=2)
```

#### Number of cPeaks from MASH

```{r, echo=F}
library(dplyr)
lfsr <- as.data.frame(res.c$result$lfsr)
lfsr$gene <- sapply(strsplit(rownames(lfsr),"\\|"), function(x){x[1]})

result <- lfsr %>%
  group_by(gene) %>%
  summarise(across(starts_with(c("0hr", "1hr", "6hr")), ~ sum(.x < 0.05), .names = "count_{.col}"))

dat <- data.frame(count=colSums(result[c(2,5,8,3,6,9,4,7,10)]>0),
                  celltype=rep(c("GABA","nmglut","npglut"),each=3),
                  timepoint=rep(c("0hr","1hr","6hr"),3))
ggplot(dat, aes(y=count,x=celltype, fill=timepoint)) + 
  geom_bar(stat = "identity", position = "dodge") + 
  theme_classic() + ylab("Number of cPeaks")
#barplot(dat$count,las=2,col=rep(1:3,each=3))
```


## Dynamic eQTL testing

### Dynamic eQTL testing by CellRegMap
#### Nominal P values for highlighted dynamic eQTL

* NPglut: ADAM10 | rs6494026    P value: 4.139992e-13
* GABA: CROT | rs13233308       P value: 0.295749
* NMglut: CPT1C | rs12104272    P value: 0.2200463

P values for CROT and CPT1C (nominated by cTWAS) do not have significant P values. But there is a SNP in weak LD that is significant for nominal values for GABA CROT (rs6946379) and CPT1C

#### Dynamic eQTL expanded to FDR<20% in marginal association testing

We expanded the scope of eQTL testing to FDR<20% with LD pruning ($R^2<0.1$). So that multiple testing burden is reduced. P values of eQTLs within a gene were corrected with Bonferroni FWER. Benjamini Hochberg FDR was applied across genes. Overall, we found 630, 953, and 1110 dynamic eQTLs in nmglut, GABA, and npglut respectively. Dynamic eQTLs are less enriched at TSS compared to eQTLs not significant for dynamic testing.

```{r, echo=F}
nmglut <- readRDS("data/nmglut_crm_fdr_loc.rds")
ggplot(nmglut[nmglut$tss_dist<2.5e5,], aes(x=tss_dist, colour=significant)) + 
  geom_density(lwd = 1.2, linetype = 1) + theme_classic() + 
  xlab("Distance to TSS") + ggtitle("dynamic eQTL in nmglut")
```


```{r, echo=F}
npglut <- readRDS("data/npglut_crm_fdr_loc.rds")
ggplot(npglut[npglut$tss_dist<2.5e5,], aes(x=tss_dist, colour=significant)) + 
  geom_density(lwd = 1.2, linetype = 1) + theme_classic() + 
  xlab("Distance to TSS") + ggtitle("dynamic eQTL in npglut")
```

```{r, echo=F}
GABA <- readRDS("data/GABA_crm_fdr_loc.rds")
ggplot(GABA[GABA$tss_dist<2.5e5,], aes(x=tss_dist, colour=significant)) + 
  geom_density(lwd = 1.2, linetype = 1) + theme_classic() + 
  xlab("Distance to TSS") + ggtitle("dynamic eQTL in GABA")
```

### Dynamic eQTL testing by interaction testing

We simply used the fixed interaction effect model to test the interaction effect between genotypes and stimulation for each cell type separately:

$$
expression \sim age+sex+disease+\sum_i^5 genotypePC + \sum_j^{15}expressionPC +genotype + timepoint + genotype \times timepoint
$$

Please note that we tested early response (1hr) and late response (6hr) against the resting state separately.

```{r,echo=F,eval=F}
library(data.table)
library(ACAT)


fdr.control <- function(dat) {
  geneP <- data.frame(gene=unique(dat$gene),geneP=1,geneFDR=1)
  for(g in unique(dat$gene)) {
    geneP[geneP$gene==g,"geneP"] <- ACAT(dat[dat$gene == g]$`p-value`)
  }
  geneP$geneFDR <- p.adjust(geneP$geneP, method="BH")
  merge(dat, geneP,by="gene")
}
dymfs <- list.files("data/dynamic_eqtl/",full.names = T)
dymres <- list()
for(f in dymfs) {
  n <- basename(f)
  dymres[[n]] <- fread(f)
}
fdr.control(dat)

```


#### Enrichment with upregulated peaks in stimulating states.

```{r,echo=T}
library(ggplot2)
res.tab <- read.table("data/pairwise_enrichment.txt",header=T)
enrich.df <- res.tab
enrich.df$log2OR <- log2(exp(res.tab$estimate))
enrich.df$lower <- log2(exp(res.tab$low))
enrich.df$upper <- log2(exp(res.tab$high))
enrich.df$se <- (enrich.df$upper - enrich.df$lower)/2/qnorm(0.975)
enrich.df$pval <- 2*(1-pnorm(enrich.df$log2OR/enrich.df$se))
enrich.df$annot <- sapply(strsplit(enrich.df$term,"\\."), function(x){x[1]})
enrich.df$annot <- gsub('_', ',', enrich.df$annot)

ggplot(enrich.df, aes(x = log2OR, y = annot)) +
  geom_errorbarh(aes(xmin = lower, xmax = upper), height = 0.2, color = "grey50") +
  geom_point(size = 4) +
  geom_vline(xintercept = 0, linetype = 'dashed', color = 'black') +
  
  # Add p-values with larger font and more space
  geom_text(aes(x = upper + 0.3, label = paste0("p=", signif(pval, 3))),
            hjust = 0, size = 4) +
  
  theme_minimal() +
  labs(x = 'log2 Odds Ratio', y = 'Annotation') +
  theme(
    legend.position = 'top',
    legend.direction = 'horizontal',
    legend.title = element_blank(),
    axis.text.x = element_text(size = 10),  # Bigger x-axis tick labels
    axis.text.y = element_text(size = 10)   # Bigger y-axis tick labels
  ) +
  # Expand x-axis to make room for labels
  scale_x_continuous(expand = expansion(mult = c(0.05, 0.2))) +
  coord_cartesian(clip = "off")



```


## Updated cTWAS results

We tried to rerun the analysis with updated version of cTWAS. So that enrichment may have standard error. However, percentage of h2g decreased from 11% to 6%. Sheng reported that old results were inflated because of sampling 10% of GWAS SNPs to estimate prior parameters (`thin=0.1`). Updated results do not require sampling any more. We have tried different thinning parameters (i.e. 10%, 20%, 50%, 100%). Results show consistent decrease of h2g

#### h2g decrease as sampling ratio of GWAS SNPs increased
```{r, echo=F}
pve <- numeric(4L)
names(pve) <- c("01","02","05","1")
param.store <- "data/new_ctwas/"
for(i in c("01","02","05","1")) {
  params <- readRDS(paste0(param.store,"NS.thin",i,".SCZ.parameters.RDS"))
  pve[i] <- sum(params$group_pve[-10] / sum(params$group_pve))
}

plot(y=pve*100,x=c(10,20,50,100), xlab="% of GWAS SNPs sampled", ylab="% of h2g", pch=16, xlim=c(0,105))
lines(y=pve*100,x=c(10,20,50,100))
```

Moreover, distribution of h2g across conditions changed a lot compared to original results. NMglut has much less heritability, and conditions of early response have higher h2g in general.

#### Enrichment of cTWAS results without thinning

```{r, echo=F}
## Relied on params read from previous block
library(ggplot2)

params <- readRDS(paste0(param.store,"NS.thin","1",".SCZ.parameters.RDS"))
enr <- exp(params$log_enrichment)
names(enr) <- sapply(strsplit(names(enr),"\\|"), function(x){x[1]})
dat <- data.frame(condition=factor(names(enr), levels=names(enr)), 
                  enrichment=enr, 
                  se=params$log_enrichment_se,
                  celltype=sapply(strsplit(names(enr), "_"), function(x){x[2]}))
dat["0hr_nmglut",]$se <- 1
#barplot(enr, las=2, col=rep(1:3,each=3), main="new cTWAS results without thinning")
ggplot(dat, aes(x=condition,y=enrichment, fill=celltype)) + 
  geom_bar(stat="identity", position = "dodge") + 
  geom_errorbar(aes(ymin=enrichment-se, ymax=enrichment+se), 
                width=0.4, position=position_dodge()) + 
  labs(y="log fold of enrichment") +
  theme_classic(base_size = 12) +
  theme(axis.text.x = element_text(angle = 45, hjust = 1))

```

```{r, echo=F}
## Relied on params read from previous block
params <- readRDS(paste0(param.store,"NS.thin","01",".SCZ.parameters.RDS"))
enr <- exp(params$log_enrichment)
names(enr) <- sapply(strsplit(names(enr),"\\|"), function(x){x[1]})
barplot(enr, las=2, col=rep(1:3,each=3), main="new cTWAS results with 10% sampling")
```


```{r, echo=F}
ctwas.params <- readRDS("data/joint_celltype_cTWAS_params.rds")
barplot(ctwas.params$enrichment[c(1,4,7,2,5,8,3,6,9)], las=2, main="Enrichment of old cTWAS results",
        col=rep(1:3,each=3))
```



#### Putative risk genes from different thinning

For the gene with PIP > 50% in previous results, we compared their PIP in the updated results.

```{r, echo=F}
library(pheatmap)

pip <- readRDS("data/new_ctwas/new_ctwas_aggr_pip.rds")

res <- readRDS("data/SCZ_cover8_noSNP_ctwas.RDS")
res <- res[res$cs_index>0,]
cond <- names(table(res$type))
res$genename <- sapply(strsplit(res$id, "\\|"), function(x) {x[1]})
res2 <- aggregate(res$susie_pip, list(res$genename), sum)
res3 <- merge(res2, res[res$type==cond[1],c("genename","susie_pip")], 
                 by.x="Group.1", by.y="genename",all.x = T)
colnames(res3) <- c("genename","PIP_sum",paste0("PIP_",cond[1]))
for(i in 2:length(cond)) {
  res3 <- merge(res3, res[res$type==cond[i],c("genename","susie_pip")], 
                    by="genename",all.x = T)
  colnames(res3)[i+2] <- paste0("PIP_",cond[i])
}
res3[is.na(res3)] <- 0

dat <- matrix(0, nrow=sum(res3$PIP_sum>0.5), ncol=5)
rownames(dat) <- res3$genename[res3$PIP_sum>0.5]
colnames(dat) <- c(colnames(pip),"old")
inters <- intersect(rownames(dat), rownames(pip))
dat[inters,1:4] <- pip[inters,]
dat[,5] <- res3$PIP_sum[res3$PIP_sum>0.5]
pheatmap(dat[order(dat[,5]),], cluster_cols = F, cluster_rows = F, color = colorRampPalette(c("white", "red"))(100))

```

Below is the heatmap of 28 putative risk gene with PIP>50% in updated cTWAS results without thinning. Highlighted genes such as CPT1C, CROT, and NAGA are still there. NGEF nominated in NMglut in previous results have PIP<50%.

```{r, echo=F}
library(ctwas)
library(pheatmap)

mapping_table <- readRDS("data/gene_mapping.RDS")
res <- readRDS("../Downloads/SCZ.finemap_regions_res.RDS")
finemap_res <- res$susie_alpha_res
finemap_res <- anno_susie_alpha_res(finemap_res,
                                      mapping_table = mapping_table,
                                      map_by = "molecular_id",
                                      drop_unmapped = TRUE)
  
total_pip <- combine_gene_pips(finemap_res,
                                    group_by = "gene_name",
                                    by = "context",
                                    method = "combine_cs",
                                    filter_cs = FALSE,
                                    include_cs_id = FALSE)

total_pip[is.na(total_pip)] <- 0
pip1 <- total_pip[total_pip$combined_pip>0.5,]
rownames(pip1) <- pip1$gene_name
idx <- expand.grid(paste0(c(0,1,6),"hr"), c("GABA", "npglut", "nmglut"))
idx <- paste(idx$Var1, idx$Var2, "pip", sep="_")
pheatmap(pip1[,idx], cluster_cols = F, cluster_rows = F, color = colorRampPalette(c("white", "red"))(100), angle_col = 315)
```


The updated results shared 3 genes with Cerebellar (i.e. SERPINI1, FOXN2, ZNF823). ZNF823 is the only gene shared by all GTEx brain tissues.

```{r, echo=F}
GTEx.res <- readRDS("data/SCZ_GTEx_brain_cTWAS_geneSymbol_res.rds")
GTEx.gene <- lapply(GTEx.res, function(x){x$hgnc_symbol[(x$susie_pip>0.5)&(x$cs_index!=0)]})
gtex.union <- unique(unlist(GTEx.gene))
GTEx.gene$Brain_union <- gtex.union
shared <- sapply(GTEx.gene, function(x) {sum(x %in% pip1$gene_name)})
names(shared) <- substr(names(shared), 7, nchar(names(shared)))
```

```{r}
library(VennDiagram)

p1 <- venn.diagram(list(neuron_stim=pip1$gene_name, Cerebellar=GTEx.gene$Brain_Cerebellar_Hemisphere),
            filename=NULL, cat.pos=c(-20,20), alpha=0.4, fill=c("red","blue"),fontface = "bold", cex=2)
grid.newpage()
grid.draw(p1)
```



### eGenes with other stimulus

Compared with mouse sensory experience DEGs from Jubao:

* a	Early response genes (|log2(FC)|>1 , FDR <0.05) whose expression increases
* b	Early response genes (|log2(FC)|>1 , FDR <0.05) whose expression decreases
* c	Late response genes (|log2(FC)|>1 , FDR <0.05) whose expression increases
* d	Late response genes (|log2(FC)|>1 , FDR <0.05) whose expression decreases

#### DEGs with excitatory neuron cells?

Out of 108 DEGs (log fold change > 1 and FDR<5%), 61 of them was an eQTL in at least one condition.

```{r, echo=F}
library(biomaRt)
library(DT)

dat <- read.csv("data/mouse_sesenory_experince_ext.csv")
map <- read.csv("data/gProfiler_mmusculus_hsapiens_ext.csv")
dat$upper <- toupper(dat$GeneName)
dat1 <- merge(dat, map[,c(2,5)], by.x="upper", by.y="initial_alias")

eGene <- readRDS("data/gene_fdr_results.RDS")
egenes <- list()
for(n in colnames(eGene)) {
  egenes[[n]] <- rownames(eGene)[eGene[,n]<0.05]
}

for(n in names(egenes)) {
  dat1[[paste0("eqtl_in_",n)]] <- dat1$ortholog_name %in% egenes[[n]]
}

dat2 <- dat1[rowSums(dat1[,11:19])>0,]

DT::datatable(dat2[,-1])
```

#### DEGs with inhibitory neuron cells?

Out of 102 DEGs (log fold change > 1 and FDR<5%), 52 of them was an eQTL in at least one condition.

```{r, echo=F}
library(biomaRt)
library(DT)

dat <- read.csv("data/mouse_sesenory_experince_inh.csv")
map <- read.csv("data/gProfiler_mmusculus_hsapiens_inh.csv")
dat$upper <- toupper(dat$GeneName)
dat1 <- merge(dat, map[,c(2,5)], by.x="upper", by.y="initial_alias")

eGene <- readRDS("data/gene_fdr_results.RDS")
egenes <- list()
for(n in colnames(eGene)) {
  egenes[[n]] <- rownames(eGene)[eGene[,n]<0.05]
}

for(n in names(egenes)) {
  dat1[[paste0("eqtl_in_",n)]] <- dat1$ortholog_name %in% egenes[[n]]
}

dat2 <- dat1[rowSums(dat1[,11:19])>0,]

DT::datatable(dat2[,-1])
```


### Number of risk genes discovered by cTWAS with different SCZ GWAS

```{r,echo=F}
pve.scz <- c(0,12,28)
names(pve.scz) <- c(2011,2014,2022)
x=1:3
plot(x, pve.scz, type = "o", col = "blue",
     main = "",
     ylab = "# of risk genes with PIP>50%",xaxt = "n",
     xlab="SCZ GWAS conducted in different years")  # suppress default x-axis
axis(1, at = x, labels = names(pve.scz))
```


